<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html>
<head>
<title>Promethease report</title>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" >
<style type="text/css">
body {
font-family: arial, helvetica, sans-serif;
color: black;
}
button {
border: 1px solid #000000;
padding: 0px;
font-size: 4px;
margin-left: 1px;
}
button:hover {
cursor: pointer;
color: #ffffff;
}
#navmenu {
background-color: #F0F0F0;
position:fixed;
right:5px;
top:5px;
border:1px solid black;
z-index:2;
}
#navbox {
border:1px solid black;
padding-left: 170px;
float: right;
clear: none;
padding: 0px;
min-width: 50%;
max-width: 69%;
}
#treeDiv1 {
max-height:300px;
overflow: auto;
background-color: #F0F0F0;
float:left;
right:5px;
top:5px;
border:1px solid black;
clear: none;
min-width: 20%;
max-width: 30%;
padding: 0px;
}
#expandcontractdiv {
border:1px dotted #dedede;
background-color:#EBE4F2;
margin:0 0 .5em 0;
padding:0.4em;
}
.majorsection {
float:left;
width:95%;
border:1px solid black;
clear:both;
margin: 5px 0px;
padding: 0px 10px;
}
.sectiontext {
clear:both;
font-size: x-large;
}
.boxresult {
width: inherit;
margin: 10px auto;
border: 2px solid grey;
}
.boxgoodresult {
width: inherit;
margin: 10px auto;
border: 2px solid #80FF90;
}
.boxbadresult {
width: inherit;
margin: 10px auto;
border: 2px solid #FF8080;
}
.boxresultcol1 {
background-color: #FFFFFF;
float: left;
width: 190px;
margin: 0;
word-wrap: break-word;
}
.boxresultcol2 {
margin-left: 190px;
}
.boxlink {
float:left;
width:130px;
}
.boxfreq {
float:left;
width:170px;
clear:both;
}
.boxmag {
float:left;
width:170px;
clear:both;
}
.boxrep {
float:left;
width:170px;
clear:both;
}
.boxeffect {
float:left;
}
.gsboxeffect {
float:left;
clear: none;
}
.boxshow {
float:left;
width:100px;
}
.rstext {
float:left;
clear: left;
background-color: #D0D0FF;
}
.diseasebutton {
margin-left: 20px;
float:left;
clear: both;
}
.member-Topic {
margin-left: 40px;
border:1px solid black;
clear: both;
}
.member-is-a-medical-condition {
margin-left: 40px;
border:1px solid black;
clear: both;
}
.member-is-a-medicine {
margin-left: 40px;
border:1px solid black;
clear: both;
}
.adiseasebox {
width: 90%;
float:left;
margin-left: 40px;
clear: both;
}
.diseaselink {
margin-left: 240px;
FONT-SIZE: 18pt;
}
.diseaseinfo {
margin-left: 650px;
}
.boxfooter {
clear: both;
margin: 0;
border-top: 1px solid gray;
}
.majorboxfooter {
clear: both;
margin: 0;
border-top: 1px solid gray;
}
</style>

<script type="text/javascript">
function toggle(element) {
if (document.getElementById(element).style.display == "none") {
   document.getElementById(element).style.display = "";
} else {
   document.getElementById(element).style.display = "none";
}
}
function toggle2(element) {
if (document.getElementById(element).style.display == "none") {
   document.getElementById(element).style.display = "";
   document.getElementById('show-'+element).style.display = "none";
} else {
   document.getElementById(element).style.display = "none";
   document.getElementById('show-'+element).style.display = "";
}
}
function toggle3(name) {
var e=getElementsByClassName(name);
for ( var i=0; i < e.length; i++ ) {
   if (e[i].style.display == "none") {
       e[i].style.display = "";
       document.getElementById('show-'+e[i].id).style.display = "none";
   } else {
       e[i].style.display = "none";
       document.getElementById('show-'+e[i].id).style.display = "";
   }
}
}

/*

Need to switch to this cleaner method
Crockford walkTheDOM http://javascript.crockford.com/hackday.ppt

function walkTheDOM(node, func) {

func(node);
node = node.firstChild;
while (node) {
    walkTheDOM(node, func);
    node = node.nextSibling;
}

}


function getElementsByClassName(className, node) {
 var results = [];
 walkTheDOM(node || document.body, function (node) {
     var a, c = node.className, i;
     if (c) {
         a = c.split(' ');
         for (i = 0; i < a.length; i += 1) {
             if (a[i] === className) {
                  results.push(node);
                  break;
              }
         }
     }
 });
 return results;
}


By now there is a more efficient method, but once upon a time this javascript served well enough

    BEGIN:http://www.robertnyman.com
    http://www.robertnyman.com/2008/05/27/the-ultimate-getelementsbyclassname-anno-2008/
    cross platform 'getElementsByClassName'
    Developed by Robert Nyman, http://www.robertnyman.com
    Code/licensing: http://code.google.com/p/getelementsbyclassname/
*/
var getElementsByClassName = function (className, tag, elm){
    if (document.getElementsByClassName) {
            getElementsByClassName = function (className, tag, elm) {
                    elm = elm || document;
                    var elements = elm.getElementsByClassName(className),
                            nodeName = (tag)? new RegExp("\b" + tag + "\b", "i") : null,
                            returnElements = [],
                            current;
                    for(var i=0, il=elements.length; i<il; i+=1){
                            current = elements[i];
                            if(!nodeName || nodeName.test(current.nodeName)) {
                                    returnElements.push(current);
                            }
                    }
                    return returnElements;
            };
    }
    else if (document.evaluate) {
            getElementsByClassName = function (className, tag, elm) {
                    tag = tag || "*";
                    elm = elm || document;
                    var classes = className.split(" "),
                            classesToCheck = "",
                            xhtmlNamespace = "http://www.w3.org/1999/xhtml",
                            namespaceResolver = (document.documentElement.namespaceURI === xhtmlNamespace)? xhtmlNamespace : null,
                            returnElements = [],
                            elements,
                            node;
                    for(var j=0, jl=classes.length; j<jl; j+=1){
                            classesToCheck += "[contains(concat(' ', @class, ' '), ' " + classes[j] + " ')]";
                    }
                    try     {
                            elements = document.evaluate(".//" + tag + classesToCheck, elm, namespaceResolver, 0, null);
                    }
                    catch (e) {
                            elements = document.evaluate(".//" + tag + classesToCheck, elm, null, 0, null);
                    }
                    while ((node = elements.iterateNext())) {
                            returnElements.push(node);
                    }
                    return returnElements;
            };
    }
    else {
            getElementsByClassName = function (className, tag, elm) {
                    tag = tag || "*";
                    elm = elm || document;
                    var classes = className.split(" "),
                            classesToCheck = [],
                            elements = (tag === "*" && elm.all)? elm.all : elm.getElementsByTagName(tag),
                            current,
                            returnElements = [],
                            match;
                    for(var k=0, kl=classes.length; k<kl; k+=1){
                            classesToCheck.push(new RegExp("(^|\s)" + classes[k] + "(\s|$)"));
                    }
                    for(var l=0, ll=elements.length; l<ll; l+=1){
                            current = elements[l];
                            match = false;
                            for(var m=0, ml=classesToCheck.length; m<ml; m+=1){
                                    match = classesToCheck[
                                        m].test(current.className);
                                    if (!match) {
                                            break;
                                    }
                            }
                            if (match) {
                                    returnElements.push(current);
                            }
                    }
                    return returnElements;
            };
    }
    return getElementsByClassName(className, tag, elm);
};
/*      END:http://www.robertnyman.com */

function ShowEverything(){//we open all of them

var sections = new Array('diseasebox',
                     'member-is-a-Topic',
                     'member-is-a-medical-condition',
                     'member-is-a-medicine',
                     'rstext',
                     'boxrstext',
                     'boxshow',
                     'boxsection'
                     );

for ( var i_sect=0; i_sect < sections.length; i_sect++ ) {
    var e=getElementsByClassName(sections[i_sect]);
    for ( var i=0; i < e.length; i++ ) {
        e[i].style.display = '';
    }
}

}
</script>
</head>
<body><p>
<a href="../report.html">main page</a><a name='top'></a>
<div id='header'>
<h1><a href='http://www.promethease.com'>Promethease</a> Report</h1>
<h2>Version: 0.1.164</h2>
<h2>Generated: 2016-01-05 12:04</h2>
<h3>Infile: genome_test.txt <br>infiles /tmp/tmpcRWXbJ</h3><h3>Reference Population GIH</h3><h3>23818 genotypes annotated</h3></div>
<div id='navmenu'>
<h3><a href='http://www.snpedia.com/index.php?title=Promethease/results'>Help</a> interpreting your results.</h3>
<a href="javascript:ShowEverything()">Show Everything</a>
</div>
    <div class="majorsection">     <div class='sectiontext'>Good news</div>
        <div class='boxshow' id="show-goodNews" style="display: inline;"><a href="javascript:toggle2('goodNews')">...more...</a></div>
        <div class="boxsection" id="goodNews" style="display: none;"><a href="javascript:toggle2('goodNews')">(hide)</a>

  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7294919'>rs7294919</a><a href='http://www.snpedia.com/index.php/rs7294919(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 3<br/>Frequency: <span style='background-color: #FFADAD;'>13.3%</span><br/>Repute:Good<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Moderately enhanced hippocampal volume</div>
        <div class='boxshow' id="show-sec-0.0462838928023" style="display: inline;"><a href="javascript:toggle2('sec-0.0462838928023')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0462838928023" style="display: none;"><a href="javascript:toggle2('sec-0.0462838928023')">(hide)</a>
        <div class='rstext'>rs7294919, showed a particularly strong link to a reduced hippocampus volume, suggesting that this gene is very important to hippocampus development or health. No associations for brain volume, but they did discover that intracranial volume was significantly associated with two loci: rs4273712, a known height locus on chromosome 6q22, and rs9915547, tagging the inversion on chromosome 17q21. The SNP is located between two genes, HRK and FBXW8] but evidence suggests that it influences the expression level of a gene 3&#8217; to FBXW8], TESC Each copy of the T allele was associated with a 107.8 mm&lt;sup&gt;3&lt;/sup&gt; decrease in hippocampal volume [http://www.nature.com/ng/journal/v44/n5/full/ng.2237.html. In European populations, the effect allele (T) is found at frequency of 0.898 The m...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1042522'>rs1042522</a><a href='http://www.snpedia.com/index.php/rs1042522(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 3<br/>Frequency: <span style='background-color: #F098F0;'>9.2%</span><br/>Repute:Good<br/>References:126<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Live 3 years longer. Chemotherapy is more effective.</div>
        <div class='boxshow' id="show-sec-0.221760773999" style="display: inline;"><a href="javascript:toggle2('sec-0.221760773999')">...more...</a></div>
        <div class='boxrstext' id="sec-0.221760773999" style="display: none;"><a href="javascript:toggle2('sec-0.221760773999')">(hide)</a>
        <div class='rstext'>This SNP, a variant in the TP53 gene, is 1 of 25 SNPs reported to represent independently minor, but cumulatively significant, increased risk for breast cancer. For details of all 25 SNPs in this group, along with the two methods used to calculate overall risk estimates for breast cancer, refer to the SNPedia breast cancer entry. For this particular SNP, the G allele encodes an arginine at position 72 of the protein (where there's normally a proline), and the SNP is commonly called the Arg72 variant, although P72R and Arg72Pro are also common in the literature. Minor allele homozygotes, i.e. rs1042522(C;C) genotypes, live on average 3 years longer than major allele (G;G) homozygotes, based on a study of 9,200+ Danish individuals. The increased longevity is speculated to be related to the i...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs295"  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs295'>gs295</a></div></div>
        <div class='boxmag'>Magnitude: 2.8</div>
        <div class='boxrep'>Repute: Good</div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>Lower heart attack risk than average</div>
        <div class='gsboxeffect'>You are in the 10 - 20 % of people (depending on the population studied) who carry three minor alleles at two SNPs, rs1108580 and rs1611115. This genoset is reported as having 0.46x the risk of a heart attack or cardiovascular incident compared to people who carry zero of the minor alleles for these two SNPs, based on a study of 3,000 African-Americans enrolled in the Jackson Heart Study.</div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1837253'>rs1837253</a><a href='http://www.snpedia.com/index.php/rs1837253(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2.8<br/>Frequency: <span style='background-color: #FF9292;'>5.3%</span><br/>Repute:Good<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>reduced risk (0.73) for late-onset (adult) asthma  Found in only 6% of all Caucasians, this genotype is associated with reduced risk (relative risk = 0.73) for late-onset (adult) asthma.</div>
        <div class='boxshow' id="show-sec-0.418787028718" style="display: inline;"><a href="javascript:toggle2('sec-0.418787028718')">...more...</a></div>
        <div class='boxrstext' id="sec-0.418787028718" style="display: none;"><a href="javascript:toggle2('sec-0.418787028718')">(hide)</a>
        <div class='rstext'>rs1837253 is a SNP found a bit upstream of the thymic stromal lymphopoietin TSLP gene on chromosome 5. The TSLP protein plays a role in inflammation of cells in the trachea and alveoli therefore may have a causative role in certain types of asthma. The rs1837253(T) allele has been found to reduce the risk of developing late-onset (adult) asthma in several studies, and it is recognized by the CPMC Study as one they report to their participants.  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs53576'>rs53576</a><a href='http://www.snpedia.com/index.php/rs53576(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Repute:Good<br/>References:44<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Optimistic and empathetic; handle stress well  The one in four subjects who inherited a variation in this allele called G/G were significantly better at accurately reading the emotions of others by observing their faces than were the remaining three-quarters of subjects, who had inherited either a pair of A's or an A and a G from their parents at this site. Compared to the three-fourths with A/A or A/G variations, the G/G individuals were also less likely to startle when blasted by a loud noise, or to become stressed at the prospect of such a noise. And by their own reports, the G/G subjects were mellower and more attuned to other people than were the A/As or A/Gs. news ------- Effect on Parental Sensitivity Controlling for maternal education, depression and marital discord, OXTR [F	(1,152...</div>
        <div class='boxshow' id="show-sec-0.913927793591" style="display: inline;"><a href="javascript:toggle2('sec-0.913927793591')">...more...</a></div>
        <div class='boxrstext' id="sec-0.913927793591" style="display: none;"><a href="javascript:toggle2('sec-0.913927793591')">(hide)</a>
        <div class='rstext'>rs53576 is a silent G to A change in the oxytocin receptor (OXTR) gene. Studies have demonstrated that individuals with the G allele are more empathetic, feel less lonely, employ more sensitive parenting techniques, and have lower rates of autism (discussed in ) The blog Not Exactly Rocket Science discusses that Americans with (G;G) tend to be more sensitive parents, more empathetic and less lonely than those with an &#8216;A&#8217;. In a Korean population people with (G;G) were less likely to seek support from their peers. Culture, distress, and oxytocin receptor polymorphism (OXTR) interact to influence emotional support seeking. Oxytocin receptor genetic variation relates to empathy and stress reactivity in humans. In brief, people with the G;G genotype were better able to discern the e...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4888378'>rs4888378</a><a href='http://www.snpedia.com/index.php/rs4888378(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2.4<br/>Frequency: <span style='background-color: #FFADAD;'>13.4%</span><br/>Repute:Good<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>For women, slower carotid thickening with age (so presumably lower atherosclerotic risk)  See papers cited for rs4888378</div>
        <div class='boxshow' id="show-sec-0.0997188531718" style="display: inline;"><a href="javascript:toggle2('sec-0.0997188531718')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0997188531718" style="display: none;"><a href="javascript:toggle2('sec-0.0997188531718')">(hide)</a>
        <div class='rstext'></div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3816873'>rs3816873</a><a href='http://www.snpedia.com/index.php/rs3816873(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2.2<br/>Frequency: <span style='background-color: #FF9292;'>5.3%</span><br/>Repute:Good<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>reduced risk of type-2 diabetes</div>
        <div class='boxshow' id="show-sec-0.819610449554" style="display: inline;"><a href="javascript:toggle2('sec-0.819610449554')">...more...</a></div>
        <div class='boxrstext' id="sec-0.819610449554" style="display: none;"><a href="javascript:toggle2('sec-0.819610449554')">(hide)</a>
        <div class='rstext'>Also known as I128T MTP rare allele of the MTP I128T polymorphism may be protective against impaired glucose tolerance, type-2 diabetes Association of polymorphisms in glutamate-cysteine ligase catalytic subunit and microsomal triglyceride transfer protein genes with nonalcoholic fatty liver disease.  [OMIM:MICROSOMAL TRIGLYCERIDE TRANSFER PROTEIN, 88-KD; MTP]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9264942'>rs9264942</a><a href='http://www.snpedia.com/index.php/rs9264942(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2.2<br/>Frequency: <span style='background-color: #FFA3A3;'>9.7%</span><br/>Repute:Good<br/>References:21<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>90% reduction in HIV viral load The rs9264942(C;C) genotype is reported to be associated with a 90% reduction in viral load in HIV-infected individuals. See also rs9264942 and HIV.</div>
        <div class='boxshow' id="show-sec-0.106201271128" style="display: inline;"><a href="javascript:toggle2('sec-0.106201271128')">...more...</a></div>
        <div class='boxrstext' id="sec-0.106201271128" style="display: none;"><a href="javascript:toggle2('sec-0.106201271128')">(hide)</a>
        <div class='rstext'>This SNP (C/T) is in 5' region of the HLA-C gene, 35 kb away from transcription initiation in or around the HLA-C gene. 'People with this tiny sequence variation, dubbed rs9264942, appear to have up to 90% less virus in their systems than those who carry other polymorphisms. About 10% of Europeans appear to carry two copies of rs9264942, which leads to an average 90% viral load reduction. About 50% of Europeans carry one copy, which gives a 60% reduction. By comparison, less than 40% of people of African descent appear to carry a single copy of the polymorphism.' Genetic variation may lower HIV load by 90% They are referring to the (C;C) genotype giving a 90% reduction and the (C;T) giving a 60% reduction. dbSNP page This SNP is also reported to account for 6.5% of the 15% variation in vir...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs189798'>rs189798</a><a href='http://www.snpedia.com/index.php/rs189798(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FF9A9A;'>7.1%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>decreased high myopia risk  Less risk of high myopia (in French people). 34.2% of French people without high myopia have T here, but only 21.1% of controls do.(table)</div>
        <div class='boxshow' id="show-sec-0.245815493449" style="display: inline;"><a href="javascript:toggle2('sec-0.245815493449')">...more...</a></div>
        <div class='boxrstext' id="sec-0.245815493449" style="display: none;"><a href="javascript:toggle2('sec-0.245815493449')">(hide)</a>
        <div class='rstext'></div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6323'>rs6323</a><a href='http://www.snpedia.com/index.php/rs6323(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFB5B5;'>16.8%</span><br/>Repute:Good<br/>References:13<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased monoamine oxidase A activity</div>
        <div class='boxshow' id="show-sec-0.543727896821" style="display: inline;"><a href="javascript:toggle2('sec-0.543727896821')">...more...</a></div>
        <div class='boxrstext' id="sec-0.543727896821" style="display: none;"><a href="javascript:toggle2('sec-0.543727896821')">(hide)</a>
        <div class='rstext'>rs6323 (R297R / Arg297Arg) is a SNP in the MAOA (monoamine oxidase A) gene. Monoamine oxidase A degrades serotonin, dopamine, epineprine, and norepinephrine. The G allele encodes for the higher activity form of the enzyme. Subjects with major depressive disorder with the highest activity form of the enzyme (G or G/G) had a significantly lower magnitude of placebo response.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6505162'>rs6505162</a><a href='http://www.snpedia.com/index.php/rs6505162(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFCBCB;'>30.4%</span><br/>Repute:Good<br/>References:14<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.43x decreased risk for esophageal cancer</div>
        <div class='boxshow' id="show-sec-0.15364570756" style="display: inline;"><a href="javascript:toggle2('sec-0.15364570756')">...more...</a></div>
        <div class='boxrstext' id="sec-0.15364570756" style="display: none;"><a href="javascript:toggle2('sec-0.15364570756')">(hide)</a>
        <div class='rstext'>rs6505162 is a SNP located in the pre-miRNA region of mir423. The major allele is rs6505162(C). A study of 346 Caucasians with esophageal cancer found that compared with the homozygous wild-type genotype of rs6505162, individuals with the heterozygous and homozygous variant genotype had a significantly reduced esophageal cancer risk with odds ratios of 0.58 (CI: 0.41-0.82) and 0.43 (CI: 0.27-0.68), respectively. After stratifying by gender, smoking, and age, the authors discovered that the protective effect was significant only for individuals under 64 years of age, regardless of sex or whether they smoked. Other unfavorable genotypes (ie SNPs) were also identified that were not as significant as rs6505162, and the overall risk of esophageal cancer increased with increasing numbers of unfa...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9273363'>rs9273363</a><a href='http://www.snpedia.com/index.php/rs9273363(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFDEDE;'>49.2%</span><br/>Repute:Good<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Much lower 0.15x risk of Type 1 Diabetes.  According to 23andMe, this SNP greatly reduces your risk of Type 1 Diabetes. Type 1 Diabetes is an autoimmune disease where your immune system attacks your own body's insulin producing cells in the pancreas, preventing your blood sugar from being able to enter cells that need energy. This has been verified for white people. 23andMe originally wanted to use rs9272346 instead, but it was deemed unreliable.</div>
        <div class='boxshow' id="show-sec-0.217255528778" style="display: inline;"><a href="javascript:toggle2('sec-0.217255528778')">...more...</a></div>
        <div class='boxrstext' id="sec-0.217255528778" style="display: none;"><a href="javascript:toggle2('sec-0.217255528778')">(hide)</a>
        <div class='rstext'>This is one of the SNPs that 23andMe uses for Type 1 diabetes. According to 23andMe, this SNP changes your risk of Type 1 Diabetes. That is an autoimmune disease where your immune system attacks your own body's insulin producing cells in the pancreas, preventing your blood sugar from being able to enter cells that need energy. This has been verified for white people. 23andMe originally wanted to use rs9272346 instead, but it was deemed unreliable.  [GWAS:Chronic lymphocytic leukemia]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1815739'>rs1815739</a><a href='http://www.snpedia.com/index.php/rs1815739(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFE6E6;'>58.4%</span><br/>Repute:Good<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Mix of muscle types. Likely sprinter.  Normal for white people. One working and one defective copy of the gene for alpha-actinin-3 in fast-twitch muscle fibres. This genotype may indicate somewhat impaired muscles, however many sprinters and some endurance athletes have this.</div>
        <div class='boxshow' id="show-sec-0.243291905907" style="display: inline;"><a href="javascript:toggle2('sec-0.243291905907')">...more...</a></div>
        <div class='boxrstext' id="sec-0.243291905907" style="display: none;"><a href="javascript:toggle2('sec-0.243291905907')">(hide)</a>
        <div class='rstext'>This SNP, in the ACTN3 gene, encodes a premature stop codon in a muscle protein called alpha-actinin-3. The polymorphism alters position 577 of the alpha-actinin-3 protein. In the (C;C) genotype is often called RR. The truncated (T;T) is often called XX. (T;T) is under-represented in elite strength athletes, consistent with previous reports indicating that alpha-actinin-3 deficiency appears to impair muscle performance. The most common nucleotide at this position, (C), encodes an arginine (amino acid code R), the alternative T allele encodes a stop codon (X). Hence, the SNP is referred to as R577X, with homozygotes being either RR or XX and heterozygotes being RX. XX individuals completely lack the expression of alpha-actinin-3. An earlier report studying a relatively small number of Austr...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs738409'>rs738409</a><a href='http://www.snpedia.com/index.php/rs738409(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #F0D0F0;'>55.4%</span><br/>Repute:Good<br/>References:112<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>most common genotype; slightly less damage from alcohol</div>
        <div class='boxshow' id="show-sec-0.148956144493" style="display: inline;"><a href="javascript:toggle2('sec-0.148956144493')">...more...</a></div>
        <div class='boxrstext' id="sec-0.148956144493" style="display: none;"><a href="javascript:toggle2('sec-0.148956144493')">(hide)</a>
        <div class='rstext'>rs738409 is a SNP in the PNPLA3 gene. In all HapMap populations the most common allele is a C on the plus strand, which means a G on the minus strand as represented for this SNP in dbSNP and in SNPedia. In the scientific literature, however, most authors appear unaware that this SNP is defined as oriented on the minus strand, so they refer incorrectly to rs738409(G) as the minor allele. The genotypes defined in SNPedia are correct for the minus orientation of this SNP, but this can be confusing since it's the opposite of what the literature indicates in many cases. In a Mestizo population, rs738409 is strongly associated with alcoholic liver disease and clinically evident alcoholic cirrhosis (unadjusted odds ratio 2.25, p=1.7e-10); ancestry-adjusted OR 1.79, p=1.9 x 10(-5)). rs738409(G) in...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs156"  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs156'>gs156</a></div></div>
        <div class='boxmag'>Magnitude: 2</div>
        <div class='boxrep'>Repute: Good</div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>NAT2 Rapid metabolizer.</div>
        <div class='gsboxeffect'>NAT2 Rapid metabolizer source This is ostensibly the 'normal' type of metabolizer, who carries two rapid (active) acetylator alleles such as NAT2*4. NAT2 rapid acetylators can be typically be administered drugs which are substrates of the NAT2 enzyme following standard dosing practices. This prediction and that of gs154 of NAT2 metabolizer phenotype is based on using 2 SNPs, rs1041983 and rs1801280. An alternative prediction of NAT2 metabolizer status based on using more SNPs (6) is used by genosets gs138, gs139, and gs140.</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs246"  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs246'>gs246</a></div></div>
        <div class='boxmag'>Magnitude: 2</div>
        <div class='boxrep'>Repute: Good</div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>APOE3/APOE3</div>
        <div class='gsboxeffect'>Apo-&#949;3/&#949;3, the most common form of APOE. It is considered the 'neutral' genotype, with E2 having lower risk and E4 having higher risk of Alzheimer's disease. Criteria: rs429358(T;T),rs7412(C;C)). Frequency *European 57.4% *African American 41.7% *East Asian 74.6%</div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7101429'>rs7101429</a><a href='http://www.snpedia.com/index.php/rs7101429(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF7777;'>1.8%</span><br/>Repute:Good<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.70x reduced risk for Alzheimer's risk</div>
        <div class='boxshow' id="show-sec-0.147223725225" style="display: inline;"><a href="javascript:toggle2('sec-0.147223725225')">...more...</a></div>
        <div class='boxrstext' id="sec-0.147223725225" style="display: none;"><a href="javascript:toggle2('sec-0.147223725225')">(hide)</a>
        <div class='rstext'>rs7101429 is a SNP in the GRB2-associated binding protein 2 GAB2 gene. In a study of 4000 DNA samples from ~1300 families affected by Alzheimer's disease, there was a reduced risk (0.70x; p = .002) among carriers of the rs7101429(G) minor allele.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3790844'>rs3790844</a><a href='http://www.snpedia.com/index.php/rs3790844(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF8787;'>3.5%</span><br/>Repute:Good<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>reduced risk (0.59x) of pancreatic cancer</div>
        <div class='boxshow' id="show-sec-0.0546887184936" style="display: inline;"><a href="javascript:toggle2('sec-0.0546887184936')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0546887184936" style="display: none;"><a href="javascript:toggle2('sec-0.0546887184936')">(hide)</a>
        <div class='rstext'>23andMe reports that the G allele of this SNP appears to lower the risk of pancreatic cancer. Genome-wide association study of pancreatic cancer in Japanese population.  [OMIM:?] [GWAS:Pancreatic cancer]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2811712'>rs2811712</a><a href='http://www.snpedia.com/index.php/rs2811712(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFABAB;'>12.4%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.5x less risk for physical impairment with age.  This reduces the rate of physical problems with aging. It also reduces the risk of Cerebral Small Vessel Disease in the brain.</div>
        <div class='boxshow' id="show-sec-0.947512273053" style="display: inline;"><a href="javascript:toggle2('sec-0.947512273053')">...more...</a></div>
        <div class='boxrstext' id="sec-0.947512273053" style="display: none;"><a href="javascript:toggle2('sec-0.947512273053')">(hide)</a>
        <div class='rstext'>rs2811712 is a SNP associated with physical function in older individuals, and thus indirectly associated with longevity. In a study ultimately totaling over 3,300 elderly Caucasian individuals, the less common rs2811712(G) allele was associated with reduced physical impairment. Overall, severely limited physical function was present in 15.0% of the more common rs2811712(A;A) homozygotes, versus 7.0% of the rs2811712(G;G) homozygotes, yielding a per risk allele odds ratio of 1.48 (CI: 1.17-1.88, p=0.001). Association of single-nucleotide polymorphisms in the cell cycle genes with breast cancer in the British population. Consortium analysis of 7 candidate SNPs for ovarian cancer. Candidate gene analysis using imputed genotypes: cell cycle single-nucleotide polymorphisms and ovarian cancer r...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1128535'>rs1128535</a><a href='http://www.snpedia.com/index.php/rs1128535(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFB6B6;'>17.2%</span><br/>Repute:Good<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>reduced risk (0.77x) for Crohn's disease</div>
        <div class='boxshow' id="show-sec-0.260355970831" style="display: inline;"><a href="javascript:toggle2('sec-0.260355970831')">...more...</a></div>
        <div class='boxrstext' id="sec-0.260355970831" style="display: none;"><a href="javascript:toggle2('sec-0.260355970831')">(hide)</a>
        <div class='rstext'>The less common allele of rs1128535, a SNP in the TRAIP gene, may have a protective effect against the development of Crohn's disease, based on a study of 469 Caucasian patients. The reported odds ratio is 0.77 (CI: 0.67-0.89, p=0.002). Note that rs1128535 is highly correlated (r&lt;sup&gt;2&lt;/sup&gt; &gt; 0.8) to nonsynonymous SNPs in the nearby MST1R gene, rs2230590 and rs1062633, also known as Arg523Gln and Gly1335Arg, respectively.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1711437'>rs1711437</a><a href='http://www.snpedia.com/index.php/rs1711437(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFB9B9;'>18.6%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>younger, healthier kidney function  In the BLSA population, the genotype of rs1711437 explains 2.1% of the variation in creatinine clearance and in the InCHIANTI population, the genotype explains 0.9% of the variation. A is the protective allele. For an individual who carries the A allele, his or her creatinine clearance is approximately that of someone 4&#8211;5 years younger who does not carry the A allele. The association between rs1711437 genotype and creatinine clearance has not been replicated in additional populations</div>
        <div class='boxshow' id="show-sec-0.441188012656" style="display: inline;"><a href="javascript:toggle2('sec-0.441188012656')">...more...</a></div>
        <div class='boxrstext' id="sec-0.441188012656" style="display: none;"><a href="javascript:toggle2('sec-0.441188012656')">(hide)</a>
        <div class='rstext'>A genomic convergence approach was used to show that rs1711437 in ''MMP20'' is correlated with creatinine clearance (uncorrected p=3.6 x 10-5, empirical p=0.01). For genomic convergence, whole transcriptional profiling was first used to identify 630 genes that either change with age in the kidney or genes belonging to pathways that are enriched for genes that change with age. From these, eQTL mapping was used to find 101 genes that contained SNPs associated with expression variation that were used in an association study for creatinine clearance. Because only 2038 candidate SNPs (including SNPs that are not eQTLs) in these 101 genes were tested, p-values less than 5 x 10-5 were considered statistically significant. Kidney function at different ages was measured by glomerular filtration rat...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7571842'>rs7571842</a><a href='http://www.snpedia.com/index.php/rs7571842(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFB9B9;'>18.6%</span><br/>Repute:Good<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>possible reduction in risk for salt-induced hypertension  possible reduction in risk for salt-induced hypertension</div>
        <div class='boxshow' id="show-sec-0.648604347123" style="display: inline;"><a href="javascript:toggle2('sec-0.648604347123')">...more...</a></div>
        <div class='boxrstext' id="sec-0.648604347123" style="display: none;"><a href="javascript:toggle2('sec-0.648604347123')">(hide)</a>
        <div class='rstext'></div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs17822931'>rs17822931</a><a href='http://www.snpedia.com/index.php/rs17822931(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC0C0;'>23.2%</span><br/>Repute:Good<br/>References:14<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Wet earwax. Slightly better body odour.  Heterozygous for wet earwax. This makes earwax wet, sticky, and honey-coloured, which is normal for non-Asians. Being heterozygous slightly improves body odour and may slightly reduce colostrum production for breast feeding. People with this usually still need to use deodorant.</div>
        <div class='boxshow' id="show-sec-0.28715420949" style="display: inline;"><a href="javascript:toggle2('sec-0.28715420949')">...more...</a></div>
        <div class='boxrstext' id="sec-0.28715420949" style="display: none;"><a href="javascript:toggle2('sec-0.28715420949')">(hide)</a>
        <div class='rstext'>rs17822931, also known as c.538G&gt;A or G180R, is a SNP in the ATP-binding cassette, sub-family C (CFTR/MRP), member 11 ABCC11 gene. The ABCC11 protein helps transport small molecules across apical membranes such as those in apocrine secretory cells. This SNP determines wet vs dry earwax as well as sweat production, and it is also associated with lipid secretion. It is commonly (T;T) for Asians and (C;C) for Europeans and Africans. A study of deodorant usage in a large (17,000+) population showed that rs17822931(T;T) individuals were at least 5-fold less likely to use deodorant, consistent with them being 'genotypically nonodorous'. Although not statistically significant, there was some evidence for a heterozygote effect, implying that individuals with a (C;C) genotype for this SNP were l...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6983267'>rs6983267</a><a href='http://www.snpedia.com/index.php/rs6983267(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC2C2;'>23.9%</span><br/>Repute:Good<br/>References:150<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>normal risk for cancers; aspirin reduces risk for colorectal cancer  This genotype is not on its own associated with higher cancer risk. Additionally, aspirin has been reported to somewhat lower the risk for colorectal cancer in individuals of this genotype.</div>
        <div class='boxshow' id="show-sec-0.528944637984" style="display: inline;"><a href="javascript:toggle2('sec-0.528944637984')">...more...</a></div>
        <div class='boxrstext' id="sec-0.528944637984" style="display: none;"><a href="javascript:toggle2('sec-0.528944637984')">(hide)</a>
        <div class='rstext'>rs6983267 is a SNP on chromosome 8q24, associated with increased risk for several cancers, particularly prostate cancer. In studies dividing the 8q24 region, this SNP falls in region 3. This SNP has also been reported to influence the cancer-risk-decreasing effect of aspirin. In a study of over 3,600 Caucasians with prostate cancer, rs6983267 is one of five SNPs used (with family history as a sixth factor) to cumulatively predict overall risk. On their own, the rs6983267(G;G) and (G;T) risk genotypes yield an odds ratio for developing prostate cancer of 1.37 (CI: 1.18-1.59, p=3.4-10e-5) and may account for 22.2% of population attributable risk. The increased risk of developing prostate cancer associated with rs6983267 now appears to be independent of the risk associated with its close neig...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3914132'>rs3914132</a><a href='http://www.snpedia.com/index.php/rs3914132(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC4C4;'>25.7%</span><br/>Repute:Good<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Lower otosclerosis risk</div>
        <div class='boxshow' id="show-sec-0.0395866142746" style="display: inline;"><a href="javascript:toggle2('sec-0.0395866142746')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0395866142746" style="display: none;"><a href="javascript:toggle2('sec-0.0395866142746')">(hide)</a>
        <div class='rstext'>Each C allele at rs3914132 decreases the likelihood of developing otosclerosis, which can cause hearing loss.  [PharmGKB:Non-Curated In a GWAS done in Belgian-Dutch cases and controls (694 of each, split into a discovery and a replication set) and then in a second replication set of 455 French cases and 480 controls, rs3914132 was found to be associated with otosclerosis.] [OMIM:OTOSCLEROSIS; OTSC1] [GWAS:Otosclerosis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7538876'>rs7538876</a><a href='http://www.snpedia.com/index.php/rs7538876(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFCCCC;'>31.2%</span><br/>Repute:Good<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.78x decreased Basal Cell Carcinoma risk.   Reduced risk of this kind of skin cancer.</div>
        <div class='boxshow' id="show-sec-0.860085924666" style="display: inline;"><a href="javascript:toggle2('sec-0.860085924666')">...more...</a></div>
        <div class='boxrstext' id="sec-0.860085924666" style="display: none;"><a href="javascript:toggle2('sec-0.860085924666')">(hide)</a>
        <div class='rstext'>2 SNPs located in different regions of chromosome 1 are likely to be associated with increased risk for basal cell carcinoma (BCC), the most common form of skin cancer. *each A at rs7538876 yields a 1.28x increased risk of developing BCC (p = 4.4 x 10e-12) *each G at rs801114 also yields a 1.28x increased odds of developing BCC (p = 5.9 x 10e-12) *2.68x increased risk of BCC for homozygous carriers of both SNPs This study was based on an analysis of ~2,000 Icelandic and Eastern European skin cancer patients (as well as a large number of controls). The authors estimate that approximately 1.6% of all individuals of European ancestry are homozygous for both risk alleles, and thus that percentage of the population is at 2.68x higher risk for BCC compared to individuals who are rs7538876(G;G) a...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs261332'>rs261332</a><a href='http://www.snpedia.com/index.php/rs261332(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD0D0;'>34.8%</span><br/>Repute:Good<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>associated with higher HDL cholesterol</div>
        <div class='boxshow' id="show-sec-0.634661885147" style="display: inline;"><a href="javascript:toggle2('sec-0.634661885147')">...more...</a></div>
        <div class='boxrstext' id="sec-0.634661885147" style="display: none;"><a href="javascript:toggle2('sec-0.634661885147')">(hide)</a>
        <div class='rstext'>rs261332 increases susceptibility to Elevated HDL-Cholesterol for carriers of the A allele A allele is associated with 1.41mg/dl increase in HDL cholesterol (good cholesterol).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3758391'>rs3758391</a><a href='http://www.snpedia.com/index.php/rs3758391(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD2D2;'>36.3%</span><br/>Repute:Good<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Less mental decline with age.  One copy of the good version of the SIRT1 longevity gene. This is believed to slow the aging of the brain, and perhaps to increase longevity.</div>
        <div class='boxshow' id="show-sec-0.595847657043" style="display: inline;"><a href="javascript:toggle2('sec-0.595847657043')">...more...</a></div>
        <div class='boxrstext' id="sec-0.595847657043" style="display: none;"><a href="javascript:toggle2('sec-0.595847657043')">(hide)</a>
        <div class='rstext'>rs3758391, a SNP in the SIRT1 gene, has been reported to potentially have some role in either longevity, or with higher statistical significance, better cognitive function in older individuals. SIRT1 is thought to regulate neuronal metabolism and survival in response to stress, and the equivalent gene in other species influences maximal life span. The study reporting improved cognition for carriers of rs3758391(T) alleles was based on over 1,000 Finnish adults over the age of 85, and a weak trend towards lower cardiovascular disease was also reported for this allele.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs8070723'>rs8070723</a><a href='http://www.snpedia.com/index.php/rs8070723(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD5D5;'>39.8%</span><br/>Repute:Good<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.18x reduced risk of developing progressive supranuclear palsy</div>
        <div class='boxshow' id="show-sec-0.826545467705" style="display: inline;"><a href="javascript:toggle2('sec-0.826545467705')">...more...</a></div>
        <div class='boxrstext' id="sec-0.826545467705" style="display: none;"><a href="javascript:toggle2('sec-0.826545467705')">(hide)</a>
        <div class='rstext'>Each copy of the G version of rs8070723 was associated with about 5.5 times lower odds of progressive supranuclear palsy. The researchers identified smaller effects at rs1411478, rs7571971 and rs1768208. 23andMe blog Genomewide association study for susceptibility genes contributing to familial Parkinson disease. Genome-wide association study confirms SNPs in SNCA and the MAPT region as common risk factors for Parkinson disease.  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs8085664'>rs8085664</a><a href='http://www.snpedia.com/index.php/rs8085664(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD6D6;'>40.7%</span><br/>Repute:Good<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.78x reduced risk of Male Pattern Baldness.  Discovered by 23andWe based on customer surveys, and considered preliminary research.</div>
        <div class='boxshow' id="show-sec-0.188902425104" style="display: inline;"><a href="javascript:toggle2('sec-0.188902425104')">...more...</a></div>
        <div class='boxrstext' id="sec-0.188902425104" style="display: none;"><a href="javascript:toggle2('sec-0.188902425104')">(hide)</a>
        <div class='rstext'>This is the 23andWe discovery: 'Male Pattern Baldness: Preliminary Research'. It affects appearance.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1864163'>rs1864163</a><a href='http://www.snpedia.com/index.php/rs1864163(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD7D7;'>41.4%</span><br/>Repute:Good<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>associated with higher HDL cholesterol</div>
        <div class='boxshow' id="show-sec-0.340423993091" style="display: inline;"><a href="javascript:toggle2('sec-0.340423993091')">...more...</a></div>
        <div class='boxrstext' id="sec-0.340423993091" style="display: none;"><a href="javascript:toggle2('sec-0.340423993091')">(hide)</a>
        <div class='rstext'>G allele is associated with 4.12mg/dl increase in HDL cholesterol (good cholesterol).  [GWAS:HDL cholesterol]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9340799'>rs9340799</a><a href='http://www.snpedia.com/index.php/rs9340799(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD8D8;'>42.4%</span><br/>Repute:Good<br/>References:92<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Women have normal (lower) risk of Endometriosis, 0.75x lower risk of Endometrial Cancer, and normal (higher) risk of cognitive impairment.  Men have zero risk, but can pass this on to their daughters. One normal and one rare version of Estrogen Receptor 1 (alpha).</div>
        <div class='boxshow' id="show-sec-0.34815911387" style="display: inline;"><a href="javascript:toggle2('sec-0.34815911387')">...more...</a></div>
        <div class='boxrstext' id="sec-0.34815911387" style="display: none;"><a href="javascript:toggle2('sec-0.34815911387')">(hide)</a>
        <div class='rstext'>rs9340799 is a SNP in the estrogen alpha receptor ESR1 gene, and it is also known as the -351A&gt;G variant. Among women, rs8179176, rs9340799, rs1256065, and rs1256030 were associated with likelihood of developing cognitive impairment with ageing. In men, rs728524, rs1255998, and rs1256030 were associated with cognitive impairment. A study of 6,200+ Dutch individuals did not replicate previously reported associations between this SNP (rs9340799) and risk of ischemic stroke. A study of 702 cases with invasive endometrial cancer concluded that the less common alleles of SNP rs9340799 lowered risk for the disease. The odds ratio was 0.75 (CI: 0.60-0.93) for (A;G) and 0.53 (CI: 0.37-0.77) for (G;G) compared to rs9340799(A;A) individuals. Endometriosis Uterine Fibroids  [PharmGKB:Curated SNP i...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3764261'>rs3764261</a><a href='http://www.snpedia.com/index.php/rs3764261(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDADA;'>44.2%</span><br/>Repute:Good<br/>References:53<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>associated with higher HDL cholesterol</div>
        <div class='boxshow' id="show-sec-0.785587236069" style="display: inline;"><a href="javascript:toggle2('sec-0.785587236069')">...more...</a></div>
        <div class='boxrstext' id="sec-0.785587236069" style="display: none;"><a href="javascript:toggle2('sec-0.785587236069')">(hide)</a>
        <div class='rstext'>T allele is associated with 3.47mg/dl increase in HDL cholesterol (good cholesterol).  [PharmGKB:Non-Curated Phenotype: In a GWAS, this SNP was significantly associated with plasma concentrations of HDL-C and apolipoprotein A1. Study size: 6382. Study population/ethnicity: Caucasian women. Significance metric(s): HDL-C: p = 1.0 x 10 (-41);ApoA1: p = 1.1 x 10(-34). Type of association: CO; GN] [OMIM:?] [GWAS:HDL cholesterol]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2802292'>rs2802292</a><a href='http://www.snpedia.com/index.php/rs2802292(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDBDB;'>45.3%</span><br/>Repute:Good<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>One copy of longevity gene. Slightly increased lifespan.  This is a well known longevity gene, FOXO3, although only one of your copies is the good version. It seems to improve insulin sensitivity, and decrease oxidative stress, among other things.</div>
        <div class='boxshow' id="show-sec-0.952478766265" style="display: inline;"><a href="javascript:toggle2('sec-0.952478766265')">...more...</a></div>
        <div class='boxrstext' id="sec-0.952478766265" style="display: none;"><a href="javascript:toggle2('sec-0.952478766265')">(hide)</a>
        <div class='rstext'>A study of 5 genes in 3,741 Japanese men, some of whom lived significantly longer than average (centarians), concluded that the FOXO3 gene (also known as FOXO3A) harbors several SNPs linked to longevity, including this SNP, rs2802292, as well as rs2764264 and rs13217795. The odds ratio for rs2802292(G;G) vs (T;T) carriers was 2.75 (p = 0.00009; adjusted p = 0.00135). It is not known yet if this correlation is specific to Japanese males. This is discussed extensively at geneticfuture as a longevity snp. *(T) shorter lifespan *(G) longer lifespan Quoted from the geneticfuture article: 'No-one should consider this association to be definitive until it's been replicated in independent cohorts. Secondly, the effects of this variant are likely to differ between groups. According to the HapMap da...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs763110'>rs763110</a><a href='http://www.snpedia.com/index.php/rs763110(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDEDE;'>48.6%</span><br/>Repute:Good<br/>References:19<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~0.80x reduced cancer risk</div>
        <div class='boxshow' id="show-sec-0.803216649458" style="display: inline;"><a href="javascript:toggle2('sec-0.803216649458')">...more...</a></div>
        <div class='boxrstext' id="sec-0.803216649458" style="display: none;"><a href="javascript:toggle2('sec-0.803216649458')">(hide)</a>
        <div class='rstext'>rs763110, also known as -844C&gt;T, is a SNP in the FAS ligand FASLG gene. A meta-analysis of 19 published studies, including 11,105 cancer cases and 11,372 controls, concluded that the rs763110(C;T) and (T;T) genotypes were associated with a significantly reduced (about 0.80x) cancer risk of all cancer types.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3738579'>rs3738579</a><a href='http://www.snpedia.com/index.php/rs3738579(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDEDE;'>49.6%</span><br/>Repute:Good<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.5x decreased risk for cervical cancer, HNSCC, and breast cancer</div>
        <div class='boxshow' id="show-sec-0.927723141783" style="display: inline;"><a href="javascript:toggle2('sec-0.927723141783')">...more...</a></div>
        <div class='boxrstext' id="sec-0.927723141783" style="display: none;"><a href="javascript:toggle2('sec-0.927723141783')">(hide)</a>
        <div class='rstext'>rs3738579 represents a SNP in the 5' UTR region upstream of the RNASEL gene. A study of patients diagnosed with carcinoma of the uterine cervix, head and neck squamous cell carcinomas (HNSCC), and breast cancer found 1.5x-2x increased risk for all three cancer types for the rs3738579(T;T) genotype, while finding decreased risk (0.5x) for rs3738579(C;T) heterozygotes. rs3738579(C;C) homozygotes had 0.6x less risk for cervical cancer but increased risk for HNSCC (1.4x) and breast cancer (1.8x). Although statistics were not reported per genotype, a combination of data from all three cancer forms over all genotypes provided strong statistical evidence for rs3738579 as a cancer marker, with a p-value of 4.43x10(-5).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6311'>rs6311</a><a href='http://www.snpedia.com/index.php/rs6311(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE1E1;'>52.2%</span><br/>Repute:Good<br/>References:51<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of sexual dysfunction when taking SSRI Antidepressants.</div>
        <div class='boxshow' id="show-sec-0.134767039925" style="display: inline;"><a href="javascript:toggle2('sec-0.134767039925')">...more...</a></div>
        <div class='boxrstext' id="sec-0.134767039925" style="display: none;"><a href="javascript:toggle2('sec-0.134767039925')">(hide)</a>
        <div class='rstext'>rs6311 (-1438A&gt;G / A-1438G or -1438G&gt;A / G-1438A) is a SNP located upstream (and within the promoter region of) HTR2A. Multiple Regulatory Variants Modulate Expression of 5-Hydroxytryptamine 2A Receptors in Human Cortex. The minor A allele of rs6311 reduces expression of a previously unannotated extension of the 5' untranslated region of HTR2A mRNA. rs6311 together with rs6314 have a modest effect on depression severity, when examined in the Sequenced Treatment Alternatives to Relieve Depression (STAR*D) study. Three (rs643627-rs594242-rs6311: A-C-T), two (rs594242-rs6311: C-T) and a single functional (rs6311: T) marker were protective against suicidal behavior. The complementary markers (rs594242-rs6311: G-C and rs6311: C) were associated with increased risk for non-violent and impu...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10468017'>rs10468017</a><a href='http://www.snpedia.com/index.php/rs10468017(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE1E1;'>52.7%</span><br/>Repute:Good<br/>References:18<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>associated with higher HDL cholesterol</div>
        <div class='boxshow' id="show-sec-0.858492306025" style="display: inline;"><a href="javascript:toggle2('sec-0.858492306025')">...more...</a></div>
        <div class='boxrstext' id="sec-0.858492306025" style="display: none;"><a href="javascript:toggle2('sec-0.858492306025')">(hide)</a>
        <div class='rstext'>T allele is associated with 1.76mg/dl increase in HDL cholesterol (good cholesterol).  [PharmGKB:Non-Curated GWAS results: Common variants at 30 loci contribute to polygenic dyslipidemia. (Initial Sample Size: 19,840 individuals; Replication Sample Size: Up to 20,623 individuals); (Region: 15q22.1; Reported Gene(s): LIPC; Risk Allele: rs10468017-T); (p-value= 8E-23).This variant is associated with HDL cholesterol.] [GWAS:HDL cholesterol]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10936599'>rs10936599</a><a href='http://www.snpedia.com/index.php/rs10936599(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE1E1;'>53.1%</span><br/>Repute:Good<br/>References:14<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>longer telomeres, longer life?</div>
        <div class='boxshow' id="show-sec-0.784455568566" style="display: inline;"><a href="javascript:toggle2('sec-0.784455568566')">...more...</a></div>
        <div class='boxrstext' id="sec-0.784455568566" style="display: none;"><a href="javascript:toggle2('sec-0.784455568566')">(hide)</a>
        <div class='rstext'>[GWAS:Celiac disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4143094'>rs4143094</a><a href='http://www.snpedia.com/index.php/rs4143094(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE6E6;'>58.5%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>no increased risk of colorectal cancer correlated with consumption of processed meats</div>
        <div class='boxshow' id="show-sec-0.216928719212" style="display: inline;"><a href="javascript:toggle2('sec-0.216928719212')">...more...</a></div>
        <div class='boxrstext' id="sec-0.216928719212" style="display: none;"><a href="javascript:toggle2('sec-0.216928719212')">(hide)</a>
        <div class='rstext'>'''rs4143094''' is a SNP identified by a genome-wide diet-gene interaction analysis (GxE), and was found to be associated with increased risk of colon cancer correlated to the dietary variable of processed meat consumption. It is located on chromosome 10p14, 7.2kb upstream of GATA binding protein 3 (''GATA3''), in the promoter region of the gene rs4143094 was identified in a screen of 2.7 million SNPs for the risk of colorectal cancer, by combining 10 studies comparing 9287 cases and 9117 controls. Data was obtained from the Genetics and Epidemiology of Colorectal Cancer Consortium (GECCO) and analyzed by conventional case-control logistic regression for multiplicative interactions between SNPs and dietary factors. rs4143094 was the most significantly implicated SNP in 10p14 for processed ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2854464'>rs2854464</a><a href='http://www.snpedia.com/index.php/rs2854464(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE7E7;'>60.2%</span><br/>Repute:Good<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>higher muscle strength  the high muscle strength mutation at ACVR1B, specifically tested on the knees</div>
        <div class='boxshow' id="show-sec-0.0400817879922" style="display: inline;"><a href="javascript:toggle2('sec-0.0400817879922')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0400817879922" style="display: none;"><a href="javascript:toggle2('sec-0.0400817879922')">(hide)</a>
        <div class='rstext'></div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1799884'>rs1799884</a><a href='http://www.snpedia.com/index.php/rs1799884(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFECEC;'>67.3%</span><br/>Repute:Good<br/>References:16<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Mothers have typical Birth-Weight babies. Slightly lower risk of Type 2 Diabetes.</div>
        <div class='boxshow' id="show-sec-0.0690187759461" style="display: inline;"><a href="javascript:toggle2('sec-0.0690187759461')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0690187759461" style="display: none;"><a href="javascript:toggle2('sec-0.0690187759461')">(hide)</a>
        <div class='rstext'>rs1799884, known also as the -30 SNP of the GCK gene, has been associated with type-2 diabetes. This SNP is one of 4 relatively common SNPs reported to represent risk for type-2 diabetes in the DESIR prospective study of 3,877 Caucasian participants. Under a recessive model, the odds ratio for rs1799884(A;A) homozygotes is 2.70 (CI: 1.51-4.83, p=0.0008); under an additive model, the odds ratio is 1.34 (CI: 1.07-1.69, p=0.01). For the 4 SNPs, each risk allele increased type-2 diabetes risk by 1.34x (p=2x10e-6), with an odds ratio of 2.48 (CI: 1.59-3.86) for carriers of 4 or more compared to those with one or none (risk alleles). A common haplotype of the glucokinase gene alters fasting glucose and birth weight: association in six studies and population-genetics analyses. Type 2 diabetes TCF...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs671'>rs671</a><a href='http://www.snpedia.com/index.php/rs671(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFFFFF;'>100.0%</span><br/>Repute:Good<br/>References:52<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Alcohol Flush: Normal, doesn't flush. Normal hangovers. Normal risk of Alcoholism. Normal risk of Esophageal Cancer. Disulfiram is effective for alcoholism.  Two working copies of Aldehyde Dehydrogenase (ALDH2) mean you won't turn bright red when you drink alcohol, unlike most Asians. After your body turns alcohol into the dangerous acetaldehyde, this gene quickly turns the acetaldehyde into vinegar. It also stops the acetaldehyde in cigarettes from causing Esophageal Cancer. It reduces hangovers compared to Asians, but less hangovers and flushing obviously increases the risk of alcoholism. Alcoholism can be treated effectively with the drug Disulfiram (Antabuse) which would stop this gene from working and cause the Asian flush in non-Asians. PMC2659709 [http://www.ncbi.nlm.nih.gov/pmc/art...</div>
        <div class='boxshow' id="show-sec-0.0722395005404" style="display: inline;"><a href="javascript:toggle2('sec-0.0722395005404')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0722395005404" style="display: none;"><a href="javascript:toggle2('sec-0.0722395005404')">(hide)</a>
        <div class='rstext'>rs671 is a classic SNP, well known in a sense through the phenomena known as the 'alcohol flush', also known as the 'Asian Flush' or 'Asian blush', in which certain individuals, often of Asian descent, have their face, neck and sometimes shoulders turn red after drinking alcohol. The rs671(A) allele of the ALDH2 gene is the culprit, in that it encodes a form of the aldehyde dehydrogenase 2 protein that is defective at metabolizing alcohol. This allele is known as the ALDH*2 form, and individuals possessing either one or two copies of it show alcohol-related sensitivity responses including facial flushing, and severe hangovers (and hence they are usually not regular drinkers). Perhaps not surprisingly they appear to suffer less from alcoholism and alcohol-related liver disease. Disulfiram (...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7927894'>rs7927894</a><a href='http://www.snpedia.com/index.php/rs7927894(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.9<br/>Frequency: <span style='background-color: #FFD9D9;'>43.4%</span><br/>Repute:Good<br/>References:12<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of Atopic Dermatitis.</div>
        <div class='boxshow' id="show-sec-0.575706533766" style="display: inline;"><a href="javascript:toggle2('sec-0.575706533766')">...more...</a></div>
        <div class='boxrstext' id="sec-0.575706533766" style="display: none;"><a href="javascript:toggle2('sec-0.575706533766')">(hide)</a>
        <div class='rstext'>23andMe blog associated rs7927894(T) with Crohn&#8217;s disease 1.16x and atopic dermatitis  [PharmGKB:Non-Curated GWAS results: A common variant on chromosome 11q13 is associated with Dermatitis, Atopic. (Initial Sample Size: 939 cases, 975 controls, 1,097 family members; Replication Sample Size: 2,637 cases, 3,957 controls); (Region: 11q13.5; Reported Gene(s): C11orf30; Risk Allele: rs7927894-A); (p-value= 0.0000000008).This variant is associated with Dermatitis, Atopic.] [OMIM:?] [GWAS:Atopic dermatitis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2200733'>rs2200733</a><a href='http://www.snpedia.com/index.php/rs2200733(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.9<br/>Frequency: <span style='background-color: #FFF3F3;'>78.8%</span><br/>Repute:Good<br/>References:37<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.86x decreased risk of Atrial Fibrillation  This is the normal version which makes the heart form properly and makes people less likely to get Atrial Fibrillation (quivering of the top chambers of the heart caused by chaotic electrical signals) and less likely to have an Ischemic Stroke (blocked blood flow to the brain). But also check the other SNP nearby (rs10033464) which slightly affects Atrial Fibrillation too. 23andMe gave this a 4 star confidence rating as it has been heavily studied. It has been confirmed in Europeans and Asians.</div>
        <div class='boxshow' id="show-sec-0.373591982584" style="display: inline;"><a href="javascript:toggle2('sec-0.373591982584')">...more...</a></div>
        <div class='boxrstext' id="sec-0.373591982584" style="display: none;"><a href="javascript:toggle2('sec-0.373591982584')">(hide)</a>
        <div class='rstext'>Two SNPs from chromosome 4q25, rs2200733 and rs10033464, were found to be associated with atrial fibrillation in a study of both European and Asian populations. The odds ratio associated with one or more copies of either risk allele was ~1.4x. The results derived from a study of 78 Italians with atrial fibrillation (AF) and atrial flutter (AFL) agree with previously reported findings from the Icelandic study (), which also found that the rs2200733(T) was associated with AF/AFL disease. In 1,661 Icelandic ischemic stroke samples and two large European replication sets combined, rs2200733 was associated with cardioembolic stroke (CES) (odds ratio 1.54, p = 8.05 x 10e-9). rs2200733 even showed association to ischemic stroke that was not classified as cardioembolic stroke. rs2200733 and rs1003...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1052576'>rs1052576</a><a href='http://www.snpedia.com/index.php/rs1052576(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.7<br/>Frequency: <span style='background-color: #FFC9C9;'>29.2%</span><br/>Repute:Good<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.5x reduced risk of multiple myeloma, general reduced risk of cancer  A number of small studies with fairly low p-values, but with low heterogeneity across studies, show a protective effect for the A allele which is greater in homozygotes.</div>
        <div class='boxshow' id="show-sec-0.0746281910304" style="display: inline;"><a href="javascript:toggle2('sec-0.0746281910304')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0746281910304" style="display: none;"><a href="javascript:toggle2('sec-0.0746281910304')">(hide)</a>
        <div class='rstext'>also known as Q221R rs1052576 is located on chromosome 1p36.21 in the coding region of CASP9. The CAG to CGG transition results in the allele Q221R. This SNP is associated with ADHD, general cancer risk, multiple myeloma, non-small cell lung cancer and non-Hodgkin lymphoma .Generally the A allele confers protection from a variety of cancers. An intact apoptotic pathway is essential for many cellular processes including development, immune response and prevention of cellular transformation. Caspases are cysteinyl aspartate proteases involved in signal transduction cascades that result in programmed cell death. CASP9 is an initiator caspase and part of the intrinsic pathway, which is an evolutionarily conserved apoptotic pathway. It is responsible for cleaving and activating the executioner ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3843763'>rs3843763</a><a href='http://www.snpedia.com/index.php/rs3843763(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.6<br/>Repute:Good<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal (higher) HDL %22Good%22 Cholesterol.  HDL ('Good') cholesterol actually cleans your arteries and transports cholesterol out of the body, so it is usually good to have higher HDL cholesterol.</div>
        <div class='boxshow' id="show-sec-0.678284899552" style="display: inline;"><a href="javascript:toggle2('sec-0.678284899552')">...more...</a></div>
        <div class='boxrstext' id="sec-0.678284899552" style="display: none;"><a href="javascript:toggle2('sec-0.678284899552')">(hide)</a>
        <div class='rstext'>The (T) allele of rs3843763 was associated with risk for lower high-density lipoprotein (HDL) cholesterol plasma levels in 3 independent population samples, including both Caucasians and African-Americans.  [PharmGKB:Curated This SNP has been associated with small changes in HDL cholesterol levels. The T allele has been correlated with lower HDL.] [OMIM:PHOSPHOLIPID TRANSFER PROTEIN; PLTP]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6427528'>rs6427528</a><a href='http://www.snpedia.com/index.php/rs6427528(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFA7A7;'>11.1%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>for rheumatoid arthritis patients, better response to etanercept treatment</div>
        <div class='boxshow' id="show-sec-0.902646942062" style="display: inline;"><a href="javascript:toggle2('sec-0.902646942062')">...more...</a></div>
        <div class='boxrstext' id="sec-0.902646942062" style="display: none;"><a href="javascript:toggle2('sec-0.902646942062')">(hide)</a>
        <div class='rstext'>rs6427528 is a SNP in the chromosome 1q23 region apparently located near enough to the CD84 locus to affect it's expression. A GWAS study of 2,700+ rheumatoid arthritis (RA) patients treated with one of three anti-TNF drugs found that rs6427528(A) carriers had higher CD84 expression and responded better to etanercept treatment. The effect was statistically significant but not that large in magnitude, totaling 2.6% of the variance in response to etanercept treatment. The result was seen in European (Caucasian) patients, but perhaps due to small patient numbers, was not statistically supported in Japanese patients.  [GWAS:Response to anti-TNF treatment in rheumatoid arthritis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6812193'>rs6812193</a><a href='http://www.snpedia.com/index.php/rs6812193(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFAFAF;'>14.2%</span><br/>Repute:Good<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly decreased risk of developing Parkinson's Disease  As reported in European populations, but with failure to reproduce in Greek or Chinese populations.</div>
        <div class='boxshow' id="show-sec-0.588486380119" style="display: inline;"><a href="javascript:toggle2('sec-0.588486380119')">...more...</a></div>
        <div class='boxrstext' id="sec-0.588486380119" style="display: none;"><a href="javascript:toggle2('sec-0.588486380119')">(hide)</a>
        <div class='rstext'>People with two copies of the T allele at rs6812193 appear to have a slightly lower risk of Parkinson's disease. The same researchers identified another risk allele for Parkinson's disease at rs11868035. plos  [GWAS:Parkinson's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs420259'>rs420259</a><a href='http://www.snpedia.com/index.php/rs420259(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFCBCB;'>31.0%</span><br/>Repute:Good<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Possibly reduced risk of Bipolar Disorder.  This appears to lower the risk of Bipolar Disorder compared to the more common version of the gene, based on preliminary research.</div>
        <div class='boxshow' id="show-sec-0.133453435081" style="display: inline;"><a href="javascript:toggle2('sec-0.133453435081')">...more...</a></div>
        <div class='boxrstext' id="sec-0.133453435081" style="display: none;"><a href="javascript:toggle2('sec-0.133453435081')">(hide)</a>
        <div class='rstext'>Linked to bipolar disorder in one of the most comprehensive studies in 2007. Risk allele with reference to dbSNP orientation is reported to be (T), with either one or two copies leading to an odds ratio of 2 (CI 1.6-2.7). Some replications, but in 2011 23andMe published as failed to replicate in Pharmacogenetics: data, concepts and tools to improve drug discovery and drug treatment. Findings from bipolar disorder genome-wide association studies replicate in a Finnish bipolar family-cohort. Simultaneous genotype calling and haplotype phasing improves genotype accuracy and reduces false-positive associations for genome-wide association studies.  [PharmGKB:Non-Curated GWAS Results: Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls (Initial Sample...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4939883'>rs4939883</a><a href='http://www.snpedia.com/index.php/rs4939883(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFCDCD;'>32.7%</span><br/>Repute:Good<br/>References:17<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>associated with higher HDL cholesterol</div>
        <div class='boxshow' id="show-sec-0.193165146444" style="display: inline;"><a href="javascript:toggle2('sec-0.193165146444')">...more...</a></div>
        <div class='boxrstext' id="sec-0.193165146444" style="display: none;"><a href="javascript:toggle2('sec-0.193165146444')">(hide)</a>
        <div class='rstext'>C allele is associated with 1.87mg/dl increase in HDL cholesterol (good cholesterol).  [PharmGKB:Non-Curated GWAS results: Common variants at 30 loci contribute to polygenic dyslipidemia. (Initial Sample Size: 19,840 individuals; Replication Sample Size: Up to 20,623 individuals); (Region: 18q21.1; Reported Gene(s): LIPG; Risk Allele: rs4939883-T); (p-value= 0.000000000000007).This variant is associated with HDL cholesterol.] [GWAS:Cholesterol, total]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs5443'>rs5443</a><a href='http://www.snpedia.com/index.php/rs5443(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFD0D0;'>34.9%</span><br/>Repute:Good<br/>References:40<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>normal, but higher risk for several conditions  ostensibly the 'normal' form, however variation is common, and each genotype seems to show some good and bad consequences.</div>
        <div class='boxshow' id="show-sec-0.588410002507" style="display: inline;"><a href="javascript:toggle2('sec-0.588410002507')">...more...</a></div>
        <div class='boxrstext' id="sec-0.588410002507" style="display: none;"><a href="javascript:toggle2('sec-0.588410002507')">(hide)</a>
        <div class='rstext'>rs5443, a SNP in the G-protein beta3 subunit (GNB3) gene that is more commonly known as the C825T variant, has been linked to a number of metabolic conditions including obesity, coronary artery disease, insulin resistance and therefore diabetes, left ventricular hypertrophy, and hypertension. It has also been linked to how well a patient responds to Viagra (sildenafil). Several studies have been unable to replicate one or more of the associations in at least some populations between this SNP and these conditions. The more notable studies include: *rs5443(T) allele carriers are 2-3 fold more likely to be obese in Caucasian, Chinese, and African American populations. *rs5443(T) carriers are clearly at higher risk for hypertension, but this review indicates that whether they are also at incre...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1892534'>rs1892534</a><a href='http://www.snpedia.com/index.php/rs1892534(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDADA;'>44.6%</span><br/>Repute:Good<br/>References:12<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly higher C-Reactive Protein levels. Possibly slightly lower cancer risk.  This causes a median hsCRP Level of 2.09 mg/L in women. Strangely this was associated with a slightly lower overall cancer risk (in Finnish people).</div>
        <div class='boxshow' id="show-sec-0.265589241764" style="display: inline;"><a href="javascript:toggle2('sec-0.265589241764')">...more...</a></div>
        <div class='boxrstext' id="sec-0.265589241764" style="display: none;"><a href="javascript:toggle2('sec-0.265589241764')">(hide)</a>
        <div class='rstext'>rs1892534 is a SNP that was originally associated with C-reactive protein levels, and given other findings that suggest C-reactive protein levels may be predictive of cancer risk, was thought to be linked to cancer risk. A large study pooling data from 3 Finnish studies totaling over 18,000 individuals concluded that while the rs1892534(A) allele was not likely to be causative (relative to cancer), it is associated with increased risk for cancer overall. The odds ratio for 1 or 2 such alleles was 1.05 (CI: 0.90 - 1.23) and 1.2 (CI: 1.01 - 1.42) overall. Two other CRP SNPs (rs1169300 and rs2464196) were specifically associated with increased lung cancer risk. CRP SNPs were not associated with colorectal, prostate or breast cancer risk.  [PharmGKB:Curated GWAS Results: Loci Related to Metabo...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3851179'>rs3851179</a><a href='http://www.snpedia.com/index.php/rs3851179(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDCDC;'>46.9%</span><br/>Repute:Good<br/>References:32<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.85x decreased risk for Alzheimer's disease</div>
        <div class='boxshow' id="show-sec-0.85684620446" style="display: inline;"><a href="javascript:toggle2('sec-0.85684620446')">...more...</a></div>
        <div class='boxrstext' id="sec-0.85684620446" style="display: none;"><a href="javascript:toggle2('sec-0.85684620446')">(hide)</a>
        <div class='rstext'>rs3851179 is a SNP upstream of the PICALM gene. A study of over 5,000 Alzheimer's disease patients (and over 10,000 controls) found a slight protective effect of the (A) allele of this SNP. rs3851179(A) carriers had a slightly decreased risk for Alzheimer's disease, with an odds ratio of 0.85 (CI: 0.8-0.9, p=1.9x10e-8). The association of this SNP and late-onset Alzheimer's disease was replicated over a total of another 1829 cases (and 2576 controls).  [GWAS:Alzheimer's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1063192'>rs1063192</a><a href='http://www.snpedia.com/index.php/rs1063192(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDDDD;'>47.8%</span><br/>Repute:Good<br/>References:17<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.71x reduced risk of myocardial infarction</div>
        <div class='boxshow' id="show-sec-0.931553041821" style="display: inline;"><a href="javascript:toggle2('sec-0.931553041821')">...more...</a></div>
        <div class='boxrstext' id="sec-0.931553041821" style="display: none;"><a href="javascript:toggle2('sec-0.931553041821')">(hide)</a>
        <div class='rstext'>rs1063192 is a SNP in the cyclin-dependent kinase inhibitor 2B CDKN2B gene. A study of 432 Han Chinese patients with myocardial infarctions concluded that male subjects carrying a rs1063192(C) allele were at 0.71x decreased risk (for MI). Myocardial Infarction  [GWAS:Vertical cup-disc ratio]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1050631'>rs1050631</a><a href='http://www.snpedia.com/index.php/rs1050631(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDFDF;'>50.4%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Mean Survival Time of 32 months for esophageal squamous-cell carcinoma</div>
        <div class='boxshow' id="show-sec-0.502643188301" style="display: inline;"><a href="javascript:toggle2('sec-0.502643188301')">...more...</a></div>
        <div class='boxrstext' id="sec-0.502643188301" style="display: none;"><a href="javascript:toggle2('sec-0.502643188301')">(hide)</a>
        <div class='rstext'>rs1050631 is a SNP on chromosome 18q12 in exon 6 of SLC39A6 gene, associated with length of survival in esophageal squamous-cell carcinoma(ESCC). The SNP is located in the 5&#8217; UTR of SLC39A6. In a study, a genome-wide scan of SNPs to identify variants associated with length of survival in 1331 individuals with ESCC was performed, and was validated by two independent sets, including 1962 individuals with ESCC (1308 and 654 individuals in two different replications). The hazard ratio for death (odds ratio) (T; T) and (C; T) from ESCC in the 1331 test cases was 1.34 (95% confidence interval (CI) = 1.17&#8722;1.53; P = 7.78 E-6). For the combined data, hazard ratio of death similarly was found to be 1.34 (95% CI = 1.17&#8722;1.53; P = 7.78 E&#8722;6). The combined discovery and replicatio...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1165205'>rs1165205</a><a href='http://www.snpedia.com/index.php/rs1165205(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #F0B5F0;'>22.3%</span><br/>Repute:Good<br/>References:9<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.85x decreased gout risk</div>
        <div class='boxshow' id="show-sec-0.355774055516" style="display: inline;"><a href="javascript:toggle2('sec-0.355774055516')">...more...</a></div>
        <div class='boxrstext' id="sec-0.355774055516" style="display: none;"><a href="javascript:toggle2('sec-0.355774055516')">(hide)</a>
        <div class='rstext'>rs1165205 is a SNP in the SLC17A3 gene on ch 6. A large study totaling 7,699 participants in the Framingham cohort and 4,148 participants in the Rotterdam cohort was conducted, with genome-wide significant SNPs then replicated in 11,000+ Caucasian and ~4,000 African-American individuals who took part in the study of Atherosclerosis Risk in Communities (ARIC). This study calculated an odds ratio of 0.85 for rs1165205, (CI: 0.77-0.94, p = 0.002). A genetic score comprised of this SNP plus 2 others may stratify risk for gout. See also: 23andMe blog gout *rs2231142(T) 1.7x *rs16890979(C) 1.7x *rs1165205(A) 1.2x  [PharmGKB:Non-Curated Association of three genetic loci with uric acid concentration and risk of gout: a genome-wide association study. (Initial Sample Size: 11,847 individuals; Replic...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10050860'>rs10050860</a><a href='http://www.snpedia.com/index.php/rs10050860(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.4<br/>Frequency: <span style='background-color: #FFD3D3;'>37.2%</span><br/>Repute:Good<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.71x lower risk for Ankylosing Spondylitis.  Very low risk (about 0.07%) of the rare autoimmune disease that causes inflammation of the spine and joints. Based on preliminary research.</div>
        <div class='boxshow' id="show-sec-0.602321533192" style="display: inline;"><a href="javascript:toggle2('sec-0.602321533192')">...more...</a></div>
        <div class='boxrstext' id="sec-0.602321533192" style="display: none;"><a href="javascript:toggle2('sec-0.602321533192')">(hide)</a>
        <div class='rstext'>rs10050860 is one of several SNPs in the ERAP1 gene that has been shown in a large (over 1,000 Caucasian patients) study to be associated with ankylosing spondylitis. The odds ratio is 0.71 (p=7.7x10e-9). Note that the text description (but not the result) at 23andMe is wrong. It is the uncommon T allele that gives protection. Association of an ERAP1 ERAP2 haplotype with familial ankylosing spondylitis. Investigating the genetic association between ERAP1 and ankylosing spondylitis. Serum cytokine receptors in ankylosing spondylitis: relationship to inflammatory markers and endoplasmic reticulum aminopeptidase polymorphisms. ERAP1 polymorphisms and haplotypes are associated with ankylosing spondylitis susceptibility and functional severity in a Spanish population. Associations between ERAP1...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs184"  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs184'>gs184</a></div></div>
        <div class='boxmag'>Magnitude: 1.2</div>
        <div class='boxrep'>Repute: Good</div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>Able to taste bitterness.</div>
        <div class='gsboxeffect'>Normal tasting ability for bitterness. This makes turnip, cabbage, brussels sprouts, broccoli, and other cruciferous vegetables taste more interesting and less horribly bland. Coffee and dark beers also tastes more bitter. You can taste propylthiouracil (PROP), PTC, and related chemicals. On the other hand, you can't taste the unpleasant bitterness of the tropical fruit from the Bignay tree, it will taste sweet to you. You probably eat healthier.</div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2070074'>rs2070074</a><a href='http://www.snpedia.com/index.php/rs2070074(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFC3C3;'>24.6%</span><br/>Repute:Good<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>harmless ancestral galactosemia variant carrier  This variant is evolutionarily ancestral, and in vitro studies fail to support an impact of this variant on enzyme activity.</div>
        <div class='boxshow' id="show-sec-0.415093931055" style="display: inline;"><a href="javascript:toggle2('sec-0.415093931055')">...more...</a></div>
        <div class='boxrstext' id="sec-0.415093931055" style="display: none;"><a href="javascript:toggle2('sec-0.415093931055')">(hide)</a>
        <div class='rstext'>Galactosemia  [OMIM:GALACTOSE-1-PHOSPHATE URIDYLYLTRANSFERASE; GALT]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11190302'>rs11190302</a><a href='http://www.snpedia.com/index.php/rs11190302(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFD3D3;'>37.2%</span><br/>Repute:Good<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'></div>
        <div class='boxshow' id="show-sec-0.196264712709" style="display: inline;"><a href="javascript:toggle2('sec-0.196264712709')">...more...</a></div>
        <div class='boxrstext' id="sec-0.196264712709" style="display: none;"><a href="javascript:toggle2('sec-0.196264712709')">(hide)</a>
        <div class='rstext'>rs11190302 increases susceptibility to late-onset Alzheimer's disease 1.39 times for heterozygotes (CT) and 2.07 times for homozygotes (TT).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4320932'>rs4320932</a><a href='http://www.snpedia.com/index.php/rs4320932(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFD3D3;'>37.2%</span><br/>Repute:Good<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.87x decreased risk for ovarian cancer</div>
        <div class='boxshow' id="show-sec-0.612151310292" style="display: inline;"><a href="javascript:toggle2('sec-0.612151310292')">...more...</a></div>
        <div class='boxrstext' id="sec-0.612151310292" style="display: none;"><a href="javascript:toggle2('sec-0.612151310292')">(hide)</a>
        <div class='rstext'>rs4320932 is a SNP in the IF2 insulin-like growth factor 2 gene. A study totaling ~5,000 non-Hispanic white patients with ovarian cancer found a 13% decreased risk of ovarian cancer per copy of the rs4320932(G) allele carried (CI: 0.81-0.93, P-trend=7.4 &#215; 10e-5). Associated with schizophrenia in a study of Croatians. Due to the limited number (~100 patients) and diversity of the studied population, more study is needed to confirm any link. The G allele was associated with an increased risk of schizophrenia with P &lt; 0.001. Small size study with 212 cases linked the G allele earlier found protective against ovarian cancer increasing relapse by 3.05 (95% confidence interval CI: 1.47&#8211;6.37, P-trend 2*10e-4) and mortality 3.28 (95% CI: 1.64&#8211;6.57, P-trend 6*10e-5). Association...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs30187'>rs30187</a><a href='http://www.snpedia.com/index.php/rs30187(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFDCDC;'>46.9%</span><br/>Repute:Good<br/>References:28<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal low risk (0.1%) for ankylosing spondylitis.  Normal very low risk (1 in a thousand) of this rare autoimmune disease that causes inflammation of the spine and joints. Verified in Europeans and Asians.</div>
        <div class='boxshow' id="show-sec-0.746007508377" style="display: inline;"><a href="javascript:toggle2('sec-0.746007508377')">...more...</a></div>
        <div class='boxrstext' id="sec-0.746007508377" style="display: none;"><a href="javascript:toggle2('sec-0.746007508377')">(hide)</a>
        <div class='rstext'>rs30187 is one of several SNPs in the ERAP1 gene that has been shown in a large (over 1,000 Caucasian patients) study to be associated with ankylosing spondylitis. Each T allele appears to increase the odds by about 1.4x (p=3.4x10e-10). A study of 872 Korean patients found that each T allele was associated with 1.5x higher odds of ankylosing spondylitis.  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11246226'>rs11246226</a><a href='http://www.snpedia.com/index.php/rs11246226(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFE4E4;'>55.8%</span><br/>Repute:Good<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>decreased risk of schizophrenia in limited study</div>
        <div class='boxshow' id="show-sec-0.0190979648445" style="display: inline;"><a href="javascript:toggle2('sec-0.0190979648445')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0190979648445" style="display: none;"><a href="javascript:toggle2('sec-0.0190979648445')">(hide)</a>
        <div class='rstext'>Associated with schizophrenia in a study of Croatians. Due to the limited number and diversity of the studied population, more study is needed to confirm any link. The C allele was associated with an decreased risk of schizophrenia.  [PharmGKB:Curated This variant along with 3 other SNPs near DRD4 (4 SNP haplotype: rs3758653; rs916457; rs762502; rs11246226) is associated with tardive dyskinesia in male schizophrenia patients.]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs610932'>rs610932</a><a href='http://www.snpedia.com/index.php/rs610932(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFE5E5;'>57.1%</span><br/>Repute:Good<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'></div>
        <div class='boxshow' id="show-sec-0.0234076274094" style="display: inline;"><a href="javascript:toggle2('sec-0.0234076274094')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0234076274094" style="display: none;"><a href="javascript:toggle2('sec-0.0234076274094')">(hide)</a>
        <div class='rstext'>Alzheimer's disease associated, based on large 2011 study Genome-wide association study identifies variants at CLU and PICALM associated with Alzheimer's disease. The prevalence of CD33 and MS4A6A variant in Chinese Han population with Alzheimer's disease.  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs13333226'>rs13333226</a><a href='http://www.snpedia.com/index.php/rs13333226(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFCACA;'>30.1%</span><br/>Repute:Good<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>slightly lower risk for hypertension</div>
        <div class='boxshow' id="show-sec-0.762860492057" style="display: inline;"><a href="javascript:toggle2('sec-0.762860492057')">...more...</a></div>
        <div class='boxrstext' id="sec-0.762860492057" style="display: none;"><a href="javascript:toggle2('sec-0.762860492057')">(hide)</a>
        <div class='rstext'>rs13333226 is a SNP in the 5' region of uromodulin UMOD gene. A large GWAS study ultimately totaling over 20,000 cases and almost as many controls concluded that, compared to the far more common allele, the rarer rs13333226(G) allele is associated with a (slightly) lower risk of hypertension (odds ratio 0.87, CI: 0.84-0.91) as well as a 7.7% reduction per allele for risk of cardiovascular events after adjusting for age, sex, BMI, and smoking status (hazard ratio?0.923, CI: 0.860-0.991; p?=?0.027). High Blood Pressure (Hypertension) Kidney Disease</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2229765'>rs2229765</a><a href='http://www.snpedia.com/index.php/rs2229765(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFCACA;'>30.1%</span><br/>Repute:Good<br/>References:14<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal IGF-1 level</div>
        <div class='boxshow' id="show-sec-0.350570729103" style="display: inline;"><a href="javascript:toggle2('sec-0.350570729103')">...more...</a></div>
        <div class='boxrstext' id="sec-0.350570729103" style="display: none;"><a href="javascript:toggle2('sec-0.350570729103')">(hide)</a>
        <div class='rstext'>rs2229765(A;A) associated with ischemic stroke (OR = 1.641, P = 0.022) in a Chinese population</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs795484'>rs795484</a><a href='http://www.snpedia.com/index.php/rs795484(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFDBDB;'>45.5%</span><br/>Repute:Good<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal morphine dose requirement and postoperative pain in tonsillectomy and adenoidectomy</div>
        <div class='boxshow' id="show-sec-0.0468289140962" style="display: inline;"><a href="javascript:toggle2('sec-0.0468289140962')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0468289140962" style="display: none;"><a href="javascript:toggle2('sec-0.0468289140962')">(hide)</a>
        <div class='rstext'>Each minor allele at rs795484 (guanine [G]&gt;adenine [A]) contributed +17.6 &#956;g/kg (95% confidence interval [CI] 10.7-24.4) to dose. Effect direction and magnitude were replicated in an independent cohort of 75 EC children (P&lt;0.05). No association with morphine dose was detected in African Americans (AA) (n=241). Postoperative pain scores &#8805; 7/10 were associated with rs795484 (G&gt;A) in the EC cohort (odds ratio 2.35, 95% CI 1.56-3.52, P&lt;0.00005) and this association replicated in AA children (odds ratio 1.76, 95% CI 1.14-2.71, P&lt;0.01).  [GWAS:Morphine dose requirement in tonsillectomy and adenoidectomy surgery]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11172113'>rs11172113</a><a href='http://www.snpedia.com/index.php/rs11172113(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFDEDE;'>48.7%</span><br/>Repute:Good<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.9x lower risk for migraines</div>
        <div class='boxshow' id="show-sec-0.476326323824" style="display: inline;"><a href="javascript:toggle2('sec-0.476326323824')">...more...</a></div>
        <div class='boxrstext' id="sec-0.476326323824" style="display: none;"><a href="javascript:toggle2('sec-0.476326323824')">(hide)</a>
        <div class='rstext'>rs11172113 is a SNP on ch 12q13.3 in the LRP1 gene. A large GWAS study of over 5,000 patients with migraines and a meta-analysis concluded that this SNP was one of three influencing the condition, albeit with only slight risk changes on its own. The odds ratio for the less common rs11172113(C) allele was 0.9 (CI: 0.87 - 0.93, p = 4.3 x 10e-9).  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs713598'>rs713598</a><a href='http://www.snpedia.com/index.php/rs713598(C;G)'>(C;G)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFE0E0;'>51.6%</span><br/>Repute:Good<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Can taste bitter.  Normal tasting ability for bitterness. This makes turnip, cabbage, brussels sprouts, broccoli, and other cruciferous vegetables taste more interesting and less horribly bland. Coffee and dark beers also tastes more bitter. You can taste propylthiouracil (PROP), PTC, and related chemicals. On the other hand, you can't taste the unpleasant bitterness of the tropical fruit from the Bignay tree, it will taste sweet to you. You probably eat healthier.</div>
        <div class='boxshow' id="show-sec-0.794184726522" style="display: inline;"><a href="javascript:toggle2('sec-0.794184726522')">...more...</a></div>
        <div class='boxrstext' id="sec-0.794184726522" style="display: none;"><a href="javascript:toggle2('sec-0.794184726522')">(hide)</a>
        <div class='rstext'>rs713598 is one of three SNPs that form the main haplotypes behind the ability to perceive as bitter the taste of the compound phenylthiocarbamide (PTC) and similar molecules in foods (like cabbage and raw broccoli) or drinks (like coffee and dark beers). The rs713598(G) allele, in the orientation shown in dbSNP, is the 'tasting' allele, and it is dominant to the 'non-tasting' allele rs713598(C), so having one copy is enough to have the bitter tasting ability. If you are a 'taster', you're also likely to carry at least one rs10246939(C) and one rs1726866(C) allele since, along with rs713598(G), these three SNPs form the most common tasting haplotype. If you lack these alleles, you're quite likely (~80%) to be a non-taster of bitterness, meaning that foods that may taste bitter to others ta...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2293347'>rs2293347</a><a href='http://www.snpedia.com/index.php/rs2293347(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFF5F5;'>81.4%</span><br/>Repute:Good<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>among NSCLC patients, better Gefitinib response; common</div>
        <div class='boxshow' id="show-sec-0.162133924623" style="display: inline;"><a href="javascript:toggle2('sec-0.162133924623')">...more...</a></div>
        <div class='boxrstext' id="sec-0.162133924623" style="display: none;"><a href="javascript:toggle2('sec-0.162133924623')">(hide)</a>
        <div class='rstext'>rs2293347, also known as D994D, is a SNP in the epidermal growth factor receptor EGFR gene. In a study of therapeutic outcome and survival in 84 advanced non-small-cell lung cancer (NSCLC) patients treated with Gefitinib, the response rate for rs2293347(G;G) patients was almost double that of other genotypes for this SNP (71.2% versus 37.5%, p=0.0043). The rs2293347(G;G) genotype was also associated with longer progression-free survival compared with the rs2293347(A;G) or (A;A) genotypes (11 months versus 3 months, p=0.0018).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10892759'>rs10892759</a><a href='http://www.snpedia.com/index.php/rs10892759(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.01<br/>Frequency: <span style='background-color: #FFABAB;'>12.5%</span><br/>Repute:Good<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>reduced risk for Alzheimers?  A single paper in 2007 found a link to reduced (0.83x) risk for Alzheimers.</div>
        <div class='boxshow' id="show-sec-0.929234487405" style="display: inline;"><a href="javascript:toggle2('sec-0.929234487405')">...more...</a></div>
        <div class='boxrstext' id="sec-0.929234487405" style="display: none;"><a href="javascript:toggle2('sec-0.929234487405')">(hide)</a>
        <div class='rstext'></div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1784931'>rs1784931</a><a href='http://www.snpedia.com/index.php/rs1784931(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.01<br/>Frequency: <span style='background-color: #FFB4B4;'>16.1%</span><br/>Repute:Good<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>reduced risk for Alzheimers?  A single paper in 2007 found a weak link to reduced (0.83x) risk for Alzheimers.</div>
        <div class='boxshow' id="show-sec-0.335349942587" style="display: inline;"><a href="javascript:toggle2('sec-0.335349942587')">...more...</a></div>
        <div class='boxrstext' id="sec-0.335349942587" style="display: none;"><a href="javascript:toggle2('sec-0.335349942587')">(hide)</a>
        <div class='rstext'>rs1784931 is among several SNPs in the SORL1 gene that show some association with increased (or decreased) risk for Alzheimer's disease in a study of 1,400 Caucasians.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4027132'>rs4027132</a><a href='http://www.snpedia.com/index.php/rs4027132(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFB3B3;'>15.9%</span><br/>Repute:Good<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing bipolar disorder</div>
        <div class='boxshow' id="show-sec-0.690122271707" style="display: inline;"><a href="javascript:toggle2('sec-0.690122271707')">...more...</a></div>
        <div class='boxrstext' id="sec-0.690122271707" style="display: none;"><a href="javascript:toggle2('sec-0.690122271707')">(hide)</a>
        <div class='rstext'>rs4027132 has been reported in a large study to be associated with bipolar disorder. The risk allele (oriented to the dbSNP entry) is (A), and the odds ratio associated with heterozygotes is 1.39 (CI 1.19-1.64), and for homozygotes, 1.51 (CI 1.27-1.79).  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1058396'>rs1058396</a><a href='http://www.snpedia.com/index.php/rs1058396(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFBDBD;'>21.2%</span><br/>Repute:Good<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Jk(a-/b+) Kidd blood group, barring other variants  Unless there are other mitigating mutations, this genotype leads to being Kidd blood group Jk(a-/b+).</div>
        <div class='boxshow' id="show-sec-0.637688094547" style="display: inline;"><a href="javascript:toggle2('sec-0.637688094547')">...more...</a></div>
        <div class='boxrstext' id="sec-0.637688094547" style="display: none;"><a href="javascript:toggle2('sec-0.637688094547')">(hide)</a>
        <div class='rstext'>rs1058396, also known as c.838G&gt;A, p.Asp280Asn and D280N, is a variant in the SLC14A1 gene on chromosome 18 that encodes the Jk(a)/Jk(b) Kidd Blood Group antigens. The rs1058396(G) allele encodes the Asp amino acid defining the Jk(a) antigen, and the (A) allele encodes the Asn defining Jk(b). Bladder Cancer Non-ABO Blood Groups  [OMIM:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs492602'>rs492602</a><a href='http://www.snpedia.com/index.php/rs492602(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFC2C2;'>23.9%</span><br/>Repute:Good<br/>References:10<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal B12 levels</div>
        <div class='boxshow' id="show-sec-0.536474942461" style="display: inline;"><a href="javascript:toggle2('sec-0.536474942461')">...more...</a></div>
        <div class='boxrstext' id="sec-0.536474942461" style="display: none;"><a href="javascript:toggle2('sec-0.536474942461')">(hide)</a>
        <div class='rstext'>From : We identified a strong association (p = 5.36 x 10(-17)) between rs492602 in the FUT2 gene and plasma vitamin B(12) levels in a genome-wide scan (n = 1,658) and an independent replication sample (n = 1,059) from the Nurses' Health Study. Women homozygous for the rs492602(C) allele (in dbSNP orientation) had higher B(12) levels. This allele is in strong linkage disequilibrium with the FUT2 nonsecretor variant encoding W143X, suggesting a plausible mechanism for altered B(12) absorption and plasma levels.  [PharmGKB:Curated GWAS Results: Common variants of FUT2 are associated with plasma vitamin B12 levels (Initial Sample Size: 1,658 women; Replication Sample Size: 1,059 women; Risk Allele: rs492602-G). This variant is associated with Plasma level of vitamin B12.] [OMIM:FUCOSYLTRANSFER...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2283123'>rs2283123</a><a href='http://www.snpedia.com/index.php/rs2283123(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFC2C2;'>24.5%</span><br/>Repute:Good<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>decreased risk of schizophrenia in limited study</div>
        <div class='boxshow' id="show-sec-0.163665514897" style="display: inline;"><a href="javascript:toggle2('sec-0.163665514897')">...more...</a></div>
        <div class='boxrstext' id="sec-0.163665514897" style="display: none;"><a href="javascript:toggle2('sec-0.163665514897')">(hide)</a>
        <div class='rstext'>Associated with schizophrenia in a study of Croatians. Due to the limited number and diversity of the studied population, more study is needed to confirm any link. The T allele was associated with a decreased risk of schizophrenia.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4939827'>rs4939827</a><a href='http://www.snpedia.com/index.php/rs4939827(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFC9C9;'>29.2%</span><br/>Repute:Good<br/>References:43<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>0.73x decreased risk for colorectal cancer</div>
        <div class='boxshow' id="show-sec-0.827553893546" style="display: inline;"><a href="javascript:toggle2('sec-0.827553893546')">...more...</a></div>
        <div class='boxrstext' id="sec-0.827553893546" style="display: none;"><a href="javascript:toggle2('sec-0.827553893546')">(hide)</a>
        <div class='rstext'>rs4939827 is one of 3 SNPs in the SMAD7 gene associated with risk for colorectal cancer, based on a large study (7,400+ cases) conducted in the UK. The odds ratios show a decreased risk for the minor rs4939827(C) allele; the OR for (C;C) homozygotes is 0.73 (CI: 0.66-0.8), and for (C;T) heterozygotes 0.86 (CI: 0.79-0.92), overall p=1x10&lt;sup&gt;-12&lt;/sup&gt;.  [PharmGKB:Non-Curated GWAS Results: Genome-wide association scan identifies a colorectal cancer susceptibility locus on 11q23 and replicates risk loci at 8q24 and 18q21 (Initial Sample Size: 981 cases, 1,002 controls; Replication Sample Size: 16,476 cases, 15,351 controls; Risk Allele: rs4939827-T).] [OMIM:MOTHERS AGAINST DECAPENTAPLEGIC, DROSOPHILA, HOMOLOG OF, 7; SMAD7] [GWAS:Colorectal cancer] [GWAS:Colorectal cancer]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2156921'>rs2156921</a><a href='http://www.snpedia.com/index.php/rs2156921(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFCACA;'>30.0%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>normal risk for depression</div>
        <div class='boxshow' id="show-sec-0.319028312618" style="display: inline;"><a href="javascript:toggle2('sec-0.319028312618')">...more...</a></div>
        <div class='boxrstext' id="sec-0.319028312618" style="display: none;"><a href="javascript:toggle2('sec-0.319028312618')">(hide)</a>
        <div class='rstext'>rs2156921, a SNP in the BCR gene on chromosome 22, has been associated with increased risk for depression. The odds ratio for carriers of the minor allele (G) are reported as 1.29 (p=0.031) based on a study of 329 Japanese patients.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs5174'>rs5174</a><a href='http://www.snpedia.com/index.php/rs5174(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFCDCD;'>32.7%</span><br/>Repute:Good<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk for heart disease</div>
        <div class='boxshow' id="show-sec-0.791755322735" style="display: inline;"><a href="javascript:toggle2('sec-0.791755322735')">...more...</a></div>
        <div class='boxrstext' id="sec-0.791755322735" style="display: none;"><a href="javascript:toggle2('sec-0.791755322735')">(hide)</a>
        <div class='rstext'>rs5174 encodes a variant of the LRP8 gene, encoding the low density lipoprotein receptor-related 8 protein (or the apolipoprotein e receptor). The variant affects the protein, changing an arginine to a glutamine; depending on the publication, it can be referred to by terms like R952Q or Arg952Gln. In three Caucasian populations combined, the odds ratio for coronary artery disease or myocardial infarction (and thus heart disease) is 1.31 (adjusted p=0.0003). Note that this association was only seen in populations with familial and premature disease, as a study of 1,231 patients with primarily late-onset, sporadic heart disease did not show any correlation with this SNP. rs5174 (or its equivalent, rs5177) was analyzed in 1,210 patients with familial MI and 1,015 controls from the German MI F...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs12752888'>rs12752888</a><a href='http://www.snpedia.com/index.php/rs12752888(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFCECE;'>33.6%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal progression to Alzheimer&#8217;s disease from mild cognitive impairment.</div>
        <div class='boxshow' id="show-sec-0.424719332681" style="display: inline;"><a href="javascript:toggle2('sec-0.424719332681')">...more...</a></div>
        <div class='boxrstext' id="sec-0.424719332681" style="display: none;"><a href="javascript:toggle2('sec-0.424719332681')">(hide)</a>
        <div class='rstext'>rs12752888 is a SNP on chromosome 1 upstream of the gene ACOT11. ACOT11 has been associated with obesity and is a member of a family of enzymes that help catalyze the de-esterification of fatty acids. One study used genotyping information from 489 patients with mild cognitive impairment (MCI) from the Vitamin E trial. The GWAS employed used Alzheimer&#8217;s disease progression measured quantitatively from Clinical Dementia Rating-sum of boxes (CRD-SB) scores over time. Additionally, the study utilized information about 200 Alzheimer&#8217;s disease patients, 300 MCI patients, and 200 controls from the Alzheimer's Disease Neuroimaging Initiative (ADNI) consortium. Significance was assessed based on the interaction term between genotype and time in a Cox proportional hazards model. rs127528...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11171739'>rs11171739</a><a href='http://www.snpedia.com/index.php/rs11171739(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFD1D1;'>35.4%</span><br/>Repute:Good<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing Type-1 diabetes</div>
        <div class='boxshow' id="show-sec-0.75569674821" style="display: inline;"><a href="javascript:toggle2('sec-0.75569674821')">...more...</a></div>
        <div class='boxrstext' id="sec-0.75569674821" style="display: none;"><a href="javascript:toggle2('sec-0.75569674821')">(hide)</a>
        <div class='rstext'>rs11171739 has been reported in a large study to be associated with type-1 diabetes. The risk allele (oriented to the dbSNP entry) is (C); the odds ratio associated with heterozygotes is 1.34 (CI 1.17-1.54), and for homozygotes, 1.75 (CI 1.48-2.06). In an expanded follow-up study of &gt;6,000 controls and 6,000 patients, in which rs2292239 was ultimately chosen to replace the nearby rs11171739 originally determined to be associated with type-1 diabetes, the heterozygote odds ratio for SNP rs11171739 was recalculated to be 1.22 (CI 1.17&#226;&#8364;&#8220;1.29).  [PharmGKB:Non-Curated A genome-wide association study in 2,000 individuals for each of 7 major diseases and a shared set of 3,000 controls found an association of this SNP with type 1 diabetes.] [GWAS:Type 1 diabetes]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs140701'>rs140701</a><a href='http://www.snpedia.com/index.php/rs140701(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFD1D1;'>35.4%</span><br/>Repute:Good<br/>References:12<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk for anxiety disorders</div>
        <div class='boxshow' id="show-sec-0.0476045616587" style="display: inline;"><a href="javascript:toggle2('sec-0.0476045616587')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0476045616587" style="display: none;"><a href="javascript:toggle2('sec-0.0476045616587')">(hide)</a>
        <div class='rstext'>A study of patients with panic disorder or social anxiety disorder, both moderately heritable anxiety disorders, concluded that the rs140701(A) allele was associated with increased risk for both disorders. This SNP was actually part of a tightly linked haplotype A-A-G (for SNPs rs3794808, rs140701 and rs4583306, respectively) that had 1.7x increased risk for panic disorder (CI: 1.2-2.3).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3825776'>rs3825776</a><a href='http://www.snpedia.com/index.php/rs3825776(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFD5D5;'>39.8%</span><br/>Repute:Good<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk for ALS</div>
        <div class='boxshow' id="show-sec-0.87984707666" style="display: inline;"><a href="javascript:toggle2('sec-0.87984707666')">...more...</a></div>
        <div class='boxrstext' id="sec-0.87984707666" style="display: none;"><a href="javascript:toggle2('sec-0.87984707666')">(hide)</a>
        <div class='rstext'>rs3825776, a SNP in the region of the LIPC gene on chromosome 15, has been associated with the sporadic form of ALS (Lou Gehrig's disease) in a study of 1000+ European patients. The odds ratio for the risk allele rs3825776(G) is 1.34 (CI: 1.12 - 1.46).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4792311'>rs4792311</a><a href='http://www.snpedia.com/index.php/rs4792311(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFD6D6;'>40.7%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of prostate cancer</div>
        <div class='boxshow' id="show-sec-0.275098943068" style="display: inline;"><a href="javascript:toggle2('sec-0.275098943068')">...more...</a></div>
        <div class='boxrstext' id="sec-0.275098943068" style="display: none;"><a href="javascript:toggle2('sec-0.275098943068')">(hide)</a>
        <div class='rstext'>Sequence variants of elaC homolog 2 (Escherichia coli) (ELAC2) gene and susceptibility to prostate cancer in the Health Professionals Follow-Up Study. Development of a fingerprinting panel using medically relevant polymorphisms. Genetic variants and prostate cancer risk: candidate replication and exploration of viral restriction genes. Population based allele frequencies of disease associated polymorphisms in the Personalized Medicine Research Project.  [OMIM:PROSTATE CANCER, SUSCEPTIBILITY TO]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs729302'>rs729302</a><a href='http://www.snpedia.com/index.php/rs729302(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDBDB;'>46.0%</span><br/>Repute:Good<br/>References:17<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing rheumatoid arthritis</div>
        <div class='boxshow' id="show-sec-0.572222981459" style="display: inline;"><a href="javascript:toggle2('sec-0.572222981459')">...more...</a></div>
        <div class='boxrstext' id="sec-0.572222981459" style="display: none;"><a href="javascript:toggle2('sec-0.572222981459')">(hide)</a>
        <div class='rstext'>A variant located 64 bp upstream of the first untranslated exon (exon 1A), consisting of a 5 bp insertion/deletion CGGGG, may be the causative SNP in this region that is most responsible for increasing SLE risk, however it lacks an rs# (i.e. it isn't registered in dbSNP). A meta-analysis comprising 5 case-control studies, totaling 6,582 rheumatoid arthritis cases and 5,375 controls, concluded that several IRF5 gene SNPs were indeed (still) significantly associated with the disease. The rs729302(C) allele was associated with a slight protective effect (odds ratio 0.89, CI: 0.8-0.977, p = 0.015).  [GWAS:Systemic lupus erythematosus]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1800860'>rs1800860</a><a href='http://www.snpedia.com/index.php/rs1800860(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDDDD;'>48.2%</span><br/>Repute:Good<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal sized kidneys as newborns</div>
        <div class='boxshow' id="show-sec-0.0522443809924" style="display: inline;"><a href="javascript:toggle2('sec-0.0522443809924')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0522443809924" style="display: none;"><a href="javascript:toggle2('sec-0.0522443809924')">(hide)</a>
        <div class='rstext'>rs1800860 is a SNP in the RET ret proto-oncogene. A study of newborns concluded that Caucasian newborns who were clinically normal and carried a rs1800860(A) allele had kidney volumes 10% smaller and cord blood cystatin C levels 9% higher than those with the rs1800860(G;G) genotype. Since individuals with lower nephron numbers have an increased lifetime risk for essential hypertension or renal insufficiency, this may have clinical consequences in adult life (but this remains to be proven).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2952768'>rs2952768</a><a href='http://www.snpedia.com/index.php/rs2952768(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDEDE;'>48.6%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>slightly less drug dependence, decreased effectiveness of analgesics</div>
        <div class='boxshow' id="show-sec-0.304797293596" style="display: inline;"><a href="javascript:toggle2('sec-0.304797293596')">...more...</a></div>
        <div class='boxrstext' id="sec-0.304797293596" style="display: none;"><a href="javascript:toggle2('sec-0.304797293596')">(hide)</a>
        <div class='rstext'>Genome-wide association study identifies a potent locus associated with human opioid sensitivity rs2952768(C) was associated with more analgesic requirements ... exhibited less vulnerability to severe methamphetamine or alcohol dependence, fewer eating disorders, and a lower &#8216;Reward Dependence&#8217; score on a personality questionnaire. The (C;C) genotype was significantly associated with the elevated expression of a neighboring gene, CREB1. SNPs in this locus are the most potent genetic factors associated with human opioid sensitivity known to date, affecting both the efficacy of opioid analgesics such as morphine and fentanyl and liability to severe substance dependence. These findings may provide valuable information for the personalized treatment of pain and drug dependence. Oth...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4464148'>rs4464148</a><a href='http://www.snpedia.com/index.php/rs4464148(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFE1E1;'>52.7%</span><br/>Repute:Good<br/>References:15<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk for colorectal cancer</div>
        <div class='boxshow' id="show-sec-0.521656855645" style="display: inline;"><a href="javascript:toggle2('sec-0.521656855645')">...more...</a></div>
        <div class='boxrstext' id="sec-0.521656855645" style="display: none;"><a href="javascript:toggle2('sec-0.521656855645')">(hide)</a>
        <div class='rstext'>rs4464148 is one of 3 SNPs in the SMAD7 gene associated with risk for colorectal cancer, based on a large study (7,400+ cases) conducted in the UK. The odds ratios show an increased risk for the minor rs4464148(C) allele; the OR for (C;C) homozygotes is 1.35 (CI: 1.2-1.51), and for (C;T) heterozygotes 1.10 (CI: 1.09-1.21), overall p=7x10&lt;sup&gt;-8&lt;/sup&gt;.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3814570'>rs3814570</a><a href='http://www.snpedia.com/index.php/rs3814570(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFE1E1;'>53.1%</span><br/>Repute:Good<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing Crohn's disease</div>
        <div class='boxshow' id="show-sec-0.177343996016" style="display: inline;"><a href="javascript:toggle2('sec-0.177343996016')">...more...</a></div>
        <div class='boxrstext' id="sec-0.177343996016" style="display: none;"><a href="javascript:toggle2('sec-0.177343996016')">(hide)</a>
        <div class='rstext'>rs3814570 is a SNP in the transcription factor 7-like 2 (T-cell specific, HMG-box) TCF7L2 gene. A study of 784 Crohn's disease patients with ileal involvement concluded that rs3814570 was associated with increased risk, but not with risk for colonic Crohn's disease or ulcerative colitis. The odds ratio was 1.27 (CI: 1.07 - 1.52, p = 0.00737). Evidence for association between polycystic ovary syndrome (PCOS) and TCF7L2 and glucose intolerance in women with PCOS and TCF7L2.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6603272'>rs6603272</a><a href='http://www.snpedia.com/index.php/rs6603272(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFE1E1;'>53.1%</span><br/>Repute:Good<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing schizophrenia</div>
        <div class='boxshow' id="show-sec-0.549256782331" style="display: inline;"><a href="javascript:toggle2('sec-0.549256782331')">...more...</a></div>
        <div class='boxrstext' id="sec-0.549256782331" style="display: none;"><a href="javascript:toggle2('sec-0.549256782331')">(hide)</a>
        <div class='rstext'>This snp is in the Pseudoautosomal region. rs6603272, part of a haplotype block spanning introns 4, 5 and 6 of the IL3RA gene, has been reported in a whole genome association study to be associated with schizophrenia. (The other SNPs in this block are rs6422441 and rs17883192.) The risk allele (oriented to the dbSNP entry) is (G); the odds ratio associated with this allele is 2.74.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10980705'>rs10980705</a><a href='http://www.snpedia.com/index.php/rs10980705(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFE2E2;'>54.0%</span><br/>Repute:Good<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk for knee osteoarthritis</div>
        <div class='boxshow' id="show-sec-0.668613097005" style="display: inline;"><a href="javascript:toggle2('sec-0.668613097005')">...more...</a></div>
        <div class='boxrstext' id="sec-0.668613097005" style="display: none;"><a href="javascript:toggle2('sec-0.668613097005')">(hide)</a>
        <div class='rstext'>rs10980705, also known as -2820G/A, is a SNP upstream of the lysophosphatidic acid receptor 1 LPAR1 gene. In a study of 2 Japanese populations of patients with knee osteoarthritis (OA), the rs10980705(T) allele (in dbSNP orientation) was associated with increased risk in two Japanese populations, with an odds ratio of 2.3 (CI: 1.6-3.3, p=2.6x10&lt;sup&gt;-5&lt;/sup&gt;). The association appears to hold only for the homozygous (T;T) genotype, and the authors state they believe this variant may be the 'disease-causing SNP'. However, 2% of individuals without knee OA are (T;T) homozygotes, and, over 50% of all patients with knee OA are (C;C) homozygotes. In a study of five collections of samples, and a meta-analysis combining them all, no particularly significant association was seen for rs10...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6048'>rs6048</a><a href='http://www.snpedia.com/index.php/rs6048(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFE3E3;'>55.4%</span><br/>Repute:Good<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>common</div>
        <div class='boxshow' id="show-sec-0.0962525658435" style="display: inline;"><a href="javascript:toggle2('sec-0.0962525658435')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0962525658435" style="display: none;"><a href="javascript:toggle2('sec-0.0962525658435')">(hide)</a>
        <div class='rstext'>Gene variants associated with deep vein thrombosis. Updated analysis of gene variants associated with deep vein thrombosis.  [OMIM:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3790565'>rs3790565</a><a href='http://www.snpedia.com/index.php/rs3790565(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFE5E5;'>57.1%</span><br/>Repute:Good<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing primary biliary cirrhosis</div>
        <div class='boxshow' id="show-sec-0.537754791001" style="display: inline;"><a href="javascript:toggle2('sec-0.537754791001')">...more...</a></div>
        <div class='boxrstext' id="sec-0.537754791001" style="display: none;"><a href="javascript:toggle2('sec-0.537754791001')">(hide)</a>
        <div class='rstext'>Primary biliary cirrhosis associated with HLA, IL12A, and IL12RB2 variants. C allele associated with increased risk of Primary biliary cirrhosis</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9357271'>rs9357271</a><a href='http://www.snpedia.com/index.php/rs9357271(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFE5E5;'>57.5%</span><br/>Repute:Good<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing restless legs syndrome</div>
        <div class='boxshow' id="show-sec-0.962272925237" style="display: inline;"><a href="javascript:toggle2('sec-0.962272925237')">...more...</a></div>
        <div class='boxrstext' id="sec-0.962272925237" style="display: none;"><a href="javascript:toggle2('sec-0.962272925237')">(hide)</a>
        <div class='rstext'>rs9357271, a SNP located in the BTBD9 gene region, has been linked to a lower frequency of restless legs syndrome, a common sleep disorder, with an overall odds ratio of 0.63 (CI: 0.52-0.75) for the (C) minor allele. This SNP may also be associated with Tourette syndrome (TS), particularly TS without obsessive-compulsive disorder, based on a study of ~600 patients.  [PharmGKB:Curated In replicated GWAS case-control studies of Caucasian(European and French-Canadian) familial and sporadic RLS, the T allele of rs9357271 was significantly associated with Restless Legs Syndrome.] [OMIM:RESTLESS LEGS SYNDROME, SUSCEPTIBILITY TO, 6] [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9561778'>rs9561778</a><a href='http://www.snpedia.com/index.php/rs9561778(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFEBEB;'>66.4%</span><br/>Repute:Good<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of adverse drug reations from cyclophosphamide treatments</div>
        <div class='boxshow' id="show-sec-0.043033690473" style="display: inline;"><a href="javascript:toggle2('sec-0.043033690473')">...more...</a></div>
        <div class='boxrstext' id="sec-0.043033690473" style="display: none;"><a href="javascript:toggle2('sec-0.043033690473')">(hide)</a>
        <div class='rstext'>rs9561778 is a SNP in the ATP-binding cassette, sub-family C (CFTR/MRP), member 4 ABCC4 gene. A stepwise case-control study of ~400 breast cancer Japanese patients receiving cyclophosphamide-based combination chemotherapy, roughly divided equally between those having or not having serious adverse drug reactions (ADRs), concluded that carriers of rs9561778(T) allele had a ~2x higher risk of ADRs. The ADRs for these patients tended to be gastrointestinal toxicity and leukopenia/neutropenia.  [PharmGKB:Curated Risk or phenotype-associated allele: T. Phenotype: In a stepwise case-control association study of breast cancer patients receiving cyclophosphamide combination therapy, rs9561778 in ABCC4 was associated with cyclophosphamide-induced ADRs. Study size: 403. Study population/ethnicity: Ja...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs663048'>rs663048</a><a href='http://www.snpedia.com/index.php/rs663048(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFECEC;'>67.3%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing lung cancer</div>
        <div class='boxshow' id="show-sec-0.112690682103" style="display: inline;"><a href="javascript:toggle2('sec-0.112690682103')">...more...</a></div>
        <div class='boxrstext' id="sec-0.112690682103" style="display: none;"><a href="javascript:toggle2('sec-0.112690682103')">(hide)</a>
        <div class='rstext'>rs663048 associated with lung cancer The homozygotes for the variant allele had more than a 3-fold risk compared with the wild-type homozygotes [combined odds ratio (OR), 3.32; 95% confidence interval (95% CI), 1.81-7.21]. Heterozygotes also had a significantly elevated risk of lung cancer from the combined replication studies with an OR of 1.15 (95% CI, 1.04-1.59). The effect remained significant after adjusting for age, gender, and pack-years of tobacco smoke.007;67(17):8406-11].</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10464059'>rs10464059</a><a href='http://www.snpedia.com/index.php/rs10464059(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFEFEF;'>72.3%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing Parkinson's Disease</div>
        <div class='boxshow' id="show-sec-0.629225270108" style="display: inline;"><a href="javascript:toggle2('sec-0.629225270108')">...more...</a></div>
        <div class='boxrstext' id="sec-0.629225270108" style="display: none;"><a href="javascript:toggle2('sec-0.629225270108')">(hide)</a>
        <div class='rstext'>[GWAS:Parkinson's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2295490'>rs2295490</a><a href='http://www.snpedia.com/index.php/rs2295490(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFF2F2;'>76.8%</span><br/>Repute:Good<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>normal Normal risk of early-onset type II diabetes</div>
        <div class='boxshow' id="show-sec-0.0151594011181" style="display: inline;"><a href="javascript:toggle2('sec-0.0151594011181')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0151594011181" style="display: none;"><a href="javascript:toggle2('sec-0.0151594011181')">(hide)</a>
        <div class='rstext'>rs2295490 is a SNP in the tribbles homolog 3 TRIB3 gene, also known as Q84R. The (A) allele encodes the Gln (Q), while the (G) allele encodes the Arg (R). A study involving four different case-control samples comprising a total of 5469 Caucasisans determined that rs2295490(G) was nominally associated with type-2 diabetes (odds ratio 1.17, CI: 1.00-1.36, p = 0.04). When stratified by age of diabetes onset, these carriers had an increased risk of early-onset T2D (OR 1.32, CI: 1.10-1.58, p = 0.002). this gain of function mutation may affect insulin signaling and increase the risk of insulin resistance  [PharmGKB:Curated This variant in the TRIB3 gene is associated with early-onset T2D in Whites. This Q84R %22gain of function%22 polymorphism, impairs insulin signaling.]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10488631'>rs10488631</a><a href='http://www.snpedia.com/index.php/rs10488631(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFF2F2;'>77.0%</span><br/>Repute:Good<br/>References:29<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing SLE, primary biliary cirrhosis, and Sj&#246;gren's syndrome</div>
        <div class='boxshow' id="show-sec-0.197004790689" style="display: inline;"><a href="javascript:toggle2('sec-0.197004790689')">...more...</a></div>
        <div class='boxrstext' id="sec-0.197004790689" style="display: none;"><a href="javascript:toggle2('sec-0.197004790689')">(hide)</a>
        <div class='rstext'>rs10488631, a SNP located 3' of the IRF5 gene, has been reported as a possibly causative SNP for systemic lupus erthymatosus (SLE), based on a study of ~700 Swedish patients. The risk allele is rs10488631(C), with a reported odds ratio of 2.07 (CI: 1.63-2.62, p = 9x10e-10). The C allele is also associated with primary biliary cirrhosis, with each C increasing the odds about 1.6 times. Each C allele increases risk of Sj&#246;gren's syndrome by 1.7 times. SLE rs10488631 and rs7582694  [OMIM:INTERFERON REGULATORY FACTOR 5; IRF5] [GWAS:Systemic lupus erythematosus]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4273077'>rs4273077</a><a href='http://www.snpedia.com/index.php/rs4273077(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFF3F3;'>78.6%</span><br/>Repute:Good<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk for development of MGUS  This was defined as an OR of 1.0 for the development of MGUS (Monoclonal Gammopathy of Uncertain Significance). A is the major allele and not the risk allele.</div>
        <div class='boxshow' id="show-sec-0.591941693579" style="display: inline;"><a href="javascript:toggle2('sec-0.591941693579')">...more...</a></div>
        <div class='boxrstext' id="sec-0.591941693579" style="display: none;"><a href="javascript:toggle2('sec-0.591941693579')">(hide)</a>
        <div class='rstext'>[GWAS:Hematological and biochemical traits]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1223271'>rs1223271</a><a href='http://www.snpedia.com/index.php/rs1223271(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFF4F4;'>80.4%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing Parkinson's Disease</div>
        <div class='boxshow' id="show-sec-0.0571522868153" style="display: inline;"><a href="javascript:toggle2('sec-0.0571522868153')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0571522868153" style="display: none;"><a href="javascript:toggle2('sec-0.0571522868153')">(hide)</a>
        <div class='rstext'>[GWAS:Parkinson's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6532197'>rs6532197</a><a href='http://www.snpedia.com/index.php/rs6532197(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFF6F6;'>84.1%</span><br/>Repute:Good<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing Parkinson's Disease</div>
        <div class='boxshow' id="show-sec-0.112259135065" style="display: inline;"><a href="javascript:toggle2('sec-0.112259135065')">...more...</a></div>
        <div class='boxrstext' id="sec-0.112259135065" style="display: none;"><a href="javascript:toggle2('sec-0.112259135065')">(hide)</a>
        <div class='rstext'>[GWAS:Parkinson's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4979462'>rs4979462</a><a href='http://www.snpedia.com/index.php/rs4979462(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFFEFE;'>98.5%</span><br/>Repute:Good<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing primary biliary cirrhosis</div>
        <div class='boxshow' id="show-sec-0.0622332245815" style="display: inline;"><a href="javascript:toggle2('sec-0.0622332245815')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0622332245815" style="display: none;"><a href="javascript:toggle2('sec-0.0622332245815')">(hide)</a>
        <div class='rstext'>[GWAS:Primary biliary cirrhosis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2494732'>rs2494732</a><a href='http://www.snpedia.com/index.php/rs2494732(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Repute:Good<br/>References:19<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>lower odds of psychosis</div>
        <div class='boxshow' id="show-sec-0.453501366857" style="display: inline;"><a href="javascript:toggle2('sec-0.453501366857')">...more...</a></div>
        <div class='boxrstext' id="sec-0.453501366857" style="display: none;"><a href="javascript:toggle2('sec-0.453501366857')">(hide)</a>
        <div class='rstext'>rs2494732 is a SNP in the AKT1 gene, encoding one of three serine/threonine-protein kinases regulating numerous processes such as cell survival, growth and angiogenesis. Medical conditions that may be linked to the rarer (minor) rs2494732(C) allele include drug response, schizophrenia, and drug-related psychoses, in particuler, cannabis-related psychosis. A case-control study of 489 first-episode psychosis patients concluded that while rs2494732 was not associated with an increased risk of a psychotic disorder, with lifetime cannabis use, or with frequency of use, there was a significant association between psychosis and cannabis use. rs2494732(C;C) individuals with a history of cannabis use had an increased likelihood of a psychotic disorder (odds ratio 2.18, CI: 1.1 - 4.3) when compared ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxgoodresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3923890'>rs3923890</a><a href='http://www.snpedia.com/index.php/rs3923890(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #F0D0F0;'>45.1%</span><br/>Repute:Good<br/>References:2<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>normal social interaction  Normal autism risk.[http://www.ncbi.nlm.nih.gov/pubmed/18270976?dopt=Abstract]</div>
        <div class='boxshow' id="show-sec-0.370540791819" style="display: inline;"><a href="javascript:toggle2('sec-0.370540791819')">...more...</a></div>
        <div class='boxrstext' id="sec-0.370540791819" style="display: none;"><a href="javascript:toggle2('sec-0.370540791819')">(hide)</a>
        <div class='rstext'>rs3923890 has been reported to be a SNP potentially associated with increased risk for autism</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>
</div>     <div class="majorboxfooter"></div>     </div><div class="majorsection">     <div class='sectiontext'>Bad news</div>
        <div class='boxshow' id="show-badNews" style="display: inline;"><a href="javascript:toggle2('badNews')">...more...</a></div>
        <div class="boxsection" id="badNews" style="display: none;"><a href="javascript:toggle2('badNews')">(hide)</a>

  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1061170'>rs1061170</a><a href='http://www.snpedia.com/index.php/rs1061170(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 3.2<br/>Frequency: <span style='background-color: #FF0A0A;'>0.0%</span><br/>Repute:Bad<br/>References:125<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>5.9x risk for AMD; higher mortality among nonagenarians  To minimize your risk of age related macular degeneration, consider a diet rich in vitamins C and E, lutein, zeaxanthin and the minerals zinc and copper, such as kale, spinach and broccoli. source</div>
        <div class='boxshow' id="show-sec-0.336954627508" style="display: inline;"><a href="javascript:toggle2('sec-0.336954627508')">...more...</a></div>
        <div class='boxrstext' id="sec-0.336954627508" style="display: none;"><a href="javascript:toggle2('sec-0.336954627508')">(hide)</a>
        <div class='rstext'>rs1061170 is a SNP in the complement factor H CFH gene; it is also known as Tyr402His. The rs1061170(T) allele encodes the more common Tyr (Y), while the generally rarer rs1061170(C) encodes the His (H). This SNP has been associated primarily with age related macular degeneration, and to a lesser extent, with longevity. This research paper shows that CFH Y402H is the relevant mutation. A haplotype of rs1061170 rs3753394 rs800292 rs1329428 (TGTC) was found to confer a significantly increased likelihood of exudative AMD. rs1061170 (aka Y402H 1277 T&gt;C) was not associated with exudative AMD. CFH variations appear to contribute to ARMD in Caucasians, but not in Japanese CFH variant rs1061170 Y402H is strongly associated with both dry and wet AMD rs1061170(C) alleles are significantly associa...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs122"  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs122'>gs122</a></div></div>
        <div class='boxmag'>Magnitude: 3.1</div>
        <div class='boxrep'>Repute: Bad</div>
      </div>
      <div class='boxresultcol2'>
        <img class="genoimage" src="http://media.snpedia.com/images/2/22/Baldness.jpg"/>
        <div class='gsboxsummary'>7x risk of baldness</div>
        <div class='gsboxeffect'>7x risk of baldness according to an article in Nature. That site may require paid access; the abstract at is accessible.</div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10830963'>rs10830963</a><a href='http://www.snpedia.com/index.php/rs10830963(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 3.1<br/>Frequency: <span style='background-color: #F084F0;'>4.6%</span><br/>Repute:Bad<br/>References:64<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased type-2 diabetes risk; higher gestational diabetes risk</div>
        <div class='boxshow' id="show-sec-0.575895199864" style="display: inline;"><a href="javascript:toggle2('sec-0.575895199864')">...more...</a></div>
        <div class='boxrstext' id="sec-0.575895199864" style="display: none;"><a href="javascript:toggle2('sec-0.575895199864')">(hide)</a>
        <div class='rstext'>rs10830963 was associated with an increased risk of type-2 diabetes in our Han Chinese cohort (OR 1.16, 95% CI 1.03-1.31, p = 0.015). As previously described, the risk variant was also associated with increased fasting plasma glucose, showing an increase of 0.068 mmol/l (95% CI 0.036-0.100, p = 4 x 10(-5)) per risk allele rs10830962, rs4753426, and rs10830963 were significantly associated with higher fasting plasma glucose concentrations and reduced OGTT- and IVGTT-induced insulin release. rs3781638 displayed significant association with lower fasting plasma glucose levels and increased OGTT-induced insulin release A study totaling 19,000+ Europeans concluded that rs10830963 had the most influence of any MTNR1B gene SNP on the risk for type-2 diabetes. Specifically, the (G) allele increase...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs151"  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs151'>gs151</a></div></div>
        <div class='boxmag'>Magnitude: 3</div>
        <div class='boxrep'>Repute: Bad</div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>CYP2C19 Intermediate Metabolizer</div>
        <div class='gsboxeffect'>CYP2C19 Intermediate Metabolizer. Your body breaks down some medicines at a slightly slower than normal rate (which is represented by gs150). Individuals with gs152 genotypes have even slower metabolism. *anti-epileptics (such as diazepam, phenytoin, and phenobarbitone) *anti-depressants (such as amitriptyline and clomipramine) *anti-platelet drug clopidogrel (Plavix) *anti-ulcer proton pump inhibitors like omeprazole (trade names Losec and Prilosec), esomeprazole (trade name Nexium), and lansoprazole (Prevacid) *hormones (estrogen, progesterone).</div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10033464'>rs10033464</a><a href='http://www.snpedia.com/index.php/rs10033464(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 3<br/>Frequency: <span style='background-color: #FFB3B3;'>15.9%</span><br/>Repute:Bad<br/>References:13<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.28x increased risk of Atrial Fibrillation and cardioembolic stroke.  This is one copy of a mutant version (and one of the normal version) which slightly affects the formation of the heart and makes people slightly more likely to get Atrial Fibrillation (quivering of the top chambers of the heart caused by chaotic electrical signals) and slightly more likely to have a cardioembolic stroke (blocked blood flow to the brain). But also check the other SNP nearby (rs2200733) which affects Atrial Fibrillation even more. 23andMe gave this a 4 star confidence rating as it has been heavily studied. It has been confirmed in Europeans.</div>
        <div class='boxshow' id="show-sec-0.964811854689" style="display: inline;"><a href="javascript:toggle2('sec-0.964811854689')">...more...</a></div>
        <div class='boxrstext' id="sec-0.964811854689" style="display: none;"><a href="javascript:toggle2('sec-0.964811854689')">(hide)</a>
        <div class='rstext'>Two SNPs from chromosome 4q25, rs2200733 and rs10033464, were found to be associated with atrial fibrillation in a study of both European and Asian populations. The odds ratio associated with one or more copies of either risk allele was ~1.4x. In 1,661 Icelandic ischemic stroke samples and two large European replication sets combined, rs10033464 was associated with cardioembolic stroke (CES) (odds ratio 1.27, p = 6.1 x 10e-4). rs2200733 and rs10033464 are strongly associated with AF in four cohorts of European descent. Variation in the 4q25 chromosomal locus predicts atrial fibrillation after coronary artery bypass graft surgery. Chromosome 4q25 variants and atrial fibrillation recurrence after catheter ablation. Common variants in KCNN3 are associated with lone atrial fibrillation. Associ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4244285'>rs4244285</a><a href='http://www.snpedia.com/index.php/rs4244285(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 3<br/>Frequency: <span style='background-color: #FFB9B9;'>19.0%</span><br/>Repute:Bad<br/>References:30<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>poorer metabolizer of several popular medicines; patients prescribed Plavix get less benefit, and have higher risk for adverse cardiovascular events</div>
        <div class='boxshow' id="show-sec-0.701458993648" style="display: inline;"><a href="javascript:toggle2('sec-0.701458993648')">...more...</a></div>
        <div class='boxrstext' id="sec-0.701458993648" style="display: none;"><a href="javascript:toggle2('sec-0.701458993648')">(hide)</a>
        <div class='rstext'>rs4244285 is a SNP in the CYP2C19 gene, potentially encoding the CYP2C19*2 variant. This variant is the most common reason for poor metabolism of compounds like mephenytoin (an anti-convulsant), some antidepressants, the anti-platelet drug Plavix, and some drugs used for ulcer conditions of various types. The risk allele is rs4244285(A). As a nonfunctioning CYP2C19, this variant would be expected to be a poor metabolizer of several commonly prescribed drugs, including anti-ulcer drugs like omeprazole (trade names Losec and Prilosec), esomeprazole (trade name Nexium), and lansoprazole (trade name Prevacid). In Caucasians, SNPs in CYP2C19 are relatively rare (in contrast to SNPs in CYP2D6), but SNPs in this gene are common in Asians. Ulcer treatment with omeprazole to reduce Helicobacter pyl...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1045642'>rs1045642</a><a href='http://www.snpedia.com/index.php/rs1045642(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 3<br/>Frequency: <span style='background-color: #FFC9C9;'>29.2%</span><br/>Repute:Bad<br/>References:106<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>altered drug metabolism and bioavailability; moderately increased risk for certain cancers  This is the homozygous variant of C3435T, on the gene encoding P-glycoprotein (called MDR1 or ABCB1). P-glycoprotein is a protein involved in pumping foreign substances out of cells. It mainly affects areas of the body that have a barrier or excretory function: blood-tissue barrier, the gastrointestinal tract, liver and kidney. Poor functioning of this cellular defense mechanism can affect elimination of carcinogens and movement of drugs in the system. == Drug Treatment Response & Toxicity == '''ABCB1 polymorphism as prognostic factor in breast cancer patients treated with docetaxel and doxorubicin neoadjuvant chemotherapy.''' (2014). Among 216 patients, those with the 3435TT genotype had a longer o...</div>
        <div class='boxshow' id="show-sec-0.910694622812" style="display: inline;"><a href="javascript:toggle2('sec-0.910694622812')">...more...</a></div>
        <div class='boxrstext' id="sec-0.910694622812" style="display: none;"><a href="javascript:toggle2('sec-0.910694622812')">(hide)</a>
        <div class='rstext'>rs1045642, also known as C3435T, is a SNP located in the ABCB1 gene. It is often studied in conjunction with rs2032582. C3435T has been mentioned by: * A 'Silent' Polymorphism in the MDR1 Gene Changes Substrate Specificity (for example, to verapamil) * (R)-lansoprazole (Prevacid) concentrations are significantly increased in CYP2C19 extensive metabolizers with ABCB1 C3435T C allele. * In a Korean population, plasma concentrations of fexofenadine (Allegra) were 17% lower in 2677AA/3435CC subjects and 47% higher in the 2677TT/3435TT subjects compared to 2677GG/3435CC subjects. * In contrast, in this study no association was observed between the C3435T polymorphism and fexofenadine plasma or urine concentrations in a German Caucasian population. * A meta-analysis including 9 case-control stud...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2145418'>rs2145418</a><a href='http://www.snpedia.com/index.php/rs2145418(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 3<br/>Frequency: <span style='background-color: #FFCBCB;'>31.0%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>5.0x increased thyroid cancer risk  Relatively common</div>
        <div class='boxshow' id="show-sec-0.88756696334" style="display: inline;"><a href="javascript:toggle2('sec-0.88756696334')">...more...</a></div>
        <div class='boxrstext' id="sec-0.88756696334" style="display: none;"><a href="javascript:toggle2('sec-0.88756696334')">(hide)</a>
        <div class='rstext'>rs2145418 is a SNP in the chromosome 1p12-13 region. In a case-control association study in a Spanish population, rs2145418 was associated with increased risk for thyroid cancer. rs2145418(G;T) heterozygotes had an odds ratio of 5.0 (CI: 2.85 - 8.83), while (G;G) homozygotes had an increased odds ratio of 9.2 (CI: 4.50 - 21.6), with reported significance of p &lt; 0.0001.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11614913'>rs11614913</a><a href='http://www.snpedia.com/index.php/rs11614913(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 3<br/>Frequency: <span style='background-color: #FFCECE;'>33.6%</span><br/>Repute:Bad<br/>References:68<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk of various types of cancer  This variant increases risk of numerous types of cancer in many studies. It is in a microRNA</div>
        <div class='boxshow' id="show-sec-0.985119263893" style="display: inline;"><a href="javascript:toggle2('sec-0.985119263893')">...more...</a></div>
        <div class='boxrstext' id="sec-0.985119263893" style="display: none;"><a href="javascript:toggle2('sec-0.985119263893')">(hide)</a>
        <div class='rstext'>miRNA sequences and non-small cell lung cancer rs11614913 SNP in hsa-mir-196a2 was associated with survival in individuals with NSCLC. Specifically, survival was significantly decreased in individuals who were homozygous CC at SNP rs11614913. microRNA 1,009 breast cancer cases and 1,093 controls in a population of Chinese women *rs11614913 in hsa-mir-196a2 T&gt;C 1.23; 95% confidence interval [CI], 1.02-1.48 *rs3746444 in hsa-mir-499 A&gt;G OR 1.25; 95% CI, 1.02-1.51 Single nucleotide polymorphisms of microRNA machinery genes modify the risk of renal cell carcinoma. Genetic variations in microRNA-related genes are novel susceptibility loci for esophageal cancer risk. Signatures of purifying and local positive selection in human miRNAs. MicroRNA polymorphisms: the future of pharmacogenomics...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7754840'>rs7754840</a><a href='http://www.snpedia.com/index.php/rs7754840(C;G)'>(C;G)</a></div>
        <div class='boxmag'>Magnitude: 3<br/>Frequency: <span style='background-color: #FFDDDD;'>47.8%</span><br/>Repute:Bad<br/>References:72<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk for type-2 diabetes</div>
        <div class='boxshow' id="show-sec-0.0189730269993" style="display: inline;"><a href="javascript:toggle2('sec-0.0189730269993')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0189730269993" style="display: none;"><a href="javascript:toggle2('sec-0.0189730269993')">(hide)</a>
        <div class='rstext'>The accuracy of claims about this snp are called into question by table 1 of discusses the process of including this snp into Coriell and raises some doubt about the value of its inclusion. rs7754840 replicated as significant for type-2 diabetes risk in 1,900 Japanese patients, with odds ratio of 1.28 (CI: 1.17-1.41, p = 4.5 x 10e-7) type-2 diabetes Carriers of the (C;G) and (C;C) genotypes of rs7754840 had 11% and 24% lower first-phase insulin release in an IVGTT compared to that in carriers of the (G;G) genotype (P=0.002). The C allele was also associated with higher glucose area under the curve in an OGTT (P=0.016). rs7754840 type-2 diabetes Ashkenazi Jewish population shows an even stronger effect for rs7754840 than the general Caucasian population, based on 1,131 cases versus 1,147 co...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1333049'>rs1333049</a><a href='http://www.snpedia.com/index.php/rs1333049(C;G)'>(C;G)</a></div>
        <div class='boxmag'>Magnitude: 3<br/>Frequency: <span style='background-color: #FFDFDF;'>50.4%</span><br/>Repute:Bad<br/>References:95<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.5x increased risk for CAD  1.5x higher risk for coronary artery disease</div>
        <div class='boxshow' id="show-sec-0.936165942792" style="display: inline;"><a href="javascript:toggle2('sec-0.936165942792')">...more...</a></div>
        <div class='boxrstext' id="sec-0.936165942792" style="display: none;"><a href="javascript:toggle2('sec-0.936165942792')">(hide)</a>
        <div class='rstext'>rs1333049 has been reported in a large study to be associated with heart disease, in particular, coronary artery disease. The risk allele (oriented to the dbSNP entry) is most likely (C); the odds ratio associated with heterozygotes is 1.47 (CI 1.27-1.70), and for homozygotes, 1.9 (CI 1.61-2.24). This SNP has also been reported to have the highest association of any SNP studied in a subsequent experiment conducted with the resources of the German MI [Myocardial Infarction] Family Study. The initial studies were conducted on Caucasian populations. A subsequent study of Japanese and Korean patients has also found rs1333049 to be associated with increased coronary artery disease risk, with roughly similar odds ratios. '''Further reading (with comments)''' - A long-term study of a cohort of 76...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs13266634'>rs13266634</a><a href='http://www.snpedia.com/index.php/rs13266634(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 3<br/>Frequency: <span style='background-color: #FFE4E4;'>56.6%</span><br/>Repute:Bad<br/>References:118<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk for type-2 diabetes</div>
        <div class='boxshow' id="show-sec-0.248708499633" style="display: inline;"><a href="javascript:toggle2('sec-0.248708499633')">...more...</a></div>
        <div class='boxrstext' id="sec-0.248708499633" style="display: none;"><a href="javascript:toggle2('sec-0.248708499633')">(hide)</a>
        <div class='rstext'>rs13266634 is a SNP in the zinc transporter protein member 8 SLC30A8 gene that has primarily been associated with type-2 diabetes in several studies. This SNP is also known as the Arg325Trp or R325W variant; the (C) allele encodes the arginine (R), and the (T) allele encodes the tryptophan (W). significantly associated p = 0.0073; in 1,630 Japanese subjects with type-2 diabetes and in 1,064 controls The major alleles of the SLC30A8 SNP rs13266634 and the HHEX SNP rs7923837 associate with reduced insulin secretion, but not with insulin resistance. 46% of European non-diabetic offspring of type-2 diabetes patients are rs13266634(C;C) homozygotes; they are diabetes-prone and characterised by a 19% decrease in first-phase insulin release following an intravenous glucose load. * Note: this SNP,...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2011077'>rs2011077</a><a href='http://www.snpedia.com/index.php/rs2011077(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 3<br/>Frequency: <span style='background-color: #FFEAEA;'>64.6%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>In Japanese, 6.2x increased risk of developing prostate cancer, a 3x increased risk of developing BPH, and a 5.5x increased risk of developing metastatic prostate cancer  see discussion at rs2011077</div>
        <div class='boxshow' id="show-sec-0.951086403858" style="display: inline;"><a href="javascript:toggle2('sec-0.951086403858')">...more...</a></div>
        <div class='boxrstext' id="sec-0.951086403858" style="display: none;"><a href="javascript:toggle2('sec-0.951086403858')">(hide)</a>
        <div class='rstext'>A study of ~500 Japanese prostate cancer patients found that individuals with a rs2011077(G;G) genotype had a 6.2- and 3-fold increased risk of prostate cancer and benign prostate hyperplasia (BPH), and a 5.5-fold increased risk of metastatic prostate cancer, compared to the (A;A) genotype.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3095318'>rs3095318</a><a href='http://www.snpedia.com/index.php/rs3095318(A;T)'>(A;T)</a></div>
        <div class='boxmag'>Magnitude: 3<br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>In Japanese populations, associated with HLA-B*5801 allele and allopurinol side effects  see discussion at rs3095318</div>
        <div class='boxshow' id="show-sec-0.154050101938" style="display: inline;"><a href="javascript:toggle2('sec-0.154050101938')">...more...</a></div>
        <div class='boxrstext' id="sec-0.154050101938" style="display: none;"><a href="javascript:toggle2('sec-0.154050101938')">(hide)</a>
        <div class='rstext'>The minor allele rs3095318(T), as defined on the minus strand, is reported to be associated in Japanese populations with the HLA-B*5801 allele and therefore susceptibility to severe side effects from taking allopurinol and perhaps other drugs. Sequence and haplotype analysis supports HLA-C as the psoriasis susceptibility 1 gene.  [PharmGKB:Curated Risk or phenotype-associated allele: allele 1. Phenotype: Tagging SNP for HLA-B*5801. Study size: 44. Study population/ethnicity: Japanese from Tokyo, Japan. Significance metric(s): Type of association: GN]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs192"  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs192'>gs192</a></div></div>
        <div class='boxmag'>Magnitude: 2.9</div>
        <div class='boxrep'>Repute: Bad</div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>MTHFR polymorphisms affecting homocysteine</div>
        <div class='gsboxeffect'>You have a combination of 2 SNP variations in MTHFR which influence homocysteine levels. This is found in ~20% of people. 0.08% of people had a double copy of a mutation in one of the SNPs, and a single mutation in the other. . gs193 indicates double mutations in both snps, and people with this are believed to be critically impacted, however several promethease users have self reported this genotype with no apparent consequences. *rs1801131 is at position 1298 *rs1801133 is at position 677 *http://www.mthfrheds.com/ *https://www.23andme.com/you/community/thread/5312/ *https://www.23andme.com/you/community/thread/2001/ *http://www.youtube.com/watch?v=ZA8GUIRqIkE MTHFR mutation frequencies in a sample of 37,000 individuals: * 677CT/1298AA 22.8% - 1 heterozygous mutation * 677CC/1298CA 20.8% ...</div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1800497'>rs1800497</a><a href='http://www.snpedia.com/index.php/rs1800497(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.9<br/>Frequency: <span style='background-color: #FFC8C8;'>28.3%</span><br/>Repute:Bad<br/>References:111<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>A1/A2: Bad at avoidance of errors. 0.5x lower OCD risk, 0.87x lower Tardive Diskinesia risk, higher ADHD risk. More Alcohol Dependence. Lower risk of Postoperative Nausea. Increased obesity. Bupropion is not effective.  The DRD2 TaqIA A1/A2 version causes less dopamine receptors. Doesn't learn from mistakes well. Reduced response to errors and increased addictive behavior Men may have half the risk of Obsessive Compulsive Disorder but higher risk of ADHD. Women have lower Persistence. 0.87x reduced risk of Tardive Diskinesia when taking dopamine receptor antagonists. Slightly increased risk of alcoholism and smoking addiction. Slower recovery from traumatic brain injury. Lower risk of postoperative nausea within 6 hours of surgery. Bupropion (Wellbutrin, Budeprion, Prexaton, Elontril, Aple...</div>
        <div class='boxshow' id="show-sec-0.489579506508" style="display: inline;"><a href="javascript:toggle2('sec-0.489579506508')">...more...</a></div>
        <div class='boxrstext' id="sec-0.489579506508" style="display: none;"><a href="javascript:toggle2('sec-0.489579506508')">(hide)</a>
        <div class='rstext'>rs1800497, a SNP also known as the TaqIA (or Taq1A) polymorphism of the dopamine D2 receptor DRD2 gene (even though it is actually located over 10,000bp downstream of the gene), gives rise to the DRD2*A1 allele. This allele (rs1800497(T)) is associated with a reduced number of dopamine binding sites in the brain . Reduced response to errors and increased addictive behavior It has been postulated to play a role in alcoholism, smoking, and certain neuropsychiatric disorders. The reduced number of dopamine binding sites may play a role in nicotine addiction by causing an 'understimulated' state that can be relieved by smoking (and/or use of other drugs). A wide variety of reports have been published over more than ten years either linking rs1800497 to aspects of nicotine use and smoking cessa...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs100"  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs100'>gs100</a></div></div>
        <div class='boxmag'>Magnitude: 2.5</div>
        <div class='boxrep'>Repute: Bad</div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>Lactose intolerance risk</div>
        <div class='gsboxeffect'>77% of Europeans with this genoset are lactose intolerant It is detected by 2 genotypes *rs4988235(C;C) *rs182549(C;C)</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs155"  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs155'>gs155</a></div></div>
        <div class='boxmag'>Magnitude: 2.5</div>
        <div class='boxrep'>Repute: Bad</div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>CYP3A5 non-expressor</div>
        <div class='gsboxeffect'>This is common for Caucasians, but rare for African populations. As a CYP3A5 non-expressor you are unable to metabolize some common medications. This influences the synthesis of cholesterol, steroids and other lipids. The enzyme metabolizes drugs such as olanzapine, tacrolimus, nifedipine and cyclosporine as well as the steroid hormones testosterone, progesterone and androstenedione.</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs281"  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs281'>gs281</a></div></div>
        <div class='boxmag'>Magnitude: 2.5</div>
        <div class='boxrep'>Repute: Bad</div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>part of the 88% of the population claimed not to maintain weight loss unless you perform high energy exercise</div>
        <div class='gsboxeffect'>An interesting hypothesis, but not well validated, this patent is summarized in a simplified blog post and suggests a genotype to suggest diet and lifestyle changes which may be beneficial. Based on 2 snps *rs4994 *rs1042713</div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs17782313'>rs17782313</a><a href='http://www.snpedia.com/index.php/rs17782313(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FF8787;'>3.5%</span><br/>Repute:Bad<br/>References:91<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>adults likely to be 0.44 BMI units higher</div>
        <div class='boxshow' id="show-sec-0.779099797512" style="display: inline;"><a href="javascript:toggle2('sec-0.779099797512')">...more...</a></div>
        <div class='boxrstext' id="sec-0.779099797512" style="display: none;"><a href="javascript:toggle2('sec-0.779099797512')">(hide)</a>
        <div class='rstext'>A study of 60,000 adults indicates that rs17782313(C) alleles are associated with higher body mass index (BMI), with even greater effect in children. The average increase in BMI is 0.22 BMI units (p = 2.8 x 10&lt;sup&gt;-15&lt;/sup&gt;). Note that rs10871777 is considered an equivalent SNP in Caucasian and Asian populations. The rs17782313(C) allele is equivalent to the rs10871777(G) allele. The rs17782313](C) allele is significantly associated with higher intakes of total energy and dietary fat. In addition, the SNP was related to greater long-term weight change and increased risk of diabetes in women. rs17782313(C) more snacking, however rs1421085(C) did not influence eating behavior obesity and type-2 diabetes *rs1121980 *rs7498665 *rs4752856 *rs17782313 *rs2815752 *rs10938397  [PharmGK...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4988235'>rs4988235</a><a href='http://www.snpedia.com/index.php/rs4988235(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFA0A0;'>8.8%</span><br/>Repute:Bad<br/>References:16<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>likely to be lactose intolerant as an adult  see also gs100 if you are a caucasian who also has rs182549(C;C) there is 77% chance of lactose intolerance * See also OMIM 601806.0001</div>
        <div class='boxshow' id="show-sec-0.653772386501" style="display: inline;"><a href="javascript:toggle2('sec-0.653772386501')">...more...</a></div>
        <div class='boxrstext' id="sec-0.653772386501" style="display: none;"><a href="javascript:toggle2('sec-0.653772386501')">(hide)</a>
        <div class='rstext'>Also known as 'C/T(-13910)', and located in the MCM6 gene but with influence on the lactase LCT gene, rs4988235 is one of two SNPs that is associated with the primary haplotype associated with hypolactasia, more commonly known as lactose intolerance in European Caucasian populations. , In these populations, the rs4988235(T) allele is both the more common allele and the one associated with lactase persistence; individuals who are rs4988235(C;C) are likely to be lactose intolerant. In populations of sub-Saharan Africans, though, the rs4988235(T) allele is so rare that it's unlikely to be predictive of lactase persistence, and other SNPs are predictive instead. * See also OMIM 601806.0001 Measuring European population stratification with microarray genotype data. Adult-type hypolactasia is no...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6441286'>rs6441286</a><a href='http://www.snpedia.com/index.php/rs6441286(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFADAD;'>13.4%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>3.08x chance of developing primary biliary cirrhosis</div>
        <div class='boxshow' id="show-sec-0.247519039602" style="display: inline;"><a href="javascript:toggle2('sec-0.247519039602')">...more...</a></div>
        <div class='boxrstext' id="sec-0.247519039602" style="display: none;"><a href="javascript:toggle2('sec-0.247519039602')">(hide)</a>
        <div class='rstext'>23andMe blog *Each G at rs2856683 increased the odds of primary biliary cirrhosis by 1.75x. *Each G at rs6441286 increased the odds of primary biliary cirrhosis by 1.54x. *Each A at rs3790567 increased the odds of primary biliary cirrhosis by 1.51x.  [PharmGKB:Curated A GWAS of 2072 Canadian and U.S. subjects (536 patients with primary biliary cirrhosis and 1536 controls) showed that primary biliary cirrhosis was significantly and reproducibly associated with this SNP (rs6441286) at the IL12A locus (P=2.42x10(-14); odds ratio, 1.54).] [OMIM:?] [GWAS:Primary biliary cirrhosis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs12777823'>rs12777823</a><a href='http://www.snpedia.com/index.php/rs12777823(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFC3C3;'>24.8%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Avoid Plavix, higher risk for adverse cardiovascular events.  Neighboring rs4244285 is a more reliable predictor, but this genotype is linked to being a poor metabolizer of several popular medicines; patients prescribed Plavix (Clopidogrel) have higher risk for adverse cardiovascular events.</div>
        <div class='boxshow' id="show-sec-0.501037606009" style="display: inline;"><a href="javascript:toggle2('sec-0.501037606009')">...more...</a></div>
        <div class='boxrstext' id="sec-0.501037606009" style="display: none;"><a href="javascript:toggle2('sec-0.501037606009')">(hide)</a>
        <div class='rstext'>JAMA rs12777823 and rs4244285 implicated in platelet response to clopidogrel  [PharmGKB:Curated A GWAS found this variant (rs12777823) was the most significantly associated SNP with clopidogrel response. This SNP was in strong linkage disequilibrium with 12 further SNPs within the CYP2C18-CYP2C19-CYP2C9-CYP2C8 cluster (rs12777823; rs10109204; rs2025445; rs1326837; rs717238; rs2860838; rs2860903; rs9332105; rs9332113; rs12572351; rs10509679; rs1934951; rs1934680) and also with the loss-of-function variant CYP2C19*2 (rs4244285). Further findings in this study showed that the CYP2C19*2 genotype accounted for most of all the initially found association with diminished platelet response to clopidogrel.]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7574865'>rs7574865</a><a href='http://www.snpedia.com/index.php/rs7574865(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFCECE;'>33.6%</span><br/>Repute:Bad<br/>References:108<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x risk of rheumatoid arthritis; 1.55x risk of SLE; 1.42x risk of Sj&#246;gren's syndrome; increased risk of type-1 diabetes; and increased risk of primary biliary cirrhosis.</div>
        <div class='boxshow' id="show-sec-0.302889369539" style="display: inline;"><a href="javascript:toggle2('sec-0.302889369539')">...more...</a></div>
        <div class='boxrstext' id="sec-0.302889369539" style="display: none;"><a href="javascript:toggle2('sec-0.302889369539')">(hide)</a>
        <div class='rstext'>rs7574865, a SNP in the third intron of the STAT4 gene, has been reported in a large study of Swedes to be associated with both rheumatoid arthritis (RA) and lupus (SLE). Among other studies, it has been confirmed in a meta-analysis of 8 studies totaling 7,381 patients and over 10,000 controls from both European and Asian populations. The risk allele (oriented to the dbSNP entry) is (T); the odds ratio associated the presence of a risk allele was 1.3 for rheumatoid arthritis and 1.55 for lupus (SLE). The paper states that, 'Homozygosity of the risk allele, as compared with absence of the allele, was associated with a more than doubled risk for lupus and a 60% increased risk for rheumatoid arthritis.' A study of 124 Caucasian patients with primary Sj&#246;gren's syndrome, an autoimmune dise...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2254958'>rs2254958</a><a href='http://www.snpedia.com/index.php/rs2254958(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFD1D1;'>35.4%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.61x increased risk for Alzheimer's</div>
        <div class='boxshow' id="show-sec-0.849939102366" style="display: inline;"><a href="javascript:toggle2('sec-0.849939102366')">...more...</a></div>
        <div class='boxrstext' id="sec-0.849939102366" style="display: none;"><a href="javascript:toggle2('sec-0.849939102366')">(hide)</a>
        <div class='rstext'>While the ApoE4 allele (rs429358(C)) is widely accepted as the predominant genetic risk factor for Alzheimer's disease, there are likely to be numerous other factors, both genetic and environmental, associated to lesser degrees with susceptibility to the disease. Genes influencing the immune system, and in particular susceptibility to viral infections such as herpes, may be among such factors. This SNP, located in the promoter region of the EIF2AK2 gene and thus implicated in the activation of HIV and HSV-1 viruses, is seen more commonly in ~300 Alzheimer patients than in the same number of controls. The risk allele is rs2254958(C). The odds ratio is reported to be 1.61 (CI: 1.02-2.55) for rs2254958(C;C) homozygotes, and 1.24 (CI: 0.80-1.93) for rs2254958(C;T) heterozygotes, compared with ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9886784'>rs9886784</a><a href='http://www.snpedia.com/index.php/rs9886784(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFD1D1;'>35.4%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>3.23x risk for Alzheimer's disease  rs9886784, a intergenic SNP on chromosome 9, is reported to influence the risk for Alzheimer's disease based on a study of ~1100 Canadian patients. The risk allele is (A); the odds ratio is 3.23 (CI: 1.79 - 5.84).</div>
        <div class='boxshow' id="show-sec-0.978308868555" style="display: inline;"><a href="javascript:toggle2('sec-0.978308868555')">...more...</a></div>
        <div class='boxrstext' id="sec-0.978308868555" style="display: none;"><a href="javascript:toggle2('sec-0.978308868555')">(hide)</a>
        <div class='rstext'>rs9886784, a intergenic SNP on chromosome 9, is reported to influence the risk for Alzheimer's disease based on a study of ~1100 Canadian patients. The risk allele is (A); the odds ratio is 3.23 (CI: 1.79 - 5.84). Meta-analysis did NOT show rs9886784 with consistent evidence for association with AD across the investigated data sets: Genetic association analysis testing of 4 GWA signals (rs7101429 [GAB2], rs7019241 [GOLM1], rs10519262 [chromosome 15q], and rs9886784 [chromosome 9p]) using family-based methods. In the combined analyses, only rs7101429 in GAB2 yielded significant evidence of association with the same allele as in the original GWA study (P =.002). The results are in agreement with recent meta-analyses of this and other GAB2 polymorphisms suggesting approximately a 30% decrease...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10490924'>rs10490924</a><a href='http://www.snpedia.com/index.php/rs10490924(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFD3D3;'>38.1%</span><br/>Repute:Bad<br/>References:58<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2.7x risk for age related macular degeneration</div>
        <div class='boxshow' id="show-sec-0.797219883943" style="display: inline;"><a href="javascript:toggle2('sec-0.797219883943')">...more...</a></div>
        <div class='boxrstext' id="sec-0.797219883943" style="display: none;"><a href="javascript:toggle2('sec-0.797219883943')">(hide)</a>
        <div class='rstext'>rs10490924 was identified as a risk factor from chromosome 10 related to age related macular degeneration. The risk allele is (T). Odds ratios for heterozygotes and homozygotes are, respectively, 2.69 (CI: 2.22-3.27) and 8.21 (CI: 5.79-11.65). Disease risk in combination with the rs1061170 SNP in the CFH gene is dramatically increased. Homozygotes for both the rare/risk alleles at both loci are estimated to be at 57 fold higher risk for age related macular degeneration than individuals homozygous for the common alleles at both loci. A subsequent study indicated that the risk based on solely the ARMS2 SNP rs10490924 is significantly higher in smokers than in non-smokers. Based on a study of ~750 patients, rs10490924 is significantly higher in participants with choroidal neovascularization t...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2180439'>rs2180439</a><a href='http://www.snpedia.com/index.php/rs2180439(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFD5D5;'>39.8%</span><br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk of Male Pattern Baldness.  One copy of the risk allele, and one copy of the normal allele. Verified in Europeans and Asians. Family Tree DNA, which tends to skew heterozygous results like this in the most positive light, reports this as less likely to go bald, but 23andMe doesn't mention it.</div>
        <div class='boxshow' id="show-sec-0.854338367203" style="display: inline;"><a href="javascript:toggle2('sec-0.854338367203')">...more...</a></div>
        <div class='boxrstext' id="sec-0.854338367203" style="display: none;"><a href="javascript:toggle2('sec-0.854338367203')">(hide)</a>
        <div class='rstext'>spitoon rs2180439(T;T) have 2x increased odds of male pattern baldness.  [PharmGKB:Non-Curated Susceptibility variants for male-pattern baldness on chromosome 20p11. (Initial Sample Size: 296 cases, 347 controls; Replication Sample Size: 319 cases, 234 controls); (Region: 20p11.22; Reported Gene: PAX1, BQ013595, BE789145; Risk Allele: rs2180439-C) This variant is associated with Male-pattern baldness.] [OMIM:ALOPECIA, ANDROGENETIC, 3; AGA3] [GWAS:Male-pattern baldness]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11190870'>rs11190870</a><a href='http://www.snpedia.com/index.php/rs11190870(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFD7D7;'>41.6%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Possibly even more increased risk of scoliosis</div>
        <div class='boxshow' id="show-sec-0.530915263568" style="display: inline;"><a href="javascript:toggle2('sec-0.530915263568')">...more...</a></div>
        <div class='boxrstext' id="sec-0.530915263568" style="display: none;"><a href="javascript:toggle2('sec-0.530915263568')">(hide)</a>
        <div class='rstext'>The T allele of this SNP is associated with an increased risk of scoliosis in adolescent Japanese girls. [http://www.nature.com/ng/journal/v43/n12/full/ng.974.html]  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2943634'>rs2943634</a><a href='http://www.snpedia.com/index.php/rs2943634(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFDADA;'>44.2%</span><br/>Repute:Bad<br/>References:12<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>higher risk of ischemic stroke</div>
        <div class='boxshow' id="show-sec-0.212034688049" style="display: inline;"><a href="javascript:toggle2('sec-0.212034688049')">...more...</a></div>
        <div class='boxrstext' id="sec-0.212034688049" style="display: none;"><a href="javascript:toggle2('sec-0.212034688049')">(hide)</a>
        <div class='rstext'>rs2943634 is a SNP found to be reproducibly associated with heart disease rs2943634 associated with high density lipoprotein (HDL) cholesterol Association studies of 22 candidate SNPs with late-onset Alzheimer's disease. Evaluation of the association of genetic variants on the chromosomal loci 9p21.3, 6q25.1, and 2q36.3 with angiographically characterized coronary artery disease. Genome-wide association of early-onset myocardial infarction with single nucleotide polymorphisms and copy number variants. Genome-wide association studies for atherosclerotic vascular disease and its risk factors. Genetics of coronary artery disease: focus on genome-wide association studies. Evaluation of population impact of candidate polymorphisms for coronary heart disease in the Framingham Heart Study Offspri...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1801133'>rs1801133</a><a href='http://www.snpedia.com/index.php/rs1801133(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFDADA;'>44.2%</span><br/>Repute:Bad<br/>References:225<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1 copy of C677T allele of MTHFR = 60% efficiency in processing folic acid = could have somewhat elevated homocysteine and low B12 and folate levels  It is found in approximately * 48% of Hispanic Americans * 45% of Caucasian Americans * 45% of Japanese * 37% of Germans * 29% of Asians * 24% of African Americans * 12% of Sub-Saharan Africans</div>
        <div class='boxshow' id="show-sec-0.644537274478" style="display: inline;"><a href="javascript:toggle2('sec-0.644537274478')">...more...</a></div>
        <div class='boxrstext' id="sec-0.644537274478" style="display: none;"><a href="javascript:toggle2('sec-0.644537274478')">(hide)</a>
        <div class='rstext'>rs1801133 is a SNP that is relatively common and has been studied for (relatively) a long time. Also known as C677T, Ala222Val, and A222V, it encodes a variant in the MTHFR gene, which encodes an enzyme involved in folate metabolism. Homozygous rs1801133(T;T) individuals have ~30% of the expected MTHFR enzyme activity, and rs1801133(C;T) heterozygotes have ~65% activity, compared to the most common genotype, rs1801133(C;C). This reduced activity (i.e. this SNP) has been linked at least once to each of the following disorders (though not necessarily reproducibly): * autism * cancer, including ** gastric cancer ** lung cancer ** head and neck cancer ** renal cancer * cleft lip and cleft palate * coronary artery disease * dementia * depression * hyperhomocysteinemia * migraine * neural tube d...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs16969968'>rs16969968</a><a href='http://www.snpedia.com/index.php/rs16969968(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFDADA;'>45.1%</span><br/>Repute:Bad<br/>References:105<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>slightly higher risk for nicotine dependence, lower risk for cocaine dependence</div>
        <div class='boxshow' id="show-sec-0.886333085479" style="display: inline;"><a href="javascript:toggle2('sec-0.886333085479')">...more...</a></div>
        <div class='boxrstext' id="sec-0.886333085479" style="display: none;"><a href="javascript:toggle2('sec-0.886333085479')">(hide)</a>
        <div class='rstext'>Associated with smoking phenotype (p=0.007) based on association study of 2,000+ individuals, and functional studies demonstrated that the risk allele decreased response to a nicotine agonist, therefore the rs16969968(A) allele is likely to be involved in nicotine dependence. In a study 500+ individuals with cocaine dependence, the same rs16969968(A) allele was associated with a 0.67x decreased risk (p = 0.0045). A study of 200 individuals replicated the association between this SNP and nicotine dependence, and also concluded that the rs16969968(A) allele was significantly associated with 'enhanced pleasurable responses' to a person's first cigarette. 23andMe blog 23andMe considers rs1051730 equivalent The association between rs16969968 and lung cancer may, in part, be confounded by chroni...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs8034191'>rs8034191</a><a href='http://www.snpedia.com/index.php/rs8034191(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFDCDC;'>47.3%</span><br/>Repute:Bad<br/>References:46<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.27x lung cancer risk</div>
        <div class='boxshow' id="show-sec-0.0416301697871" style="display: inline;"><a href="javascript:toggle2('sec-0.0416301697871')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0416301697871" style="display: none;"><a href="javascript:toggle2('sec-0.0416301697871')">(hide)</a>
        <div class='rstext'>23andMe blog rs8034191(C;C) decreased response to alcohol rs8034191 increases susceptibility to Lung cancer 1.27 times for heterozygotes (CT) and 1.80 times for homozygotes (CC) rs8034191 increases susceptibility to Lung cancer 1.28 times for heterozygotes (CT) and 1.81 times for homozygotes (CC)  [PharmGKB:Non-Curated GWAS results: Familial Aggregation of Common Sequence Variants on 15q24-25.1 in Lung Cancer. (Initial Sample Size: 194 cases, 219 controls; Replication Sample Size: 3,878 cases, 4,831 controls); (Region: 15q25.1; Reported Gene(s): LOC123688, PSMA4, CHRNA3, CHRNA5, CHRNB4; Risk Allele: rs8034191-G); (p-value= 0.00000001).This variant is associated with Lung cancer.] [OMIM:LUNG CANCER SUSCEPTIBILITY 2; LNCR2] [GWAS:Lung cancer] [GWAS:Lung cancer]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1421085'>rs1421085</a><a href='http://www.snpedia.com/index.php/rs1421085(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFE1E1;'>53.1%</span><br/>Repute:Bad<br/>References:51<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~1.3x increased obesity risk</div>
        <div class='boxshow' id="show-sec-0.444996762404" style="display: inline;"><a href="javascript:toggle2('sec-0.444996762404')">...more...</a></div>
        <div class='boxrstext' id="sec-0.444996762404" style="display: none;"><a href="javascript:toggle2('sec-0.444996762404')">(hide)</a>
        <div class='rstext'>rs1421085 is a SNP located in the first intron of the FTO (fat mass and obesity associated) gene on chromosome 16. This SNP showed the most association with obesity in the original work by Dina et al . The rs1421085 T-to- C single-nucleotide alteration underlies the association between FTO and obesity by disrupting ARID5B-mediated repression of IRX3 and IRX5. This disruption leads to a developmental shift from browning to whitening programs and loss of mitochondrial thermogenesis Within the FTO gene, many SNPs appear to be co-inherited. The SNP showing the strongest association with body weight (i.e. body mass index, BMI) is not rs1421085, although this SNP is one of co-inherited SNPs in the FTO gene region. For more information, refer to the FTO gene or the most studied of FTO SNPs, rs993...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1121980'>rs1121980</a><a href='http://www.snpedia.com/index.php/rs1121980(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFE2E2;'>54.0%</span><br/>Repute:Bad<br/>References:28<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.67x risk for obesity</div>
        <div class='boxshow' id="show-sec-0.946521036932" style="display: inline;"><a href="javascript:toggle2('sec-0.946521036932')">...more...</a></div>
        <div class='boxrstext' id="sec-0.946521036932" style="display: none;"><a href="javascript:toggle2('sec-0.946521036932')">(hide)</a>
        <div class='rstext'>rs1121980, a SNP in the FTO gene, showed the strongest association of several SNPs in the region with early onset obesity in a study of ~1,000 Caucasians. The odds ratio for the rs1121980(T) risk allele was 1.66 (CI: 1.37&#226;&#8364;&#8220;2.01); and the odds ratios for heterozygotes and homozygotes (rs1121980(T;T)) were estimated (with a correct p&lt;0.0001) at 1.67 (CI: 1.22&#226;&#8364;&#8220;2.27) and 2.76 (CI: 1.88&#226;&#8364;&#8220;4.03), respectively. Note that although rs1121980 showed the strongest association, it is in a block in the FTO gene of at least 5 other SNPs that are in linkage disequilibrium (r&lt;sup&gt;2&lt;/sup&gt;&gt;=0.88 for all). The other 5 SNPs are: * rs9939973 * rs7193144 * rs9940128 * rs8050136 * rs9939609 This SNP is also associated with (severe) obesity i...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs664143'>rs664143</a><a href='http://www.snpedia.com/index.php/rs664143(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Frequency: <span style='background-color: #FFE2E2;'>54.0%</span><br/>Repute:Bad<br/>References:11<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Higher risk for number of cancers</div>
        <div class='boxshow' id="show-sec-0.648134145042" style="display: inline;"><a href="javascript:toggle2('sec-0.648134145042')">...more...</a></div>
        <div class='boxrstext' id="sec-0.648134145042" style="display: none;"><a href="javascript:toggle2('sec-0.648134145042')">(hide)</a>
        <div class='rstext'>600+ Korean lung cancer patients were part of a study that concluded that the rs664143(T) allele, as oriented vis-a-vis dbSNP, located in the ATM gene and also known as IVS62+60G&gt;A, was associated with increased risk, with an odds ratio of 1.68 (CI: 1.1-2.1, p&lt;0.05). In a study of 119 American patients with potentially resectable pancreatic cancer, the rs664143(C) allele was associated with poorer overall survival (ie response) to combined gerncitabine radiation therapy. Also, as 1 of 3 SNPs its risk allele is associated with poorer overall survival for pancreatic cancer patients being treated with combined gerncitabine radiation therapy; median overall survival times of 31.0, 16.2, and 10.5 months were calculated for pancreatic cancer patients carrying &lt; or = 1, 2, and 3 risk all...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs12979860'>rs12979860</a><a href='http://www.snpedia.com/index.php/rs12979860(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Repute:Bad<br/>References:184<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~20-40% of such hepatitis C patients respond to treatment  This is considered an unfavorable response genotype for hepatitis C patients considering the use of PEG-interferon-alpha containing therapeutic treatment. This basically means the odds of a successful response are lower than for patients with the rs12979860(C;C) genotype.</div>
        <div class='boxshow' id="show-sec-0.261571069096" style="display: inline;"><a href="javascript:toggle2('sec-0.261571069096')">...more...</a></div>
        <div class='boxrstext' id="sec-0.261571069096" style="display: none;"><a href="javascript:toggle2('sec-0.261571069096')">(hide)</a>
        <div class='rstext'>rs12979860 is a SNP near the IL28B gene, encoding interferon-lambda-3 (IFN-lambda-3). Several studies now indicate that this SNP predicts hepatitis C treatment-induced viral clearance. It is associated with an approximately twofold change in response to pegylated interferon-alpha (PEG-IFN-alpha) plus ribavirin (RBV) treatment, both among patients of European ancestry (p = 1.06 x 10e-25) and African-Americans (p = 2.06 x 10e-3). These studies indicate that the virus was eradicated in ~80% of (C;C) patients, compared to only about 25% of those with (T;T), while (C;T) response was intermediate. FDA PGx mentions about rs12979860 with respect to Boceprevir, Simeprevir, Sofosbuvir, and Telaprevir 23andMe blog rs8099917(C) less likely to respond to Hepatitis C treatment. A functional SNP rs368234...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs59912467'>rs59912467</a><a href='http://www.snpedia.com/index.php/rs59912467(C;G)'>(C;G)</a></div>
        <div class='boxmag'>Magnitude: 2.5<br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>somewhat possible association with Peutz-Jeghers syndrome  see discussion at rs59912467</div>
        <div class='boxshow' id="show-sec-0.0446237722737" style="display: inline;"><a href="javascript:toggle2('sec-0.0446237722737')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0446237722737" style="display: none;"><a href="javascript:toggle2('sec-0.0446237722737')">(hide)</a>
        <div class='rstext'>rs59912467, also known as F354L or Phe354Leu, is a SNP in the STK11 gene on chromosome 19. One report links the minor allele of this SNP to Peutz-Jeghers syndrome (PJS), however, that report also mentions that the patient's mother was asymptomatic, and since PJS is usually inherited in a dominant manner, this implies quite variable penetrance. In ClinVar, this mutation is listed as of uncertain significance, since there are reports indicating it is benign as well as reports indicating it is pathogenic. See also OMIM 602216.0024 This variant meets the criteria published in 2013 by the ACMG regarding incidental findings in exome or genome sequencing, as a variant that they do recommend informing a patient about.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9939609'>rs9939609</a><a href='http://www.snpedia.com/index.php/rs9939609(A;T)'>(A;T)</a></div>
        <div class='boxmag'>Magnitude: 2.4<br/>Frequency: <span style='background-color: #FFE4E4;'>56.6%</span><br/>Repute:Bad<br/>References:300<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x risk for T2D; obesity risk  one copy of the genotype which increases production of the appetite stimulating hormone ghrelin. This increases your risk for obesity and Type-2 diabetes by approximately 20%.</div>
        <div class='boxshow' id="show-sec-0.6642320714" style="display: inline;"><a href="javascript:toggle2('sec-0.6642320714')">...more...</a></div>
        <div class='boxrstext' id="sec-0.6642320714" style="display: none;"><a href="javascript:toggle2('sec-0.6642320714')">(hide)</a>
        <div class='rstext'>rs9939609 is a SNP in the fat mass and obesity associated FTO gene, aka the 'Fat Gene' The original paper describing it is here . 2015 research suggests rs1421085] may be the causal snp. The researchers identified 10 different FTO] SNPs in the first intron of the gene that associated with both BMI and type-2 diabetes. Because they all correlated with each other, they chose to examine one of the SNPs (rs9939609) in detail. rs9939609 has also been reported in a large study to be associated with type-2 diabetes. The risk allele (oriented to the dbSNP entry) is (A); the odds ratio associated with heterozygotes is 1.34 (CI 1.17-1.52), and for homozygotes, 1.55 (CI 1.3-1.84). In a different study, this SNP was found in a linkage block in the FTO gene with rs1121980; see rs1121980 for the associa...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1143679'>rs1143679</a><a href='http://www.snpedia.com/index.php/rs1143679(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2.4<br/>Repute:Bad<br/>References:21<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.78x increased risk for SLE</div>
        <div class='boxshow' id="show-sec-0.887423502144" style="display: inline;"><a href="javascript:toggle2('sec-0.887423502144')">...more...</a></div>
        <div class='boxrstext' id="sec-0.887423502144" style="display: none;"><a href="javascript:toggle2('sec-0.887423502144')">(hide)</a>
        <div class='rstext'>Causal SNP in the ITGAM (CD11b) gene in patients with SLE who are of European or African descent;(P = 1.7 x 10(-17), odds ratio = 1.78). reconfirmed  [PharmGKB:Curated This nonsynonymous SNP is an associated with risk of systemic lupus erythematosus in a study with 3,818 individuals of European descent.] [OMIM:INTEGRIN, ALPHA-M; ITGAM]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3825942'>rs3825942</a><a href='http://www.snpedia.com/index.php/rs3825942(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2.3<br/>Frequency: <span style='background-color: #FFEDED;'>68.3%</span><br/>Repute:Bad<br/>References:46<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>common but 10x higher glaucoma risk in most (but not all) populations  This genotype is found in 2 out of 3 people, but has been repeatedly linked to a much higher risk of glaucoma. However a 2010 paper in a south african population reports reduced risk, casting some doubts.</div>
        <div class='boxshow' id="show-sec-0.953553888208" style="display: inline;"><a href="javascript:toggle2('sec-0.953553888208')">...more...</a></div>
        <div class='boxrstext' id="sec-0.953553888208" style="display: none;"><a href="javascript:toggle2('sec-0.953553888208')">(hide)</a>
        <div class='rstext'>rs3825942, also known as G153D, a SNP causing an amino acid change in the lysyl oxidase 1 LOXL1 gene, has been linked to exfoliation glaucoma (also known as exfoliation syndrome). This form of glaucoma causes up to 10% of the cases of blindness in many countries, including the US. From the abstract of this study: 'Approximately 25% of the general population is homozygous for the highest risk haplotype [C;C here combined with other risk SNPs] and their risk of suffering XFG (exfoliation glaucoma) is over 100 times that of those only carrying low-risk haplotypes.' The risk allele for this SNP is rs3825942(C), as oriented with respect to the dbSNP entry, and it confers a estimated relative risk (by itself) of 27 compared to the (T) allele. The odds ratio is 20.10 (CI 10.80-37.41). [Note that ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7913069'>rs7913069</a><a href='http://www.snpedia.com/index.php/rs7913069(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.2<br/>Frequency: <span style='background-color: #FF7777;'>1.8%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.47x risk for uterine fibroids</div>
        <div class='boxshow' id="show-sec-0.947942607744" style="display: inline;"><a href="javascript:toggle2('sec-0.947942607744')">...more...</a></div>
        <div class='boxrstext' id="sec-0.947942607744" style="display: none;"><a href="javascript:toggle2('sec-0.947942607744')">(hide)</a>
        <div class='rstext'>rs7913069 is a snp located in an intergenic region of chr 10 at position 105704639. It has been shown in one study to be associated with increased risk for Uterine fibroids. In an exploratory study of 1612 cases and 1428 controls from Japan, the T allele of rs7913069 was shown to be associated with increased odds of developing uterine fibroids with a pvalue of 7.9x10-8 and an odds ratio of 1.47. The association validated in a second cohort of 3,466 cases and 3,245 controls (also Japanese) by the same authors, with a combined pvalue from both studies of 8.65x10-14. rs7913069 is located in the L1 repeat L1ME2. Its closest gene neighbors are SLK (12kb downstream), OBFC1 (36.6kb upstream). OBFC1 is a subunit of an alpha accessory factor (AAF) that stimulates the activity of DNA olymerase-alpha...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2272783'>rs2272783</a><a href='http://www.snpedia.com/index.php/rs2272783(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.2<br/>Frequency: <span style='background-color: #FFA6A6;'>10.7%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Low expression variant of FECH enzyme  If your other FECH allele is normal, you are most likely asymptomatic. However, if you harbor this rs2272783(C;T) genotype AND your other FECH allele is actually nonfunctional, you are likely to have symptoms associated with Erythropoietic protoporphyria.</div>
        <div class='boxshow' id="show-sec-0.870482094061" style="display: inline;"><a href="javascript:toggle2('sec-0.870482094061')">...more...</a></div>
        <div class='boxrstext' id="sec-0.870482094061" style="display: none;"><a href="javascript:toggle2('sec-0.870482094061')">(hide)</a>
        <div class='rstext'>rs2272783, also known as IVS3-48T-C, is a mutation in the ferrochelatase FECH gene leading to lowered expression of that enzyme. When coupled in trans with a nonfunctional FECH allele, this is likely to lead to erythropoietic protoporphyria. See also: OMIM 612386.0015</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs944289'>rs944289</a><a href='http://www.snpedia.com/index.php/rs944289(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2.2<br/>Frequency: <span style='background-color: #FFD4D4;'>38.9%</span><br/>Repute:Bad<br/>References:16<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2.6x increased thyroid cancer risk</div>
        <div class='boxshow' id="show-sec-0.190972168335" style="display: inline;"><a href="javascript:toggle2('sec-0.190972168335')">...more...</a></div>
        <div class='boxrstext' id="sec-0.190972168335" style="display: none;"><a href="javascript:toggle2('sec-0.190972168335')">(hide)</a>
        <div class='rstext'>Each A at rs965513 increased the odds of thyroid cancer by 1.75 times. Each T at rs944289 increased the odds of thyroid cancer by 1.37 times. A genoset, Gs137, has also been created in SNPedia to represent the increased risk reported for carriers of both variants. See also: 23andMe blog  [PharmGKB:Non-Curated GWAS results: Common variants on 9q22.33 and 14q13.3 predispose to thyroid cancer in European populations. (Initial Sample Size: 192 cases, 37,196 controls; Replication Sample Size: 432 cases, 1,727 controls); (Region: 14q13.3; Reported Gene(s): NKX2-1; Risk Allele: rs944289-T); (p-value= 0.000000002).This variant is associated with Thyroid cancer.] [OMIM:THYROID CARCINOMA, PAPILLARY]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2231137'>rs2231137</a><a href='http://www.snpedia.com/index.php/rs2231137(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FF8484;'>3.1%</span><br/>Repute:Bad<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~1.5-3x increased risk for ischemic stroke</div>
        <div class='boxshow' id="show-sec-0.204476258671" style="display: inline;"><a href="javascript:toggle2('sec-0.204476258671')">...more...</a></div>
        <div class='boxrstext' id="sec-0.204476258671" style="display: none;"><a href="javascript:toggle2('sec-0.204476258671')">(hide)</a>
        <div class='rstext'>rs2231137, also known as Val12Met, is a SNP in the ATP-binding cassette, sub-family G (WHITE), member 2 ABCG2 gene. The (G) allele encodes the Val. In a study of incident ischemic stroke during 14 years of follow-up in a population-based study of older adults known as the Cardiovascular Health Study (CHS), rs2231137 was associated with stroke in both white (hazard ratio, 1.46, CI: 1.05 - 2.03) and black (hazard ratio, 3.59, CI, 1.11 - 11.6) participants. The risk of ischemic stroke was higher in Val allele homozygotes than in Met allele carriers. The adjusted hazard ratio for Val allele homozygotes, compared with Met allele carriers, was 1.50 (90% CI, 1.06 to 2.12) in whites and 3.62 (90% CI, 1.11 to 11.9) in black participants (Table 4).  [PharmGKB:Curated control size=7; M12 cases=6; PK=...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10871777'>rs10871777</a><a href='http://www.snpedia.com/index.php/rs10871777(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FF8787;'>3.5%</span><br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>adults likely to be 0.44 BMI units higher</div>
        <div class='boxshow' id="show-sec-0.409386124872" style="display: inline;"><a href="javascript:toggle2('sec-0.409386124872')">...more...</a></div>
        <div class='boxrstext' id="sec-0.409386124872" style="display: none;"><a href="javascript:toggle2('sec-0.409386124872')">(hide)</a>
        <div class='rstext'>See rs17782313 for details. Measures of Obesity  [GWAS:Obesity (extreme)]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4693596'>rs4693596</a><a href='http://www.snpedia.com/index.php/rs4693596(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFAFAF;'>14.2%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2x odds of myopathy if taking statins</div>
        <div class='boxshow' id="show-sec-0.0345616209424" style="display: inline;"><a href="javascript:toggle2('sec-0.0345616209424')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0345616209424" style="display: none;"><a href="javascript:toggle2('sec-0.0345616209424')">(hide)</a>
        <div class='rstext'>rs4693596(C;C) 2x odds of myopathy when taking statins. Statin Response</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2383207'>rs2383207</a><a href='http://www.snpedia.com/index.php/rs2383207(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFBFBF;'>22.1%</span><br/>Repute:Bad<br/>References:31<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk for heart disease</div>
        <div class='boxshow' id="show-sec-0.721877719398" style="display: inline;"><a href="javascript:toggle2('sec-0.721877719398')">...more...</a></div>
        <div class='boxrstext' id="sec-0.721877719398" style="display: none;"><a href="javascript:toggle2('sec-0.721877719398')">(hide)</a>
        <div class='rstext'>discussed in this blog post as possibly playing a role in coronary heart disease This SNP was also associated with increased risk for coronary artery disease in a Korean population. The association remained significant after adjusting for significant clinical covariates (P=0.001 to 0.024). We identified one risk haplotype (GGGG; P=0.017) and one protective haplotype (AAAA; P=0.007) for development of CAD. Further analysis suggested that the SNPs probably confer susceptibility to CAD in a dominance model (covariates-adjusted P=0.001 to 0.024; OR=2.37 to 1.54). Also found to be significant in a study of 416 Italian myocardial infarction patients. Found to be associated with coronary heart disease in a study of 1360 Chinese Han patients, with an odds ratio of 1.52 (CI: 1.13 - 2.04) for rs2383...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs279858'>rs279858</a><a href='http://www.snpedia.com/index.php/rs279858(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFC3C3;'>24.8%</span><br/>Repute:Bad<br/>References:19<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>apparently more prone to alcoholism  interacts with the medicine Finasteride</div>
        <div class='boxshow' id="show-sec-0.67562180371" style="display: inline;"><a href="javascript:toggle2('sec-0.67562180371')">...more...</a></div>
        <div class='boxrstext' id="sec-0.67562180371" style="display: none;"><a href="javascript:toggle2('sec-0.67562180371')">(hide)</a>
        <div class='rstext'>This SNP in the GABRA2 gene has been linked to Alcoholism. The effect of this SNP, a synonymous change at amino acid residue 132, is unknown, but it does not change the GABRA2 protein sequence. Carriers of at least one rs279858(G) allele respond slower to the effects of alcohol and are thereby apparently more prone to alcoholism than carriers of two rs279858(A) alleles. Finasteride, an FDA approved drug for the treatment of benign prostatic hypertrophy and a modulator of GABRA2, was shown to have more of a blocking effect on rs279858(A) homozygotes. Note that nearby SNPs have been grouped into a haplotype that shows statistically stronger association with alcohol dependence, namely the haplotype consisting of the rs279871(A) + rs279845(T) + rs279836(A) alleles . 1050 nicotine-dependent cas...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1052133'>rs1052133</a><a href='http://www.snpedia.com/index.php/rs1052133(C;G)'>(C;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFC8C8;'>28.6%</span><br/>Repute:Bad<br/>References:64<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>normal for bladder cancer risk; 1.9x increased risk for gallbladder cancer</div>
        <div class='boxshow' id="show-sec-0.618774464404" style="display: inline;"><a href="javascript:toggle2('sec-0.618774464404')">...more...</a></div>
        <div class='boxrstext' id="sec-0.618774464404" style="display: none;"><a href="javascript:toggle2('sec-0.618774464404')">(hide)</a>
        <div class='rstext'>rs1052133, also known as Ser326Cys, is a SNP in the oxoguanine glycosylase 1 OGG1 gene, involved in the repair of DNA damage (for example, damage caused by tobacco smoke). The minor (rarer) allele, encoding the cysteine (Cys), is rs1052133(G). Note that associations reported for this SNP are not always replicating in different studies or populations. A study of 250 Japanese patients with bladder cancer found an adjusted odds ratio of 2.05 (CI: 1.36-3.08; p = 0.01) for (G;G) homozygotes compared to the other genotypes. In addition, if evaluated only in non-smokers, the adjusted odds ratio increased to 2.78 (CI: 1.39-5.60; p &lt; 0.01). A study of 200 Chinese patients with gallbladder cancer concluded that increased risk was associated with both the (C;G) genotype (odds ratio 1.9, (CI: = 1.0...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1329428'>rs1329428</a><a href='http://www.snpedia.com/index.php/rs1329428(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFCECE;'>33.0%</span><br/>Repute:Bad<br/>References:19<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2x increased risk for macular degeneration</div>
        <div class='boxshow' id="show-sec-0.890687917804" style="display: inline;"><a href="javascript:toggle2('sec-0.890687917804')">...more...</a></div>
        <div class='boxrstext' id="sec-0.890687917804" style="display: none;"><a href="javascript:toggle2('sec-0.890687917804')">(hide)</a>
        <div class='rstext'>linked to blindness in age related macular degeneration rs3753394, rs800292, rs1061147, rs1061170, rs380390, and rs1329428 Significant associations were detected for AMD with rs3753394 rs800292 rs1329428 A haplotype of rs1061170 rs3753394 rs800292 rs1329428 (TGTC) was found to confer a significantly increased likelihood of exudative AMD CFH variations appear to contribute to ARMD in Caucasians, but not in Japanese  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7903146'>rs7903146</a><a href='http://www.snpedia.com/index.php/rs7903146(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFD0D0;'>34.5%</span><br/>Repute:Bad<br/>References:296<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x increased risk for diabetes (and perhaps colon cancer).  Higher risk for Type 2 Diabetes and Gestational Diabetes. Reduced insulin secretion in the pancreas. This has been confirmed in most races.</div>
        <div class='boxshow' id="show-sec-0.158033134428" style="display: inline;"><a href="javascript:toggle2('sec-0.158033134428')">...more...</a></div>
        <div class='boxrstext' id="sec-0.158033134428" style="display: none;"><a href="javascript:toggle2('sec-0.158033134428')">(hide)</a>
        <div class='rstext'>This SNP in TCF7L2 influences the risk of Type-2 diabetes (T2D). This SNP is also known as IVS3C&gt;T. rs7903146(C;T) rs7903146(T;T) strongly predicted future type-2 diabetes. Considered in context with rs7903146 rs12255372 rs10885406. Note: this is one of two SNPs within the TCF7L2 gene that have been reported to be associated with type-2 diabetes, the other being rs4506565. They have approximately equal power to estimate risk for type-2 diabetes, and the results from one correlate 92% of the time with the other. Full text of the paper is available from from Plos Medicine. Or from NCBI as . TCF7L2 polymorphisms and progression to diabetes in the Diabetes Prevention Program. reconfirmed in a diverse population Associated in a study of ~1000 Hispanic-Americans with reduced insulin secretion...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10821936'>rs10821936</a><a href='http://www.snpedia.com/index.php/rs10821936(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFD8D8;'>42.5%</span><br/>Repute:Bad<br/>References:13<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Higher risk of childhood acute lymphoblastic leukemia  Consistently, when we further classified children into age groups of those aged less than 5 years, those aged 5 to 10 years, and those aged greater than 10 years, there was a trend for decreasing allelic odds ratio (ie, relative risk of ALL conferred by each copy of the C allele at rs10821936) as age increased: 2.01 (95% CI = 1.85 to 2.19), 1.8 (95% CI = 1.6 to 2.02), and 1.48 (95% CI = 1.3 to 1.68), respectively.</div>
        <div class='boxshow' id="show-sec-0.0809575765749" style="display: inline;"><a href="javascript:toggle2('sec-0.0809575765749')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0809575765749" style="display: none;"><a href="javascript:toggle2('sec-0.0809575765749')">(hide)</a>
        <div class='rstext'>[PharmGKB:Curated Phenotype: This SNP is associated with risk of childhood acute lymphoblastic leukemia in blacks and also with racial differences in frequency of the disease . The frequency of the risk allele, C, was significantly higher(0.33) in black children with ALL enrolled in St. Jude clinical trials than in controls of African ancestry (0.163 ASW,0.20 YRI). Risk Allelle: C allele. Study size/population/ethnicity: patients-93 black children with newly diagnosed ALL; controls- 112 unrelated subjects of African ancestry from the HapMap project(52 from American Southwest and 60 from Yoruba) . Significance metric(s): OR = 2.08; 95% CI = 1.32-3.27; p = 0.0015 . Type of association: CO; GN. The association of the C allele (one or two copies) with risk for the ALL subtype B-hyperdiploid AL...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs560887'>rs560887</a><a href='http://www.snpedia.com/index.php/rs560887(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFDADA;'>44.6%</span><br/>Repute:Bad<br/>References:39<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Avg. fasting Plasma Glucose 5.18 mmol/L (93 mg/dl).  Slightly higher blood glucose. Fasting Plasma Glucose is the amount of sugar in your blood when you wake up. It's required to power your body, but too much is very bad for you. Fasting plasma glucose between 5.6 mmol/L and 6.9 mmol/L (101 &#8211; 125 mg/dl) is considered pre-diabetes. Fasting plasma glucose higher than 7 mmol/L (126 mg/dl) is considered diabetes.</div>
        <div class='boxshow' id="show-sec-0.310861946752" style="display: inline;"><a href="javascript:toggle2('sec-0.310861946752')">...more...</a></div>
        <div class='boxrstext' id="sec-0.310861946752" style="display: none;"><a href="javascript:toggle2('sec-0.310861946752')">(hide)</a>
        <div class='rstext'>rs560887, p = 4 x 10(-7)) *654 normoglycemic subjects, replicated in 9,353 subjects *fasting plasma glucose (FPG) (beta = -0.06 mmol/l per A-allele, combined p = 4 x 10(-23)) and with pancreatic beta-cell function (Homa-B model, combined p = 3 x 10(-13)) in three populations; however it was not associated with type 2 diabetes risk. (ageing) Blood Glucose  [PharmGKB:Non-Curated Phenotype 1: In a meta-analysis of 21 GWAS cohorts followed by analysis in additional individuals, this SNP was found to be associated with fasting glucose level. Study size: 119,169. Significance metric(s): p = 8.7 x 10(-218). Phenotype 2: In the same study, this SNP was found to be associated with HOMA-B (homeostasis model assessment of beta-cell function). Study size: 94,839. Significance metric(s): p = 1.5 x 10(-...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1801131'>rs1801131</a><a href='http://www.snpedia.com/index.php/rs1801131(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFDADA;'>45.1%</span><br/>Repute:Bad<br/>References:123<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Possible risks. Complex.  MTHFR rs1801131 is involved in converting 5-methylfolate (5MTHF) to tetrahydrofolate (THF). Unlike MTHFR C677T, this mutation does not lead to elevated homocysteine levels.</div>
        <div class='boxshow' id="show-sec-0.656088543787" style="display: inline;"><a href="javascript:toggle2('sec-0.656088543787')">...more...</a></div>
        <div class='boxrstext' id="sec-0.656088543787" style="display: none;"><a href="javascript:toggle2('sec-0.656088543787')">(hide)</a>
        <div class='rstext'>rs1801131 is a SNP in the MTHFR gene, representing an A&gt;C mutation at mRNA position 1298, resulting in a glu429-to-ala (E429A) substitution (hence this SNP is also known as A1298C or E429A). A diplotype (according to the authors, but perhaps actually a genoset) of rs1801131 and rs1801133 has been linked to increased risk for several types of brain cancer. The highest risk of meningioma was associated with heterozygosity for both MTHFR SNPs (odds ratio 2.11, CI: 1.42-3.12, p=0.002). The corresponding odds ratio for glioma was 1.23 (CI: 0.91-1.66, p=0.02. In general, risks were increased with genotypes associated with reduced MTHFR activity. rs1801131 shows no consistent association with schizophrenia overall in 12 studies totaling 3,000+ patients  [PharmGKB:Curated Risk or phenotype-asso...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs17070145'>rs17070145</a><a href='http://www.snpedia.com/index.php/rs17070145(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFDCDC;'>46.9%</span><br/>Repute:Bad<br/>References:16<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>reduced memory abilities</div>
        <div class='boxshow' id="show-sec-0.761266108831" style="display: inline;"><a href="javascript:toggle2('sec-0.761266108831')">...more...</a></div>
        <div class='boxrstext' id="sec-0.761266108831" style="display: none;"><a href="javascript:toggle2('sec-0.761266108831')">(hide)</a>
        <div class='rstext'>In a Swiss cohort: 'Carriers of KIBRA rs17070145 T allele had 24% better free recall performance 5 min after word presentation (P = 0.000004) and 19% better free recall performance 24 hours after word presentation (P = 0.0008) than did noncarriers.' In a US cohort: 'T allele carriers had significantly better memory scores than non-carriers in the Buschke's Selective Reminding Test (SRT). Performance on another episodic memory task, the Rey Auditory Verbal Learning Test (AVLT), was also significantly different between allele groups.' In another Swiss cohort, with a visual episodic memory task: 'T allele carriers also performed significantly better than did noncarriers in this population.' 'Functional magnetic resonance imaging detected KIBRA allele-dependent differences in hippocampal activ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9272346'>rs9272346</a><a href='http://www.snpedia.com/index.php/rs9272346(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFDDDD;'>47.8%</span><br/>Repute:Bad<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>5.5x risk type-1 diabetes</div>
        <div class='boxshow' id="show-sec-0.0857946066978" style="display: inline;"><a href="javascript:toggle2('sec-0.0857946066978')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0857946066978" style="display: none;"><a href="javascript:toggle2('sec-0.0857946066978')">(hide)</a>
        <div class='rstext'>rs9272346 has been reported in a large study to be associated with type-1 diabetes. The risk allele (oriented to the dbSNP entry) is (A); the odds ratio associated with heterozygotes is 5.49 (CI 4.83-6.24), and for homozygotes, 18.52 (CI 12.69-27.03). Note that the (A) allele is the most common in various human populations; instead of labeling (A) as a risk allele, we could consider the (G) allele protective. Risk alleles for multiple sclerosis identified by a genomewide study. Pharmacogenetics: data, concepts and tools to improve drug discovery and drug treatment. Candidate causal regulatory effects by integration of expression QTLs with complex trait genetic associations. A study of CNVs as trait-associated polymorphisms and as expression quantitative trait loci.  [PharmGKB:Curated GWAS ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3761847'>rs3761847</a><a href='http://www.snpedia.com/index.php/rs3761847(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFE3E3;'>54.9%</span><br/>Repute:Bad<br/>References:40<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.32 x risk of rheumatoid arthritis.  A common genetic variant at the TRAF1-C5 locus on chromosome 9 is associated with an increased risk of anti-CCP-positive rheumatoid arthritis. rs3761847, a SNP located between two genes associated with chronic inflammation (TRAF1 and C5), is associated with increased risk of anti CCP-positive rheumatoid arthritis. The risk allele is (G); the odds ratio reported is 1.32 (95% confidence interval, 1.23 to 1.42; P=4x10-14) .</div>
        <div class='boxshow' id="show-sec-0.295806248036" style="display: inline;"><a href="javascript:toggle2('sec-0.295806248036')">...more...</a></div>
        <div class='boxrstext' id="sec-0.295806248036" style="display: none;"><a href="javascript:toggle2('sec-0.295806248036')">(hide)</a>
        <div class='rstext'>rs3761847, a SNP located between two genes associated with chronic inflammation (TRAF1 and C5), is associated with increased risk of anti CCP-positive rheumatoid arthritis. The risk allele is (G); the odds ratio reported is 1.32 (95% confidence interval, 1.23 to 1.42; P=4x10-14) . rs3761847(G;G) associated with an increased risk of death (hazard ratio 3.96 [95% confidence interval 1.24-12.6], P = 0.020) as compared with (A;A) homozygote status. The excess mortality was attributed to deaths due to malignancies and sepsis but not cardiovascular disease (CVD).  [PharmGKB:Non-Curated GWAS Results: TRAF1-C5 as a risk locus for rheumatoid arthritis--a genomewide study (Initial Sample Size: 1,493 cases, 1,831 controls; Replication Sample Size: 1,053 cases, 1,858 controls; Risk Allele: rs3761847-G...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10811661'>rs10811661</a><a href='http://www.snpedia.com/index.php/rs10811661(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFEBEB;'>65.5%</span><br/>Repute:Bad<br/>References:109<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.2x increased risk for type-2 diabetes</div>
        <div class='boxshow' id="show-sec-0.343205239418" style="display: inline;"><a href="javascript:toggle2('sec-0.343205239418')">...more...</a></div>
        <div class='boxrstext' id="sec-0.343205239418" style="display: none;"><a href="javascript:toggle2('sec-0.343205239418')">(hide)</a>
        <div class='rstext'>rs10811661 replicated as significant for type-2 diabetes risk in 1,900 Japanese patients, with odds ratio of 1.27 (CI: 1.15-1.40, p = 1.9 x 10e-6) rs10811661 is mentioned in part 2 of a series significantly associated with type-2 diabetes p = 0.0024; in 1,630 Japanese subjects with type-2 diabetes and in 1,064 controls 1,638 type 2 diabetes patients and 1,858 controls *rs10811661 in the vicinity of CDKN2B (OR 1.20, 95% CI: 1.06-1.37, p = 0.004) T2D and normal glucose tolerant (NGT) individuals. (3,295 T2D and 3,595 NGT), strong associations with T2D were found for *CDKAL1 (OR(rs7756992) = 1.30[1.19-1.42], P = 2.3x10(-9)) *CDKN2A/2B (OR(rs10811661) = 0.74[0.66-0.82], P = 3.5x10(-8)) *IGFBP2 (OR(rs1470579) = 1.17[1.07-1.27], P = 0.0003) SNPs. T2D risk increased strongly when risk alleles, in...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7837688'>rs7837688</a><a href='http://www.snpedia.com/index.php/rs7837688(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFF7F7;'>85.0%</span><br/>Repute:Bad<br/>References:10<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.7x increased risk for prostate cancer</div>
        <div class='boxshow' id="show-sec-0.88016768854" style="display: inline;"><a href="javascript:toggle2('sec-0.88016768854')">...more...</a></div>
        <div class='boxrstext' id="sec-0.88016768854" style="display: none;"><a href="javascript:toggle2('sec-0.88016768854')">(hide)</a>
        <div class='rstext'>rs7837688 is one of 4 tightly linked SNPs in the 'locus 1' region of 8q24 chromosomal region, which has been linked in several studies to prostate cancer, initially through rs1447295. In a study of 1,563 patients of European ancestry, the odds ratio for prostate cancer associated with risk genotypes of locus 1 were reported as 1.70 (CI: 1.39-2.07). Two other regions of chromosome 8q24 were also studied.  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4444903'>rs4444903</a><a href='http://www.snpedia.com/index.php/rs4444903(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Repute:Bad<br/>References:19<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>3.5x risk of hep-cancer in cirrhosis patients; higher glioma risk; 3x higher gallbladder cancer risk in females</div>
        <div class='boxshow' id="show-sec-0.0677933344358" style="display: inline;"><a href="javascript:toggle2('sec-0.0677933344358')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0677933344358" style="display: none;"><a href="javascript:toggle2('sec-0.0677933344358')">(hide)</a>
        <div class='rstext'>rs4444903 is a SNP, also known as +61, located in the promoter region of the epidermal growth factor EGF gene that influences the amount of EGF produced. The rs4444903(G) allele appears to produce higher amounts of EGF than the (A) allele. A study of 2 Caucasian populations of patients with alcoholic cirrhosis indicates that, for these patients, the presence of the risk allele rs4444903(G) increases the chances of developing hepatocellular carcinoma (liver cancer). After adjusting for age, sex, race, etiology, and severity of cirrhosis the number of (G) copies led to an odds ratio for either (G;G) or (A;G) genotypes versus the (A;A) genotype of 3.49 (CI: 1.29-9.44, p=.01). In another study, analysis of 197 glioma patients showed that the rs4444903(G) allele conferred higher risks for gliom...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs543749'>rs543749</a><a href='http://www.snpedia.com/index.php/rs543749(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>if a smoker, somewhat more likely to develop COPD  if a smoker, somewhat more likely to develop COPD</div>
        <div class='boxshow' id="show-sec-0.138281738521" style="display: inline;"><a href="javascript:toggle2('sec-0.138281738521')">...more...</a></div>
        <div class='boxrstext' id="sec-0.138281738521" style="display: none;"><a href="javascript:toggle2('sec-0.138281738521')">(hide)</a>
        <div class='rstext'>rs543749, also known as IVS21_22 or V-1, is a SNP in the ADAM metallopeptidase domain 33 ADAM33 gene. A study of ~600 non-Hispanic white males who smoked concluded that rs543749(G;G) carriers, i.e. those homozygous for the most common allele, were more likely (p&lt; 0.0026, after Bonferroni correction) to develop chronic obstructive pulmonary disease (COPD) than carriers of one or two rs543749(T) alleles.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs5751876'>rs5751876</a><a href='http://www.snpedia.com/index.php/rs5751876(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Repute:Bad<br/>References:19<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>significantly higher anxiety levels after moderate caffeine consumption</div>
        <div class='boxshow' id="show-sec-0.00064153082002" style="display: inline;"><a href="javascript:toggle2('sec-0.00064153082002')">...more...</a></div>
        <div class='boxrstext' id="sec-0.00064153082002" style="display: none;"><a href="javascript:toggle2('sec-0.00064153082002')">(hide)</a>
        <div class='rstext'>Variation at this SNP, rs5751876, also referred to as 1976C&gt;T or 1976T&gt;C, appears to influence how an individual responds to moderate levels (150mg) of caffeine. T/T carriers displaying significantly increased subjective levels of anxiety after consumption. However a PharmGKB variant annotation suggests it is (C;C) genotypes who are more sensitive to caffeine. rs5751876 is not associated with how much coffee one drinks, though, nor with risk for Parkinson's disease, based on a study of 1,200 subjects. 'rs2298383 shows functional potential based on in silico analyses and might therefore represent the true underlying causal variant'</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs380390'>rs380390</a><a href='http://www.snpedia.com/index.php/rs380390(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #F0B4F0;'>21.5%</span><br/>Repute:Bad<br/>References:22<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk for ARMD</div>
        <div class='boxshow' id="show-sec-0.462239654969" style="display: inline;"><a href="javascript:toggle2('sec-0.462239654969')">...more...</a></div>
        <div class='boxrstext' id="sec-0.462239654969" style="display: none;"><a href="javascript:toggle2('sec-0.462239654969')">(hide)</a>
        <div class='rstext'>linked to blindness in old age rs3753394, rs800292, rs1061147, rs1061170, rs380390, and rs1329428 CFH variations appear to contribute to ARMD in Caucasians, but not in Japanese  [PharmGKB:Non-Curated GWAS Results: Complement factor H polymorphism in age-related macular degeneration (Initial Sample Size: 96 cases, 50 controls; Replication Sample Size: NR; Risk Allele: rs380390-C).] [OMIM:EXCISION-REPAIR CROSS-COMPLEMENTING, GROUP 6; ERCC6] [GWAS:Age-related macular degeneration]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs104"  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs104'>gs104</a></div></div>
        <div class='boxmag'>Magnitude: 2</div>
        <div class='boxrep'>Repute: Bad</div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>Restless legs syndrome risk</div>
        <div class='gsboxeffect'>People with this genoset may carry the haplotype described in gs103 for restless leg syndrome</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs221"  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs221'>gs221</a></div></div>
        <div class='boxmag'>Magnitude: 2</div>
        <div class='boxrep'>Repute: Bad</div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>Autoimmune disorder risk in Europeans</div>
        <div class='gsboxeffect'>This genoset tags the DQ2.2 haplotype in Europeans, and thus increased risk gluten intolerance and for autoimmune disorders such as celiac disease.</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs276"  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs276'>gs276</a></div></div>
        <div class='boxmag'>Magnitude: 2</div>
        <div class='boxrep'>Repute: Bad</div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>Slightly increased risk (17.2% of white women) of Atrial Fibrillation in the 2 SNPs mentioned by 23andMe.</div>
        <div class='gsboxeffect'>The less important of the two atrial fibrillation SNPs mentioned by 23andMe has one defective copy. This slightly affects the formation of the heart and slightly increases risk of Atrial Fibrillation (quivering of the top part of the heart) that 23andMe reports. It also seems to slightly increase the risk of Cardioembolic ischemic stroke (blocked blood flow to the brain) although 23andMe doesn't mention that. For European women, 23andMe reports this as a 17.2% risk, compared to the 15.9% average. The risk is higher for men. Based on many quality studies, and confirmed for Europeans. It's recommended to eat a heart-healthy diet, and only drink alcohol in moderation, to lower the risk.</div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs12567232'>rs12567232</a><a href='http://www.snpedia.com/index.php/rs12567232(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF0A0A;'>0.0%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk for Crohn's Disease</div>
        <div class='boxshow' id="show-sec-0.643569551858" style="display: inline;"><a href="javascript:toggle2('sec-0.643569551858')">...more...</a></div>
        <div class='boxrstext' id="sec-0.643569551858" style="display: none;"><a href="javascript:toggle2('sec-0.643569551858')">(hide)</a>
        <div class='rstext'>[PharmGKB:Non-Curated GWAS results: Genome-wide association study for Crohn's disease in the Quebec Founder Population identifies multiple validated disease loci (Initial Sample Size: 382 trios; Replication Sample Size: 521 trios, 750 cases, 828 controls). This variant is associated with Crohn's disease. This variant is associated with Crohn's disease.] [GWAS:Crohn's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs17115100'>rs17115100</a><a href='http://www.snpedia.com/index.php/rs17115100(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF6969;'>0.9%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk of developing Parkinson's Disease</div>
        <div class='boxshow' id="show-sec-0.556212888079" style="display: inline;"><a href="javascript:toggle2('sec-0.556212888079')">...more...</a></div>
        <div class='boxrstext' id="sec-0.556212888079" style="display: none;"><a href="javascript:toggle2('sec-0.556212888079')">(hide)</a>
        <div class='rstext'>[GWAS:Parkinson's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4538475'>rs4538475</a><a href='http://www.snpedia.com/index.php/rs4538475(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF6969;'>0.9%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk of developing Parkinson's Disease</div>
        <div class='boxshow' id="show-sec-0.952651878268" style="display: inline;"><a href="javascript:toggle2('sec-0.952651878268')">...more...</a></div>
        <div class='boxrstext' id="sec-0.952651878268" style="display: none;"><a href="javascript:toggle2('sec-0.952651878268')">(hide)</a>
        <div class='rstext'>[OMIM:?] [GWAS:Parkinson's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6445975'>rs6445975</a><a href='http://www.snpedia.com/index.php/rs6445975(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF9292;'>5.3%</span><br/>Repute:Bad<br/>References:18<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>disease risk associated</div>
        <div class='boxshow' id="show-sec-0.0149702255622" style="display: inline;"><a href="javascript:toggle2('sec-0.0149702255622')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0149702255622" style="display: none;"><a href="javascript:toggle2('sec-0.0149702255622')">(hide)</a>
        <div class='rstext'>rs6445975 is located on chromosome 3p14.3 and corresponds to intron 4 in the PXK gene (Phox homology domain containing serine/threonine kinase) and is associated with systemic lupus erythematosus (SLE). PXK is expressed in a variety of tissues including brain, heart, skeletal muscle, and peripheral blood lymphocytes. This SNP has also been implicated as being an important modulator of cancer cell growth . For the rs6445975 SNP, the ancestral allele is a T and the disease associated risk allele is a G. rs6445975 was first found to be associated with SLE in 2008 in women of European ancestry. The study included 720 women with SLE and 2,337 controls and showed strong evidence for association (P=7.1x10&#8211;9; OR=1.25) . Additionally, a study in 2010 found that many SLE SNPs including rs64459...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs642961'>rs642961</a><a href='http://www.snpedia.com/index.php/rs642961(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF9292;'>5.4%</span><br/>Repute:Bad<br/>References:12<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2.40x increased risk of cleft lip</div>
        <div class='boxshow' id="show-sec-0.662740249829" style="display: inline;"><a href="javascript:toggle2('sec-0.662740249829')">...more...</a></div>
        <div class='boxrstext' id="sec-0.662740249829" style="display: none;"><a href="javascript:toggle2('sec-0.662740249829')">(hide)</a>
        <div class='rstext'>risk of cleft lip 23andMe considers rs861020 a proxy Associated with European Facial Appearance *rs7590268 *rs16903544 *rs987525 *rs9574565 *rs227731 *rs642961 *rs1258763 Genome scan, fine-mapping, and candidate gene analysis of non-syndromic cleft lip with or without cleft palate reveals phenotype-specific differences in linkage and association results. Lack of association between IRF6 polymorphisms (rs2235371 and rs642961) and non-syndromic cleft lip and/or palate in a Brazilian population. Current concepts in genetics of nonsyndromic clefts. Studies of palatine rugae and interferon regulatory factor 6 variations in a group of families with sporadic hypodontia. IRF6 gene variants in Central European patients with non-syndromic cleft lip with or without cleft palate. Family-based study sh...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1169300'>rs1169300</a><a href='http://www.snpedia.com/index.php/rs1169300(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF9A9A;'>7.1%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~2x increased lung cancer risk</div>
        <div class='boxshow' id="show-sec-0.655395011404" style="display: inline;"><a href="javascript:toggle2('sec-0.655395011404')">...more...</a></div>
        <div class='boxrstext' id="sec-0.655395011404" style="display: none;"><a href="javascript:toggle2('sec-0.655395011404')">(hide)</a>
        <div class='rstext'>rs1169300 is a SNP that was originally associated with C-reactive protein levels, and given other findings that suggest C-reactive protein levels may be predictive of cancer risk, was thought to be linked to cancer risk. A large study pooling data from 3 Finnish studies totaling over 18,000 individuals concluded that while this SNP is not likely to be causative (relative to cancer), it and one other CRP SNP (rs2464196) are associated with increased risk for lung cancer. The odds ratio for minor alleles of either SNP was about 1.5x and 2x for heterozygotes and homozygotes, respectively. One other CRP SNP (rs1892534) was associated with increased overall cancer risk. CRP SNPs were not associated with colorectal, prostate or breast cancer risk.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2464196'>rs2464196</a><a href='http://www.snpedia.com/index.php/rs2464196(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF9A9A;'>7.1%</span><br/>Repute:Bad<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~2x increased lung cancer risk</div>
        <div class='boxshow' id="show-sec-0.973191867039" style="display: inline;"><a href="javascript:toggle2('sec-0.973191867039')">...more...</a></div>
        <div class='boxrstext' id="sec-0.973191867039" style="display: none;"><a href="javascript:toggle2('sec-0.973191867039')">(hide)</a>
        <div class='rstext'>rs2464196 is a SNP that was originally associated with C-reactive protein levels, and given other findings that suggest C-reactive protein levels may be predictive of cancer risk, was thought to be linked to cancer risk. A large study pooling data from 3 Finnish studies totaling over 18,000 individuals concluded that while this SNP is not likely to be causative (relative to cancer), it and one other CRP SNP (rs1169300) are associated with increased risk for lung cancer. The odds ratio for minor alleles of either SNP was about 1.5x and 2x for heterozygotes and homozygotes, respectively. One other CRP SNP (rs1892534) was associated with increased overall cancer risk. CRP SNPs were not associated with colorectal, prostate or breast cancer risk.  [PharmGKB:Curated In a candidate gene-based ass...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10096097'>rs10096097</a><a href='http://www.snpedia.com/index.php/rs10096097(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF9D9D;'>8.0%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased Anorexia Nervosa risk</div>
        <div class='boxshow' id="show-sec-0.65282768279" style="display: inline;"><a href="javascript:toggle2('sec-0.65282768279')">...more...</a></div>
        <div class='boxrstext' id="sec-0.65282768279" style="display: none;"><a href="javascript:toggle2('sec-0.65282768279')">(hide)</a>
        <div class='rstext'>rs10096097 is a SNP near the membrane bound O-acyltransferase domain containing 4 MBOAT4 gene. A study of 543 German anorexia nervosa patients observed some association between the disorder and rs10096097(G;G) (nominal two-sided p = 0.031).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1532268'>rs1532268</a><a href='http://www.snpedia.com/index.php/rs1532268(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF9D9D;'>8.0%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk  of ventricular septal defect</div>
        <div class='boxshow' id="show-sec-0.678345776171" style="display: inline;"><a href="javascript:toggle2('sec-0.678345776171')">...more...</a></div>
        <div class='boxrstext' id="sec-0.678345776171" style="display: none;"><a href="javascript:toggle2('sec-0.678345776171')">(hide)</a>
        <div class='rstext'></div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1799966'>rs1799966</a><a href='http://www.snpedia.com/index.php/rs1799966(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF9E9E;'>8.2%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk for breast cancer</div>
        <div class='boxshow' id="show-sec-0.721212069214" style="display: inline;"><a href="javascript:toggle2('sec-0.721212069214')">...more...</a></div>
        <div class='boxrstext' id="sec-0.721212069214" style="display: none;"><a href="javascript:toggle2('sec-0.721212069214')">(hide)</a>
        <div class='rstext'>This SNP, a variant in the BRCA1 gene, is 1 of 25 SNPs reported to represent independently minor, but cumulatively significant, increased risk for breast cancer. For details of all 25 SNPs in this group, along with the two methods used to calculate overall risk estimates for breast cancer, refer to the SNPedia breast cancer entry. For this particular SNP, the risk (minor) allele is (G).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1375144'>rs1375144</a><a href='http://www.snpedia.com/index.php/rs1375144(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFA0A0;'>8.8%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.59x increased risk of developing bipolar disorder</div>
        <div class='boxshow' id="show-sec-0.169256391279" style="display: inline;"><a href="javascript:toggle2('sec-0.169256391279')">...more...</a></div>
        <div class='boxrstext' id="sec-0.169256391279" style="display: none;"><a href="javascript:toggle2('sec-0.169256391279')">(hide)</a>
        <div class='rstext'>rs1375144, a SNP in the DPP10 gene, has been reported in a large study to be associated with bipolar disorder. The risk allele (oriented to the dbSNP entry) is (T); the odds ratio associated with heterozygotes is 1.32 (CI 1.07-1.63), and for homozygotes, 1.59 (CI 1.29-1.96). Findings from bipolar disorder genome-wide association studies replicate in a Finnish bipolar family-cohort.  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10937823'>rs10937823</a><a href='http://www.snpedia.com/index.php/rs10937823(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFA3A3;'>9.7%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>some association with bipolar disorder</div>
        <div class='boxshow' id="show-sec-0.642917417839" style="display: inline;"><a href="javascript:toggle2('sec-0.642917417839')">...more...</a></div>
        <div class='boxrstext' id="sec-0.642917417839" style="display: none;"><a href="javascript:toggle2('sec-0.642917417839')">(hide)</a>
        <div class='rstext'>rs10937823 is a SNP associated with the gene SORCS2, variants of which may play a role in bipolar disorder. The allele most frequently occurring is C, and the variant is T. In the HapMap-CEU population, CC, CT, and TT genotype frequencies were 0.894, 0.097, and 0.009, respectively. However, in the HapMap-CHB population, CT was the most frequently expressed genotype; CC, CT, and TT genotype frequencies were 0.366, 0.439, and 0.195, respectively. In some sample populations, the TT genotype did not exist (TSI, MKK, LWK, ASW). [dbSNP] The SORCS2 (sortillin-related VPS10 domain containing receptor 2) gene on chromosome 4p16.1 is found to be highly expressed in the mature human brain and kidney. High levels of expression are also seen in the brain during mouse development. [GeneCards] This gene ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs16942'>rs16942</a><a href='http://www.snpedia.com/index.php/rs16942(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFA6A6;'>10.6%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk for breast cancer</div>
        <div class='boxshow' id="show-sec-0.80696247616" style="display: inline;"><a href="javascript:toggle2('sec-0.80696247616')">...more...</a></div>
        <div class='boxrstext' id="sec-0.80696247616" style="display: none;"><a href="javascript:toggle2('sec-0.80696247616')">(hide)</a>
        <div class='rstext'>This SNP, a variant in the BRCA1 gene, is 1 of 25 SNPs reported to represent independently minor, but cumulatively significant, increased risk for breast cancer. For this particular SNP, the risk (minor) allele is (G). For details of all 25 SNPs in this group, along with the two methods used to calculate overall risk estimates for breast cancer, refer to the SNPedia breast cancer entry or BRCA1 and BRCA2.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs17576'>rs17576</a><a href='http://www.snpedia.com/index.php/rs17576(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFA6A6;'>10.6%</span><br/>Repute:Bad<br/>References:19<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>higher risk for lung cancer, and COPD in smokers; normal risk for MI</div>
        <div class='boxshow' id="show-sec-0.320804444834" style="display: inline;"><a href="javascript:toggle2('sec-0.320804444834')">...more...</a></div>
        <div class='boxrstext' id="sec-0.320804444834" style="display: none;"><a href="javascript:toggle2('sec-0.320804444834')">(hide)</a>
        <div class='rstext'>rs17576, also known as Gln279Arg or Q279R, is a SNP in exon 6 of the matrix metalloproteinase-9 MMP9 gene. The rs17576(G) allele encodes the Arg (R). In a study of 385 male veterans with greater than 20 pack-years of cigarette smoking, rs17576(G) allele carriers were at higher risk for chronic obstructive pulmonary disease (COPD). The rs17576(G;G) homozygous genotype was at 3-fold increased risk for COPD. A study of 744 Chinese patients with lung cancer found that the rs17576(G;G) genotype was associated with higher risk of lung cancer with metastasis (adjusted OR, 1.79, CI: 1.03-3.08) compared to the (A;A) genotype. rs17576 is also one of two SNPs in the MMP9 gene associated with increased risk for myocardial infarction, but not coronary artery disease. The relatively weak odds ratios for...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1061147'>rs1061147</a><a href='http://www.snpedia.com/index.php/rs1061147(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFABAB;'>12.4%</span><br/>Repute:Bad<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2.76x increased risk for Age Related Macular Degeneration.  When the retina cells in the eye get damaged with age, it is known as Age Related Macular Degeneration. This SNP has about three times the normal risk. This SNP wrecks the brakes on the complementary immune system, so it can become out of control, resulting in inflamation that damages the cells in the eye. The SENS Research Foundation is working on a promising cure for Age Related Macular Degeneration: http://www.sens.org/outreach/conferences/lysosens-synthetic-biology-approach-treat-age-related-macular-degeneration</div>
        <div class='boxshow' id="show-sec-0.137838167232" style="display: inline;"><a href="javascript:toggle2('sec-0.137838167232')">...more...</a></div>
        <div class='boxrstext' id="sec-0.137838167232" style="display: none;"><a href="javascript:toggle2('sec-0.137838167232')">(hide)</a>
        <div class='rstext'>may influence age related macular degeneration (damage to the eye with ageing) when part of the haplotype of rs1061170 rs3753394 rs800292 rs1329428 (TGTC), but does not appear to do so alone. age related macular degeneration  [GWAS:Age-related macular degeneration]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10993994'>rs10993994</a><a href='http://www.snpedia.com/index.php/rs10993994(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFABAB;'>12.4%</span><br/>Repute:Bad<br/>References:56<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased prostate cancer risk (odds ratio 1.6)  Compared to the (C;C) genotype for this SNP, (T;T) genotypes are estimated to be at 1.6x increased risk for prostate cancer.</div>
        <div class='boxshow' id="show-sec-0.535860537383" style="display: inline;"><a href="javascript:toggle2('sec-0.535860537383')">...more...</a></div>
        <div class='boxrstext' id="sec-0.535860537383" style="display: none;"><a href="javascript:toggle2('sec-0.535860537383')">(hide)</a>
        <div class='rstext'>rs10993994 linked to prostate cancer cancer-genetics these snps influence genetic risk for prostate cancer *the haplotype rs6983267 rs1016343 rs4242384 *rs7501939 *rs1859962 *rs2660753 *rs9364554 *rs6465657 *rs10993994 *rs7931342 *rs2735839 *rs5945619  [PharmGKB:Non-Curated GWAS Results: Multiple loci identified in a genome-wide association study of prostate cancer (Initial Sample Size: 1,172 cases, 1,157 controls; Replication Sample Size: 3,941 cases, 3,964 controls; Risk Allele: rs10993994-T).] [OMIM:MICROSEMINOPROTEIN, BETA; MSMB] [GWAS:Prostate cancer] [GWAS:Prostate cancer (aggressive)]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs16944'>rs16944</a><a href='http://www.snpedia.com/index.php/rs16944(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFADAD;'>13.3%</span><br/>Repute:Bad<br/>References:102<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk for osteoarthritis</div>
        <div class='boxshow' id="show-sec-0.638647259456" style="display: inline;"><a href="javascript:toggle2('sec-0.638647259456')">...more...</a></div>
        <div class='boxrstext' id="sec-0.638647259456" style="display: none;"><a href="javascript:toggle2('sec-0.638647259456')">(hide)</a>
        <div class='rstext'>rs16944 is a SNP in the interleukin 1, beta gene, a member of the cytokine family involved in the inflammatory response. The rs16944 G;G genotype is associated with risk for multiple mental illnesses. An increased risk for schizophrenia was found in a genetic association study of 2111 Caucasian individuals with an odds ratio of 1.24 (95% CI 1.09, 1.41) . Studies examining the brains of schizophrenic and bipolar individuals found reduced gray matter and hypoactivity associated with the G;G genotype. Additionally, this genotype is associated with impaired ability to achieve remission in a study of 256 Caucasian individuals with depression (odds ratio = 1.74; 95% confidence interval 1.2-4.3). The rs16944 A allele increases susceptibility to Osteoarthritis, with 1.80 times for heterozygotes (A...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2303138'>rs2303138</a><a href='http://www.snpedia.com/index.php/rs2303138(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFB1B1;'>15.0%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.5x increased risk (0.15%) for Ankylosing Spondylitis.  Increased, but still very low, risk of the rare autoimmune disease that causes inflammation of the spine and joints. Based on preliminary research.</div>
        <div class='boxshow' id="show-sec-0.00543546270561" style="display: inline;"><a href="javascript:toggle2('sec-0.00543546270561')">...more...</a></div>
        <div class='boxrstext' id="sec-0.00543546270561" style="display: none;"><a href="javascript:toggle2('sec-0.00543546270561')">(hide)</a>
        <div class='rstext'>Each A allele at rs2303138 in the LNPEP gene is associated with 1.5x higher odds of ankylosing spondylitis, according to a 2007 report.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6700998'>rs6700998</a><a href='http://www.snpedia.com/index.php/rs6700998(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFB3B3;'>15.9%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.2x increased risk of Atopic Dermatitis.  This seems to be a proxy that 23andMe uses for the filaggrin gene variants that they can't measure directly.</div>
        <div class='boxshow' id="show-sec-0.557645213254" style="display: inline;"><a href="javascript:toggle2('sec-0.557645213254')">...more...</a></div>
        <div class='boxrstext' id="sec-0.557645213254" style="display: none;"><a href="javascript:toggle2('sec-0.557645213254')">(hide)</a>
        <div class='rstext'>Atopic Dermatitis</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7774434'>rs7774434</a><a href='http://www.snpedia.com/index.php/rs7774434(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFB7B7;'>17.6%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk of developing primary biliary cirrhosis</div>
        <div class='boxshow' id="show-sec-0.66153416825" style="display: inline;"><a href="javascript:toggle2('sec-0.66153416825')">...more...</a></div>
        <div class='boxrstext' id="sec-0.66153416825" style="display: none;"><a href="javascript:toggle2('sec-0.66153416825')">(hide)</a>
        <div class='rstext'>On the association between rheumatoid arthritis and classical HLA class I and class II alleles predicted from single-nucleotide polymorphism data. A two-stage search strategy for detecting multiple loci associated with rheumatoid arthritis.  [GWAS:Primary biliary cirrhosis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs602662'>rs602662</a><a href='http://www.snpedia.com/index.php/rs602662(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFB9B9;'>18.6%</span><br/>Repute:Bad<br/>References:11<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Lower vitamin B12 levels  We found that the SNP '''rs602662''' was significantly associated with the levels of vitamin B(12) (p value&lt;0.0001). We also found that individuals adhering to a vegetarian diet with GG (homozygous major genotype) have significantly '''lower''' levels of vitamin B(12) in these individuals.</div>
        <div class='boxshow' id="show-sec-0.779280069393" style="display: inline;"><a href="javascript:toggle2('sec-0.779280069393')">...more...</a></div>
        <div class='boxrstext' id="sec-0.779280069393" style="display: none;"><a href="javascript:toggle2('sec-0.779280069393')">(hide)</a>
        <div class='rstext'>[PharmGKB:Non-Curated GWAS results: Genome-wide Association Study of Vitamin B6, Vitamin B12, Folate, and Homocysteine Blood Concentrations. (Initial Sample Size: 2,934 individuals; Replication Sample Size: 686 individuals); (Region: 19q13.33; Reported Gene(s): FUT2; Risk Allele: rs602662-A); (p-value= 3E-20).This variant is associated with Folate pathway vitamins.] [OMIM:VITAMIN B12 PLASMA LEVEL QUANTITATIVE TRAIT LOCUS 1; B12QTL1] [GWAS:Folate pathway vitamins]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10757272'>rs10757272</a><a href='http://www.snpedia.com/index.php/rs10757272(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFBDBD;'>21.2%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.54x increased risk for Coronary artery disease</div>
        <div class='boxshow' id="show-sec-0.0250812907266" style="display: inline;"><a href="javascript:toggle2('sec-0.0250812907266')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0250812907266" style="display: none;"><a href="javascript:toggle2('sec-0.0250812907266')">(hide)</a>
        <div class='rstext'>rs10757272 increases susceptibility to Coronary artery disease 1.30 times for heterozygotes (CT) and 1.54 times for homozygotes (TT)</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2736100'>rs2736100</a><a href='http://www.snpedia.com/index.php/rs2736100(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFBDBD;'>21.2%</span><br/>Repute:Bad<br/>References:64<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>higher risk of Interstitial lung disease, and testicular cancer</div>
        <div class='boxshow' id="show-sec-0.370146774593" style="display: inline;"><a href="javascript:toggle2('sec-0.370146774593')">...more...</a></div>
        <div class='boxrstext' id="sec-0.370146774593" style="display: none;"><a href="javascript:toggle2('sec-0.370146774593')">(hide)</a>
        <div class='rstext'>23andMe blog rs2736100 C 1.27 Glioma Lung cancer susceptibility locus at 5p15.33. Common genetic variants on 5p15.33 contribute to risk of lung adenocarcinoma in a Chinese population. Variants in the CDKN2B and RTEL1 regions are associated with high-grade glioma susceptibility. Deciphering the impact of common genetic variation on lung cancer risk: a genome-wide association study. Role of 5p15.33 (TERT-CLPTM1L), 6p21.33 and 15q25.1 (CHRNA5-CHRNA3) variation and lung cancer risk in never-smokers. Individuals susceptible to lung adenocarcinoma defined by combined HLA-DQA1 and TERT genotypes. New insights into susceptibility to glioma. Replication of lung cancer susceptibility loci at chromosomes 15q25, 5p15, and 6p21: a pooled analysis from the International Lung Cancer Consortium. Evaluatio...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1544410'>rs1544410</a><a href='http://www.snpedia.com/index.php/rs1544410(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFBFBF;'>22.1%</span><br/>Repute:Bad<br/>References:132<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk of low bone mineral density disorders</div>
        <div class='boxshow' id="show-sec-0.913850321984" style="display: inline;"><a href="javascript:toggle2('sec-0.913850321984')">...more...</a></div>
        <div class='boxrstext' id="sec-0.913850321984" style="display: none;"><a href="javascript:toggle2('sec-0.913850321984')">(hide)</a>
        <div class='rstext'>rs1544410, also known as the BsmI polymorphism, is a SNP in the Vitamin D receptor (VDR). Multiple studies examining bone mineral density (BMD) in women have associated the A allele with increased risk of low BMD , , & . Additionally, A meta-analysis of 26 studies estimated a decreased risk of osteoporosis associated with the G;G genotype with an odds ratio of 0.61, 95% CI, 0.40-0.92 . rs1544410 increases susceptibility to Shorter stature for carriers of the A allele  [PharmGKB:Curated This variant is associated with etidronate response in Caucasian (Slovenian) postmenopausal women (24 patients), with bb genotype associated with lower increase in bone mineral density (BMD) after bisphosphonate treatments.]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs909525'>rs909525</a><a href='http://www.snpedia.com/index.php/rs909525(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFBFBF;'>22.1%</span><br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Perhaps MAOA 3 repeats: Warrior Gene?  People with this SNP may also have the 3 repeat version of MAOA, known as the Warrior Gene. It makes people more aggressive and antisocial. If you are male, you inherited this from your mother and your sons won't get it from you. Do not confuse this with the completely different warrior/worrier COMT gene.</div>
        <div class='boxshow' id="show-sec-0.445633259973" style="display: inline;"><a href="javascript:toggle2('sec-0.445633259973')">...more...</a></div>
        <div class='boxrstext' id="sec-0.445633259973" style="display: none;"><a href="javascript:toggle2('sec-0.445633259973')">(hide)</a>
        <div class='rstext'>This is the best proxy for the number of repeats of the MAOA warrior gene. In 105 samples (69 males, 36 females) from the Stanley foundation brain collection, it was always A in people with the 4 or 5 repeat non-Warrior version and always G in people with the 3 repeat Warrior version. The combination rs909525(A) AND rs6323(G) AND rs3027399(G) indicate specifically that it is the 5 repeat version. The 2 repeat version and the 3.5 repeat version weren't mentioned. mentioned as potentially affecting white matter volume, sample size tiny</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2056116'>rs2056116</a><a href='http://www.snpedia.com/index.php/rs2056116(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC2C2;'>23.9%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.41x risk for breast cancer</div>
        <div class='boxshow' id="show-sec-0.976819315183" style="display: inline;"><a href="javascript:toggle2('sec-0.976819315183')">...more...</a></div>
        <div class='boxrstext' id="sec-0.976819315183" style="display: none;"><a href="javascript:toggle2('sec-0.976819315183')">(hide)</a>
        <div class='rstext'>rs2056116 is a somewhat unusual SNP, in that it is from an 'ultraconserved element' of the genome, meaning it is from a stretch of over 200 base pairs that is absolutely identical between equivalent regions of human, mouse, and rat genomes. The functions of these elements is unknown, but they are thought to have regulatory roles of various sorts. They usually have less SNPs on average than other regions of the genome. The (G) allele of rs2056116 was found to be associated with higher risk for breast cancer, with an odds ratio of 1.18 (CI: 1.06-1.30, p=0.002). The odds ratio for the (G;G) genotype compared to the (A;A) genotype was 1.41 (CI: 1.15-1.74, p=0.0011). When patients were divided into younger vs older women, the odds ratios were increased for women under 50 years of age; for the (...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs828907'>rs828907</a><a href='http://www.snpedia.com/index.php/rs828907(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC2C2;'>23.9%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk of bladder cancer and 2x risk of developing bladder cancer in smokers</div>
        <div class='boxshow' id="show-sec-0.468571444815" style="display: inline;"><a href="javascript:toggle2('sec-0.468571444815')">...more...</a></div>
        <div class='boxrstext' id="sec-0.468571444815" style="display: none;"><a href="javascript:toggle2('sec-0.468571444815')">(hide)</a>
        <div class='rstext'>rs828907, also known as G-1401T, is a SNP in the Ku80 gene. A study of 288 Chinese patients with bladder cancer and 288 age- and gender-matched healthy controls found that the rs828907(T) allele conferred a significant (p=0.0055) increased risk of bladder cancer. In association with smoking, the (G;T) and (T;T) genotypes conferred a 2x risk (CI:1.232-3.419) for bladder cancer. correlated with oral cancer susceptibility</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1434536'>rs1434536</a><a href='http://www.snpedia.com/index.php/rs1434536(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC3C3;'>24.8%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.94x increased breast cancer risk</div>
        <div class='boxshow' id="show-sec-0.301563717422" style="display: inline;"><a href="javascript:toggle2('sec-0.301563717422')">...more...</a></div>
        <div class='boxrstext' id="sec-0.301563717422" style="display: none;"><a href="javascript:toggle2('sec-0.301563717422')">(hide)</a>
        <div class='rstext'>rs1434536 is a SNP reported to potentially represent a miR-125b binding site. A study of 459 probands from a breast cancer affected sibling pair cohort were recruited from the ECOG study, and in conjunction with 1,145 patient samples from the CGEMS study, led to the conclusion that rs1434536 was associated with increased risk for breast cancer. The odds ratios for rs1434536(A;G) and (A;A) individuals were 1.29 (CI: 0.95-1.74) and 1.94 (CI: 1.40-2.71), respectively (p=1.6x10e-4). A miR-125b binding site polymorphism in bone morphogenetic protein membrane receptor type IB gene and prostate cancer risk in China.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs8099917'>rs8099917</a><a href='http://www.snpedia.com/index.php/rs8099917(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC3C3;'>24.8%</span><br/>Repute:Bad<br/>References:201<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Moderately lower odds of responding to PEG-IFNalpha/RBV treatment  Moderately lower odds of responding to PEG-IFNalpha/RBV treatment for hepatitis C.</div>
        <div class='boxshow' id="show-sec-0.86213683657" style="display: inline;"><a href="javascript:toggle2('sec-0.86213683657')">...more...</a></div>
        <div class='boxrstext' id="sec-0.86213683657" style="display: none;"><a href="javascript:toggle2('sec-0.86213683657')">(hide)</a>
        <div class='rstext'>23andMe blog rs8099917(C) less likely to respond to Hepatitis C treatment. rs12979860(C;C) better able to become naturally Hepatitis C virus-free. rs12980275(A) usually corresponds to the rs12979860(C)  [PharmGKB:Curated Risk or phenotype-associated allele: G. Phenotype: This variant is associated with null virological response (NVR) and sustained virologic response (SVR) in Janapese patients with hepatitis C virus infection treated with PEG-INF-alpha plus ribavirin. Study size: Initial sample:154; Replication Sample: 172. Study population/ethnicity: Japanese. Significance metric(s): P=3.11 x 10(-15) (NVR); P = 1.11 x 10(-27)(SVR). Type of association: GN; CO] [OMIM:INTERLEUKIN 28B; IL28B] [GWAS:Response to Hepatitis C treatment]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11045585'>rs11045585</a><a href='http://www.snpedia.com/index.php/rs11045585(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC4C4;'>25.9%</span><br/>Repute:Bad<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>63% chance (higher than average) of docetaxel-induced leukopenia/neutropenia</div>
        <div class='boxshow' id="show-sec-0.828922875255" style="display: inline;"><a href="javascript:toggle2('sec-0.828922875255')">...more...</a></div>
        <div class='boxrstext' id="sec-0.828922875255" style="display: none;"><a href="javascript:toggle2('sec-0.828922875255')">(hide)</a>
        <div class='rstext'>rs12762549 and rs11045585 can be used to predict whether docetaxel will induce leukopenia/neutropenia, according to a study of ~100 Japanese patients. When patients were classified into three groups by the scoring system based on the genotypes of these two SNPs, patients with a score of 1 or 2 were shown to have a significantly higher risk of docetaxel-induced leukopenia/neutropenia as compared to those with a score of 0 (P = 0.0000057; odds ratio [OR], 7.00; 95% CI [confidence interval], 2.95-16.59). This prediction system correctly classified 69.2% of severe leukopenia/neutropenia and 75.7% of non-leukopenia/neutropenia into the respective categories. [Effect percentages provided in table are raw population fractions taken from the referenced study based on rs11045585 only. Total populat...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4968451'>rs4968451</a><a href='http://www.snpedia.com/index.php/rs4968451(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC5C5;'>26.5%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.61x increased risk for meningioma</div>
        <div class='boxshow' id="show-sec-0.121758767545" style="display: inline;"><a href="javascript:toggle2('sec-0.121758767545')">...more...</a></div>
        <div class='boxrstext' id="sec-0.121758767545" style="display: none;"><a href="javascript:toggle2('sec-0.121758767545')">(hide)</a>
        <div class='rstext'>rs4968451, which is in a breast cancer susceptibility gene, was associated with increased risk of meningioma, a common form of brain tumor, in a combination of five studies totaling 631 European patients. 'Meningioma risks associated with hetero- and homozygosity for the minor allele of were 1.61 (95% CI = 1.26 to 2.06) and 2.33 (95% CI = 1.25 to 4.34), respectively, and were thus compatible with a multiplicative model of action.' The risk allele and minor allele is rs4968451(C). Around 30% of Europeans carry one of risk genotypes, i.e. rs4968451(A;C) or (C;C); and perhaps 16% of meningiomas might be associated with this SNP.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2143340'>rs2143340</a><a href='http://www.snpedia.com/index.php/rs2143340(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC7C7;'>27.7%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk of dyslexia and poor reading performance</div>
        <div class='boxshow' id="show-sec-0.325102906554" style="display: inline;"><a href="javascript:toggle2('sec-0.325102906554')">...more...</a></div>
        <div class='boxrstext' id="sec-0.325102906554" style="display: none;"><a href="javascript:toggle2('sec-0.325102906554')">(hide)</a>
        <div class='rstext'>genetic risk for poor reading performance. Rs2143340, a SNP in the TTRAP gene, is in a region that crops up in several independent studies as likely to associated with dyslexia. The risk allele in the US/UK Caucasian populations studied is (C), and it is indicative of a risk haplotype found in ~18% of the general population but up to ~28% of severely dyslexic individuals. It is not thought that rs2143340 is a functional SNP; instead, it marks the haplotype known as rs4504469-rs2038137-rs2143340 '1-1-2' since the other two SNPs are the common forms whereas the significant form for rs2143340 is the rare form. The functional effect of this haplotype appears to be on the KIAA0319 gene, in that KIAA0319 gene activity associated with the 1-1-2 risk haplotype is 40% lower than for other haplotype...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7639618'>rs7639618</a><a href='http://www.snpedia.com/index.php/rs7639618(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC8C8;'>28.3%</span><br/>Repute:Bad<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.45x increased osteoarthritis risk</div>
        <div class='boxshow' id="show-sec-0.647844309237" style="display: inline;"><a href="javascript:toggle2('sec-0.647844309237')">...more...</a></div>
        <div class='boxrstext' id="sec-0.647844309237" style="display: none;"><a href="javascript:toggle2('sec-0.647844309237')">(hide)</a>
        <div class='rstext'>rs7639618 is one of several SNPs in the DVWA gene found to be associated with osteoarthritis based on a study of several Japanese patient populations. The allelic odds ratio is 1.45 (CI: 1.27-1.65, p = 2.2 x 10e-8) for the minor allele, which is the (T) allele (in dbSNP orientation). Our results suggest that the effect that DVWA amino acid changes have on tubulin binding is unlikely to influence risk of OA in Caucasians. rs7639618 with knee osteoarthritis with an odds ratio (OR) of 1.29, 95% confidence interval (CI) of 1.15 -1.45 and P of 2.70x10(-5), but not for Europeans  [OMIM:OSTEOARTHRITIS SUSCEPTIBILITY 6; OS6]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7743761'>rs7743761</a><a href='http://www.snpedia.com/index.php/rs7743761(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC8C8;'>28.3%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>4.6x increased risk of Ankylosing Spondylitis.  Substantially increased (but still only 0.5%) risk of this rare autoimmune disease that causes inflammation of the spine and joints. Based on preliminary research. Age of onset is usually between 15 and 25, so if you are older than that, there's not much need for worry.</div>
        <div class='boxshow' id="show-sec-0.938630439078" style="display: inline;"><a href="javascript:toggle2('sec-0.938630439078')">...more...</a></div>
        <div class='boxrstext' id="sec-0.938630439078" style="display: none;"><a href="javascript:toggle2('sec-0.938630439078')">(hide)</a>
        <div class='rstext'>Each A allele at rs7743761 is associated with 4.6x higher risk of ankylosing spondylitis, according to a 2007 publication.  [GWAS:Ankylosing spondylitis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1801394'>rs1801394</a><a href='http://www.snpedia.com/index.php/rs1801394(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC8C8;'>28.6%</span><br/>Repute:Bad<br/>References:16<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x higher risk for meningiomas  This common variant (HapMap allele frequency of 31.3%) in a protein involved in folate (B9) and cobalamin (B12) metabolism and is often reported as 'MTRR I22M' (for the Ile to Met substitution at amino acid 22). Mothers homozygous for this variant were found in a few studies to have an increased risk for Down syndrome babies (risk of 0.4%, average risk in population is 0.25%), but other studies (including in Italian, Irish, French, and Indian-Gujarati women) found no increased risk. Notably, age plays a far larger role in the rate of Down syndrome (risk is 4.5% for a mother 45-years-of-age), and, it is unknown how this variant may combine with the effect of age. There are conflicting reports associating this variant with incidence of neural tube defects, p...</div>
        <div class='boxshow' id="show-sec-0.314167334491" style="display: inline;"><a href="javascript:toggle2('sec-0.314167334491')">...more...</a></div>
        <div class='boxrstext' id="sec-0.314167334491" style="display: none;"><a href="javascript:toggle2('sec-0.314167334491')">(hide)</a>
        <div class='rstext'>rs1801394, also known as A66G or Ile22Met, is a SNP in the methionine synthase MTRR gene. This gene encodes one of the two enzymes involved in the production of methionine (the other is MTR). The protein encoded by an rs1801394 allele has a lower affinity for MTR () and is inconsistently associated with homocysteine level, although it is a risk factor for neural tube defects () and Down syndrome () in conditions of higher homocysteine. Based on a study of British patients with primary brain tumors, (1,005 glioma cases and 631 meningioma cases), rs1801394(G;G) individuals were at higher risk for meningioma (odds ratio 1.41, CI: 1.02-1.94). In general, genotypes associated with increased 5,10-methylenetetrahydrofolate levels are associated with elevated risk for these types of brain cancer. ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3129934'>rs3129934</a><a href='http://www.snpedia.com/index.php/rs3129934(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC8C8;'>28.6%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk of Multiple Sclerosis.</div>
        <div class='boxshow' id="show-sec-0.827814203584" style="display: inline;"><a href="javascript:toggle2('sec-0.827814203584')">...more...</a></div>
        <div class='boxrstext' id="sec-0.827814203584" style="display: none;"><a href="javascript:toggle2('sec-0.827814203584')">(hide)</a>
        <div class='rstext'>rs3129934 is a SNP near the HLA-class II region that may be associated with several autoimmune diseases. In a study of two cohorts (Spanish and American multiple sclerosis patients, each numbering several hundred), rs3129934 was the SNP most associated with increased risk for multiple sclerosis. The odds ratio was 3.3 (CI: 2.3 - 4.9, p = 9 x 10e-11). Seven other SNPs were also identified in this study as associated with increased risk for MS. associated with type-1 diabetes. 1.6x risk of type-1 diabetes? 23andMe uses rs9273363 instead of rs3129934 in its reports on type 1 diabetes risk. This SNP was formerly used as a proxy for rs9272346, with this explanation: 'Note: the 2007 Wellcome Trust paper reported a strong association between type 1 diabetes and the SNP rs9272346. Our quality cont...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3819331'>rs3819331</a><a href='http://www.snpedia.com/index.php/rs3819331(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC9C9;'>29.2%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk of autism  Increased risk of autism.[http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2784255/]</div>
        <div class='boxshow' id="show-sec-0.394153504897" style="display: inline;"><a href="javascript:toggle2('sec-0.394153504897')">...more...</a></div>
        <div class='boxrstext' id="sec-0.394153504897" style="display: none;"><a href="javascript:toggle2('sec-0.394153504897')">(hide)</a>
        <div class='rstext'>Examination of tetrahydrobiopterin pathway genes in autism. Examination of association to autism of common genetic variation in genes related to dopamine.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2230199'>rs2230199</a><a href='http://www.snpedia.com/index.php/rs2230199(C;G)'>(C;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC9C9;'>29.2%</span><br/>Repute:Bad<br/>References:48<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.6x+ risk of ARMD</div>
        <div class='boxshow' id="show-sec-0.471292333321" style="display: inline;"><a href="javascript:toggle2('sec-0.471292333321')">...more...</a></div>
        <div class='boxrstext' id="sec-0.471292333321" style="display: none;"><a href="javascript:toggle2('sec-0.471292333321')">(hide)</a>
        <div class='rstext'>rs2230199, a SNP in the complement component C3 gene, has been reported by several investigators to be associated with ARMD. The common allele at this SNP is known as Arg102; the variant and risk allele is known as Gly102. The risk allele, in orientation to the dbSNP entry for this rs#, is (G). In one of the largest case-control studies, the odds ratio associated with heterozygotes is 1.61, and for homozygotes, 3.26 (p = 4.5 x 10e-12). NEJM reports significant associate with ARMD. blog post  [PharmGKB:Curated rs2230199 was found to be associated with risk of age-related macular degeneration in English and Scottish case-control studies.] [OMIM:MACULAR DEGENERATION, AGE-RELATED, 9; ARMD9] [GWAS:Age-related macular degeneration]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs12255372'>rs12255372</a><a href='http://www.snpedia.com/index.php/rs12255372(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFCACA;'>30.1%</span><br/>Repute:Bad<br/>References:106<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased type-2 diabetes risk  see discussion at rs12255372</div>
        <div class='boxshow' id="show-sec-0.596578368735" style="display: inline;"><a href="javascript:toggle2('sec-0.596578368735')">...more...</a></div>
        <div class='boxrstext' id="sec-0.596578368735" style="display: none;"><a href="javascript:toggle2('sec-0.596578368735')">(hide)</a>
        <div class='rstext'>rs12255372, also known as IVS4G&gt;T and c.483+9017G&gt;T, is a well-studied SNP in the TCF7L2 gene on chromosome 10. In some studies, it has been linked to slight increases in risk for type-2 diabetes, breast cancer and aggressive prostate cancer. '''* Type-2 Diabetes Risk Overview:''' numerous papers report an increase in risk, on the order of 1.5 - 2x for the risk genotypes compared to the non-risk genotype. Key papers include: ** reports the association of rs12255372 and rs7903146 with Type-2 diabetes in a Finnish sample. **rs12255372 Common variants (rs12255372 and rs7903146) in TCF7L2 seem to be associated with an increased risk of diabetes among persons with impaired glucose tolerance ** Considered for type-2 diabetes with rs7903146 rs12255372 rs10885406. ** Associated in a study of...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9525638'>rs9525638</a><a href='http://www.snpedia.com/index.php/rs9525638(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFCBCB;'>30.4%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>weaker bones</div>
        <div class='boxshow' id="show-sec-0.197391935823" style="display: inline;"><a href="javascript:toggle2('sec-0.197391935823')">...more...</a></div>
        <div class='boxrstext' id="sec-0.197391935823" style="display: none;"><a href="javascript:toggle2('sec-0.197391935823')">(hide)</a>
        <div class='rstext'>Osteoporosis  [GWAS:Bone mineral density (paediatric, upper limb)]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs629242'>rs629242</a><a href='http://www.snpedia.com/index.php/rs629242(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFCBCB;'>31.0%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>somewhat higher risk for prostate cancer</div>
        <div class='boxshow' id="show-sec-0.160250384112" style="display: inline;"><a href="javascript:toggle2('sec-0.160250384112')">...more...</a></div>
        <div class='boxrstext' id="sec-0.160250384112" style="display: none;"><a href="javascript:toggle2('sec-0.160250384112')">(hide)</a>
        <div class='rstext'>rs629242 is one of seven SNPs found in a combined study of over 1,000 patients to be associated with increased risk for prostate cancer. The risk allele for this SNP is (T); and while the odds ratio was not specifically reported, the probability of false significance (not permuted though) was given as p=7.2 x 10e-7, using an additive model of risk.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1800566'>rs1800566</a><a href='http://www.snpedia.com/index.php/rs1800566(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFCCCC;'>31.9%</span><br/>Repute:Bad<br/>References:59<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Higher cancer risk  rs1800566 increases susceptibility to Lung cancer 0.48 times for heterozygotes (CT)</div>
        <div class='boxshow' id="show-sec-0.815396381524" style="display: inline;"><a href="javascript:toggle2('sec-0.815396381524')">...more...</a></div>
        <div class='boxrstext' id="sec-0.815396381524" style="display: none;"><a href="javascript:toggle2('sec-0.815396381524')">(hide)</a>
        <div class='rstext'>rs1800566 (C609T, Pro187Ser) is a snp within NQO1 (NAD(P)H dehydrogenase (quinone 1)). A Ser (T) at this location denotes the NQO1*2 allele. influences the risk of breast cancer according to the 23andMe blog rs1800566 increases susceptibility to Lung cancer 0.48 times for heterozygotes (CT)  [PharmGKB:Curated Risk or phenotype-associated allele: C/T. Phenotype: There was no association between the NQO1*2 polymorphism and the risk of anthracycline-related CHF. Study size: 145. Study population/ethnicity: Nested case-control study was conducted within a cohort of 1979 patients enrolled in the Childhood Cancer Survivor Study who received treatment with anthracyclines and had available DNA. Significance metric(s): OR = 1.04; p = 0.97. Type of association: CO; TOX; ADR.] [OMIM:NAD(P)H DEHYDROGE...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs27388'>rs27388</a><a href='http://www.snpedia.com/index.php/rs27388(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFCDCD;'>32.1%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk of developing schizophrenia</div>
        <div class='boxshow' id="show-sec-0.0211551251891" style="display: inline;"><a href="javascript:toggle2('sec-0.0211551251891')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0211551251891" style="display: none;"><a href="javascript:toggle2('sec-0.0211551251891')">(hide)</a>
        <div class='rstext'>rs27388 is a SNP in the MEGF10 gene. In a case-control study involving 652 Irish patients, rs27388(A) allele carriers were at increased risk for schizophrenia. (Note that the (A) allele is quite common in many populations.) No association between schizophrenia and rs27388 of the MEGF10 gene in Chinese case-control sample.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2352028'>rs2352028</a><a href='http://www.snpedia.com/index.php/rs2352028(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD1D1;'>35.4%</span><br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'></div>
        <div class='boxshow' id="show-sec-0.421407411296" style="display: inline;"><a href="javascript:toggle2('sec-0.421407411296')">...more...</a></div>
        <div class='boxrstext' id="sec-0.421407411296" style="display: none;"><a href="javascript:toggle2('sec-0.421407411296')">(hide)</a>
        <div class='rstext'>news increased risk of lung cancer in non smokers. Analysis showed that the presence of this variant significantly increased the risk of a never smoker having lung cancer by 1.46 times (odds ratio 1.46, 95% confidence interval 1.26 to 1.70).  [PharmGKB:Curated Risk or phenotype-associated allele: Not specified. Phenotype: This variant was asssociated with risk for lung cancer in never smokers in a GWAS and validated and replicated in two further cohorts. Study size: 754 (GWAS); 988 (validation); 530 (replication). Study population/ethnicity: Never smokers; USA. Significance metric(s): OR = 1.46 (95% CI 1.26-1.70; p = 5.94x10(-6). Type of association: CO.]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1495741'>rs1495741</a><a href='http://www.snpedia.com/index.php/rs1495741(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD2D2;'>36.3%</span><br/>Repute:Bad<br/>References:14<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>NAT2 intermediate metabolizer (predicted)  see discussion at rs1495741</div>
        <div class='boxshow' id="show-sec-0.279826747032" style="display: inline;"><a href="javascript:toggle2('sec-0.279826747032')">...more...</a></div>
        <div class='boxrstext' id="sec-0.279826747032" style="display: none;"><a href="javascript:toggle2('sec-0.279826747032')">(hide)</a>
        <div class='rstext'>With 99% sensitivity and 95% specificity, at least in Caucasian populations, rs1495741 is reported to classify the NAT2 phenotype of an individual; it therefore is a candidate to replace the 7-SNP panel used in genosets such as gs138, gs139, and gs140. The rs1495741(A;A), (A;G) and (G;G) genotypes predicted slow, intermediate, and rapid NAT2 acetylation phenotypes, respectively.  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3212227'>rs3212227</a><a href='http://www.snpedia.com/index.php/rs3212227(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD2D2;'>36.3%</span><br/>Repute:Bad<br/>References:72<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Significantly increased risk of developing cervical cancer</div>
        <div class='boxshow' id="show-sec-0.995947899852" style="display: inline;"><a href="javascript:toggle2('sec-0.995947899852')">...more...</a></div>
        <div class='boxrstext' id="sec-0.995947899852" style="display: none;"><a href="javascript:toggle2('sec-0.995947899852')">(hide)</a>
        <div class='rstext'>IL12B gene SNP, part of a haplotype with rs6887695 associated with psoriasis. A study of 500+ patients with psoriatic arthritis (PsA) concluded that the odds ratio for rs3212227(A;A) carriers was 1.43 (CI: 1.17-1.76). The effect sizes observed in patients with PsA appear to be smaller than those previously reported in patients with psoriasis, suggesting that both loci are primarily associated with psoriasis susceptibility rather than primarily with psoriatic arthritis. rs568408 GA/AA and rs3212227 AC/CC variant genotypes were associated with a significantly increased risk of cervical cancer [adjusted odds ratio, 1.43; 95% confidence interval (CI), 1.06-1.93; and adjusted odds ratio, 1.30; 95% CI, 0.97-1.75, respectively]. subjects carrying variant genotypes of both loci had a 1.82-fold (95...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs31489'>rs31489</a><a href='http://www.snpedia.com/index.php/rs31489(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD3D3;'>37.5%</span><br/>Repute:Bad<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk for lung cancer and emphysema  The rs31489 C-allele correlated with reduced forced expiratory volume in 1 s (p=0.006). Homozygous carriers of the rs31489 C-allele exhibited increased susceptibility to bronchial obstruction (OR 1.82, 95% CI 1.24&#8211;2.69; p=0.002). A similar association was observed for diffusing capacity of the lung for carbon monoxide (p=0.004). Consistent with this, CC-carriers had an increased risk of emphysema (OR 2.04, 95% CI 1.41&#8211;2.94; p=1.73&#215;10&#8722;4) and displayed greater alveolar destruction. Finally, CC-carriers also had an increased risk for lung cancer (OR 1.90, 95% CI 1.21&#8211;2.99; p=0.005), and were more susceptible to developing both lung cancer and bronchial obstruction than lung cancer alone (OR 2.11, 95% CI 1.04&#8211;4.26...</div>
        <div class='boxshow' id="show-sec-0.464439263207" style="display: inline;"><a href="javascript:toggle2('sec-0.464439263207')">...more...</a></div>
        <div class='boxrstext' id="sec-0.464439263207" style="display: none;"><a href="javascript:toggle2('sec-0.464439263207')">(hide)</a>
        <div class='rstext'>[GWAS:Lung adenocarcinoma]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2201841'>rs2201841</a><a href='http://www.snpedia.com/index.php/rs2201841(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD3D3;'>38.1%</span><br/>Repute:Bad<br/>References:37<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.5x increased risk for Crohn's disease; 2x increased risk for Graves' disease</div>
        <div class='boxshow' id="show-sec-0.850088070573" style="display: inline;"><a href="javascript:toggle2('sec-0.850088070573')">...more...</a></div>
        <div class='boxrstext' id="sec-0.850088070573" style="display: none;"><a href="javascript:toggle2('sec-0.850088070573')">(hide)</a>
        <div class='rstext'>SNP rs2201841, in the IL23R gene, is associated with increased risk for Crohn's disease in both Jewish and non-Jewish populations. Another study found that the 'A allele' and 'AA genotype' were significantly overrepresented in Graves' disease patients with Graves ophthalmopathy, based on 216 North American patients. The odds ratios reported were 2.04 for the allele (p=1x10&lt;sup&gt;-4&lt;/sup&gt;), and for the so-called 'AA' genotype (presumably rs2201841(T;T) when correctly oriented to the dbSNP orientation), 2.4 (p=1x10&lt;sup&gt;-4&lt;/sup&gt;). 23andMe blog psoriasis Europeans *rs2201841(G) 1.13x risk  [PharmGKB:Non-Curated GWAS results: Genome-wide scan reveals association of psoriasis with IL-23 and NF-kappaB pathways. (Initial Sample Size: 1,359 cases, 1,400 controls; Replication S...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10889677'>rs10889677</a><a href='http://www.snpedia.com/index.php/rs10889677(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD4D4;'>38.9%</span><br/>Repute:Bad<br/>References:42<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.5x increased risk for certain autoimmune diseases; 2x increased risk for Graves disease</div>
        <div class='boxshow' id="show-sec-0.263290109057" style="display: inline;"><a href="javascript:toggle2('sec-0.263290109057')">...more...</a></div>
        <div class='boxrstext' id="sec-0.263290109057" style="display: none;"><a href="javascript:toggle2('sec-0.263290109057')">(hide)</a>
        <div class='rstext'>SNP rs10889677, in the IL23R gene, is associated with increased risk for Crohn's disease in both Jewish and non-Jewish populations. The same risk allele for this SNP has been associated with increased risk for ankylosing spondylitis in a large study of over 1,000 Caucasian patients. The odds ratio is 1.3 (p=1.3x10e-6). In a study of 216 North American patients with Graves' disease, the C allele of rs10889677 was 2.03x overrepresented (p=1.3x10&lt;sup&gt;&#226;&#8364;&#8220;4&lt;/sup&gt;), and the homozygous rs10889677(C;C) genotype was also overrepresented (2.36x; p=1.4x10&lt;sup&gt;-4&lt;/sup&gt;) in Graves ophthalmopathy patients. A genome-wide association study using DNA samples from 1,052 individuals with ulcerative colitis and preexisting data from 2,571 controls, all of European ance...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3775291'>rs3775291</a><a href='http://www.snpedia.com/index.php/rs3775291(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD4D4;'>38.9%</span><br/>Repute:Bad<br/>References:21<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>common but higher risk of age related macular degeneration</div>
        <div class='boxshow' id="show-sec-0.494552911995" style="display: inline;"><a href="javascript:toggle2('sec-0.494552911995')">...more...</a></div>
        <div class='boxrstext' id="sec-0.494552911995" style="display: none;"><a href="javascript:toggle2('sec-0.494552911995')">(hide)</a>
        <div class='rstext'>rs3775291 is a SNP in the TLR3 gene associated with an amino acid change in the corresponding protein. In the orientation as shown in dbSNP, the more common rs3775291(G) allele encodes a leucine while the rarer rs3775291(A) allele encodes a phenylalanine. rs3775291 has been associated with one form of age related macular degeneration (ARMD), specifically, the form known as 'dry' ARMD, also known as geographic atrophy. A study of 3 case-control groups of Americans of European descent, comprising about 900 ARMD patients in total, indicated that having one rs3775291(A) allele reduces the odds of having dry ARMD about 30%, and being a rs3775291(A;A) homozygote cuts your odds by more than half. Specifically, the odds ratio of having dry ARMD with one minor allele is 0.712 (CI: 0.50-1.0), and fo...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4129148'>rs4129148</a><a href='http://www.snpedia.com/index.php/rs4129148(C;G)'>(C;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD4D4;'>38.9%</span><br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>3x risk of schizophrenia.</div>
        <div class='boxshow' id="show-sec-0.79193245336" style="display: inline;"><a href="javascript:toggle2('sec-0.79193245336')">...more...</a></div>
        <div class='boxrstext' id="sec-0.79193245336" style="display: none;"><a href="javascript:toggle2('sec-0.79193245336')">(hide)</a>
        <div class='rstext'>Located in the Pseudoautosomal region rs4129148, located near the CSF2RA gene, has been reported in a whole genome association study to be associated with schizophrenia. The risk allele (oriented to the dbSNP entry) is (G); the odds ratio associated with this allele is 3.23 (CI 2.04 - 5.15). Given the results from the recent massive Psychiatric Genomics Consortium GWAS, it appears very unlikely that any large effect common SNPs exist for schizophrenia. It is not completely clear, though, how the previously replicated results for rs4129148 might be explained. It is currently predicted that 10,000 SNPs contribute to schizophrenic illness. The diagnosis for schizophrenia has now possibly shifted from asking: Schizophrenic? Yes or No to How much Schizophrenic?  [GWAS:Schizophrenia]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs5848'>rs5848</a><a href='http://www.snpedia.com/index.php/rs5848(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD4D4;'>38.9%</span><br/>Repute:Bad<br/>References:26<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased dementia risk  Increased risk of frontotemporal dementia, alzheimer's disease and parkinson's disease.</div>
        <div class='boxshow' id="show-sec-0.844852431748" style="display: inline;"><a href="javascript:toggle2('sec-0.844852431748')">...more...</a></div>
        <div class='boxrstext' id="sec-0.844852431748" style="display: none;"><a href="javascript:toggle2('sec-0.844852431748')">(hide)</a>
        <div class='rstext'>rs5848(T;T) found a 3.2-fold increased risk vs C-allele carriers (95% CI: 1.50-6.73, p=0.003) for ubiquitin- and TAR DNA-binding protein 43 (TDP-43)-positive frontotemporal dementia (FTLD-U also knows as FTLD-TDP43), a progressive neurodegenerative disease affecting approximately 10% of early-onset dementia patients. Detailed haplotype analysis was conducted, indicating rs5848 as most likely causal variant. FTLD-TDP43 is divided into four subtypes, some sources suggest association with type 3 only . Association between rs5848(T;T) and frontotemporal dementia is now generally assumed in literature, despite some studies failing to reproduce it. First meta-analysis of rs5848 association with Alzheimer's disease combining previous five studies for a total of 2502 Alzheimer's disease cases and ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs744373'>rs744373</a><a href='http://www.snpedia.com/index.php/rs744373(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD5D5;'>39.8%</span><br/>Repute:Bad<br/>References:15<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.17x risk of Alzheimer's</div>
        <div class='boxshow' id="show-sec-0.705377008167" style="display: inline;"><a href="javascript:toggle2('sec-0.705377008167')">...more...</a></div>
        <div class='boxrstext' id="sec-0.705377008167" style="display: none;"><a href="javascript:toggle2('sec-0.705377008167')">(hide)</a>
        <div class='rstext'>rs6733839 and rs744373 are less than 2,000 base pairs apart,and they have similar ORs and MAFs. Therefore, it seems reasonable to suspect that these 2 SNPs are actually reporting the same signal.  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11672691'>rs11672691</a><a href='http://www.snpedia.com/index.php/rs11672691(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD7D7;'>41.6%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.18x increased risk for PCSM in patients with prostate cancer  For a person already diagnosed with prostate cancer, this relatively common genotype is associated with a more aggressive form of the cancer and an increased risk (by 1.18x) of dying from causes related to the cancer.</div>
        <div class='boxshow' id="show-sec-0.853862675905" style="display: inline;"><a href="javascript:toggle2('sec-0.853862675905')">...more...</a></div>
        <div class='boxrstext' id="sec-0.853862675905" style="display: none;"><a href="javascript:toggle2('sec-0.853862675905')">(hide)</a>
        <div class='rstext'>rs11672691 is a SNP on chromosome 19 located between the ATP5SL and CEACAM21 genes and within LOC100505495, a possible noncoding RNA. A large study of over 10,000 patients with prostate cancer concluded that each rs11672691(G) allele was associated with an increased risk (by 1.18x) of prostate cancer specific mortality (PCSM), i.e. death from the cancer. A meta-analysis of 4 studies totaling ~6000 patients with prostate cancer and follow up in 49,000 samples concluded that rs11672691 was associated with increased susceptibility (odds ratio 1.12, CI:1.03&#8211;1.21, p = 1.4 x 10e-8) but also with aggressive prostate cancer and therefore a poorer prognosis.  [GWAS:Prostate cancer]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7794745'>rs7794745</a><a href='http://www.snpedia.com/index.php/rs7794745(A;T)'>(A;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD7D7;'>41.6%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>slightly increased risk for autism</div>
        <div class='boxshow' id="show-sec-0.254545795375" style="display: inline;"><a href="javascript:toggle2('sec-0.254545795375')">...more...</a></div>
        <div class='boxrstext' id="sec-0.254545795375" style="display: none;"><a href="javascript:toggle2('sec-0.254545795375')">(hide)</a>
        <div class='rstext'>A common SNP in the CNTNAP2 gene, rs7794745, is associated with increased risk for autism based on a study of 148 affected children from families with two more autistic children. Related article A genome-wide association study of autism reveals a common novel risk locus at 5p14.1. Do candidate genes discriminate patients with an autism spectrum disorder from those with attention deficit/hyperactivity disorder and is there an effect of lifetime substance use disorders? A common genetic variant in the neurexin superfamily member CNTNAP2 is associated with increased risk for selective mutism and social anxiety-related traits. Genetic variation in CNTNAP2 alters brain function during linguistic processing in healthy individuals.  [PharmGKB:Curated This common variant in CNTNAP2 is significantl...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs17228212'>rs17228212</a><a href='http://www.snpedia.com/index.php/rs17228212(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD8D8;'>42.9%</span><br/>Repute:Bad<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.26x increased risk for heart disease</div>
        <div class='boxshow' id="show-sec-0.388429314595" style="display: inline;"><a href="javascript:toggle2('sec-0.388429314595')">...more...</a></div>
        <div class='boxrstext' id="sec-0.388429314595" style="display: none;"><a href="javascript:toggle2('sec-0.388429314595')">(hide)</a>
        <div class='rstext'>rs17228212 is a SNP found to be associated with heart disease by the German MI (Myocardial infarction) Family Study group in two populations. The odds ratio per allele is 1.26 (CI: 1.11-1.44, p=0.0178). rs599839, rs4970834 and rs17228212 associated with non-HDL cholesterol</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs699'>rs699</a><a href='http://www.snpedia.com/index.php/rs699(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD9D9;'>43.4%</span><br/>Repute:Bad<br/>References:49<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk of hypertension</div>
        <div class='boxshow' id="show-sec-0.0189978128861" style="display: inline;"><a href="javascript:toggle2('sec-0.0189978128861')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0189978128861" style="display: none;"><a href="javascript:toggle2('sec-0.0189978128861')">(hide)</a>
        <div class='rstext'>rs699 is a SNP in the angiotensin AGT gene that encodes a functional change. In most published literature, the name for this SNP is M235T, or perhaps Met235Thr, however its amino acid 268 (not 235) that varies based on the numbering in todays databases. rs699 is also occasionally known as C4072T. In any case, the rs699(C) allele encodes the threonine variant, which is associated with higher plasma angiotensin levels, and ultimately higher blood pressure leading to increased risk for hypertension associated disorders. This association was first reported in 1992 , and many studies, though not all, have replicated these findings. rs699 is also (generally) reported to be in tight linkage with rs5051, a neighboring SNP in the promoter of the AGT gene. rs699(C) has also been implicated as a risk...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3793784'>rs3793784</a><a href='http://www.snpedia.com/index.php/rs3793784(C;G)'>(C;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD9D9;'>43.8%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.5x risk for ARMD</div>
        <div class='boxshow' id="show-sec-0.0848113546608" style="display: inline;"><a href="javascript:toggle2('sec-0.0848113546608')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0848113546608" style="display: none;"><a href="javascript:toggle2('sec-0.0848113546608')">(hide)</a>
        <div class='rstext'>rs3793784, a SNP upstream of the ERCC6 DNA repair gene, has been linked independently and in conjunction with another SNP (rs380390) to age related macular degeneration. On it's own, the odds ratios for increased risk for heterozygotes and homozygotes, were, respectively, 1.51 (CI 1.07-2.11) and 1.60 (CI 0.99-2.61). In dbSNP orientation, individuals who are CFH gene SNP genotype rs380390(C;C) and are also rs3793784(G;G) for this (ERCC6) SNP are calculated to be at 23 fold higher risk for developing age related macular degeneration compared to individuals who are homozygous for the more common alleles (among Caucasians) at both loci (ie who are rs380390(G;G) and rs3793784(C;C). Genetic markers and biomarkers for age-related macular degeneration. ERCC6/CSB gene polymorphisms and lung cancer ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs5882'>rs5882</a><a href='http://www.snpedia.com/index.php/rs5882(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDADA;'>44.2%</span><br/>Repute:Bad<br/>References:31<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Faster aging. Increased risk for Dementia. Less good cholesterol.  This reduces HDL ('good') cholesterol. HDL cholesterol removes cholesterol from artery walls and improves health. This SNP seems to be associated with higher blood pressure, less longevity, and more diseases of aging, in particular memory decline and dementia, and lower odds of living to 100, especially in Ashkenazi Jews. Attempting to raise HDL cholesterol may (or may not) be useful to mitigate the effects of this SNP.</div>
        <div class='boxshow' id="show-sec-0.877394335009" style="display: inline;"><a href="javascript:toggle2('sec-0.877394335009')">...more...</a></div>
        <div class='boxrstext' id="sec-0.877394335009" style="display: none;"><a href="javascript:toggle2('sec-0.877394335009')">(hide)</a>
        <div class='rstext'>This SNP rs5882 in the CETP gene referred to as ILE405VAL and RSAI405V has been related to ageing and exceptional long life. A study in Ashkenazi Jews showed that individuals homozygotic for the G allele have significantly longer lifespans. New research also indicates that G homozygotes have an average of 70% less risk of dementia and Alzheimer's disease. The study also found that heterozygotes have a lower risk of dementia and Alzheimer's disease but more study is needed to determine the risk reduction for heterozygotes. See Association of a Functional Polymorphism in the Cholesteryl Ester Transfer Protein (CETP) Gene With Memory Decline and Incidence of Dementia and CETP Variant Linked to Slower Cognitive Decline and Reduced Dementia Risk .  [PharmGKB:Curated Risk or phenotype-associated...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs17487223'>rs17487223</a><a href='http://www.snpedia.com/index.php/rs17487223(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDADA;'>44.6%</span><br/>Repute:Bad<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Higher lung cancer risk?</div>
        <div class='boxshow' id="show-sec-0.0679790711798" style="display: inline;"><a href="javascript:toggle2('sec-0.0679790711798')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0679790711798" style="display: none;"><a href="javascript:toggle2('sec-0.0679790711798')">(hide)</a>
        <div class='rstext'>rs17487223 increases susceptibility to Lung cancer 1.28 times for carriers of the T allele plos Risk Factors for Age-Dependent Nicotine Addiction</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6855911'>rs6855911</a><a href='http://www.snpedia.com/index.php/rs6855911(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDADA;'>44.6%</span><br/>Repute:Bad<br/>References:10<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>normal, but increased risk of gout</div>
        <div class='boxshow' id="show-sec-0.706323703141" style="display: inline;"><a href="javascript:toggle2('sec-0.706323703141')">...more...</a></div>
        <div class='boxrstext' id="sec-0.706323703141" style="display: none;"><a href="javascript:toggle2('sec-0.706323703141')">(hide)</a>
        <div class='rstext'>risk for cardiovascular morbidity and mortality rs6855911 (p = 1.84E-16) although no risk for coronary artery disease or myocardial infarction was observed, rs6855911 was associated with gout. The minor allele had a protective effect, with an allelic odds ratio of 0.62 (CI: 0.52-0.75, p = 3.2x10(-7)).  [PharmGKB:Non-Curated GWAS Results: The GLUT9 Gene Is Associated with Serum Uric Acid Levels in Sardinia and Chianti Cohorts (Initial Sample Size: 4,305 Sardinian individuals; Replication Sample Size: 1,301 Tuscan individuals; Risk Allele: rs6855911-A). This variant is associated with serum urate levels.] [GWAS:Serum urate]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs710521'>rs710521</a><a href='http://www.snpedia.com/index.php/rs710521(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDADA;'>45.1%</span><br/>Repute:Bad<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x increased risk of developing bladder cancer.  Slightly increased risk of bladder cancer, particularly in Europeans and in people who already have tumors with low risk of progression.</div>
        <div class='boxshow' id="show-sec-0.900928460843" style="display: inline;"><a href="javascript:toggle2('sec-0.900928460843')">...more...</a></div>
        <div class='boxrstext' id="sec-0.900928460843" style="display: none;"><a href="javascript:toggle2('sec-0.900928460843')">(hide)</a>
        <div class='rstext'>rs710521 is a SNP located in chromosome region 3q38, near the TP63 gene. A study ultimately totaling over 4,000 bladder cancer patients of European ancestry (and 40,000 controls) concluded that individuals carrying two copies of rs710521(A) were at 1.4x increased risk (allele-specific odds ratio = 1.19, p = 1 x 10e-7) for the disease. An additional 1,425 bladder cancer patients, supplemented by a meta-analysis of 5,695 additional cases (and 40,000+ controls), did replicate a (somewhat) increased risk associated with the rs710521(A) allele. The overall odds ratio was 1.18 (OR: 1.12 - 1.25, p &lt; 0.0001). Another study in Chinese populations did not find an increased risk of bladder cancer for this SNP.  [PharmGKB:Non-Curated GWAS Results: Sequence variant on 8q24 confers susceptibility to ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1051730'>rs1051730</a><a href='http://www.snpedia.com/index.php/rs1051730(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDADA;'>45.1%</span><br/>Repute:Bad<br/>References:95<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk of lung cancer</div>
        <div class='boxshow' id="show-sec-0.89635974494" style="display: inline;"><a href="javascript:toggle2('sec-0.89635974494')">...more...</a></div>
        <div class='boxrstext' id="sec-0.89635974494" style="display: none;"><a href="javascript:toggle2('sec-0.89635974494')">(hide)</a>
        <div class='rstext'>rs1051730, also known as D398N, is a SNP in the nicotinic acetylcholine receptor alpha 3 subunit CHRNA3 gene. In two recent (2008) studies, together comprising over 6,000 lung cancer patients of European ancestry, the rs1051730(T) allele was very significantly associated with increased risk. Having one copy (i.e. being a rs1051730(C;T) genotype) increased risk for lung cancer about 1.3x, and having two copies (rs1051730(T;T) individuals) represented 1.8x increased risk. Up to 14% of lung cancer incidence may be attributable to this allele. An independent study published at the same time concluded that (T) allele carriers for SNP rs1051730 are not at higher risk of becoming smokers compared to (C) carriers. However, if they do smoke, (T) carriers are quite likely to smoke more cigarettes th...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2395185'>rs2395185</a><a href='http://www.snpedia.com/index.php/rs2395185(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDBDB;'>45.5%</span><br/>Repute:Bad<br/>References:18<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2.72x higher risk of Type 1 Diabetes, but 0.88x lower risk of Ulcerative Colitis.  This lowers the risk of Ulcerative Colitis (according to 23andMe), and increases the risk of Type 1 Diabetes. It may also affect other autoimmune diseases.</div>
        <div class='boxshow' id="show-sec-0.171876291415" style="display: inline;"><a href="javascript:toggle2('sec-0.171876291415')">...more...</a></div>
        <div class='boxrstext' id="sec-0.171876291415" style="display: none;"><a href="javascript:toggle2('sec-0.171876291415')">(hide)</a>
        <div class='rstext'>A genome-wide association study using DNA samples from 1,052 individuals with ulcerative colitis and preexisting data from 2,571 controls, all of European ancestry, concluded that this SNP and several others were associated with altered risk for the disease.  [PharmGKB:Non-Curated GWAS results: Ulcerative colitis-risk loci on chromosomes 1p36 and 12q15 found by genome-wide association study. (Initial Sample Size: 1,022 cases, 2,503 controls; Replication Sample Size: 1,387 cases, 1,115 controls); (Region: 6p21.32; Reported Gene(s): BTNL2, HLA-DRA, HLA-DRB5, HLA-DRB1, HLA-DQA1, HLA-DQB1; Risk Allele: rs2395185-?); (p-value= 0.0000000000000001).This variant is associated with Ulcerative colitis.] [OMIM:?] [GWAS:Ulcerative colitis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4633'>rs4633</a><a href='http://www.snpedia.com/index.php/rs4633(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDBDB;'>45.9%</span><br/>Repute:Bad<br/>References:50<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>higher risk for endometrial cancer</div>
        <div class='boxshow' id="show-sec-0.413949159898" style="display: inline;"><a href="javascript:toggle2('sec-0.413949159898')">...more...</a></div>
        <div class='boxrstext' id="sec-0.413949159898" style="display: none;"><a href="javascript:toggle2('sec-0.413949159898')">(hide)</a>
        <div class='rstext'>rs4633 is a variant at codon 62 of the COMT gene, however, it does not change the amino acid sequence of the COMT protein. In a study of 150 (Caucasian) cases of endometrial cancer, a significant increase in rs4633(T;T) genotype was observed in patients compared to controls (OR = 2.39, CI: 1.31-4.37, p = 0.004). Furthemore, the frequency of the C-G haplotype of rs4633-rs4680 was significantly higher in controls (p &lt; 0.0001) than in patients. This correlated with lower expression levels of the COMT protein in carriers of these alleles. pubmed 1180576 Schizophrenia Susceptibility Genetic basis for individual variations in pain perception and the development of a chronic pain condition  [PharmGKB:Curated On the basis of subjects' pain responsiveness, haplotypes involving rs6269 (A/G), rs46...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3738919'>rs3738919</a><a href='http://www.snpedia.com/index.php/rs3738919(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDBDB;'>46.0%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.94x risk of developing rheumatoid arthritis</div>
        <div class='boxshow' id="show-sec-0.865891691914" style="display: inline;"><a href="javascript:toggle2('sec-0.865891691914')">...more...</a></div>
        <div class='boxrstext' id="sec-0.865891691914" style="display: none;"><a href="javascript:toggle2('sec-0.865891691914')">(hide)</a>
        <div class='rstext'>rs3738919, a SNP located in the ITGAV gene, was identified in a European study to be associated with rheumatoid arthritis (RA). The risk allele for rs3738919 is the more common allele, (C). For the three European Caucasian populations studied (372 RA patients + 330 controls), and combining the (C;C) and (A;C) genotypes in comparison to the (A;A) genotype, the odds ratio for RA = 1.94 (CI: 1.3&#226;&#8364;&#8220;2.9, p = 0.002). There was no significant difference in RA risk between those carrying one or two (C) alleles. An editorial about this finding has been published.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1160312'>rs1160312</a><a href='http://www.snpedia.com/index.php/rs1160312(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDDDD;'>47.8%</span><br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.6x increased risk of Male Pattern Baldness.</div>
        <div class='boxshow' id="show-sec-0.495012713616" style="display: inline;"><a href="javascript:toggle2('sec-0.495012713616')">...more...</a></div>
        <div class='boxrstext' id="sec-0.495012713616" style="display: none;"><a href="javascript:toggle2('sec-0.495012713616')">(hide)</a>
        <div class='rstext'>nature Baldness OR = 1.60, P = 1.1 times 10-14). see also gs122 The risk allele is assumed to be rs1160312(A). Note that a haplotype consisting of rs201571(T) - rs6036025(G) has a perfect correlation (r2 = 1) with rs1160312(A), at least in the Caucasian population studied.  [PharmGKB:Non-Curated Male-pattern baldness susceptibility locus at 20p11. (Initial Sample Size: 578 cases, 547 controls; Replication Sample Size: 1,351 cases, 2,485 controls); (Region: 20p11.22; Reported Gene: PAX1; Risk Allele: rs1160312-A) This variant is associated with Male-pattern baldness.] [OMIM:ALOPECIA, ANDROGENETIC, 3; AGA3] [GWAS:Male-pattern baldness]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2073963'>rs2073963</a><a href='http://www.snpedia.com/index.php/rs2073963(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDDDD;'>47.8%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk of baldness</div>
        <div class='boxshow' id="show-sec-0.721332664478" style="display: inline;"><a href="javascript:toggle2('sec-0.721332664478')">...more...</a></div>
        <div class='boxrstext' id="sec-0.721332664478" style="display: none;"><a href="javascript:toggle2('sec-0.721332664478')">(hide)</a>
        <div class='rstext'>Baldness paper from 23andMe June 2012  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7807268'>rs7807268</a><a href='http://www.snpedia.com/index.php/rs7807268(C;G)'>(C;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDDDD;'>47.8%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x risk for Crohn's disease</div>
        <div class='boxshow' id="show-sec-0.337354860887" style="display: inline;"><a href="javascript:toggle2('sec-0.337354860887')">...more...</a></div>
        <div class='boxrstext' id="sec-0.337354860887" style="display: none;"><a href="javascript:toggle2('sec-0.337354860887')">(hide)</a>
        <div class='rstext'>rs7807268 has been reported in a large study to be associated with Crohn's disease. The risk allele (oriented to the dbSNP entry) is (G); the odds ratio associated with heterozygotes is 1.38 (CI 1.20-1.60), and for homozygotes, 1.47 (CI 1.24-1.74). * Note: there is a slight amount of ambiguity over the orientation of this SNP information relative to the dbSNP entry.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7501331'>rs7501331</a><a href='http://www.snpedia.com/index.php/rs7501331(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDEDE;'>49.5%</span><br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Reduced conversion of beta-carotene to retinol  Reduced BCMO1 activity results in 32% lower ability to convert Beta-carotene to retinyl esters and higher serum beta-carotene levels. See also rs12934922.</div>
        <div class='boxshow' id="show-sec-0.556951786291" style="display: inline;"><a href="javascript:toggle2('sec-0.556951786291')">...more...</a></div>
        <div class='boxrstext' id="sec-0.556951786291" style="display: none;"><a href="javascript:toggle2('sec-0.556951786291')">(hide)</a>
        <div class='rstext'>rs12934922 (R267S) and rs7501331 (A379V) double mutant have a reduced catalytic activity of beta-carotene by 57%. Female volunteers carrying the T variant of rs7501331 (379V) had a 32% lower ability to convert Beta-carotene, and those carrying at least one T in both SNPs show a 69% lower ability to convert Beta-carotene into retinyl esters. rs7501331: the frequency of the wild-type C allele and variant T allele was 76 and 24%, respectively; 56% of the population was CC wild-type genotype, and 39% was heterozygote CT with the TT variant present in 5% of the population. rs12934922: the frequency of the wild-type A allele and variant T allele was 58 and 42%, respectively; 38% of the population was AA wild-type genotype, 40% was heterozygote AT, and 22% was TT homozygote</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1333048'>rs1333048</a><a href='http://www.snpedia.com/index.php/rs1333048(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDEDE;'>49.6%</span><br/>Repute:Bad<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased coronary artery disease risk</div>
        <div class='boxshow' id="show-sec-0.818253760858" style="display: inline;"><a href="javascript:toggle2('sec-0.818253760858')">...more...</a></div>
        <div class='boxrstext' id="sec-0.818253760858" style="display: none;"><a href="javascript:toggle2('sec-0.818253760858')">(hide)</a>
        <div class='rstext'>rs1333048 increases susceptibility to coronary artery disease 1.30 times for heterozygotes (AC) and 1.54 times for homozygotes (CC) rs1333048 2x risk for periodontitis and a confirmation of coronary heart disease 23andMe blog  [OMIM:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2073711'>rs2073711</a><a href='http://www.snpedia.com/index.php/rs2073711(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDEDE;'>49.6%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Back Pain: normal (higher) risk of Lumbar Disc Disease.  This normal version appears to have a slightly higher risk of LDD, at least in Japanese people and women.</div>
        <div class='boxshow' id="show-sec-0.389578704706" style="display: inline;"><a href="javascript:toggle2('sec-0.389578704706')">...more...</a></div>
        <div class='boxrstext' id="sec-0.389578704706" style="display: none;"><a href="javascript:toggle2('sec-0.389578704706')">(hide)</a>
        <div class='rstext'>The rs2073711 SNP (1184T&#8594;C ) is located in exon 8 of the cartilage intermediate layer protein gene (CILP). However, CILP gene is primarily known to be expressed abundantly in intervertebral discs, and was reported to have increased expression when degeneration occurs . For the rs2073711 SNP, the ancestral allele is T and the disease associated risk allele is C. A study of Asian patients with lumbar disc disease (LDD) implicates each copy of a (C) allele of SNP rs2073711 (as oriented with respect to dbSNP) as increasing risk about 1.6 fold. ==Background== Low back pain (LBP) is an enervating condition that affects 70-85% of all people during their lifetime . It can lead to decreased physical activity and quality of life, presenting considerable negative socioeconomic and health impact...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4307059'>rs4307059</a><a href='http://www.snpedia.com/index.php/rs4307059(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDEDE;'>49.6%</span><br/>Repute:Bad<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.19x risk of Autism  Slightly increased risk of Autism due to worse brain cell adhesion.</div>
        <div class='boxshow' id="show-sec-0.185898047702" style="display: inline;"><a href="javascript:toggle2('sec-0.185898047702')">...more...</a></div>
        <div class='boxrstext' id="sec-0.185898047702" style="display: none;"><a href="javascript:toggle2('sec-0.185898047702')">(hide)</a>
        <div class='rstext'>23andMe blog rs4307059 &#8212; compared to two copies of a C, each copy of the more common T version increased the odds of autism by 1.19 times. [http://dx.doi.org/10.1038/nature07999]  [PharmGKB:Curated This SNP on 5p14.1 was associated with autism spectrum disorder in a combined analysis of four GWAS data sets with a total of more than 10,000 subjects of European ancestry. This SNP is in an intergenic region between CDH10 and CDH9 that contains several highly conserved genomic elements.] [OMIM:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs520354'>rs520354</a><a href='http://www.snpedia.com/index.php/rs520354(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDEDE;'>49.6%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk in men for biliary conditions</div>
        <div class='boxshow' id="show-sec-0.731062116813" style="display: inline;"><a href="javascript:toggle2('sec-0.731062116813')">...more...</a></div>
        <div class='boxrstext' id="sec-0.731062116813" style="display: none;"><a href="javascript:toggle2('sec-0.731062116813')">(hide)</a>
        <div class='rstext'>rs520354 is a SNP in an intron of the apolipoprotein B APOB gene; it is also known as the IVS6+360C&gt;T variant. In a study of ~1,000 Chinese patients with various biliary tract conditions, including cancer and gallstones, men carrying a rs520354(A) allele had a 2x risk of bile duct cancer (CI: 1.2-3.4).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10757278'>rs10757278</a><a href='http://www.snpedia.com/index.php/rs10757278(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDFDF;'>50.0%</span><br/>Repute:Bad<br/>References:78<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x risk for Heart Attack. Normal risk for Abdominal Aortic Aneurysm and Brain Aneurysm.  One copy of an SNP that increases risk of Myocardial Infarction (Heart Attack) and other Coronary Heart Disease. It has been shown that regularly eating raw vegetables and fruit can reduce the risk to the same level as people without this SNP. This SNP also has the normal medium risk of abdominal aortic aneurysm (weakened artery to abdomen and legs) or brain aneurysm (weakened artery to brain).</div>
        <div class='boxshow' id="show-sec-0.184292023026" style="display: inline;"><a href="javascript:toggle2('sec-0.184292023026')">...more...</a></div>
        <div class='boxrstext' id="sec-0.184292023026" style="display: none;"><a href="javascript:toggle2('sec-0.184292023026')">(hide)</a>
        <div class='rstext'>This SNP, rs10757278, is one of several clustered together in a region of chromosome 9 that has been linked to increased risk for heart disease and potentially diabetes. The overall estimate of heart disease cases that may involve this SNP (or related ones nearby) is said to be 20-30%. The risk allele, rs10757278(G), shows an increased association for myocardial infarctions ('MI'; heart attacks) both in general and more specifically in so-called early onset MI. The odds ratio relative to rs10757278(A:A) 'noncarrier' individuals for rs10757278(G;G) individuals is 1.64 (CI: 1.47-1.82), and for carriers of one risk allele, i.e. rs10757278(A;G) individuals, 1.26 (CI: 1.16-1.36). For early onset MI, the odds are slightly higher; homozygote rs10757278(G;G) individuals have an odds ratio of 2.02 ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs25487'>rs25487</a><a href='http://www.snpedia.com/index.php/rs25487(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDFDF;'>50.0%</span><br/>Repute:Bad<br/>References:43<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2x higher risk for skin cancer; possibly other cancers</div>
        <div class='boxshow' id="show-sec-0.765469764746" style="display: inline;"><a href="javascript:toggle2('sec-0.765469764746')">...more...</a></div>
        <div class='boxrstext' id="sec-0.765469764746" style="display: none;"><a href="javascript:toggle2('sec-0.765469764746')">(hide)</a>
        <div class='rstext'>rs25487 is a SNP also known as Gln399Arg, located in the DNA repair gene XRCC1. The (A) allele encodes the Gln amino acid. In one study of ~1000 Caucasians, the rs25487(A;A) genotype had significantly reduced risk of both basal cell [BCC; odds ratio 0.7, CI: 0.4-1.0] and squamous cell cancer (SCC; odds ratio 0.6, CI: 0.3-0.9). In a study of ~300 Koreans, rs25487(A;G) and (G;G) genotypes had an approximately 2-fold increased risk of basal cell cancer compared to rs25487(A;A) individuals (adjusted odds ratio 2.324, CI: 1.11-4.86, respectively). A meta-analysis of 10+ studies found that rs25487(G) carriers had increased childhood risk of acute lymphoblastic leukemia among Asians under the dominant (odds ratio 2.11, CI: 1.50&#8211;2.97, p&lt;0.0001) as well as other models. No association was found ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3923809'>rs3923809</a><a href='http://www.snpedia.com/index.php/rs3923809(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDFDF;'>50.4%</span><br/>Repute:Bad<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.9x risk for developing restless legs syndrome  rs3923809 is a A/G variation on human chromosome 6. Associated with: Restless legs syndrome. This SNP, located in an intron of the BTBD9 gene, has a variant that is seen somewhat more frequently in individuals with restless legs syndrome. The risk allele is rs3923809(A); carriers of two such alleles, i.e. those with rs3923809(A;A) genotypes, are estimated to be 1.9 fold more likely to have restless legs syndrome than rs3923809(G;G) individuals. The authors of this study suggest that perhaps half of the cases of restless legs syndrome may involve the rs3923809 risk genotypes. Note that 70-80% of all individuals in European populations carry one or two copies of the (A) major allele, yet restless legs syndrome is thought to affect less than 10...</div>
        <div class='boxshow' id="show-sec-0.562597097306" style="display: inline;"><a href="javascript:toggle2('sec-0.562597097306')">...more...</a></div>
        <div class='boxrstext' id="sec-0.562597097306" style="display: none;"><a href="javascript:toggle2('sec-0.562597097306')">(hide)</a>
        <div class='rstext'>This SNP, located in an intron of the BTBD9 gene, has a variant that is seen somewhat more frequently in individuals with restless legs syndrome. The risk allele is rs3923809(A); carriers of two such alleles, i.e. those with rs3923809(A;A) genotypes, are estimated to be 1.9 fold more likely to have restless legs syndrome than rs3923809(G;G) individuals. The authors of this study suggest that perhaps half of the cases of restless legs syndrome may involve the rs3923809 risk genotypes. Consistent with this finding, another report about rs3923809 links the (G) minor allele to a lower frequency of restless legs syndrome with an overall odds ratio of 0.57 (CI: 0.48-0.68). Note that 70-80% of all individuals in European populations carry one or two copies of the (A) major allele, yet restless le...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1024611'>rs1024611</a><a href='http://www.snpedia.com/index.php/rs1024611(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDFDF;'>50.4%</span><br/>Repute:Bad<br/>References:19<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk of exercise induced ischemia</div>
        <div class='boxshow' id="show-sec-0.0584866971037" style="display: inline;"><a href="javascript:toggle2('sec-0.0584866971037')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0584866971037" style="display: none;"><a href="javascript:toggle2('sec-0.0584866971037')">(hide)</a>
        <div class='rstext'>rs1024611, also known as the -2578A&gt;G SNP due to its position in the promoter of the monocyte chemoattractant protein-1 MCP-1 CCL2 gene, influences the production of its corresponding protein, a chemokine involved in inflammatory responses. In a study focusing on 679 apparently healthy siblings of people with premature heart disease, investigators found that carriers of an rs1024611(C) allele - oriented as in dbSNP, not as published - independently predicted the risk of exercise induced ischemia in general. The odds ratio was reported to be 1.86 (CI: 1.14-3.04, p=0.014), regardless of race, age, sex and other factors. However, it is not clear if this risk carries over to individuals who lack siblings with heart disease. discussed in the 23andMe blog as being relevant to HIV rs1024611 is...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1800896'>rs1800896</a><a href='http://www.snpedia.com/index.php/rs1800896(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE0E0;'>51.3%</span><br/>Repute:Bad<br/>References:121<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.6x increased prostate cancer risk</div>
        <div class='boxshow' id="show-sec-0.818933783993" style="display: inline;"><a href="javascript:toggle2('sec-0.818933783993')">...more...</a></div>
        <div class='boxrstext' id="sec-0.818933783993" style="display: none;"><a href="javascript:toggle2('sec-0.818933783993')">(hide)</a>
        <div class='rstext'>This SNP is upstream of the IL10 gene, and is also known as the -1082G&gt;A IL10 SNP. Dust mite exposure modifies the effect of functional IL10 polymorphisms on allergy and asthma exacerbations. Dust mite exposure significantly modified the relation between 3 SNPs in IL10 (rs1800896, rs3024492, and rs3024496). Homozygosity for the minor allele of each of the 3 SNPs was associated with increased risk of occurrence ( approximately 3-fold to 39-fold increase). A study of 258 prostate cancer cases concluded that the rs1800896(A) allele, known to result in lower levels of this anti-inflammatory cytokine, was positively associated with risk (AG vs. GG, odds ratio of 1.69, CI: 1.10-2.60; AA vs. GG, OR of 1.81, CI: 1.11-2.96).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1015362'>rs1015362</a><a href='http://www.snpedia.com/index.php/rs1015362(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE1E1;'>52.2%</span><br/>Repute:Bad<br/>References:11<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2-4x higher risk of sun sensitivity if part of risk haplotype.  Increased risk of sunburns and freckles instead of tanning.</div>
        <div class='boxshow' id="show-sec-0.749236375291" style="display: inline;"><a href="javascript:toggle2('sec-0.749236375291')">...more...</a></div>
        <div class='boxrstext' id="sec-0.749236375291" style="display: none;"><a href="javascript:toggle2('sec-0.749236375291')">(hide)</a>
        <div class='rstext'>rs1015362 is a SNP near the ASIP (agouti signaling protein) gene on chromosome 20. This SNP is one of a tightly-linked pair that increases the likelihood of an individual being prone to sun sensitivity, in other words, freckles and sunburn, based on a study of 6,000+ Caucasians (5,000+ Icelanders + 1,000+ Dutch). The odds ratios, presumably on a dominant basis, and at least in the largest population (Icelanders) for 'freckles and burns vs. no freckles and tans' for the haplotype pair rs1015362(G) - rs4911414(T) is 3.91 (CI: 2.54-6.03) for males and 2.42 (CI: 1.52-3.86) for females, with an overall p=0.051. Based on a study by the same authors of 4,000+ skin cancer patients, this haplotype was seen to confer significant increased risk for cutaneous malignant melanoma (odds ratio 1.45, p = 1...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10984447'>rs10984447</a><a href='http://www.snpedia.com/index.php/rs10984447(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE1E1;'>52.2%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>>1.17x increased risk for multiple sclerosis</div>
        <div class='boxshow' id="show-sec-0.709906480868" style="display: inline;"><a href="javascript:toggle2('sec-0.709906480868')">...more...</a></div>
        <div class='boxrstext' id="sec-0.709906480868" style="display: none;"><a href="javascript:toggle2('sec-0.709906480868')">(hide)</a>
        <div class='rstext'>rs10984447 has been reported in a large study to be associated with multiple sclerosis. The risk allele (oriented to the dbSNP entry) is (A); the odds ratio associated with this allele is 1.17 (CI 1.09-1.25).  [PharmGKB:Curated GWAS Results: Risk alleles for multiple sclerosis identified by a genomewide study (Initial Sample Size: 931 trios, 2,431 controls; Replication Sample Size: 609 trios, 2,322 cases, 2,987 controls; Risk Allele: rs10984447-A).] [GWAS:Multiple sclerosis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs351855'>rs351855</a><a href='http://www.snpedia.com/index.php/rs351855(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE2E2;'>53.4%</span><br/>Repute:Bad<br/>References:16<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.2x increased risk for prostate cancer</div>
        <div class='boxshow' id="show-sec-0.665818184676" style="display: inline;"><a href="javascript:toggle2('sec-0.665818184676')">...more...</a></div>
        <div class='boxrstext' id="sec-0.665818184676" style="display: none;"><a href="javascript:toggle2('sec-0.665818184676')">(hide)</a>
        <div class='rstext'>rs351855, a SNP in the fibroblast growth factor receptor 4 (FGFR4) gene, is also known as the Gly388Arg variant. The rs351855(T) allele encodes the risk (Arg) allele. The Arg form of this SNP is likely to cause a harder to treat version of node-positive breast cancer, including reducing the efficacy of Herceptin, based on a study of 372 patients. A study of ~500 Japanese prostate cancer patients found that individuals with a rs351855(T;T) genotype had a 2.2- and 1.9-fold increased risk of prostate cancer and benign prostate hyperplasia (BPH), and a 1.8-fold increased risk of metastatic prostate cancer compared to the (C;C) genotype. A meta-analysis published in 2011, surveying a total of 2,618 cases of prostate cancer, concluded that the odds ratio per rs351855(T) allele was 1.17 (CI: 1.07...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6997709'>rs6997709</a><a href='http://www.snpedia.com/index.php/rs6997709(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE2E2;'>54.0%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.5x higher risk for hypertension</div>
        <div class='boxshow' id="show-sec-0.541996651865" style="display: inline;"><a href="javascript:toggle2('sec-0.541996651865')">...more...</a></div>
        <div class='boxrstext' id="sec-0.541996651865" style="display: none;"><a href="javascript:toggle2('sec-0.541996651865')">(hide)</a>
        <div class='rstext'>rs6997709 has been reported in a large study to be associated with high blood pressure. The risk allele (oriented to the dbSNP entry) is (G); the odds ratio associated with heterozygotes is 1.20 (CI 0.94-1.52), and for homozygotes, 1.49 (CI 1.18-1.89). In a study of 7,551 Koreans, this SNP was associated with systolic and diastolic blood pressures in a quantitative trait association test but no associations were found using a case-control association test.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2235015'>rs2235015</a><a href='http://www.snpedia.com/index.php/rs2235015(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE4E4;'>55.8%</span><br/>Repute:Bad<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>somewhat less likely to respond to certain antidepressants</div>
        <div class='boxshow' id="show-sec-0.338404809781" style="display: inline;"><a href="javascript:toggle2('sec-0.338404809781')">...more...</a></div>
        <div class='boxrstext' id="sec-0.338404809781" style="display: none;"><a href="javascript:toggle2('sec-0.338404809781')">(hide)</a>
        <div class='rstext'>rs2235015 is a SNP in the ABCB1 gene (also known as the MDR1 gene), which encodes a protein that transports certain molecules across the blood-brain barrier. SNPs in ABCB1 may thus influence the intracerebral concentrations of certain drugs and thus their efficacy or potential for adverse side effects. rs2032583 is near 9 SNPs found within a tight linkage block (r&lt;sup&gt;2&lt;/sup&gt; &gt;= 0.8 ) such that the minor allele at any one of them predicts (with ~80%+ accuracy) that the other SNPs will also be the minor allele. The list of the 9 SNPs is shown below. rs2235015 has an r&lt;sup&gt;2&lt;/sup&gt; value of &gt;0.5 with this group. When treated for depression with substrates of the protein encoded by ABCB1, carriers of one or two minor alleles at these ABCB1 SNPs have been reported ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2383206'>rs2383206</a><a href='http://www.snpedia.com/index.php/rs2383206(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE4E4;'>56.9%</span><br/>Repute:Bad<br/>References:31<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x increased risk for heart disease</div>
        <div class='boxshow' id="show-sec-0.242081154736" style="display: inline;"><a href="javascript:toggle2('sec-0.242081154736')">...more...</a></div>
        <div class='boxrstext' id="sec-0.242081154736" style="display: none;"><a href="javascript:toggle2('sec-0.242081154736')">(hide)</a>
        <div class='rstext'>rs10757274 and rs2383206 can significantly increase the risk of heart diseaseAbout one in every four Caucasians are thought to carry the variants, and their risk of coronary heart disease is increased by 30 to 40%. rs10757278] in the same region has been linked to diabetes The chromosomal region where these SNPs are located is 9p21, and has no known genes. a [http://suicyte.wordpress.com/2007/05/26/soul-searching-i/ blog post about investigating rs10757274 and rs2383206 This SNP was also associated with increased risk for coronary artery disease in a Korean population. Also found to be significant in a study of 416 Italian myocardial infarction patients. A study of 1,000+ patients with early-onset angiographic coronary artery disease (CAD) concluded that rs2383206(G) was associated with an...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1585215'>rs1585215</a><a href='http://www.snpedia.com/index.php/rs1585215(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE4E4;'>56.9%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2x increased risk for Hodgkin lymphoma</div>
        <div class='boxshow' id="show-sec-0.257651643962" style="display: inline;"><a href="javascript:toggle2('sec-0.257651643962')">...more...</a></div>
        <div class='boxrstext' id="sec-0.257651643962" style="display: none;"><a href="javascript:toggle2('sec-0.257651643962')">(hide)</a>
        <div class='rstext'>rs1585215 is a SNP in the NFKB1 gene. A population-based case-control study including 473 cases of Hodgkin lymphoma concluded that increased risk for the disease was significantly associated with rs1585215. The odds ratio for rs1585215(G;G) homozygotes was 3.5 (CI: 2.2-5.7, Ptrend = 1.7 x 10(-8)), and for (A;G) heterozygotes, 2.1 (CI: 1.5-2.9).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs13153971'>rs13153971</a><a href='http://www.snpedia.com/index.php/rs13153971(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE5E5;'>57.5%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal (higher) risk of Asthma.  This is normal for white people. But this (C;T) has 1.58x the risk compared to T;T (normal for non-white people). Tested in Koreans.</div>
        <div class='boxshow' id="show-sec-0.427533077864" style="display: inline;"><a href="javascript:toggle2('sec-0.427533077864')">...more...</a></div>
        <div class='boxrstext' id="sec-0.427533077864" style="display: none;"><a href="javascript:toggle2('sec-0.427533077864')">(hide)</a>
        <div class='rstext'>Asthma</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs358806'>rs358806</a><a href='http://www.snpedia.com/index.php/rs358806(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE6E6;'>59.3%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.78x increased risk of developing Type-2 diabetes</div>
        <div class='boxshow' id="show-sec-0.978244839651" style="display: inline;"><a href="javascript:toggle2('sec-0.978244839651')">...more...</a></div>
        <div class='boxrstext' id="sec-0.978244839651" style="display: none;"><a href="javascript:toggle2('sec-0.978244839651')">(hide)</a>
        <div class='rstext'>rs358806 has been reported in a large study to be associated with type-2 diabetes. The risk allele is (C); the odds ratio associated with heterozygotes is 0.86 (CI 0.75-0.97), and for homozygotes, 1.78 (CI 1.34-2.36).  [PharmGKB:Non-Curated GWAS results: Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls (Initial Sample Size: 1,924 cases, 2,938 controls; Replication Sample Size: (see Zeggini 2007)). This variant is associated with Type 2 diabetes.] [GWAS:Type 2 diabetes]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6897932'>rs6897932</a><a href='http://www.snpedia.com/index.php/rs6897932(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE6E6;'>59.3%</span><br/>Repute:Bad<br/>References:53<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.5x increased risk for multiple sclerosis</div>
        <div class='boxshow' id="show-sec-0.610795620273" style="display: inline;"><a href="javascript:toggle2('sec-0.610795620273')">...more...</a></div>
        <div class='boxrstext' id="sec-0.610795620273" style="display: none;"><a href="javascript:toggle2('sec-0.610795620273')">(hide)</a>
        <div class='rstext'>The (C) allele of rs6897932, located in the alternatively spliced exon 6 of IL7RA gene and encoding the amino acid threonine rather than isoleucine at amino acid position 244, is associated with a slight increase (18%) in risk of developing multiple sclerosis. [PMID 17660817; Nature Genetics 39, 1083 - 1091 (2007) SG Gregory et al.] Note that the (C) allele is the most common at this position in all known populations and influences the ratios of the alternative isoforms (membrane bound and soluble) of the gene. blog post giving perspective on the significance of this snp a significant risk factor for multiple sclerosis in four independent (overall P = 2.9 x 10(-7)) influences the amount of soluble and membrane-bound isoforms of the protein by putatively disrupting an exonic splicing silenc...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7442295'>rs7442295</a><a href='http://www.snpedia.com/index.php/rs7442295(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE7E7;'>60.2%</span><br/>Repute:Bad<br/>References:13<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~4x higher risk for hyperuracemia</div>
        <div class='boxshow' id="show-sec-0.710450087856" style="display: inline;"><a href="javascript:toggle2('sec-0.710450087856')">...more...</a></div>
        <div class='boxrstext' id="sec-0.710450087856" style="display: none;"><a href="javascript:toggle2('sec-0.710450087856')">(hide)</a>
        <div class='rstext'>rs7442295 is a SNP in the SLC2A9 gene; this gene encodes a glucose transporter. Based on an initial study of 1900+ hypertensive Caucasian Europeans, the more common allele rs7442295(A) allele was associated with higher serum urate and hyperuracemia (defined as urate &gt;0.4 mMol/l), with a reported odds ratio of 1.89 (CI: 1.36-2.61, p=5 x 10&lt;sup&gt;-5&lt;/sup&gt;). This was also seen in an associated meta-analysis conducted with a broadly representative population and using a surrogate SNP, rs6449213, which is in fairly tight linkage (r&lt;sup&gt;2&lt;/sup&gt;=0.88) with rs7442295. To put it another way, 79% of white Europeans carry one or two alleles leading to higher serum urate levels. Each copy leads to an average increase of 0.02mMol/l, and, an ~doubling of risk for hyperuracemia. ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs800292'>rs800292</a><a href='http://www.snpedia.com/index.php/rs800292(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE9E9;'>62.8%</span><br/>Repute:Bad<br/>References:28<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>5% higher risk of Age related macular degeneration</div>
        <div class='boxshow' id="show-sec-0.467408410632" style="display: inline;"><a href="javascript:toggle2('sec-0.467408410632')">...more...</a></div>
        <div class='boxrstext' id="sec-0.467408410632" style="display: none;"><a href="javascript:toggle2('sec-0.467408410632')">(hide)</a>
        <div class='rstext'>rs800292 is a SNP in the complement factor H CFH gene; it has been linked to blindness in age related macular degeneration. This SNP is also known as 184G&gt;A, I62V, or Val62Ile. A haplotype of rs1061170 rs3753394 rs800292 rs1329428 (TGTC) was found to confer a significantly increased likelihood of exudative AMD CFH variations appear to contribute to ARMD in Caucasians, but not in Japanese A study of Chinese AMD patients reports that carriers of both rs11200638 and rs800292 risk alleles pushes the odds ratio for AMD up to 23x. Overall, an 'extremely high' population attributable risk (PAR) of 78% reported for these SNPs. rs800292 was associated with polypoidal choroidal vasculopathy (PCV) in a study of 130 Japanese patients. age related macular degeneration Susceptibility genes for age-re...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10248420'>rs10248420</a><a href='http://www.snpedia.com/index.php/rs10248420(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFECEC;'>67.3%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>7x less likely to respond to certain antidepressants</div>
        <div class='boxshow' id="show-sec-0.850072115224" style="display: inline;"><a href="javascript:toggle2('sec-0.850072115224')">...more...</a></div>
        <div class='boxrstext' id="sec-0.850072115224" style="display: none;"><a href="javascript:toggle2('sec-0.850072115224')">(hide)</a>
        <div class='rstext'>rs10248420 is a SNP in the ABCB1 gene (also known as the MDR1 gene), which encodes a protein that transports certain molecules across the blood-brain barrier. SNPs in ABCB1 may thus influence the intracerebral concentrations of certain drugs and thus their efficacy or potential for adverse side effects. rs10248420 is one of 9 SNPs found within a tight linkage block (r&lt;sup&gt;2&lt;/sup&gt; &gt;= 0.8 ) such that the minor allele at any one of them predicts (with ~80%+ accuracy) that the other SNPs will also be the minor allele. The list of the 9 SNPs is shown below. When treated for depression with substrates of the protein encoded by ABCB1, carriers of one or two minor alleles at these ABCB1 SNPs have been reported to respond better than non-carriers. The antidepressant drugs that are kn...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9652490'>rs9652490</a><a href='http://www.snpedia.com/index.php/rs9652490(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFECEC;'>67.3%</span><br/>Repute:Bad<br/>References:22<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~2x increased risk for Parkinson's disease, and, essential tremor</div>
        <div class='boxshow' id="show-sec-0.568427099366" style="display: inline;"><a href="javascript:toggle2('sec-0.568427099366')">...more...</a></div>
        <div class='boxrstext' id="sec-0.568427099366" style="display: none;"><a href="javascript:toggle2('sec-0.568427099366')">(hide)</a>
        <div class='rstext'>rs9652490 is a SNP in the LINGO1 gene. Note: it is unclear why one set of authors report the risk allele for essential tremors to be (G), while the others just as clearly report the risk allele to be (A). A follow-up case-control study replicated the original association of rs9652490(A;A) to an ~2x increased risk for Parkinson's disease, but also found the same risk for essential tremor (ET), thus linking the two diseases. The ET study was based on ~300 patients. 23andMe blog 'In a combined analysis of 752 subjects with essential tremor from Iceland, Austria, Germany and the United States and 15,797 controls, Stefansson et al found that the G version of rs9652490 in the LINGO1 gene is associated with increased odds of the condition. Carrying one G increases the odds of essential tremor by ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs27072'>rs27072</a><a href='http://www.snpedia.com/index.php/rs27072(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFECEC;'>67.9%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2x risk of severe alcohol withdrawal. Possible increased odds of ADHD.  Alcoholics with this are twice as likely to have seizures during alcohol withdrawal.</div>
        <div class='boxshow' id="show-sec-0.121727042953" style="display: inline;"><a href="javascript:toggle2('sec-0.121727042953')">...more...</a></div>
        <div class='boxrstext' id="sec-0.121727042953" style="display: none;"><a href="javascript:toggle2('sec-0.121727042953')">(hide)</a>
        <div class='rstext'>rs27072, a SNP in the dopamine transporter SLC6A3 gene, has been associated with more severe symptoms upon alcohol withdrawal, such as seizures, in a study of 250 Caucasian alcohol-dependent patients. Two haplotypes appear to be tagged by this SNP and a neighbor, rs27048. The rs27072 polymorphism has been shown to be significantly associated with ADHD in clinical samples of Canadian children aged 6 to 16 [Feng et al., 2005] and the significant association was also reported by Ouellet-Morin et al. in a Canadian twin study, 2008 (Am. J. of Medical Genetics Part B, 147B:1442-1449 (2008). The interaction between the dopamine transporter gene and age at onset in relation to tobacco and alcohol use among 19-year-olds. Association of promoter variants of human dopamine transporter gene with schiz...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7250581'>rs7250581</a><a href='http://www.snpedia.com/index.php/rs7250581(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFEDED;'>69.0%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x risk  1.4x higher risk for coronary artery disease</div>
        <div class='boxshow' id="show-sec-0.97118810737" style="display: inline;"><a href="javascript:toggle2('sec-0.97118810737')">...more...</a></div>
        <div class='boxrstext' id="sec-0.97118810737" style="display: none;"><a href="javascript:toggle2('sec-0.97118810737')">(hide)</a>
        <div class='rstext'>rs7250581 has been reported in a large study to be associated with heart disease, in particular, coronary artery disease. The risk allele (oriented to the dbSNP entry) is (G); the odds ratio associated with heterozygotes is 1.06 (CI 0.79-1.43), and for homozygotes, 1.40 (CI 1.05-1.86).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1385699'>rs1385699</a><a href='http://www.snpedia.com/index.php/rs1385699(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFEEEE;'>70.8%</span><br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk of baldness  slightly higher risk of baldness</div>
        <div class='boxshow' id="show-sec-0.307889451498" style="display: inline;"><a href="javascript:toggle2('sec-0.307889451498')">...more...</a></div>
        <div class='boxrstext' id="sec-0.307889451498" style="display: none;"><a href="javascript:toggle2('sec-0.307889451498')">(hide)</a>
        <div class='rstext'>rs1385699, also known as Arg57Lys, is a SNP in the ectodysplasin A2 receptor EDAR2 gene. This gene is located on the X chromosome. The rs1385699(T) allele, which is the most common in European populations, was found to be associated with androgenetic alopecia in a case-control study of 400 males.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6896702'>rs6896702</a><a href='http://www.snpedia.com/index.php/rs6896702(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFEEEE;'>70.8%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk of developing Parkinson's Disease</div>
        <div class='boxshow' id="show-sec-0.17556652349" style="display: inline;"><a href="javascript:toggle2('sec-0.17556652349')">...more...</a></div>
        <div class='boxrstext' id="sec-0.17556652349" style="display: none;"><a href="javascript:toggle2('sec-0.17556652349')">(hide)</a>
        <div class='rstext'>rs6896702 is in linkage disequilibrium with a polymorphism that increases susceptibility to Parkinson's disease 1.74 times for carriers of the T allele</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11983225'>rs11983225</a><a href='http://www.snpedia.com/index.php/rs11983225(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFEFEF;'>71.7%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>7x less likely to respond to certain antidepressants</div>
        <div class='boxshow' id="show-sec-0.535632320092" style="display: inline;"><a href="javascript:toggle2('sec-0.535632320092')">...more...</a></div>
        <div class='boxrstext' id="sec-0.535632320092" style="display: none;"><a href="javascript:toggle2('sec-0.535632320092')">(hide)</a>
        <div class='rstext'>rs11983225 is a SNP in the ABCB1 gene (also known as the MDR1 gene), which encodes a protein that transports certain molecules across the blood-brain barrier. SNPs in ABCB1 may thus influence the intracerebral concentrations of certain drugs and thus their efficacy or potential for adverse side effects. rs11983225 is one of 9 SNPs found within a tight linkage block (r&lt;sup&gt;2&lt;/sup&gt; &gt;= 0.8 ) such that the minor allele at any one of them predicts (with ~80%+ accuracy) that the other SNPs will also be the minor allele. The list of the 9 SNPs is shown below. When treated for depression with substrates of the protein encoded by ABCB1, carriers of one or two minor alleles at these ABCB1 SNPs have been reported to respond better than non-carriers. The antidepressant drugs that are kn...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2032583'>rs2032583</a><a href='http://www.snpedia.com/index.php/rs2032583(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFEFEF;'>71.7%</span><br/>Repute:Bad<br/>References:11<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>7x less likely to respond to certain antidepressants.  This version of a blood brain barrier protein blocks many common antidepressants from entering the brain, including: amitriptyline (Elavil), citalopram (Celexa), paroxetine (Paxil), and venlafaxine (Effexor). That makes those antidepressants 7 times less effective.</div>
        <div class='boxshow' id="show-sec-0.681229037559" style="display: inline;"><a href="javascript:toggle2('sec-0.681229037559')">...more...</a></div>
        <div class='boxrstext' id="sec-0.681229037559" style="display: none;"><a href="javascript:toggle2('sec-0.681229037559')">(hide)</a>
        <div class='rstext'>rs2032583 is a SNP in the ABCB1 gene (also known as the MDR1 gene), which encodes a protein that transports certain molecules across the blood-brain barrier. SNPs in ABCB1 may thus influence the intracerebral concentrations of certain drugs and thus their efficacy or potential for adverse side effects. rs2032583 is one of 9 SNPs found within a tight linkage block (r&lt;sup&gt;2&lt;/sup&gt; &gt;= 0.8 ) such that the minor allele at any one of them predicts (with ~80%+ accuracy) that the other SNPs will also be the minor allele. The list of the 9 SNPs is shown below. When treated for depression with substrates of the protein encoded by ABCB1, carriers of one or two minor alleles at these ABCB1 SNPs have been reported to respond better than non-carriers. The antidepressant drugs that are know...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2235040'>rs2235040</a><a href='http://www.snpedia.com/index.php/rs2235040(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFEFEF;'>71.7%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>7x less likely to respond to certain antidepressants</div>
        <div class='boxshow' id="show-sec-0.227811045535" style="display: inline;"><a href="javascript:toggle2('sec-0.227811045535')">...more...</a></div>
        <div class='boxrstext' id="sec-0.227811045535" style="display: none;"><a href="javascript:toggle2('sec-0.227811045535')">(hide)</a>
        <div class='rstext'>rs2235040 is a SNP in the ABCB1 gene (also known as the MDR1 gene), which encodes a protein that transports certain molecules across the blood-brain barrier. SNPs in ABCB1 may thus influence the intracerebral concentrations of certain drugs and thus their efficacy or potential for adverse side effects. rs2235040 is one of 9 SNPs found within a tight linkage block (r&lt;sup&gt;2&lt;/sup&gt; &gt;= 0.8 ) such that the minor allele at any one of them predicts (with ~80%+ accuracy) that the other SNPs will also be the minor allele. The list of the 9 SNPs is shown below. When treated for depression with substrates of the protein encoded by ABCB1, carriers of one or two minor alleles at these ABCB1 SNPs have been reported to respond better than non-carriers. The antidepressant drugs that are know...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2235067'>rs2235067</a><a href='http://www.snpedia.com/index.php/rs2235067(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFEFEF;'>71.7%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>7x less likely to respond to certain antidepressants</div>
        <div class='boxshow' id="show-sec-0.993425121004" style="display: inline;"><a href="javascript:toggle2('sec-0.993425121004')">...more...</a></div>
        <div class='boxrstext' id="sec-0.993425121004" style="display: none;"><a href="javascript:toggle2('sec-0.993425121004')">(hide)</a>
        <div class='rstext'>rs2235067 is a SNP in the ABCB1 gene (also known as the MDR1 gene), which encodes a protein that transports certain molecules across the blood-brain barrier. SNPs in ABCB1 may thus influence the intracerebral concentrations of certain drugs and thus their efficacy or potential for adverse side effects. rs2235067 is one of 9 SNPs found within a tight linkage block (r&lt;sup&gt;2&lt;/sup&gt; &gt;= 0.8 ) such that the minor allele at any one of them predicts (with ~80%+ accuracy) that the other SNPs will also be the minor allele. The list of the 9 SNPs is shown below. When treated for depression with substrates of the protein encoded by ABCB1, carriers of one or two minor alleles at these ABCB1 SNPs have been reported to respond better than non-carriers. The antidepressant drugs that are know...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4148739'>rs4148739</a><a href='http://www.snpedia.com/index.php/rs4148739(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFEFEF;'>71.7%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>7x less likely to respond to certain antidepressants</div>
        <div class='boxshow' id="show-sec-0.119513073364" style="display: inline;"><a href="javascript:toggle2('sec-0.119513073364')">...more...</a></div>
        <div class='boxrstext' id="sec-0.119513073364" style="display: none;"><a href="javascript:toggle2('sec-0.119513073364')">(hide)</a>
        <div class='rstext'>rs4148739 is a SNP in the ABCB1 gene (also known as the MDR1 gene), which encodes a protein that transports certain molecules across the blood-brain barrier. SNPs in ABCB1 may thus influence the intracerebral concentrations of certain drugs and thus their efficacy or potential for adverse side effects. rs4148739 is one of 9 SNPs found within a tight linkage block (r&lt;sup&gt;2&lt;/sup&gt; &gt;= 0.8 ) such that the minor allele at any one of them predicts (with ~80%+ accuracy) that the other SNPs will also be the minor allele. The list of the 9 SNPs is shown below. When treated for depression with substrates of the protein encoded by ABCB1, carriers of one or two minor alleles at these ABCB1 SNPs have been reported to respond better than non-carriers. The antidepressant drugs that are know...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs17672135'>rs17672135</a><a href='http://www.snpedia.com/index.php/rs17672135(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFF1F1;'>75.0%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x risk</div>
        <div class='boxshow' id="show-sec-0.891012443781" style="display: inline;"><a href="javascript:toggle2('sec-0.891012443781')">...more...</a></div>
        <div class='boxrstext' id="sec-0.891012443781" style="display: none;"><a href="javascript:toggle2('sec-0.891012443781')">(hide)</a>
        <div class='rstext'>rs17672135 has been reported in a large study to be associated with heart disease, in particular, coronary artery disease. The risk allele (oriented to the dbSNP entry) is (T); the odds ratio associated with heterozygotes is 0.70 (CI 0.61-0.81), and for homozygotes, 1.32 (CI 0.79-2.22).  [PharmGKB:Non-Curated GWAS Results: Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls (Initial Sample Size: 1,926 cases, 2,938 controls; Replication Sample Size: (see Samani 2007); Risk Allele: rs17672135-C). This variant is associated with coronary disease.] [GWAS:Coronary disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2056202'>rs2056202</a><a href='http://www.snpedia.com/index.php/rs2056202(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFF2F2;'>77.0%</span><br/>Repute:Bad<br/>References:10<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>common but increased risk of autism</div>
        <div class='boxshow' id="show-sec-0.348556259527" style="display: inline;"><a href="javascript:toggle2('sec-0.348556259527')">...more...</a></div>
        <div class='boxrstext' id="sec-0.348556259527" style="display: none;"><a href="javascript:toggle2('sec-0.348556259527')">(hide)</a>
        <div class='rstext'>[OMIM:SOLUTE CARRIER FAMILY 25 (MITOCHONDRIAL CARRIER, ARALAR), MEMBER 12;]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2188380'>rs2188380</a><a href='http://www.snpedia.com/index.php/rs2188380(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFFFFF;'>100.0%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>slightly higher risk for gout  Over half of the people in any population have this genotype, which is reported to be at slightly higher risk (&gt;2x) for gout compared to people who have zero (A) alleles for this SNP.</div>
        <div class='boxshow' id="show-sec-0.888732728415" style="display: inline;"><a href="javascript:toggle2('sec-0.888732728415')">...more...</a></div>
        <div class='boxrstext' id="sec-0.888732728415" style="display: none;"><a href="javascript:toggle2('sec-0.888732728415')">(hide)</a>
        <div class='rstext'>gout association, based on ; note that this publication refers to risk allele (T) based on the minus strand, whereas for dbSNP and SNPedia, rs72552713 is defined as on the plus strand, so therefore the risk allele is (A).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1799895'>rs1799895</a><a href='http://www.snpedia.com/index.php/rs1799895(C;G)'>(C;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Repute:Bad<br/>References:15<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly (~1.5x) higher relative risk for ischemic heart disease due to higher extracellular SOD levels  Carrying one or two rs1799895(G) alleles appears to definitely lead to 10 fold higher concentrations of extracellular superoxide dismutase (EC-SOD), and by mechanisms that are unclear, this leads to a slight increase (~1.5x) in relative risk for ischemic heart disease. See rs1799895; it is estimated that 2-3% of most populations carry a G allele for this SNP.</div>
        <div class='boxshow' id="show-sec-0.548684732571" style="display: inline;"><a href="javascript:toggle2('sec-0.548684732571')">...more...</a></div>
        <div class='boxrstext' id="sec-0.548684732571" style="display: none;"><a href="javascript:toggle2('sec-0.548684732571')">(hide)</a>
        <div class='rstext'>rs1799895, also known as c.691C&gt;G, p.Arg231Gly and R231G, is a SNP in the SOD3 gene on chromosome 4. About 2-3% of the population carries a rs1799895(G) allele, which leads to ~10 fold higher concentrations of extracellular superoxide dismutase (EC-SOD). This in turn leads to a slightly higher (~1.5x) relative risk for ischemic heart disease. See also OMIM 185490.0001  [OMIM:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs182549'>rs182549</a><a href='http://www.snpedia.com/index.php/rs182549(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Repute:Bad<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>possibly lactose intolerant  see also gs100 if you are a caucasian who also has rs4988235(C;C) there is 77% chance of lactose intolerance * See also OMIM 601806.0001</div>
        <div class='boxshow' id="show-sec-0.560379707245" style="display: inline;"><a href="javascript:toggle2('sec-0.560379707245')">...more...</a></div>
        <div class='boxrstext' id="sec-0.560379707245" style="display: none;"><a href="javascript:toggle2('sec-0.560379707245')">(hide)</a>
        <div class='rstext'>Also known as 'G/A(-22018)' and located in the MCM6 but with influence on the lactase LCT gene, rs182549 is one of two SNPs that is associated with the primary haplotype associated with hypolactasia, more commonly known as lactose intolerance in European Caucasian populations. In these populations, the rs182549(C) allele (as named in accordance with dbSNP) is both the more common allele and the one associated with lactose intolerance. In populations of sub-Saharan Africans, though, the rs182549(C) allele is unlikely to be predictive of lactose intolerance, and other SNPs are predictive instead. * See also OMIM 601806.0002 Gender differences in genetic risk profiles for cardiovascular disease. Natriuretic peptide system gene variants are associated with ventricular dysfunction after coronar...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1065852'>rs1065852</a><a href='http://www.snpedia.com/index.php/rs1065852(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Repute:Bad<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Carrier of one CYP2D6 decreased or non-functioning variant.  Your body may have a reduced ability to process some medicines. You are a carrier for rs1065852(T;T)</div>
        <div class='boxshow' id="show-sec-0.664807270375" style="display: inline;"><a href="javascript:toggle2('sec-0.664807270375')">...more...</a></div>
        <div class='boxrstext' id="sec-0.664807270375" style="display: none;"><a href="javascript:toggle2('sec-0.664807270375')">(hide)</a>
        <div class='rstext'>The wild type (normal) allele at this SNP is (C). The (T) variant indicates the presence of a non-wild type CYP2D6 variant, but it appears in many different variants so it can not be used to determine the presence of any particular variant. The most common variants it appears in are CYP2D6*10 and CYP2D6*4, but these are not the only ones. While this is the defining mutation for CYP2D6*10, it is not possible by itself for this mutation to determine a particular variant. In addition, the CYP2D6*4 variant includes this mutation, but the defining mutation 1846G&gt;A SNP is not available in the common direct to consumer testing services. While it is not possible for this mutation to identify one particular variant, all the variants in which it appears have reduced or no CYP2D6 activity. If two ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3746444'>rs3746444</a><a href='http://www.snpedia.com/index.php/rs3746444(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Repute:Bad<br/>References:66<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~1.2x increased risk for cancer</div>
        <div class='boxshow' id="show-sec-0.32902574957" style="display: inline;"><a href="javascript:toggle2('sec-0.32902574957')">...more...</a></div>
        <div class='boxrstext' id="sec-0.32902574957" style="display: none;"><a href="javascript:toggle2('sec-0.32902574957')">(hide)</a>
        <div class='rstext'>microRNA 1,009 breast cancer cases and 1,093 controls in a population of Chinese women *rs3746444 in hsa-mir-499 A&gt;G OR 1.25; 95% CI, 1.02-1.51 Genetic variants of miRNA sequences and non-small cell lung cancer survival. MicroRNA polymorphisms: the future of pharmacogenomics, molecular epidemiology and individualized medicine. Comprehensive analysis of the impact of SNPs and CNVs on human microRNAs and their regulatory genes. SNPs in human miRNA genes affect biogenesis and function. Common genetic variants in pre-microRNAs are associated with risk of coal workers' pneumoconiosis. MicroRNA polymorphisms: a giant leap towards personalized medicine. Common genetic variants in pre-microRNAs and risk of gallbladder cancer in North Indian population. Association of pre-microRNAs genetic varia...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs17435'>rs17435</a><a href='http://www.snpedia.com/index.php/rs17435(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #F0ADF0;'>17.9%</span><br/>Repute:Bad<br/>References:4<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x increased risk for lupus</div>
        <div class='boxshow' id="show-sec-0.995756944638" style="display: inline;"><a href="javascript:toggle2('sec-0.995756944638')">...more...</a></div>
        <div class='boxrstext' id="sec-0.995756944638" style="display: none;"><a href="javascript:toggle2('sec-0.995756944638')">(hide)</a>
        <div class='rstext'>rs17435 is one of several SNPs in the methyl CpG binding protein 2 (MECP2) that have been associated with risk for systemic lupus erythromatosis (SLE). The MECP2 gene is located on the X chromosome, which may be instructive since lupus is a predominantly female disease. An initial survey of 600 Korean patients, followed by 1,000 Caucasian patients, ultimately led to a meta-analysis indicating an odds ratio for the rs17435(T) risk allele of 1.39 (CI: 1.24&#226;&#8364;&#8220;1.56, p = 1.2&#195;&#8212;10e-08). Variants within MECP2, a key transcription regulator, are associated with increased susceptibility to lupus and differential gene expression in patients with systemic lupus erythematosus. Replication of recently identified systemic lupus erythematosus genetic associations: a case-contro...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1800795'>rs1800795</a><a href='http://www.snpedia.com/index.php/rs1800795(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #F0B9F0;'>24.8%</span><br/>Repute:Bad<br/>References:181<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>more IL6; certain risks, see details  rs1800795 is a SNP in the promoter of the interleukin-6 IL6 gene, affecting the levels made of this important cytokine. In the literature, it is almost universally referred to as the IL6 '-174' polymorphism. It tends to be quite polymorphic in Caucasians, but Asian and African populations are almost monomorphic (for the (G) allele). The rs1800795(G) allele, generally associated with higher levels of IL6, has been associated with increased risk in the following studies: * The rs1800795(G) allele was significantly associated with type-2 diabetes (odds ratio 1.51, CI: 1.11-2.07, p=0.0096) in a study of 700 elderly Caucasians. * Following a kidney transplant, patients with rs1800795(G;G) genotypes have a higher risk of new-onset diabetes and higher C-react...</div>
        <div class='boxshow' id="show-sec-0.888867211227" style="display: inline;"><a href="javascript:toggle2('sec-0.888867211227')">...more...</a></div>
        <div class='boxrstext' id="sec-0.888867211227" style="display: none;"><a href="javascript:toggle2('sec-0.888867211227')">(hide)</a>
        <div class='rstext'>rs1800795 is a SNP in the promoter of the interleukin-6 IL6 gene, affecting the levels made of this important cytokine. In the literature, it is almost universally referred to as the IL6 '-174' polymorphism. It tends to be quite polymorphic in Caucasians, but Asian and African populations are almost monomorphic (for the (G) allele). It was first described in 1998, when it was shown that the rs1800795(C) allele produces less IL6 than the (G) allele, which supported the hypothesis that a protective genotype against systemic onset juvenile rheumatoid arthritis would be rs1800795(C;C), and indeed, few juvenile RA patients had that genotype. Studies on rs1800795 now include potential associations with heart disease, Kaposi's sarcoma, type-2 diabetes, stroke, obesity, Hodgkin's lymphoma, sudden ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6807362'>rs6807362</a><a href='http://www.snpedia.com/index.php/rs6807362(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #F0BFF0;'>29.2%</span><br/>Repute:Bad<br/>References:0<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased autism risk</div>
        <div class='boxshow' id="show-sec-0.983847129807" style="display: inline;"><a href="javascript:toggle2('sec-0.983847129807')">...more...</a></div>
        <div class='boxrstext' id="sec-0.983847129807" style="display: none;"><a href="javascript:toggle2('sec-0.983847129807')">(hide)</a>
        <div class='rstext'>rs6807362 is a SNP in the 5-hydroxytryptamine (serotonin) receptor 3, family member C HTR3C gene. In a study of 97 familes with at least one individual with autism, rs6807362 was significantly associated with the disorder (p = 0.0012).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1734791'>rs1734791</a><a href='http://www.snpedia.com/index.php/rs1734791(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #F0BAF0;'>74.1%</span><br/>Repute:Bad<br/>References:0<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x increased risk for lupus</div>
        <div class='boxshow' id="show-sec-0.661075441855" style="display: inline;"><a href="javascript:toggle2('sec-0.661075441855')">...more...</a></div>
        <div class='boxrstext' id="sec-0.661075441855" style="display: none;"><a href="javascript:toggle2('sec-0.661075441855')">(hide)</a>
        <div class='rstext'>rs1734791 is one of several SNPs in the methyl CpG binding protein 2 (MECP2) that have been associated with risk for systemic lupus erythromatosis (SLE). The MECP2 gene is located on the X chromosome, which may be instructive since lupus is a predominantly female disease. An initial survey of 600 Korean patients, followed by 1,000 Caucasian patients, ultimately led to a meta-analysis indicating an odds ratio for the rs1734791(A) risk allele of 1.39 (CI: 1.24&#226;&#8364;&#8220;1.57, p = 7.2&#215;10&#8722;8). Variants within MECP2, a key transcription regulator, are associated with increased susceptibility to lupus and differential gene expression in patients with systemic lupus erythematosus. A common MECP2 haplotype associates with reduced cortical surface area in humans in two independent populatio...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs187238'>rs187238</a><a href='http://www.snpedia.com/index.php/rs187238(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Repute:Bad<br/>References:28<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>hypertension increases risk 3.75x for sudden cardiac death</div>
        <div class='boxshow' id="show-sec-0.636867587696" style="display: inline;"><a href="javascript:toggle2('sec-0.636867587696')">...more...</a></div>
        <div class='boxrstext' id="sec-0.636867587696" style="display: none;"><a href="javascript:toggle2('sec-0.636867587696')">(hide)</a>
        <div class='rstext'>rs187238, also known as -137 G/C, is a SNP in the interleukin 18 IL18 gene. Based on studying autopsies of 700 men in the Helsinki Sudden Death Study, there was a significant interaction between IL-18 genotype and hypertension impacting on the risk of sudden cardiac death (SCD) due to coronary heart disease (CHD). Among rs187238(G;G) homozygotes, hypertension was a major risk factor for SCD due to CHD (adjusted odds ratio 3.75, CI:1.78-7.91, p &lt; 0.001) and hypertension also associated with larger coronary atherosclerotic plaque areas (p = 0.002) and the occurrence of complicated plaques (adjusted OR 8.38, CI: 2.39-29.33, p &lt; 0.001). Among rs187238(C) allele carriers, hypertension was not a significant risk factor for CHD-related SCD or CAD and did not associate with the development o...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs449647'>rs449647</a><a href='http://www.snpedia.com/index.php/rs449647(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Repute:Bad<br/>References:5<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>lower levels of ApoE</div>
        <div class='boxshow' id="show-sec-0.709015174243" style="display: inline;"><a href="javascript:toggle2('sec-0.709015174243')">...more...</a></div>
        <div class='boxrstext' id="sec-0.709015174243" style="display: none;"><a href="javascript:toggle2('sec-0.709015174243')">(hide)</a>
        <div class='rstext'>Significant Alzheimer's disease associations with were found for rs449647(A;A) and rs405509(G;G) genotypes (positive), and rs449647(A;T) and rs405509(T;T)</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4073582'>rs4073582</a><a href='http://www.snpedia.com/index.php/rs4073582(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.9<br/>Frequency: <span style='background-color: #FFD5D5;'>39.8%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>slightly higher risk for gout  Although ~80% of almost any group of people are of this genotype, it is associated with about twice the risk for developing gout as are people with the (A;A) genotype for this SNP.</div>
        <div class='boxshow' id="show-sec-0.527178480555" style="display: inline;"><a href="javascript:toggle2('sec-0.527178480555')">...more...</a></div>
        <div class='boxrstext' id="sec-0.527178480555" style="display: none;"><a href="javascript:toggle2('sec-0.527178480555')">(hide)</a>
        <div class='rstext'>gout association, based on</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs402710'>rs402710</a><a href='http://www.snpedia.com/index.php/rs402710(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.9<br/>Frequency: <span style='background-color: #FFDADA;'>44.4%</span><br/>Repute:Bad<br/>References:26<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Common, but higher lung cancer risk ?</div>
        <div class='boxshow' id="show-sec-0.56349924407" style="display: inline;"><a href="javascript:toggle2('sec-0.56349924407')">...more...</a></div>
        <div class='boxrstext' id="sec-0.56349924407" style="display: none;"><a href="javascript:toggle2('sec-0.56349924407')">(hide)</a>
        <div class='rstext'>[PharmGKB:Non-Curated GWAS results: Lung cancer susceptibility locus at 5p15.33. (Initial Sample Size: 2,971 cases, 3,746 controls; Replication Sample Size: 2,899 cases, 5,573 controls); (Region: 5p15.33; Reported Gene(s): TERT, CLPTM1; Risk Allele: rs402710-C); (p-value= 0.000004).This variant is associated with Lung cancer.] [OMIM:LUNG CANCER SUSCEPTIBILITY 3; LNCR3] [GWAS:Lung cancer]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4950928'>rs4950928</a><a href='http://www.snpedia.com/index.php/rs4950928(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.9<br/>Frequency: <span style='background-color: #F0C6F0;'>65.5%</span><br/>Repute:Bad<br/>References:24<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal (higher) risk of Asthma.</div>
        <div class='boxshow' id="show-sec-0.854338753635" style="display: inline;"><a href="javascript:toggle2('sec-0.854338753635')">...more...</a></div>
        <div class='boxrstext' id="sec-0.854338753635" style="display: none;"><a href="javascript:toggle2('sec-0.854338753635')">(hide)</a>
        <div class='rstext'>rs4950928 in CHI3L1 is likely to play roles in the predisposition to schizophrenia. An attempt to repeat the association between rs4950928 and risk for schizophrenia in an ethnically identical population (Japanese) as well as a new population (Han Chinese) failed to show any significant association. In contrast to previous studies, in this study of 6514 Danish adults the rs4950928(G) allele - rather than the (C) allele - was found to be associated with asthma.  [PharmGKB:Non-Curated GWAS Results: Effect of Variation in CHI31 on Serum YKL-40 Level, Risk of Asthma, and Lung Function (Initial Sample Size: 632 individuals; Replication Sample Size: 206 children; Risk Allele: rs4950928-G). This variant is associated with YKL-40 levels.] [OMIM:CHITINASE 3-LIKE 1; CHI3L1] [GWAS:YKL-40 levels]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs801114'>rs801114</a><a href='http://www.snpedia.com/index.php/rs801114(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.8<br/>Frequency: <span style='background-color: #FFDDDD;'>48.2%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal (slightly higher) risk of Basal Cell Carcinoma.  Normal risk of this kind of skin cancer.</div>
        <div class='boxshow' id="show-sec-0.122920326397" style="display: inline;"><a href="javascript:toggle2('sec-0.122920326397')">...more...</a></div>
        <div class='boxrstext' id="sec-0.122920326397" style="display: none;"><a href="javascript:toggle2('sec-0.122920326397')">(hide)</a>
        <div class='rstext'>2 SNPs located in different regions of chromosome 1 are likely to be associated with increased risk for basal cell carcinoma (BCC), the most common form of skin cancer. *each A at rs7538876 yields a 1.28x increased risk of developing BCC (p = 4.4 x 10e-12) *each G at rs801114 also yields a 1.28x increased odds of developing BCC (p = 5.9 x 10e-12) *2.68x increased risk of BCC for homozygous carriers of both SNPs This study was based on an analysis of ~2,000 Icelandic and Eastern European skin cancer patients (as well as a large number of controls). The authors estimate that approximately 1.6% of all individuals of European ancestry are homozygous for both risk alleles, and thus that percentage of the population is at 2.68x higher risk for BCC compared to individuals who are rs7538876(G;G) a...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs37973'>rs37973</a><a href='http://www.snpedia.com/index.php/rs37973(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.8<br/>Frequency: <span style='background-color: #FFE4E4;'>56.2%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Among asthmatics, 1.5x more likely to show less response to inhaled glucocorticoids</div>
        <div class='boxshow' id="show-sec-0.275118414555" style="display: inline;"><a href="javascript:toggle2('sec-0.275118414555')">...more...</a></div>
        <div class='boxrstext' id="sec-0.275118414555" style="display: none;"><a href="javascript:toggle2('sec-0.275118414555')">(hide)</a>
        <div class='rstext'>rs37973 is a SNP in the promoter region of the glucocorticoid induced transcript 1 GLCCI1 gene on chromosome 7. Based on 4 populations totaling 900+ asthma patients, the rs37973(G) allele was associated (p = 0.0007, pooled data) with lower amounts of improvement of lung function in response to inhaled glucocorticoids such as budesonide. The mean (&#177;SE) increase in forced expiratory volume in 1 second (FEV1) in treated rs37973(G;G) patients was about one third of that seen in rs37973(A;A) patients (3.2&#177;1.6% vs. 9.4&#177;1.1%). Based on dividing the response into highest vs lowest quartiles of response, the allelic odds ratio of falling into the lowest responding quartile based on an additive genetic model was calculated to be 1.52 (CI: 1.13 - 2.03). Genotype accounted for about 7% ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2003046'>rs2003046</a><a href='http://www.snpedia.com/index.php/rs2003046(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.8<br/>Frequency: <span style='background-color: #FFE7E7;'>59.8%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal (higher) risk of Male Pattern Baldness.  Discovered by 23andWe based on customer surveys, and considered preliminary research.</div>
        <div class='boxshow' id="show-sec-0.146915296777" style="display: inline;"><a href="javascript:toggle2('sec-0.146915296777')">...more...</a></div>
        <div class='boxrstext' id="sec-0.146915296777" style="display: none;"><a href="javascript:toggle2('sec-0.146915296777')">(hide)</a>
        <div class='rstext'>This is the 23andWe discovery: 'Male Pattern Baldness: Preliminary Research'. It affects appearance.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4474514'>rs4474514</a><a href='http://www.snpedia.com/index.php/rs4474514(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.8<br/>Frequency: <span style='background-color: #FFEBEB;'>65.5%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>>3x increased testicular cancer risk for men  found in 65% of caucasians but linked 3x higher risk of testicular cancer than the (G;G) form.</div>
        <div class='boxshow' id="show-sec-0.0913291541913" style="display: inline;"><a href="javascript:toggle2('sec-0.0913291541913')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0913291541913" style="display: none;"><a href="javascript:toggle2('sec-0.0913291541913')">(hide)</a>
        <div class='rstext'>Testicular germ cell tumor risk I('TGCT'; i.e. testicular cancer risk) was increased 3x per copy of the major (A) allele at rs3782179 and rs4474514 (OR = 3.08, CI: 2.29&#8211;4.13; OR = 3.07, CI = 2.29&#8211;4.13, respectively), based on a genome-wide scan of 277 primarily Caucasian TGCT cases and 919 controls. Men who have two copies of the common version of the c-KIT ligand KITLG gene have a 4.5-fold higher risk of testicular cancer than men who have two copies of the less common or minor version of the gene. SNPs from the same gene were also implicated in testicular cancer risk in a study of 730 cases and 1,435 controls from the UK and replicating associations in a further 571 cases and 1,806 controls. A second independent locus within DMRT1 is associated with testicular germ cell tumor...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1799930'>rs1799930</a><a href='http://www.snpedia.com/index.php/rs1799930(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.7<br/>Frequency: <span style='background-color: #FFD1D1;'>35.4%</span><br/>Repute:Bad<br/>References:36<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased risk of hearing loss  5.2x risk of Age Related Hearing Impairment(ARHI), also known as presbycusis. This applies to the European and Turkish, but not Finnish populations.</div>
        <div class='boxshow' id="show-sec-0.0472212325825" style="display: inline;"><a href="javascript:toggle2('sec-0.0472212325825')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0472212325825" style="display: none;"><a href="javascript:toggle2('sec-0.0472212325825')">(hide)</a>
        <div class='rstext'>rs1799930 is a SNP in the NAT2 gene, potentially encoding a variant detoxifying protein known as an N-acetyltransferase, but which NAT2 variant depends on which other NAT2 SNPs were also inherited. See the discussion of the NAT2 gene for a more complete explanation. Also known as G590A. The risk allele for this SNP is rs1799930(A).  [PharmGKB:Curated in vitro study; PK: 22-fold reduction in Vmax vs wild type] [OMIM:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4132601'>rs4132601</a><a href='http://www.snpedia.com/index.php/rs4132601(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.7<br/>Frequency: <span style='background-color: #FFDBDB;'>46.0%</span><br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>somewhat (1.7x) increased risk for acute lymphoblastic leukemia  see discussion at rs4132601</div>
        <div class='boxshow' id="show-sec-0.874996858511" style="display: inline;"><a href="javascript:toggle2('sec-0.874996858511')">...more...</a></div>
        <div class='boxrstext' id="sec-0.874996858511" style="display: none;"><a href="javascript:toggle2('sec-0.874996858511')">(hide)</a>
        <div class='rstext'>23andMe blog Acute lymphoblastic leukemia *each rs4132601(G) 1.69x odds of ALL *each rs7089424(G) 1.65x odds of ALL *each rs2239633(G) 1.34x odds of ALL A meta-analysis published in 2014 based on a total of 15 case-control studies with 8333 cases and 36036 controls concluded that rs4132601 was associated with increased ALL risk in Caucasians and Hispanics but not among Asians.  [OMIM:?] [GWAS:Acute lymphoblastic leukemia (childhood)]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs8055236'>rs8055236</a><a href='http://www.snpedia.com/index.php/rs8055236(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.7<br/>Frequency: <span style='background-color: #FFE9E9;'>63.7%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>common, but 2.2x higher risk for heart disease</div>
        <div class='boxshow' id="show-sec-0.774460576405" style="display: inline;"><a href="javascript:toggle2('sec-0.774460576405')">...more...</a></div>
        <div class='boxrstext' id="sec-0.774460576405" style="display: none;"><a href="javascript:toggle2('sec-0.774460576405')">(hide)</a>
        <div class='rstext'>rs8055236 has been reported in a large study to be associated with heart disease, in particular, coronary artery disease. The risk allele (oriented to the dbSNP entry) is (G); the odds ratio associated with heterozygotes is 1.91 (CI 1.33-2.74), and for homozygotes, 2.23 (CI 1.56-3.17).  [PharmGKB:Non-Curated GWAS Results: Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls (Initial Sample Size: 1,926 cases, 2,938 controls; Replication Sample Size: (see Samani 2007); Risk Allele: rs8055236-G). This variant is associated with coronary disease.] [GWAS:Coronary disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3775948'>rs3775948</a><a href='http://www.snpedia.com/index.php/rs3775948(C;G)'>(C;G)</a></div>
        <div class='boxmag'>Magnitude: 1.6<br/>Frequency: <span style='background-color: #FFDDDD;'>47.7%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>slightly higher risk for gout</div>
        <div class='boxshow' id="show-sec-0.449209139056" style="display: inline;"><a href="javascript:toggle2('sec-0.449209139056')">...more...</a></div>
        <div class='boxrstext' id="sec-0.449209139056" style="display: none;"><a href="javascript:toggle2('sec-0.449209139056')">(hide)</a>
        <div class='rstext'>gout association, based on note that due to the possibility of ambiguous flipping between the C/G alleles of this SNP, care should be taken in interpreting alleles  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11523871'>rs11523871</a><a href='http://www.snpedia.com/index.php/rs11523871(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 1.6<br/>Frequency: <span style='background-color: #FFDEDE;'>49.2%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.6x increased breast cancer risk for women over 60</div>
        <div class='boxshow' id="show-sec-0.500400163281" style="display: inline;"><a href="javascript:toggle2('sec-0.500400163281')">...more...</a></div>
        <div class='boxrstext' id="sec-0.500400163281" style="display: none;"><a href="javascript:toggle2('sec-0.500400163281')">(hide)</a>
        <div class='rstext'>rs11523871, also known as 124A&gt;C or Thr42Pro, is a SNP in the Deleted in Malignant Brain Tumors 1 DMBT1 gene. For more information about increased breast cancer risk for this SNP, see the SNPedia entry for rs2981745, to which it is tightly linked.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2981745'>rs2981745</a><a href='http://www.snpedia.com/index.php/rs2981745(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.6<br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.6x increased risk for breast cancer in females over 60</div>
        <div class='boxshow' id="show-sec-0.724807355279" style="display: inline;"><a href="javascript:toggle2('sec-0.724807355279')">...more...</a></div>
        <div class='boxrstext' id="sec-0.724807355279" style="display: none;"><a href="javascript:toggle2('sec-0.724807355279')">(hide)</a>
        <div class='rstext'>rs2981745, also known as -93C&gt;T, is a SNP in a gene known as Deleted in Malignant Brain Tumors 1 DMBT1. The (overly descriptive?) name refers to the hypothesis that this gene has a role as a tumor suppressor. A study of 1,195 BRCA1/2 mutation negative German breast cancer families and 1,466 unrelated German controls concluded that rs2981745(T) carriers over the age of 60 are at increased risk for breast cancer, with an odds ratio of 1.66 (CI: 1.21-2.29, p=0.0017). rs2981745 was also said to be tightly linked to rs11523871, which showed the same odds ratios in this study.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs16973225'>rs16973225</a><a href='http://www.snpedia.com/index.php/rs16973225(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FF7575;'>1.6%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>no reduction in colorectal cancer risk from taking aspirin  see discussion at rs16973225</div>
        <div class='boxshow' id="show-sec-0.821724525711" style="display: inline;"><a href="javascript:toggle2('sec-0.821724525711')">...more...</a></div>
        <div class='boxrstext' id="sec-0.821724525711" style="display: none;"><a href="javascript:toggle2('sec-0.821724525711')">(hide)</a>
        <div class='rstext'>rs16973225 is a SNP on chromosome 15q25.2 near the IL16 gene. In a long-term study of ~17,000 individuals and aspirin or NSAID use, the few (9%) with either an (A;C) or (C;C) genotype showed no risk reduction for colorectal cancer compared with (A;A) individuals among those taking anti-inflammatory drugs, who did benefit (prevalence, 28% vs 38%; (A;A) odds ratio 0.66 when taking aspirin/NSAID, CI: 0.62-0.71, p&#8201;=&#8201;1.9&#8201;&#215;&#8201;10e&#8722;30). If this data is robust and all other factors are equal, this indicates that most people will lower their colorectal cancer risk by taking aspirin or NSAIDs, since most people have rs16973225(A;A) genotypes. However, the few people who are rs16973225(A;C) or are rs16973225(C;C) will not change their risk one way or the other by taking aspirin. This same study reached ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10859871'>rs10859871</a><a href='http://www.snpedia.com/index.php/rs10859871(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFA3A3;'>9.7%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slight (~1.4x) increase in endometriosis risk</div>
        <div class='boxshow' id="show-sec-0.61602495922" style="display: inline;"><a href="javascript:toggle2('sec-0.61602495922')">...more...</a></div>
        <div class='boxrstext' id="sec-0.61602495922" style="display: none;"><a href="javascript:toggle2('sec-0.61602495922')">(hide)</a>
        <div class='rstext'>==='''Summary of trait'''=== Endometriosis is a disorder in which the endometrium, which lines the inside of a woman&#8217;s uterus grows outside the uterus (etopic). It affects 6% - 10% of women of reproductive age. In a patient with endometriosis, the endometrial tissue behaves normally, whereby during a menstrual cycle, it would thicken, break down and bleed out. Being outside the uterus, the broken down endometrial tissue cannot leave the body. This blood will irritate surrounding tissue, causing scar tissue and adhesions and may potentially bind organs together. Endometriosis may involve ovaries, bowel or tissue lining pelvis. Cysts (or endometriomas) may form if endometriosis involves ovaries . The exact cause of endometriosis is unknown, but the most widely accepted theory is the retrogra...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1801020'>rs1801020</a><a href='http://www.snpedia.com/index.php/rs1801020(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFBDBD;'>21.2%</span><br/>Repute:Bad<br/>References:12<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.31x increased risk of heart disease</div>
        <div class='boxshow' id="show-sec-0.313902220253" style="display: inline;"><a href="javascript:toggle2('sec-0.313902220253')">...more...</a></div>
        <div class='boxrstext' id="sec-0.313902220253" style="display: none;"><a href="javascript:toggle2('sec-0.313902220253')">(hide)</a>
        <div class='rstext'>Affects heart disease risk for both men and women, based on a study of two Finnish population cohorts (HR = 1.31 (1.08-1.60) for CVD, uncorrected p = 0.006 multiplicative model). A meta-analysis of publications through July 2009, totaling 4,386 cases (vs 40,089 controls) concluded that apart from a very weak association with myocardial infarction for the rs1801020 (T;T) + (C;T) compared with (C;C) genotype, (odds ratio 1.13, CI: 1.00 - 1.27), the evidence for an association between F12 -4C&gt;T and venous thromboembolism and myocardial infarction was weak.  [OMIM:COAGULATION FACTOR XII; F12]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4626664'>rs4626664</a><a href='http://www.snpedia.com/index.php/rs4626664(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFC1C1;'>23.6%</span><br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.44x increased risk of developing restless legs syndrome</div>
        <div class='boxshow' id="show-sec-0.508278093185" style="display: inline;"><a href="javascript:toggle2('sec-0.508278093185')">...more...</a></div>
        <div class='boxrstext' id="sec-0.508278093185" style="display: none;"><a href="javascript:toggle2('sec-0.508278093185')">(hide)</a>
        <div class='rstext'>rs4626664, a SNP in the motor neuron PTPRD gene, has been linked to restless legs syndrome based on a study of several European populations totaling ~2,500 patients. The odds ratio associated with the rs4626664(A) risk allele is 1.44 (CI: 1.31-1.59, adjusted p = 5 x 10e-10). Note that a nearby SNP, rs1975197, was also shown to independently confer risk for restless legs syndrome in this same study. Restless Legs Syndrome: Preliminary Research  [PharmGKB:Curated GWAS Results: PTPRD (protein tyrosine phosphatase receptor type delta) is associated with restless legs syndrome (Initial Sample Size: 628 cases, 1,644 controls; Replication Sample Size: 1,835 cases, 3,111 controls; Risk Allele: rs4626664-A). This variant is associated with Restless legs syndrome.] [OMIM:RESTLESS LEGS SYNDROME, SUSC...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4656461'>rs4656461</a><a href='http://www.snpedia.com/index.php/rs4656461(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFC2C2;'>23.9%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.5x increased risk for open angle glaucoma</div>
        <div class='boxshow' id="show-sec-0.785783204361" style="display: inline;"><a href="javascript:toggle2('sec-0.785783204361')">...more...</a></div>
        <div class='boxrstext' id="sec-0.785783204361" style="display: none;"><a href="javascript:toggle2('sec-0.785783204361')">(hide)</a>
        <div class='rstext'>rs4656461 is a SNP near the TMCO1 gene. A (GWA) study for open-angle glaucoma (OAG) blindness ultimately based on ~1500 cases concluded that the rather rare rs4656461(G) allele was statistically associated with the condition, with a per allele odds ratio of 1.5 (CI: 1.35&#8211;1.68, p = 6 &#215; 10e?14).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2697962'>rs2697962</a><a href='http://www.snpedia.com/index.php/rs2697962(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFC3C3;'>24.8%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk of developing Parkinson's Disease</div>
        <div class='boxshow' id="show-sec-0.619671913649" style="display: inline;"><a href="javascript:toggle2('sec-0.619671913649')">...more...</a></div>
        <div class='boxrstext' id="sec-0.619671913649" style="display: none;"><a href="javascript:toggle2('sec-0.619671913649')">(hide)</a>
        <div class='rstext'>rs2697962 is in linkage disequilibrium with a polymorphism that increases susceptibility to Parkinson's disease 1.67 times for carriers of the A allele</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6710341'>rs6710341</a><a href='http://www.snpedia.com/index.php/rs6710341(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFC4C4;'>25.9%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk of developing restless legs syndrome</div>
        <div class='boxshow' id="show-sec-0.998011538338" style="display: inline;"><a href="javascript:toggle2('sec-0.998011538338')">...more...</a></div>
        <div class='boxrstext' id="sec-0.998011538338" style="display: none;"><a href="javascript:toggle2('sec-0.998011538338')">(hide)</a>
        <div class='rstext'>rs6710341, a SNP located in the MEIS1 gene, has been linked to restless legs syndrome, a common sleep disorder, with an overall odds ratio of 0.96 (CI: 0.77-1.17) for the (G) risk allele. The association from this region that gives the highest association to restless legs syndrome, however, is a haplotype consisting of the rs6710341(A) and rs12469063(G) alleles. This haplotype gives rise to an odds ratio of 2.75 (CI: 2.23-3.41).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs464049'>rs464049</a><a href='http://www.snpedia.com/index.php/rs464049(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFC8C8;'>28.3%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk of schizophrenia in limited study</div>
        <div class='boxshow' id="show-sec-0.406234246609" style="display: inline;"><a href="javascript:toggle2('sec-0.406234246609')">...more...</a></div>
        <div class='boxrstext' id="sec-0.406234246609" style="display: none;"><a href="javascript:toggle2('sec-0.406234246609')">(hide)</a>
        <div class='rstext'>Associated with schizophrenia in a study of Croatians. Due to the limited number and diversity of the studied population, more study is needed to confirm any link. The T allele was associated with an increased risk of schizophrenia. A network of dopaminergic gene variations implicated as risk factors for schizophrenia. Family based association study of pediatric bipolar disorder and the dopamine transporter gene (SLC6A3). Genome-wide and candidate gene association study of cigarette smoking behaviors. Financial and psychological risk attitudes associated with two single nucleotide polymorphisms in the nicotine receptor (CHRNA4) gene. Two loci control tuberculin skin test reactivity in an area hyperendemic for tuberculosis.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3125923'>rs3125923</a><a href='http://www.snpedia.com/index.php/rs3125923(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFC9C9;'>28.8%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>slightly (50%) increased risk for Taxane-induced peripheral neuropathy  Slightly (50%) increased risk for peripheral neuropathy in patients undergoing chemotherapeutic treatment involving Taxane</div>
        <div class='boxshow' id="show-sec-0.677519268352" style="display: inline;"><a href="javascript:toggle2('sec-0.677519268352')">...more...</a></div>
        <div class='boxrstext' id="sec-0.677519268352" style="display: none;"><a href="javascript:toggle2('sec-0.677519268352')">(hide)</a>
        <div class='rstext'></div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1975197'>rs1975197</a><a href='http://www.snpedia.com/index.php/rs1975197(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFC9C9;'>29.2%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk of developing restless legs syndrome</div>
        <div class='boxshow' id="show-sec-0.795125328365" style="display: inline;"><a href="javascript:toggle2('sec-0.795125328365')">...more...</a></div>
        <div class='boxrstext' id="sec-0.795125328365" style="display: none;"><a href="javascript:toggle2('sec-0.795125328365')">(hide)</a>
        <div class='rstext'>rs1975197, a SNP in the motor neuron PTPRD gene, has been linked to restless legs syndrome based on a study of several European populations totaling ~2,500 patients. The odds ratio associated with the rs1975197(T) risk allele is 1.31 (CI: 1.20-1.44, adjusted p = 5 x10e-9). Note that a nearby SNP, rs4626664, was also shown to independently confer risk for restless legs syndrome in this same study. Restless Legs Syndrome: Preliminary Research  [PharmGKB:Curated GWAS Results: PTPRD (protein tyrosine phosphatase receptor type delta) is associated with restless legs syndrome (Initial Sample Size: 628 cases, 1,644 controls; Replication Sample Size: 1,835 cases, 3,111 controls; Risk Allele: rs1975197-T). This variant is associated with Restless legs syndrome.] [OMIM:RESTLESS LEGS SYNDROME, SUSCEP...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2881766'>rs2881766</a><a href='http://www.snpedia.com/index.php/rs2881766(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFCBCB;'>31.0%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk for pregnancy-induced hypertension</div>
        <div class='boxshow' id="show-sec-0.522235116172" style="display: inline;"><a href="javascript:toggle2('sec-0.522235116172')">...more...</a></div>
        <div class='boxrstext' id="sec-0.522235116172" style="display: none;"><a href="javascript:toggle2('sec-0.522235116172')">(hide)</a>
        <div class='rstext'>rs2881766 is a SNP in the promoter region of the estrogen receptor alpha ESR1 gene. Carriers of rs2881766(T) alleles are at somewhat (slightly) higher risk for pregnancy-induced hypertension.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10784502'>rs10784502</a><a href='http://www.snpedia.com/index.php/rs10784502(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFCCCC;'>31.9%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>less intracranial volume?</div>
        <div class='boxshow' id="show-sec-0.299433111018" style="display: inline;"><a href="javascript:toggle2('sec-0.299433111018')">...more...</a></div>
        <div class='boxrstext' id="sec-0.299433111018" style="display: none;"><a href="javascript:toggle2('sec-0.299433111018')">(hide)</a>
        <div class='rstext'>associated with intracranial volume (12q14.3; N = 15,782; P = 1.12 &#215; 10&#8722;12) http://www.genomesunzipped.org/2012/04/another-iq-gene-new-methods-old-flaws.php (C;C) has higher cranial capacity (about 2 teaspoons) and over 2% higher IQ. (C;T) is half this positive effect. Intelligence</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1994090'>rs1994090</a><a href='http://www.snpedia.com/index.php/rs1994090(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFCDCD;'>32.7%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk of developing Parkinson's Disease</div>
        <div class='boxshow' id="show-sec-0.965753170313" style="display: inline;"><a href="javascript:toggle2('sec-0.965753170313')">...more...</a></div>
        <div class='boxrstext' id="sec-0.965753170313" style="display: none;"><a href="javascript:toggle2('sec-0.965753170313')">(hide)</a>
        <div class='rstext'>[GWAS:Parkinson's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs401681'>rs401681</a><a href='http://www.snpedia.com/index.php/rs401681(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFCDCD;'>32.7%</span><br/>Repute:Bad<br/>References:47<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~1.2x increased risk for several types of cancer</div>
        <div class='boxshow' id="show-sec-0.466793142637" style="display: inline;"><a href="javascript:toggle2('sec-0.466793142637')">...more...</a></div>
        <div class='boxrstext' id="sec-0.466793142637" style="display: none;"><a href="javascript:toggle2('sec-0.466793142637')">(hide)</a>
        <div class='rstext'>rs401681 is a SNP in the CLPTM1-like CLPTM1 gene. The following studies span several types of cancer, and over 30,000 cases and 45,000 controls. deCODE press release reports rs401681 linked to basal cell carcinoma and many cancers for individuals with shorter telomeres, whereas those with long telomeres are at greater risk of melanoma. Neighboring rs2736098 also increases risk of cancer by affecting telomerase reverse transcriptase. The risk of these different cancers conferred by these two SNPs appears to be independent. 23andMe blog rs2736098 and rs401681 linked to many cancers, including susceptibility to pancreatic cancer However, a very large study (encompassing thousands of cases and controls) found no association between rs401681 and either (1) mean telomere length, or, (2) breast c...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs309375'>rs309375</a><a href='http://www.snpedia.com/index.php/rs309375(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFCECE;'>33.6%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Larger mosquito bites</div>
        <div class='boxshow' id="show-sec-0.309387056262" style="display: inline;"><a href="javascript:toggle2('sec-0.309387056262')">...more...</a></div>
        <div class='boxrstext' id="sec-0.309387056262" style="display: none;"><a href="javascript:toggle2('sec-0.309387056262')">(hide)</a>
        <div class='rstext'>http://blog.23andme.com/23andme-research/bug-bite-genetics/ mosquito bite size</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6908425'>rs6908425</a><a href='http://www.snpedia.com/index.php/rs6908425(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFCECE;'>33.6%</span><br/>Repute:Bad<br/>References:13<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.63x increased risk of developing Crohn's disease</div>
        <div class='boxshow' id="show-sec-0.66312932216" style="display: inline;"><a href="javascript:toggle2('sec-0.66312932216')">...more...</a></div>
        <div class='boxrstext' id="sec-0.66312932216" style="display: none;"><a href="javascript:toggle2('sec-0.66312932216')">(hide)</a>
        <div class='rstext'>rs6908425 has been reported in a large study to be associated with Crohn's disease. The risk allele (oriented to the dbSNP entry) is (C); the odds ratio associated with heterozygotes is 1.63 (CI 1.38-2.25), and for homozygotes, 1.95 (CI 1.43-2.67).  [OMIM:?] [GWAS:Crohn's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4506565'>rs4506565</a><a href='http://www.snpedia.com/index.php/rs4506565(A;T)'>(A;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFD0D0;'>34.5%</span><br/>Repute:Bad<br/>References:24<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x increased risk for type-2 diabetes</div>
        <div class='boxshow' id="show-sec-0.314493993429" style="display: inline;"><a href="javascript:toggle2('sec-0.314493993429')">...more...</a></div>
        <div class='boxrstext' id="sec-0.314493993429" style="display: none;"><a href="javascript:toggle2('sec-0.314493993429')">(hide)</a>
        <div class='rstext'>rs4506565 has been reported in a large study to be associated with type-2 diabetes. The risk allele (oriented to the dbSNP entry) is (T); the odds ratio associated with heterozygotes is 1.36 (CI 1.2-1.54), and for homozygotes, 1.88 (CI 1.56-2.27). Note: this is one of two SNPs within the TCF7L2 gene that have been reported to be associated with type-2 diabetes, the other being rs7903146. They have approximately equal power to estimate risk for type-2 diabetes, and the results from one correlate 92% of the time with the other.  [PharmGKB:Curated Phenotype: In a meta-analysis of 21 GWAS cohorts, this SNP was found to be associated with fasting glucose level. Study size: 46,181. Significance metric(s): p = 1.2 x 10(-8). Study population/ethnicity: Non-diabetic Individuals of European descent....</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3745516'>rs3745516</a><a href='http://www.snpedia.com/index.php/rs3745516(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFD2D2;'>36.3%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk of developing primary biliary cirrhosis</div>
        <div class='boxshow' id="show-sec-0.839393679235" style="display: inline;"><a href="javascript:toggle2('sec-0.839393679235')">...more...</a></div>
        <div class='boxrstext' id="sec-0.839393679235" style="display: none;"><a href="javascript:toggle2('sec-0.839393679235')">(hide)</a>
        <div class='rstext'>Each A allele of this SNP in the SPIB gene is associated with 1.46 times higher odds of primary biliary cirrhosis.  [GWAS:Primary biliary cirrhosis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs199533'>rs199533</a><a href='http://www.snpedia.com/index.php/rs199533(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFD3D3;'>37.2%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk of developing Parkinson's Disease</div>
        <div class='boxshow' id="show-sec-0.206929330948" style="display: inline;"><a href="javascript:toggle2('sec-0.206929330948')">...more...</a></div>
        <div class='boxrstext' id="sec-0.206929330948" style="display: none;"><a href="javascript:toggle2('sec-0.206929330948')">(hide)</a>
        <div class='rstext'>[GWAS:Parkinson's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10509305'>rs10509305</a><a href='http://www.snpedia.com/index.php/rs10509305(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFD4D4;'>38.9%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>carrier for pre-eclampsia associated variant  See the rs10509305 SNPedia page for discussion. This genotype represents a carrier for a variant that, if inherited in two copies by a fetus, is associated with causing pre-eclampsia in the mother.</div>
        <div class='boxshow' id="show-sec-0.94298489668" style="display: inline;"><a href="javascript:toggle2('sec-0.94298489668')">...more...</a></div>
        <div class='boxrstext' id="sec-0.94298489668" style="display: none;"><a href="javascript:toggle2('sec-0.94298489668')">(hide)</a>
        <div class='rstext'>rs10509305, also known as Glu608Asp or E608D, is a variant in the storkhead box 1 STOX1 gene. In and subsequent publications, it is reported that when fetuses are rs10509305(C;C), their mothers are likely to have pre-eclampsia and possibly also pregnancy-induced hypertension.  [OMIM:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs12469063'>rs12469063</a><a href='http://www.snpedia.com/index.php/rs12469063(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFD4D4;'>38.9%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk of developing restless legs syndrome</div>
        <div class='boxshow' id="show-sec-0.298211299625" style="display: inline;"><a href="javascript:toggle2('sec-0.298211299625')">...more...</a></div>
        <div class='boxrstext' id="sec-0.298211299625" style="display: none;"><a href="javascript:toggle2('sec-0.298211299625')">(hide)</a>
        <div class='rstext'>rs12469063, a SNP located in the MEIS1 gene, has been linked to restless legs syndrome, a common sleep disorder, with an overall odds ratio of 1.78 (CI: 1.52-2.10) for the (G) risk allele. The association from this region that gives the highest association to restless legs syndrome, however, is a haplotype consisting of the rs6710341(A) and rs12469063(G) alleles. This haplotype gives rise to an odds ratio of 2.75 (CI: 2.23-3.41). MEIS1 intronic risk haplotype associated with restless legs syndrome affects its mRNA and protein expression levels. Replication of restless legs syndrome loci in three European populations. Genotyping sleep disorders patients.  [OMIM:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2300478'>rs2300478</a><a href='http://www.snpedia.com/index.php/rs2300478(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFD6D6;'>40.2%</span><br/>Repute:Bad<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.7x risk for developing restless legs syndrome  Variants in MEIS1 rs2300478, BTBD9 rs9357271, and MAP2K5/SKOR1 rs1026732 confer a significant risk of RLS in a US population. Association studies of variants in MEIS1, BTBD9, and MAP2K5/SKOR1 with restless legs syndrome in a US population.</div>
        <div class='boxshow' id="show-sec-0.767730339443" style="display: inline;"><a href="javascript:toggle2('sec-0.767730339443')">...more...</a></div>
        <div class='boxrstext' id="sec-0.767730339443" style="display: none;"><a href="javascript:toggle2('sec-0.767730339443')">(hide)</a>
        <div class='rstext'>rs2300478, a SNP located in the MEIS1 gene, has been linked to restless legs syndrome, a common sleep disorder, with an overall odds ratio of 1.78 (CI: 1.52-2.09) for the (G) risk allele . The association between this SNP and RLS has been replicated in three European populations, within family lines but not in sporadic cases . The highest association to restless legs syndrome is a haplotype consisting of the rs6710341(A) and rs12469063(G) alleles. This haplotype gives rise to an odds ratio of 2.75 (CI: 2.23-3.41) . Variants in MEIS1 rs2300478, BTBD9 rs9357271, and MAP2K5/SKOR1 rs1026732 confer a significant risk of RLS in a US population. In another study, rs2300478 was not associated with attention deficit hyperactivity disorder (ADHD). Association studies of variants in MEIS1, BTBD9, and...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7850258'>rs7850258</a><a href='http://www.snpedia.com/index.php/rs7850258(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFD6D6;'>40.7%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly higher odds of developing primary hypothyroidism.</div>
        <div class='boxshow' id="show-sec-0.296454470556" style="display: inline;"><a href="javascript:toggle2('sec-0.296454470556')">...more...</a></div>
        <div class='boxrstext' id="sec-0.296454470556" style="display: none;"><a href="javascript:toggle2('sec-0.296454470556')">(hide)</a>
        <div class='rstext'>rs7850258 is located on chromosome 9 and is 67 kb from the nearest gene, ''FOXE1''. ''FOXE1'', also known as thyroid transcription factor 2, is an intronless gene with a crucial role in thyroid morphogenesis. This SNP is associated with primary hypothyroidism. Primary hypothyroidism is the most common thyroid disorder, affecting 1-5% of the population, and is characterized by deficiencies of thyroid hormones T3 (triiodothyronine) and T4 (thyroxine). Using electronic medical record-linked DNA biobanks, a study with 1,317 cases and 5,053 controls of European American ancestry identified rs7850258 to have the strongest association with hypothyroidism. The odds ratio for the minor allele A was 0.74 (p-value = 3.96 x 10-9). This SNP was in strong pair-wise linkage disequilibrium (r2 &gt; 0.98) ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6604026'>rs6604026</a><a href='http://www.snpedia.com/index.php/rs6604026(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFD8D8;'>42.5%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.15x risk of multiple sclerosis  Slightly higher risk of multiple sclerosis</div>
        <div class='boxshow' id="show-sec-0.97304428344" style="display: inline;"><a href="javascript:toggle2('sec-0.97304428344')">...more...</a></div>
        <div class='boxrstext' id="sec-0.97304428344" style="display: none;"><a href="javascript:toggle2('sec-0.97304428344')">(hide)</a>
        <div class='rstext'>rs6604026 has been reported in a large study to be associated with multiple sclerosis. The risk allele (oriented to the dbSNP entry) is (C); the odds ratio associated with this allele is 1.15 (CI 1.08-1.22).  [PharmGKB:Curated GWAS Results: Risk alleles for multiple sclerosis identified by a genomewide study (Initial Sample Size: 931 trios, 2,431 controls; Replication Sample Size: 609 trios, 2,322 cases, 2,987 controls; Risk Allele: rs6604026-C).] [GWAS:Multiple sclerosis] [GWAS:Multiple sclerosis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7089424'>rs7089424</a><a href='http://www.snpedia.com/index.php/rs7089424(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFD8D8;'>42.9%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>somewhat (~2x) increased risk for acute lymphoblastic leukemia  see references at rs7089424, including meta-analyses</div>
        <div class='boxshow' id="show-sec-0.691912302337" style="display: inline;"><a href="javascript:toggle2('sec-0.691912302337')">...more...</a></div>
        <div class='boxrstext' id="sec-0.691912302337" style="display: none;"><a href="javascript:toggle2('sec-0.691912302337')">(hide)</a>
        <div class='rstext'>Replication analysis confirms the association of ARID5B with childhood B-cell acute lymphoblastic leukemia. Variation at 7p12.2 and 10q21.2 influences childhood acute lymphoblastic leukemia risk in the Thai population and may contribute to racial differences in leukemia incidence.  [OMIM:?] [GWAS:Acute lymphoblastic leukemia (childhood)]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs700651'>rs700651</a><a href='http://www.snpedia.com/index.php/rs700651(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFD9D9;'>43.2%</span><br/>Repute:Bad<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~1.18x increased risk of aneurysm</div>
        <div class='boxshow' id="show-sec-0.627773978476" style="display: inline;"><a href="javascript:toggle2('sec-0.627773978476')">...more...</a></div>
        <div class='boxrstext' id="sec-0.627773978476" style="display: none;"><a href="javascript:toggle2('sec-0.627773978476')">(hide)</a>
        <div class='rstext'>23andMe blog aneurysm *rs700651 2 G 1.18x risk 1.56x risk *rs10958409 8 A 1.37x risk 1.79x risk *rs1333040 9 T 1.29x risk 1.67x risk Increased risk of intracranial (brain aneurysm) with certain SNPs. rs700651(G;G) associated with 1.56x risk of developing an intracranial aneurysm (IA). Heterozygote risk is 1.18x. Other SNPs associated with increased risk of developing IA include rs1333040 and rs10958409. rs10958409 is located on chromosome 9; rs1333040 is located on chromosome 8; rs700651 is located on chromosome 2. Unfortunately, the first sign of an intracranial aneurysm is rupture. Catastrophic bleeding and death may result. Specific knowledge of risk may influence screening decision. More information: Intracranial aneurysm screening: indications and advice for practice. .  [PharmGKB:Non...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9642880'>rs9642880</a><a href='http://www.snpedia.com/index.php/rs9642880(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDADA;'>44.2%</span><br/>Repute:Bad<br/>References:14<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.2x increased bladder cancer risk</div>
        <div class='boxshow' id="show-sec-0.137340808568" style="display: inline;"><a href="javascript:toggle2('sec-0.137340808568')">...more...</a></div>
        <div class='boxrstext' id="sec-0.137340808568" style="display: none;"><a href="javascript:toggle2('sec-0.137340808568')">(hide)</a>
        <div class='rstext'>rs9642880 is a SNP located in chromosome region 8q24, a region that has been linked to prostate, breast, and colon cancer. A study ultimately totaling over 4,000 bladder cancer patients of European ancestry (and 40,000 controls) concluded that individuals carrying two copies of rs9642880(T) were at 1.49x increased risk (allele-specific odds ratio = 1.22, p = 9 x 10e-12) for the disease. This SNP was not associated, however, with risk for prostate, breast, or colon cancer. Another study in Chinese populations found similar results in regard to bladder cancer. rs9642880(T;T) estimated risk of developing 1.49 times that of noncarriers. 1,803 urinary bladder cancer cases and 34,336 controls. No association with prostate, colorectal and breast cancers. In contrast to GSTM1 SNPs, the rs9642880(T...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6435862'>rs6435862</a><a href='http://www.snpedia.com/index.php/rs6435862(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDADA;'>44.6%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.7x higher risk of aggressive neuroblastoma</div>
        <div class='boxshow' id="show-sec-0.599185776011" style="display: inline;"><a href="javascript:toggle2('sec-0.599185776011')">...more...</a></div>
        <div class='boxrstext' id="sec-0.599185776011" style="display: none;"><a href="javascript:toggle2('sec-0.599185776011')">(hide)</a>
        <div class='rstext'>Each G allele has been associated with an allelic odds ratio of 1.68 (p = 8.65 x 10(-18)) for risk of aggressive neuroblastoma.  [OMIM:NEUROBLASTOMA]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10883365'>rs10883365</a><a href='http://www.snpedia.com/index.php/rs10883365(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDADA;'>45.1%</span><br/>Repute:Bad<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.2x increased risk for developing Crohn's disease</div>
        <div class='boxshow' id="show-sec-0.89183393115" style="display: inline;"><a href="javascript:toggle2('sec-0.89183393115')">...more...</a></div>
        <div class='boxrstext' id="sec-0.89183393115" style="display: none;"><a href="javascript:toggle2('sec-0.89183393115')">(hide)</a>
        <div class='rstext'>rs10883365 has been reported in a large study to be associated with Crohn's disease. The risk allele (oriented to the dbSNP entry) is (G); the odds ratio associated with heterozygotes is 1.2 (CI 1.03-1.39), and for homozygotes, 1.62 (CI 1.37-1.92). The association between rs10883365 and Crohn's disease was replicated in a Japanese population. Pharmacogenetics: data, concepts and tools to improve drug discovery and drug treatment. Genetic determinants of ulcerative colitis include the ECM1 locus and five loci implicated in Crohn's disease. Worldwide population differentiation at disease-associated SNPs. Gene variants influencing measures of inflammation or predisposing to autoimmune and inflammatory diseases are not associated with the risk of type 2 diabetes. Unbiased estimation of odds ra...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1801274'>rs1801274</a><a href='http://www.snpedia.com/index.php/rs1801274(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDADA;'>45.1%</span><br/>Repute:Bad<br/>References:57<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>complex; generally greater risk for cancer progression</div>
        <div class='boxshow' id="show-sec-0.944126768467" style="display: inline;"><a href="javascript:toggle2('sec-0.944126768467')">...more...</a></div>
        <div class='boxrstext' id="sec-0.944126768467" style="display: none;"><a href="javascript:toggle2('sec-0.944126768467')">(hide)</a>
        <div class='rstext'>rs1801274 is a SNP in the Fc fragment of IgG, low affinity IIa, receptor (CD32) FCGR2A gene. rs1801274(C) encodes the arginine (R) allele, with the (T) allele encoding the variant histidine (H). The (H) isoform is considered high-binding to IgG2 and IgG3, while the (R) isoform is considered low-binding. This SNP is known in the literature by many names, including A519C and H131R. What's the importance of this? FcgR isoforms expressed on immune system cells have been linked to the pathogenic consequences triggered by autoantibodies or immune complexes in autoimmune diseases such as rheumatoid arthritis (RA) and systemic lupus erythematosus (SLE), as well as to the efficacy of some immunotherapeutic treatments such as rituximab. Many studies have been published about this FCGR2A SNP, roughly...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6498169'>rs6498169</a><a href='http://www.snpedia.com/index.php/rs6498169(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDBDB;'>45.5%</span><br/>Repute:Bad<br/>References:18<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.14x risk of multiple sclerosis</div>
        <div class='boxshow' id="show-sec-0.461066703837" style="display: inline;"><a href="javascript:toggle2('sec-0.461066703837')">...more...</a></div>
        <div class='boxrstext' id="sec-0.461066703837" style="display: none;"><a href="javascript:toggle2('sec-0.461066703837')">(hide)</a>
        <div class='rstext'>rs6498169 has been reported in a large study to be associated with multiple sclerosis. The risk allele (oriented to the dbSNP entry) is (G); the odds ratio associated with this allele is 1.14 (CI 1.08-1.21).  [PharmGKB:Curated GWAS Results: Risk alleles for multiple sclerosis identified by a genomewide study (Initial Sample Size: 931 trios, 2,431 controls; Replication Sample Size: 609 trios, 2,322 cases, 2,987 controls; Risk Allele: rs6498169-G).] [OMIM:?] [GWAS:Multiple sclerosis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10492519'>rs10492519</a><a href='http://www.snpedia.com/index.php/rs10492519(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDBDB;'>46.2%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk of developing prostate cancer</div>
        <div class='boxshow' id="show-sec-0.947038116769" style="display: inline;"><a href="javascript:toggle2('sec-0.947038116769')">...more...</a></div>
        <div class='boxrstext' id="sec-0.947038116769" style="display: none;"><a href="javascript:toggle2('sec-0.947038116769')">(hide)</a>
        <div class='rstext'>rs10492519 is one of seven SNPs found in a combined study of over 1,000 patients to be associated with increased risk for prostate cancer. The risk allele for this SNP is (G); and while the odds ratio was not specifically reported, the probability of false significance (not permuted though) was given as p=5.6 x 10e-6, using an additive model of risk.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs12037606'>rs12037606</a><a href='http://www.snpedia.com/index.php/rs12037606(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDCDC;'>46.9%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.22x risk of developing Crohn's disease</div>
        <div class='boxshow' id="show-sec-0.578529754098" style="display: inline;"><a href="javascript:toggle2('sec-0.578529754098')">...more...</a></div>
        <div class='boxrstext' id="sec-0.578529754098" style="display: none;"><a href="javascript:toggle2('sec-0.578529754098')">(hide)</a>
        <div class='rstext'>rs12037606 has been reported in a large study to be associated with Crohn's disease. The risk allele (oriented to the dbSNP entry) is (A); the odds ratio associated with heterozygotes is 1.22 (CI 1.07-1.40), and for homozygotes, 1.52 (CI 1.28-1.82).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs12431733'>rs12431733</a><a href='http://www.snpedia.com/index.php/rs12431733(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDCDC;'>46.9%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk of developing Parkinson's Disease</div>
        <div class='boxshow' id="show-sec-0.167818396929" style="display: inline;"><a href="javascript:toggle2('sec-0.167818396929')">...more...</a></div>
        <div class='boxrstext' id="sec-0.167818396929" style="display: none;"><a href="javascript:toggle2('sec-0.167818396929')">(hide)</a>
        <div class='rstext'>[GWAS:Parkinson's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2236225'>rs2236225</a><a href='http://www.snpedia.com/index.php/rs2236225(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDDDD;'>47.8%</span><br/>Repute:Bad<br/>References:10<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly (~1.5x) higher risk for Caucasian mothers to give birth to NTD children  There is a slightly higher risk (~1.5x at most) for Caucasian mothers with this genotype to give birth to children with neural tube defects; see discussion at rs2236225.</div>
        <div class='boxshow' id="show-sec-0.371547215783" style="display: inline;"><a href="javascript:toggle2('sec-0.371547215783')">...more...</a></div>
        <div class='boxrstext' id="sec-0.371547215783" style="display: none;"><a href="javascript:toggle2('sec-0.371547215783')">(hide)</a>
        <div class='rstext'>rs2236225, also known as c.1958G&gt;A, G1958A, p.Arg653Gln and R653Q, is a SNP in the MTHFD1 gene on chromosome 14. This SNP is best known for it's potential role in raising risk for neural tube defects, with some studies finding a slight effect in certain populations (for example, Irish and Italians) and other studies finding no effect (for example, in the Dutch). A 2014 meta-analysis of nine studies totaling 4,302 cases of children born with neural tube defects concluded that Caucasian mothers carrying one or two rs2236225(T) alleles were at slight increased risk, of about 1.5 - 1.7x, of having affected children compared to mothers with no (T) alleles. However, they cautioned that their results should be 'viewed with caution' given the statistical uncertainties inherent in these studies....</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10260404'>rs10260404</a><a href='http://www.snpedia.com/index.php/rs10260404(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDEDE;'>48.7%</span><br/>Repute:Bad<br/>References:10<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.20x risk of developing ALS</div>
        <div class='boxshow' id="show-sec-0.050152291755" style="display: inline;"><a href="javascript:toggle2('sec-0.050152291755')">...more...</a></div>
        <div class='boxrstext' id="sec-0.050152291755" style="display: none;"><a href="javascript:toggle2('sec-0.050152291755')">(hide)</a>
        <div class='rstext'>rs10260404, a SNP in the region of the DPP6 gene on chromosome 7, has been associated with the sporadic form of ALS (Lou Gehrig's disease) in a study of 1000+ European patients. The overall odds ratio for the risk allele rs10260404(C) is 1.30 (CI: 1.18-1.43, p=0.017). When broken down by genotype, the odds ratios for heterozygotes are 1.20 (CI: 1.06-1.41), and for rs10260404(C;C) homozygotes, 1.60 (CI: 1.32-1.92). A 'C-C' haplotype for this SNP and that of it's neighbor rs10239794 is also highly (statistically; p=10e-9) associated with ALS. However, in an expanded study pooling 4 populations (Irish, Dutch, US, Polish) rs10260404 failed to reach Bonferroni significance. although it did remain significant in the (expanded) Irish-only population.  [PharmGKB:Non-Curated GWAS results: Screening...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs17319721'>rs17319721</a><a href='http://www.snpedia.com/index.php/rs17319721(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDEDE;'>48.7%</span><br/>Repute:Bad<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>reported to have some negative associations with kidney function  Although it is a fairly common genotype in Caucasians, donor kidneys with this genotype appear to be rejected in transplant cases roughly twice as much as (G;G) donor kidneys.</div>
        <div class='boxshow' id="show-sec-0.074091255563" style="display: inline;"><a href="javascript:toggle2('sec-0.074091255563')">...more...</a></div>
        <div class='boxrstext' id="sec-0.074091255563" style="display: none;"><a href="javascript:toggle2('sec-0.074091255563')">(hide)</a>
        <div class='rstext'>rs17319721 is a SNP located in an intron of the SHROOM3 gene on chromosome 4, apparently in the middle of a transcription factor 7&#8211;like 2&#8211;dependent (TCF7L2-dependent) enhancer element. The rs17319721(A) allele is associated with increased SHROOM3 transcription, and this increased expression may explain both the preliminary reports that this SNP is associated with increased risk for chronic kidney disease (CKD) and with increased odds of kidney transplant rejection, when the (A) allele is present in the donor (allograft) kidney.  [GWAS:Renal function and chronic kidney disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7536563'>rs7536563</a><a href='http://www.snpedia.com/index.php/rs7536563(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDEDE;'>48.7%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.12x risk of multiple sclerosis</div>
        <div class='boxshow' id="show-sec-0.868487405023" style="display: inline;"><a href="javascript:toggle2('sec-0.868487405023')">...more...</a></div>
        <div class='boxrstext' id="sec-0.868487405023" style="display: none;"><a href="javascript:toggle2('sec-0.868487405023')">(hide)</a>
        <div class='rstext'>rs7536563 has been reported in a large study to be associated with multiple sclerosis. The risk allele (oriented to the dbSNP entry) is (A); the odds ratio associated with this allele is 1.12 (CI 1.06-1.18).  [PharmGKB:Curated In a replicated GWAS of U.K. and U.S. case/parent trios, this variant was shown to be associated with multiple sclerosis risk.]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs144848'>rs144848</a><a href='http://www.snpedia.com/index.php/rs144848(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDEDE;'>48.7%</span><br/>Repute:Bad<br/>References:18<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>possible increased risk for breast cancer</div>
        <div class='boxshow' id="show-sec-0.27516889029" style="display: inline;"><a href="javascript:toggle2('sec-0.27516889029')">...more...</a></div>
        <div class='boxrstext' id="sec-0.27516889029" style="display: none;"><a href="javascript:toggle2('sec-0.27516889029')">(hide)</a>
        <div class='rstext'>This SNP, a variant in the BRCA2 gene, is 1 of 25 SNPs reported to represent independently minor, but cumulatively significant, increased risk for breast cancer. For details of all 25 SNPs in this group, along with the two methods used to calculate overall risk estimates for breast cancer, refer to the SNPedia breast cancer entry. For this particular SNP, the risk (minor) allele is (G), encoding the His; the SNP is also known as N372H or Asn372His. In a separate study, rs144848(G;G) homozygotes were determined to have a 1.31x increased risk (CI: 1.07-1.61) for breast cancer greater risk compared to (T;T) genotypes. Interestingly, in normal females of all ages there is a significant deficiency of homozygotes compared with that expected from Hardy-Weinberg equilibrium, whereas in males there...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs699473'>rs699473</a><a href='http://www.snpedia.com/index.php/rs699473(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDEDE;'>49.1%</span><br/>Repute:Bad<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~1.5x increased brain tumor risk</div>
        <div class='boxshow' id="show-sec-0.474683976108" style="display: inline;"><a href="javascript:toggle2('sec-0.474683976108')">...more...</a></div>
        <div class='boxrstext' id="sec-0.474683976108" style="display: none;"><a href="javascript:toggle2('sec-0.474683976108')">(hide)</a>
        <div class='rstext'>A study of non-Hispanic Caucasians with various types of brain tumors concluded that there was an increased risk of glioma (odds ratio 1.3, CI: 1.0-1.7) and meningioma (odds ratio 1.7, CI: 1.1-2.7) associated with the rs699473(C) allele.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs486907'>rs486907</a><a href='http://www.snpedia.com/index.php/rs486907(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFDFDF;'>50.4%</span><br/>Repute:Bad<br/>References:10<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.5x increased prostate cancer risk</div>
        <div class='boxshow' id="show-sec-0.534525396817" style="display: inline;"><a href="javascript:toggle2('sec-0.534525396817')">...more...</a></div>
        <div class='boxrstext' id="sec-0.534525396817" style="display: none;"><a href="javascript:toggle2('sec-0.534525396817')">(hide)</a>
        <div class='rstext'>rs486907 is a SNP in the RNase L RNASEL gene that has been associated with cancer risk. The basic rationale behind most of these studies is that RNase L is responsible for deactivating RNA-based viruses that are associated with certain cancers, and therefore SNPs that lead to lower RNase L activity may lead to increased cancer risk. This SNP is also known as R462Q or Arg462Gln. In a study of prostate cancer patients, rs486907(A;G) heterozygotes were calculated to be at 1.5x increased risk, and rs486907(A;A) homozygotes 2x risk (p=0.007). This SNP was also associated with hereditary-prostate-cancer (HPC) predisposition with an odds ratio of 1.97 (p=0.07) in a study of 116 affected Finnish families. See also:  [OMIM:PROSTATE CANCER, SUSCEPTIBILITY TO]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs763361'>rs763361</a><a href='http://www.snpedia.com/index.php/rs763361(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE0E0;'>50.9%</span><br/>Repute:Bad<br/>References:36<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk for multiple autoimmune diseases, such as type-1 diabetes  Higher risk of type-1 diabetes.</div>
        <div class='boxshow' id="show-sec-0.301789789041" style="display: inline;"><a href="javascript:toggle2('sec-0.301789789041')">...more...</a></div>
        <div class='boxrstext' id="sec-0.301789789041" style="display: none;"><a href="javascript:toggle2('sec-0.301789789041')">(hide)</a>
        <div class='rstext'>rs763361, also known as Gly307Ser, is a SNP in the immune response CD226 gene. The rs763361(T) allele (in dbSNP orientation) encodes the Ser. Based on multiple studies (by one group) now totaling over 2,000 patients, the rs763361(T) allele is associated with increased risk for multiple autoimmune diseases, including type-1 diabetes (p = 3 x 10e-9), multiple sclerosis (p = 4 x 10e-4), and possibly rheumatoid arthritis (p = 0.017). Based on tag SNP analysis, the authors conclude that this SNP could be the causal variant. Tested in a large multiple sclerosis data set consisting of 2369 trio families, 5737 cases and 10 296 unrelated controls, SNP rs763361 was associated with disease risk (p = 5.4 x 10e-8)  [PharmGKB:Non-Curated GWAS Results: Robust associations of four new chromosome regions f...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3087243'>rs3087243</a><a href='http://www.snpedia.com/index.php/rs3087243(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE0E0;'>51.3%</span><br/>Repute:Bad<br/>References:82<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk for auto-immune diseases</div>
        <div class='boxshow' id="show-sec-0.43768911384" style="display: inline;"><a href="javascript:toggle2('sec-0.43768911384')">...more...</a></div>
        <div class='boxrstext' id="sec-0.43768911384" style="display: none;"><a href="javascript:toggle2('sec-0.43768911384')">(hide)</a>
        <div class='rstext'>The rs3087243 SNP is also known in the literature as the CT60 G&gt;A or the +6230G&gt;A polymorphism, and it is located in the CTLA4 gene. In Asian (Japanese) populations, the presence of an rs3087243(G) allele represents a 1.3 fold increased risk of autoimmune thyroid disease, and for those with autoimmune thyroid disease, a 1.5 fold increased risk of type-1 diabetes. However, in individuals without autoimmune thyroid disease, no association was seen between this SNP and type-1 diabetes. The authors speculate that earlier studies may have reported associations between this SNP and type-1 diabetes that were actually primarily based on the association with autoimmune thyroid disease. This same SNP, rs3087243, has also been implicated as a (minor) risk factor for developing rheumatoid arthri...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4845618'>rs4845618</a><a href='http://www.snpedia.com/index.php/rs4845618(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE0E0;'>51.3%</span><br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.7x increased melanoma risk</div>
        <div class='boxshow' id="show-sec-0.0736884386456" style="display: inline;"><a href="javascript:toggle2('sec-0.0736884386456')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0736884386456" style="display: none;"><a href="javascript:toggle2('sec-0.0736884386456')">(hide)</a>
        <div class='rstext'>A study of 219 female melanoma patients and an equal number of controls concluded that 4 linked SNPs within the IL6R gene were associated with increased melanoma risk, but only when heterozygous. The odds ratio associated with the minor allele of this SNP, rs4845618, was 1.72 (CI: 1.04-2.84). The 4 linked SNPs from identified in this study are: rs6684439, rs4845618, rs4845622, and rs8192284.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2736990'>rs2736990</a><a href='http://www.snpedia.com/index.php/rs2736990(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE0E0;'>51.3%</span><br/>Repute:Bad<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk of developing Parkinson's Disease</div>
        <div class='boxshow' id="show-sec-0.472239945538" style="display: inline;"><a href="javascript:toggle2('sec-0.472239945538')">...more...</a></div>
        <div class='boxrstext' id="sec-0.472239945538" style="display: none;"><a href="javascript:toggle2('sec-0.472239945538')">(hide)</a>
        <div class='rstext'>rs2736990 is a SNP associated with Parkinson&#8217;s disease (PD), a neurodegenerative disorder characterized by the degradation of dopamine-producing neurons in the substantia nigra. Rs273990 is located on chromosome 4q22 in the synuclein-alpha SNCA locus. The function of SNCA is not fully understood, but it predominately found in the presynaptic terminals of neurons, and it is thought that it plays a role in maintaining adequate supplies of synaptic vesicles. SNCA is also a major component of Lewy Bodies (LB), protein accumulations often found in neurons in patients with PD. The best evidence for the association of rs2736990 with PD was provided in a genome-wide association study (GWAS) that found an increased frequency of the minor C allele (T is the major allele) in position 90897564 a...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs13181'>rs13181</a><a href='http://www.snpedia.com/index.php/rs13181(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE1E1;'>52.2%</span><br/>Repute:Bad<br/>References:43<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.12x increased risk for cutaneous melanoma</div>
        <div class='boxshow' id="show-sec-0.984802386421" style="display: inline;"><a href="javascript:toggle2('sec-0.984802386421')">...more...</a></div>
        <div class='boxrstext' id="sec-0.984802386421" style="display: none;"><a href="javascript:toggle2('sec-0.984802386421')">(hide)</a>
        <div class='rstext'>rs13181, also known as c.2251A&gt;C, p.Lys751Gln and K751Q, is a variant in the ERCC2 gene on chromosome 19. A meta-analysis of 20 studies, comprising 2,308 cases, ultimately concluded that the rs13181(G) allele is associated with increased cutaneous melanoma risk (odds ratio 1.12, CI: 1.03-1.21, p = 0.01; population attributable risk = 9.6%). A preliminary (as yet unreplicated) study of 430 patients with ovarian cancer concluded that rs13181(C;C) is associated with an increased risk of ovarian cancer (OR 5.01 relative to rs13181(A;A), CI: 3.37-7.43; p &lt; 0.0001). Interestingly, heterozygotes were reported to have a lower odds ratio for developing ovarian cancer (of 0.43; CI: 0.27&#8211;0.62, p &lt; 0.0001) compared to rs13181(A;A) individuals. Polymorphisms in XPD and TP53 and mutation in hum...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2280714'>rs2280714</a><a href='http://www.snpedia.com/index.php/rs2280714(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE1E1;'>52.2%</span><br/>Repute:Bad<br/>References:22<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x increased risk of SLE</div>
        <div class='boxshow' id="show-sec-0.578078764191" style="display: inline;"><a href="javascript:toggle2('sec-0.578078764191')">...more...</a></div>
        <div class='boxrstext' id="sec-0.578078764191" style="display: none;"><a href="javascript:toggle2('sec-0.578078764191')">(hide)</a>
        <div class='rstext'>The rs2004640(A) allele had a higher frequency in SLE cases (0.385) than controls (0.321; odds ratio (OR) = 1.32, P = 0.0003). In combined analysis, including all seven independent cohorts from the three studies so far, robust and consistent associations of the rs2004640(A) allele with SLE were observed. The estimate of risk was OR = 1.44 (CI: 1.34-1.55), with an overall P = 1.85 x 10(-23) for the rs2004640(A) allele. The haplotype (rs2004640T-rs2280714A) involved in both the alternative splice donor site and the elevated expression of IRF5 also had a highly significant association with SLE (pooled, P = 2.11 x 10(-16)). Our results indicate that the genetic effect on the risk of SLE mediated by IRF5 variants can be generally accepted in both white and Asian populations. A meta-analysis com...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6601764'>rs6601764</a><a href='http://www.snpedia.com/index.php/rs6601764(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE3E3;'>54.9%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.16x increased risk of developing Crohn's disease</div>
        <div class='boxshow' id="show-sec-0.26293902765" style="display: inline;"><a href="javascript:toggle2('sec-0.26293902765')">...more...</a></div>
        <div class='boxrstext' id="sec-0.26293902765" style="display: none;"><a href="javascript:toggle2('sec-0.26293902765')">(hide)</a>
        <div class='rstext'>rs6601764 has been reported in a large study to be associated with Crohn's disease. The risk allele (oriented to the dbSNP entry) is (C); the odds ratio associated with heterozygotes is 1.16 (CI 1.01-1.33), and for homozygotes, 1.52 (CI 1.28-1.80).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs807701'>rs807701</a><a href='http://www.snpedia.com/index.php/rs807701(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE3E3;'>55.4%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased dyslexia risk</div>
        <div class='boxshow' id="show-sec-0.536595340682" style="display: inline;"><a href="javascript:toggle2('sec-0.536595340682')">...more...</a></div>
        <div class='boxrstext' id="sec-0.536595340682" style="display: none;"><a href="javascript:toggle2('sec-0.536595340682')">(hide)</a>
        <div class='rstext'>Rs807701, a SNP in the DCDC2 gene, is in a region that crops up in several independent studies as likely to associated with dyslexia. The risk allele in the Caucasian populations studied is (C). One study reports that the odds ratio for rs807701 genotypes increases if calculated from subsets of more severely dyslexic individuals as compared to more heterogenous, larger groups of dyslexic individuals. The genotype relative risk (GRR) for rs807701(C;C) increased from 1.88 (95% CI 0.89-3.97; P=.058) for the larger group up to 5.04 (95% CI 1.35-18.88; P=.002) for the most severely affected group. Combined with another SNP marker in the DCDC2 gene, rs793862, the (haplotype) GRR also increased for the homozygous haplotype rs793862(A)-rs807701(C), from 4.11 (95% CI 2.77-6.08; P&lt;.0001) for the ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4785763'>rs4785763</a><a href='http://www.snpedia.com/index.php/rs4785763(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE4E4;'>55.8%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.5x higher risk for melanoma</div>
        <div class='boxshow' id="show-sec-0.977232003803" style="display: inline;"><a href="javascript:toggle2('sec-0.977232003803')">...more...</a></div>
        <div class='boxrstext' id="sec-0.977232003803" style="display: none;"><a href="javascript:toggle2('sec-0.977232003803')">(hide)</a>
        <div class='rstext'>rs4785763 is a SNP near the MC1R gene, which encodes the melanocortin 1 receptor. rs4785763 was found to be associated with melanoma based on a study by the eMERGE network in which electronic medical records were searched to find clinical associations associated with selected SNPs. A genome-wide association study identifies novel alleles associated with hair color and skin pigmentation. Genome-wide association study of tanning phenotype in a population of European ancestry.  [GWAS:Melanoma]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2241880'>rs2241880</a><a href='http://www.snpedia.com/index.php/rs2241880(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE4E4;'>56.2%</span><br/>Repute:Bad<br/>References:29<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x increased risk for Crohn's disease in Caucasians</div>
        <div class='boxshow' id="show-sec-0.977043380488" style="display: inline;"><a href="javascript:toggle2('sec-0.977043380488')">...more...</a></div>
        <div class='boxrstext' id="sec-0.977043380488" style="display: none;"><a href="javascript:toggle2('sec-0.977043380488')">(hide)</a>
        <div class='rstext'>rs2241880, a SNP in the ATG16L1 gene encoding a threonine to alanine substitution ('T300A') in a protein known to be involved in the function of the epithelial cells lining the intestine, has been associated with Crohn's disease in several recent studies. In another recent (2007) report, rs2241880 is confirmed to be associated with both Crohn's disease and ileal disease, but additionally, the authors calculate risk for individuals who are homozygotes for this SNP plus 2 others (in the IBD5 and NOD2 genes). Individuals homozygous for the risk alleles for all 3 of these SNPs are estimated to be at 20 fold higher risk (CI ~9-49) for Crohn's disease. From the largest most recent survey, the Crohn's disease-associated SNPs for IBD5 and NOD2 are, respectively, rs6596075 and rs17221417. associate...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9303277'>rs9303277</a><a href='http://www.snpedia.com/index.php/rs9303277(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE4E4;'>56.6%</span><br/>Repute:Bad<br/>References:12<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk of developing primary biliary cirrhosis</div>
        <div class='boxshow' id="show-sec-0.071879664968" style="display: inline;"><a href="javascript:toggle2('sec-0.071879664968')">...more...</a></div>
        <div class='boxrstext' id="sec-0.071879664968" style="display: none;"><a href="javascript:toggle2('sec-0.071879664968')">(hide)</a>
        <div class='rstext'>[OMIM:?] [GWAS:Primary biliary cirrhosis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs12498742'>rs12498742</a><a href='http://www.snpedia.com/index.php/rs12498742(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE5E5;'>57.5%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.25 increased risk for gout</div>
        <div class='boxshow' id="show-sec-0.446502136714" style="display: inline;"><a href="javascript:toggle2('sec-0.446502136714')">...more...</a></div>
        <div class='boxrstext' id="sec-0.446502136714" style="display: none;"><a href="javascript:toggle2('sec-0.446502136714')">(hide)</a>
        <div class='rstext'>Each copy of the minor (G) allele appears to confer reduced risk for gout based on combined data from over 140,000 individuals of European ancestry within the Global Urate Genetics Consortium (GUGC), replicating an association reported earlier for a linked SNP in this gene.  [GWAS:Urate levels]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs13149290'>rs13149290</a><a href='http://www.snpedia.com/index.php/rs13149290(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE9E9;'>63.3%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slightly increased risk of developing prostate cancer</div>
        <div class='boxshow' id="show-sec-0.175589596681" style="display: inline;"><a href="javascript:toggle2('sec-0.175589596681')">...more...</a></div>
        <div class='boxrstext' id="sec-0.175589596681" style="display: none;"><a href="javascript:toggle2('sec-0.175589596681')">(hide)</a>
        <div class='rstext'>rs13149290 is one of seven SNPs found in a combined study of over 1,000 patients to be associated with increased risk for prostate cancer. The risk allele for this SNP is (C); and while the odds ratio was not specifically reported, the probability of false significance (not permuted though) was given as p=2.5 x 10e-5, using a dominant model of risk.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4570625'>rs4570625</a><a href='http://www.snpedia.com/index.php/rs4570625(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFEAEA;'>64.6%</span><br/>Repute:Bad<br/>References:18<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>maybe: higher scores on anxiety-related personality traits; greater placebo response  One study of (only) 25 people shows this genotype was a significant predictor of clinical placebo response, being associated with greater improvement in anxiety symptoms. A criticism of this study has also been published, and it has not been replicated since the original publication years ago. This SNP has been linked to several psychiatric and/or behavioral phenomena, including: *higher scores on anxiety-related personality traits *obsessive compulsive disorder *panic disorder with a possible gender-dependent effect</div>
        <div class='boxshow' id="show-sec-0.715062843631" style="display: inline;"><a href="javascript:toggle2('sec-0.715062843631')">...more...</a></div>
        <div class='boxrstext' id="sec-0.715062843631" style="display: none;"><a href="javascript:toggle2('sec-0.715062843631')">(hide)</a>
        <div class='rstext'>rs4570625, also known as G-703T, is a SNP in the tryptophan hydroxylase 2 TPH2 gene. This SNP has been linked to several psychiatric and/or behavioural phenomena, including: * rs4570625 and rs4565946 linked to the pathogenesis of early-onset obsessive compulsive disorder * rs4570625 and rs11178997 in TPH2's regulatory region display preferential transmission * rs4570625(G;G) homozygosity was a significant predictor of clinical placebo response, being associated with greater improvement in anxiety symptoms. g2b2mh is rs4570625(G;G) is a significant predictor of clinical placebo response? maybe not panic disorder with a possible gender-dependent effect Transmission disequilibrium of polymorphic variants in the tryptophan hydroxylase-2 gene in attention-deficit/hyperactivity disorder. Associa...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3782179'>rs3782179</a><a href='http://www.snpedia.com/index.php/rs3782179(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFEBEB;'>65.5%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>common, but increased risk of testicular cancer</div>
        <div class='boxshow' id="show-sec-0.438975829651" style="display: inline;"><a href="javascript:toggle2('sec-0.438975829651')">...more...</a></div>
        <div class='boxrstext' id="sec-0.438975829651" style="display: none;"><a href="javascript:toggle2('sec-0.438975829651')">(hide)</a>
        <div class='rstext'>Testicular germ cell tumor risk I('TGCT'; i.e. testicular cancer risk) was increased 3x per copy of the major (T) allele at rs3782179 and rs4474514 (OR = 3.08, CI: 2.29&#8211;4.13; OR = 3.07, CI = 2.29&#8211;4.13, respectively), based on a genome-wide scan of 277 primarily Caucasian TGCT cases and 919 controls. Men who have two copies of the common version of the c-KIT ligand KITLG gene have a 4.5-fold higher risk of testicular cancer than men who have two copies of the less common or minor version of the gene. https://www.23andme.com/you/community/thread/7837/ The original study was also only done in 277 people with the disease and 900 controls, so it's a very small study.  [OMIM:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs995030'>rs995030</a><a href='http://www.snpedia.com/index.php/rs995030(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFEBEB;'>66.4%</span><br/>Repute:Bad<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>non-protective against testicular cancer</div>
        <div class='boxshow' id="show-sec-0.78545237238" style="display: inline;"><a href="javascript:toggle2('sec-0.78545237238')">...more...</a></div>
        <div class='boxrstext' id="sec-0.78545237238" style="display: none;"><a href="javascript:toggle2('sec-0.78545237238')">(hide)</a>
        <div class='rstext'>23andMe blog each rs4624820(G) increases the odds of testicular cancer by 2.5x. &lt;p&gt;&lt;br&gt; As reported by the CPMC, relative to men who are rs995030(A;A), men who are rs995030(A;G) have a 0.38 relative risk, and those who are (G;G) have a 0.15 relative risk. Association of genetic markers within the KIT and KITLG genes with human male infertility. Common variation in KITLG and at 5q31.3 predisposes to testicular germ cell cancer.  [OMIM:?] [GWAS:Testicular germ cell tumor]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1012053'>rs1012053</a><a href='http://www.snpedia.com/index.php/rs1012053(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFF2F2;'>77.0%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal (higher) risk of Bipolar Disorder.  This is the most common version of this SNP, so most people have this. But it is suspected of having a higher risk of Bipolar Disorder compared to the other rare versions of this SNP.</div>
        <div class='boxshow' id="show-sec-0.842402582403" style="display: inline;"><a href="javascript:toggle2('sec-0.842402582403')">...more...</a></div>
        <div class='boxrstext' id="sec-0.842402582403" style="display: none;"><a href="javascript:toggle2('sec-0.842402582403')">(hide)</a>
        <div class='rstext'>Bipolar disorder failed to replicate in  [PharmGKB:Non-Curated GWAS Results: A genome-wide association study implicates diacylglycerol kinase eta (DGKH) and several other genes in the etiology of bipolar disorder (Initial Sample Size: 461 cases, 563 controls; Replication Sample Size: 772 cases, 876 controls; Risk Allele: rs1012053-A).] [GWAS:Bipolar disorder]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs28694718'>rs28694718</a><a href='http://www.snpedia.com/index.php/rs28694718(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>>2x higher risk for schizophrenia</div>
        <div class='boxshow' id="show-sec-0.370468758717" style="display: inline;"><a href="javascript:toggle2('sec-0.370468758717')">...more...</a></div>
        <div class='boxrstext' id="sec-0.370468758717" style="display: none;"><a href="javascript:toggle2('sec-0.370468758717')">(hide)</a>
        <div class='rstext'>In the Pseudoautosomal region rs28694718, located in intron 8 of the CSF2RA gene, is part of a haplotype block that has been reported in a whole genome association study to be associated with schizophrenia. (The other SNP in this block is rs28414810.) The risk allele (oriented to the dbSNP entry) is (A); the odds ratio associated with this allele is 2.11.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs5219'>rs5219</a><a href='http://www.snpedia.com/index.php/rs5219(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Repute:Bad<br/>References:79<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk for type-2 diabetes</div>
        <div class='boxshow' id="show-sec-0.601803924511" style="display: inline;"><a href="javascript:toggle2('sec-0.601803924511')">...more...</a></div>
        <div class='boxrstext' id="sec-0.601803924511" style="display: none;"><a href="javascript:toggle2('sec-0.601803924511')">(hide)</a>
        <div class='rstext'>rs5219, also known as E23K, is a SNP in the KCNJ11 gene. The association between type-1 diabetes and rs5219 is mentioned as being replicated in ; there is also a 90% correlation between rs5219 and rs5215. This SNP is one of 4 relatively common SNPs reported to represent risk for type-2 diabetes in the DESIR prospective study of 3,877 Caucasian participants. Under an additive model, the odds ratio for the minor (risk) allele (T) is 1.15 (CI: 0.95-1.4, p=0.2, ie not significant). But for the 4 SNPs, each risk allele increased type-2 diabetes risk by 1.34x (p=2x10e-6), with an odds ratio of 2.48 (CI: 1.59-3.86) for carriers of 4 or more compared to those with one or none (risk alleles). Genotyping of ~4000 Chinese as well a meta-analysis concluded that rs5219 was associated with type-2 diabet...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2272127'>rs2272127</a><a href='http://www.snpedia.com/index.php/rs2272127(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #F0B9F0;'>75.0%</span><br/>Repute:Bad<br/>References:1<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Associated with herpes and schizophrenia</div>
        <div class='boxshow' id="show-sec-0.545391168376" style="display: inline;"><a href="javascript:toggle2('sec-0.545391168376')">...more...</a></div>
        <div class='boxrstext' id="sec-0.545391168376" style="display: none;"><a href="javascript:toggle2('sec-0.545391168376')">(hide)</a>
        <div class='rstext'>associations with schizophrenia and herpes rs2272127 with schizophrenia(P = 0.0007, odds ratio for C allele 1.49, 95% CI: 1.18-1.87; P = 0.03 following correction for multiple comparisons). rs2272127 was also associated with herpes simplex virus 1 (HSV1) seropositivity in cases (P = 0.04, OR for G allele 1.58, 95% CI: 1.04-2.39).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2230201'>rs2230201</a><a href='http://www.snpedia.com/index.php/rs2230201(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.4<br/>Frequency: <span style='background-color: #FF0A0A;'>0.0%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x risk of lupus</div>
        <div class='boxshow' id="show-sec-0.731449899017" style="display: inline;"><a href="javascript:toggle2('sec-0.731449899017')">...more...</a></div>
        <div class='boxrstext' id="sec-0.731449899017" style="display: none;"><a href="javascript:toggle2('sec-0.731449899017')">(hide)</a>
        <div class='rstext'>In ~500 Japanese SLE patients, the (G) allele of rs2230201, a SNP in the C3 gene, is associated with higher risk for systemic lupus erythematosus (SLE). The odds ratio is 1.19 (CI: 1.01-1.41, p=0.038).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10865331'>rs10865331</a><a href='http://www.snpedia.com/index.php/rs10865331(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.4<br/>Frequency: <span style='background-color: #FF9696;'>6.2%</span><br/>Repute:Bad<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x higher risk for ankylosing spondylitis</div>
        <div class='boxshow' id="show-sec-0.817556206705" style="display: inline;"><a href="javascript:toggle2('sec-0.817556206705')">...more...</a></div>
        <div class='boxrstext' id="sec-0.817556206705" style="display: none;"><a href="javascript:toggle2('sec-0.817556206705')">(hide)</a>
        <div class='rstext'>rs10865331 is a SNP in chromosomal region 2p15. After initial GWAS reports linked this SNP to ankylosing spondylitis (AS), a follow-up study in 456 Spanish patients replicated the association between the rs10865331(A) allele and higher risk for AS (p = 0.039). In a 2010 report, the AA genotype was associated with 1.3x odds of ankylosing spondylitis, and the GG genotype with 0.8x odds, compared to the AG genotype.  [OMIM:?] [GWAS:Ankylosing spondylitis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs498872'>rs498872</a><a href='http://www.snpedia.com/index.php/rs498872(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1.4<br/>Frequency: <span style='background-color: #FFA7A7;'>11.1%</span><br/>Repute:Bad<br/>References:17<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x higher risk for glioma development</div>
        <div class='boxshow' id="show-sec-0.0257273024401" style="display: inline;"><a href="javascript:toggle2('sec-0.0257273024401')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0257273024401" style="display: none;"><a href="javascript:toggle2('sec-0.0257273024401')">(hide)</a>
        <div class='rstext'>23andMe blog rs498872 A 1.18 Glioma  [GWAS:Glioma]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4959039'>rs4959039</a><a href='http://www.snpedia.com/index.php/rs4959039(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.4<br/>Frequency: <span style='background-color: #FFDEDE;'>49.6%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x higher risk for multiple sclerosis</div>
        <div class='boxshow' id="show-sec-0.698049801502" style="display: inline;"><a href="javascript:toggle2('sec-0.698049801502')">...more...</a></div>
        <div class='boxrstext' id="sec-0.698049801502" style="display: none;"><a href="javascript:toggle2('sec-0.698049801502')">(hide)</a>
        <div class='rstext'>rs4959039 is a SNP located within the Class I HLA-G region of chromosome 6. Independently of the influence of the (nearby) HLA-DRB1*1501 allele, the rs4959039(G) was shown to be associated with increased risk for multiple sclerosis in a study of ~2,300 patients.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1126497'>rs1126497</a><a href='http://www.snpedia.com/index.php/rs1126497(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.4<br/>Frequency: <span style='background-color: #FFE3E3;'>54.9%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x increased risk for breast cancer</div>
        <div class='boxshow' id="show-sec-0.853688248593" style="display: inline;"><a href="javascript:toggle2('sec-0.853688248593')">...more...</a></div>
        <div class='boxrstext' id="sec-0.853688248593" style="display: none;"><a href="javascript:toggle2('sec-0.853688248593')">(hide)</a>
        <div class='rstext'>rs1126497, also known as Thr115Met, is a SNP in the epithelial cell adhesion molecule EPCAM gene. In a study of case control study of 1643 individuals with breast cancer and 1818 control subjects in Eastern and Southern Chinese populations, the rs1126497(T) allele was associated with a 1.4x increased risk (CI: 1.16-1.57), and it was also associated with early breast cancer onset (p=0.0023). Functional polymorphism in the EpCAM gene is associated with occurrence and advanced disease status of cervical cancer in Chinese population.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6010620'>rs6010620</a><a href='http://www.snpedia.com/index.php/rs6010620(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.4<br/>Frequency: <span style='background-color: #FFE3E3;'>55.0%</span><br/>Repute:Bad<br/>References:31<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.4x higher risk for glioma development; but this is the common allele</div>
        <div class='boxshow' id="show-sec-0.401811678831" style="display: inline;"><a href="javascript:toggle2('sec-0.401811678831')">...more...</a></div>
        <div class='boxrstext' id="sec-0.401811678831" style="display: none;"><a href="javascript:toggle2('sec-0.401811678831')">(hide)</a>
        <div class='rstext'>'''rs6010620''' is a SNP associated with atopic dermatitis (AD). Also known as eczema, AD is one of the most common chronic inflammatory skin disorders with a prevalence estimated up to 20% in children and 3% in adults . It is a complex and highly heterogeneous disease that likely involves various genetic factors. The SNP rs6010620 locates on chromosome 20q13.33, at which there are eight genes &#8212; ''PRIC285'', ''STMN3'', ''RTEL1'', ''TNFRSF6B'', ''ARFRP1'', ''ZGPAT'', and ''LIME1'' &#8212;, within a single 200-kb LD block around rs6010620. Among them, ''TNFRSF6B'' and ''ZGPAT'' might be the susceptibility candidates whose expressions correlate with the SNP as suggested by expression quantitative trait loci (eQTL) analysis . rs6010620&#8217;s association with AD was first identified in ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6495446'>rs6495446</a><a href='http://www.snpedia.com/index.php/rs6495446(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.4<br/>Frequency: <span style='background-color: #FFE8E8;'>61.6%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2.5x increased risk for chronic kidney disease  in a small unreplicated study</div>
        <div class='boxshow' id="show-sec-0.813112753655" style="display: inline;"><a href="javascript:toggle2('sec-0.813112753655')">...more...</a></div>
        <div class='boxrstext' id="sec-0.813112753655" style="display: none;"><a href="javascript:toggle2('sec-0.813112753655')">(hide)</a>
        <div class='rstext'>rs6495446, an intronic SNP in the MTHFS gene, was associated with increased risk for chronic kidney disease based on a study associated with the Framingham Heart Study of ~1,000 Caucasian individuals. The odds ratio rs6495446(C) allele was 1.24 (CI: 1.09-1.41, p=0.001).news 23andMe reports this same information with a different emphasis: each T allele lowers the odds of chronic kidney disease by 0.8 times.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs13361189'>rs13361189</a><a href='http://www.snpedia.com/index.php/rs13361189(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.3<br/>Frequency: <span style='background-color: #FF9A9A;'>7.1%</span><br/>Repute:Bad<br/>References:23<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk for Crohn's disease</div>
        <div class='boxshow' id="show-sec-0.429078797979" style="display: inline;"><a href="javascript:toggle2('sec-0.429078797979')">...more...</a></div>
        <div class='boxrstext' id="sec-0.429078797979" style="display: none;"><a href="javascript:toggle2('sec-0.429078797979')">(hide)</a>
        <div class='rstext'>rs13361189 is a SNP in the IRGM gene. A study including 557 Crohn's disease (CD) patients, 425 ulcerative colitis (UC) patients, and 672 ethnically matched Spanish controls as well as a meta-analysis concluded that rs13361189(C) was associated with CD (p=1.07 x 10(-19), pooled odds ratio 1.34) and UC (p=0.0069, pooled odds ratio 1.16). 23andMe blog Each copy of a C at rs13361189 increases a person's risk of developing Crohn's disease 1.33 times compared to someone with two copies of a T  [PharmGKB:Non-Curated GWAS results: Sequence variants in the autophagy gene IRGM and multiple other replicating loci contribute to Crohn's disease susceptibility (Initial Sample Size: 1,748 cases, 2,938 controls; Replication Sample Size: 1,182 cases, 2,024 controls). This variant is associated with Crohn's...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4958847'>rs4958847</a><a href='http://www.snpedia.com/index.php/rs4958847(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.3<br/>Frequency: <span style='background-color: #FFB7B7;'>17.7%</span><br/>Repute:Bad<br/>References:9<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk for Crohn's disease</div>
        <div class='boxshow' id="show-sec-0.850135008699" style="display: inline;"><a href="javascript:toggle2('sec-0.850135008699')">...more...</a></div>
        <div class='boxrstext' id="sec-0.850135008699" style="display: none;"><a href="javascript:toggle2('sec-0.850135008699')">(hide)</a>
        <div class='rstext'>rs4958847 is a SNP in the IRGM gene. A study of 507 CD patients, 475 UC patients and 576 controls from New Zealand Caucasians found that rs4958847, a SNP in the IRGM gene, was associated specifically with ileal Crohn's disease but not with other forms of inflammatory bowel disease. The reported odds ratio was 1.767, CI: 1.224-2.558, p=0.0022. Another study, including 557 Crohn's disease (CD) patients, 425 ulcerative colitis (UC) patients, and 672 ethnically matched Spanish controls as well as a meta-analysis concluded that rs4958847(A) was associated with CD (p=2.78 x 10(-17), pooled odds ratio 1.31) and UC (p=0.014, pooled odds ratio 1.13). Genetic determinants of ulcerative colitis include the ECM1 locus and five loci implicated in Crohn's disease. Gene variants influencing measures of i...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3736228'>rs3736228</a><a href='http://www.snpedia.com/index.php/rs3736228(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.3<br/>Frequency: <span style='background-color: #FFC2C2;'>24.3%</span><br/>Repute:Bad<br/>References:25<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk for osteoporosis  Carried by 21% of people with European ancestry, this genotype has one copy of the higher risk variant (T), which leads to being 1.33 times as likely to develop osteoporosis as someone with no (T) variants for this SNP.</div>
        <div class='boxshow' id="show-sec-0.741946660114" style="display: inline;"><a href="javascript:toggle2('sec-0.741946660114')">...more...</a></div>
        <div class='boxrstext' id="sec-0.741946660114" style="display: none;"><a href="javascript:toggle2('sec-0.741946660114')">(hide)</a>
        <div class='rstext'>rs3736228 is a SNP in the LRP5 gene that is also known as Ala1330Val or A1330V; the more common (C) allele encodes the Ala (alanine), while the rarer (T) allele encodes the Val (valine), which is the risk allele. rs3736228(T) is associated with vertebral fractures and reduced bone mineral density (BMD), with an overall odds ratio for fractures of 1.06 per allele (CI: 1.01-1.12; 7802 fractures among 31,199 individuals studied). More specifically, the rs3736228(T) allele was associated with reduced lumbar spine BMD (p = 2.6 x 10&lt;sup&gt;-9&lt;/sup&gt;), and femoral neck BMD (p = 5.0 x 10&lt;sup&gt;-6&lt;/sup&gt;). Note that rs4988321, another LRP5 SNP (also known as Val667Met), was independently associated with BMD in this same study. news rs3736228 was associated with a 30% increase in th...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs16847548'>rs16847548</a><a href='http://www.snpedia.com/index.php/rs16847548(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.3<br/>Frequency: <span style='background-color: #FFC5C5;'>26.6%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk for sudden cardiac death in Caucasians</div>
        <div class='boxshow' id="show-sec-0.168706710996" style="display: inline;"><a href="javascript:toggle2('sec-0.168706710996')">...more...</a></div>
        <div class='boxrstext' id="sec-0.168706710996" style="display: none;"><a href="javascript:toggle2('sec-0.168706710996')">(hide)</a>
        <div class='rstext'>rs16847548 and rs16856785 strongly associated with QT interval Based on a large study of 19,000+ adults, SNPs rs10494366, rs4657139 and rs16847548 were significantly associated with adjusted QT interval in whites but not in African-Americans. The relative hazard of sudden cardiac death (SCD) associated with each rs16847548(C) allele was 1.31 (CI: 1.10 - 1.56, p=0.002). rs12567209 was also independently associated with SCD in whites (relative hazard 0.57, CI: 0.39 - 0.83, p=0.003).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs501120'>rs501120</a><a href='http://www.snpedia.com/index.php/rs501120(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.3<br/>Frequency: <span style='background-color: #FFCFCF;'>33.8%</span><br/>Repute:Bad<br/>References:10<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk for heart disease</div>
        <div class='boxshow' id="show-sec-0.155996544504" style="display: inline;"><a href="javascript:toggle2('sec-0.155996544504')">...more...</a></div>
        <div class='boxrstext' id="sec-0.155996544504" style="display: none;"><a href="javascript:toggle2('sec-0.155996544504')">(hide)</a>
        <div class='rstext'>rs501120 is a SNP found to be associated with heart disease in two populations by the German MI (Myocardial infarction) Family Study group. The risk allele in dbSNP orientation is rs501120(A). The odds ratio per allele is 1.33 (CI: 1.20-1.48, adjusted p=0.0248). Association of genetic variants and incident coronary heart disease in multiethnic cohorts: the PAGE study. Comparative analysis of genome-wide association studies signals for lipids, diabetes, and coronary heart disease: Cardiovascular Biomarker Genetics Collaboration. Genetic risk score and risk of myocardial infarction in Hispanics. |OA=1 }} Coronary artery disease-related genetic variant on chromosome 10q11 is associated with carotid intima-media thickness and atherosclerosis. Genetics and cardiovascular disease: Design and dev...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2059693'>rs2059693</a><a href='http://www.snpedia.com/index.php/rs2059693(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.3<br/>Frequency: <span style='background-color: #FFD5D5;'>39.3%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk for testicular cancer</div>
        <div class='boxshow' id="show-sec-0.867307787803" style="display: inline;"><a href="javascript:toggle2('sec-0.867307787803')">...more...</a></div>
        <div class='boxrstext' id="sec-0.867307787803" style="display: none;"><a href="javascript:toggle2('sec-0.867307787803')">(hide)</a>
        <div class='rstext'>A study of 577 cases of testicular germ cell tumors, the most common testicular cancer in young men, concluded that the rs2059693(T) allele increased risk. Among Caucasians, the odds ratio for the rs2059693(C;T) genotype was 1.33 (CI: 1.04-1.71), and for the (T;T) genotype, 1.60 (CI: 1.01-2.52, p(trend) = 0.008). The association with rs2059693 was even stronger for nonseminomas, and for teratomas and teratocarcinomas in particular (N = 58; (C;T) odds ratio 1.63, CI: 0.89-2.99; (T;T) odds ratio 4.54, CI: 2.00-10.3, p(trend) = 0.0008).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1042713'>rs1042713</a><a href='http://www.snpedia.com/index.php/rs1042713(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.3<br/>Frequency: <span style='background-color: #FFDCDC;'>46.9%</span><br/>Repute:Bad<br/>References:66<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk that pediatric inhaler use may make asthma worse</div>
        <div class='boxshow' id="show-sec-0.916347121303" style="display: inline;"><a href="javascript:toggle2('sec-0.916347121303')">...more...</a></div>
        <div class='boxrstext' id="sec-0.916347121303" style="display: none;"><a href="javascript:toggle2('sec-0.916347121303')">(hide)</a>
        <div class='rstext'>rs1042713, also known as Arg16, G16R, and 16Arg&gt;Gly, is a SNP in the adrenergic, beta-2-, surface receptor ADRB2 gene. The rs1042713(G) allele encodes the G/glycine form that is the more common one in most populations, while rs1042713(A) encodes the R/arginine residue at this position of the ADRB2 protein. A study of 1,182 young Scottish asthma patients (age, 3-22 years) concluded that rs1042713(A) alleles are significantly associated with 'exacerbations' of their condition, regardless of treatment regimens (odds ratio 1.30, CI: 1.09-1.55, p=0.003). This didn't hold true for patients using inhalers less than once per day; and it was particularly true for those receiving daily inhaled long- or short-acting beta(2)-agonist treatment (OR, 1.64, CI: 1.22-2.20, p=.001). This may therefore in...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2549782'>rs2549782</a><a href='http://www.snpedia.com/index.php/rs2549782(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.3<br/>Frequency: <span style='background-color: #FFDDDD;'>48.2%</span><br/>Repute:Bad<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk for preeclampsia in most populations  Note that this is relevant to the fetal genotype, not the maternal genotype</div>
        <div class='boxshow' id="show-sec-0.749701459456" style="display: inline;"><a href="javascript:toggle2('sec-0.749701459456')">...more...</a></div>
        <div class='boxrstext' id="sec-0.749701459456" style="display: none;"><a href="javascript:toggle2('sec-0.749701459456')">(hide)</a>
        <div class='rstext'>rs2549782 is a SNP in the endoplasmic reticulum aminopeptidase 2 ERAP2 gene. A case-control study of 1103 Chilean maternal-fetal dyads and 1637 unpaired African American samples (836 maternal, 837 fetal) concluded that the fetal minor allele rs2549782(G) was associated with 1.3x increased risk (CI: 1.075-1.619) for pre-eclampsia in the African American population (p = 0.009), but not in the Chilean population. }}</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs356219'>rs356219</a><a href='http://www.snpedia.com/index.php/rs356219(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.3<br/>Frequency: <span style='background-color: #FFDFDF;'>50.8%</span><br/>Repute:Bad<br/>References:14<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk for Parkinson's disease</div>
        <div class='boxshow' id="show-sec-0.345890888178" style="display: inline;"><a href="javascript:toggle2('sec-0.345890888178')">...more...</a></div>
        <div class='boxrstext' id="sec-0.345890888178" style="display: none;"><a href="javascript:toggle2('sec-0.345890888178')">(hide)</a>
        <div class='rstext'>rs356219, a tagging SNP in the SNCA gene for a Parkinson's disease associated haplotype, predicts the amount of SNCA (alpha-synuclein) mRNA found in post mortem substantia nigra and cerebellum brain tissue. replicates the association of rs356219 with late-onset Parkinson's disease, calculating an odds ratio of 1.3 (CI: 1.13 - 1.5, p=2x10e-4) per rs356219(G) allele among the 1145 patients from northern Spain studied. Linkage disequilibrium patterns and tagSNP transferability among European populations. Tau and alpha-synuclein in susceptibility to, and dementia in, Parkinson's disease. Genetic variability in the SNCA gene influences alpha-synuclein levels in the blood and brain. Cerebellar alpha-synuclein levels are decreased in Parkinson's disease and do not correlate with SNCA polymorphism...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10947262'>rs10947262</a><a href='http://www.snpedia.com/index.php/rs10947262(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.3<br/>Frequency: <span style='background-color: #FFF9F9;'>88.5%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk for osteoarthritis</div>
        <div class='boxshow' id="show-sec-0.424539827794" style="display: inline;"><a href="javascript:toggle2('sec-0.424539827794')">...more...</a></div>
        <div class='boxrstext' id="sec-0.424539827794" style="display: none;"><a href="javascript:toggle2('sec-0.424539827794')">(hide)</a>
        <div class='rstext'>rs10947262(C) was found to be associated with increased risk for osteoarthritis among Japanese and Europeans, with an odds ratio of 1.32 (CI: 1.19-1.46, combined p = 6.73&#215;10e-8). This SNP is also in linkage disequilibrium with rs7775228, which was found to be associated in Japanese with osteoarthritis.  [GWAS:Knee osteoarthritis]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1047286'>rs1047286</a><a href='http://www.snpedia.com/index.php/rs1047286(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.3<br/>Repute:Bad<br/>References:12<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x increased risk for age-related macular degeneration</div>
        <div class='boxshow' id="show-sec-0.0295609239792" style="display: inline;"><a href="javascript:toggle2('sec-0.0295609239792')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0295609239792" style="display: none;"><a href="javascript:toggle2('sec-0.0295609239792')">(hide)</a>
        <div class='rstext'>rs1047286 is a SNP in the complement component C3 gene. A 2011 meta-analysis of seven studies concluded that, at least for Caucasians, the rs1047286(C;T) and (T;T) genotypes had 1.27 (CI: 1.15 - 1.41) and 1.70 (CI: 1.27 - 2.11) times higher risk of ARMD than did (G;G) genotypes.  [OMIM:C3 POLYMORPHISM, HAV 4-1 PLUS/MINUS TYPE]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs13387042'>rs13387042</a><a href='http://www.snpedia.com/index.php/rs13387042(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.25<br/>Frequency: <span style='background-color: #FFCCCC;'>31.9%</span><br/>Repute:Bad<br/>References:41<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.24x increased risk for breast cancer</div>
        <div class='boxshow' id="show-sec-0.853793864925" style="display: inline;"><a href="javascript:toggle2('sec-0.853793864925')">...more...</a></div>
        <div class='boxrstext' id="sec-0.853793864925" style="display: none;"><a href="javascript:toggle2('sec-0.853793864925')">(hide)</a>
        <div class='rstext'>rs13387042(A;A) has an estimated 1.44-fold greater risk of breast cancer than the GG genotype. This initial association was later tested among patients in the Breast Cancer Association Consortium, including 31,510 women with invasive breast cancer, 1101 women with ductal carcinoma in situ (DCIS), and 35,969 controls. In Caucasian women of European origin, each A allele is associated with 1.12 higher odds (CI: 1.09-1.15). This association held regardless of ER or PR status.  [PharmGKB:Non-Curated GWAS results: A multistage genome-wide association study in Breast Cancer identifies two new risk alleles at 1p11.2 and 14q24.1 (RAD51L1). (Initial Sample Size: 1,145 cases, 1,142 controls; Replication Sample Size: 8,625 cases, 9,657 controls); (Region: 2q35; Reported Gene(s): Intergenic; Risk Alle...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs748404'>rs748404</a><a href='http://www.snpedia.com/index.php/rs748404(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1.25<br/>Frequency: <span style='background-color: #FFE5E5;'>57.7%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>slightly  increased risk (1.25) for lung cancer</div>
        <div class='boxshow' id="show-sec-0.381840650686" style="display: inline;"><a href="javascript:toggle2('sec-0.381840650686')">...more...</a></div>
        <div class='boxrstext' id="sec-0.381840650686" style="display: none;"><a href="javascript:toggle2('sec-0.381840650686')">(hide)</a>
        <div class='rstext'>In a GWAS dataset of 1,447 lung cancer cases, and then replicated in another 1300 cases, the rs748404(T) allele was significantly correlated with increased risk (odds ratio 1.15, p = 1.1 &#215; 10e-9), in agreement with a previous meta-analysis.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10937273'>rs10937273</a><a href='http://www.snpedia.com/index.php/rs10937273(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFAFAF;'>14.2%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>In Type 2 diabetics, possibly earlier onset of disease  Among a population of Taiwanese type 2 diabetics, this genotype is statistically associated with earlier rather than later onset (of T2DM).</div>
        <div class='boxshow' id="show-sec-0.981488609283" style="display: inline;"><a href="javascript:toggle2('sec-0.981488609283')">...more...</a></div>
        <div class='boxrstext' id="sec-0.981488609283" style="display: none;"><a href="javascript:toggle2('sec-0.981488609283')">(hide)</a>
        <div class='rstext'>[GWAS:Adiponectin levels]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3131296'>rs3131296</a><a href='http://www.snpedia.com/index.php/rs3131296(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFB5B5;'>17.0%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.2x increased risk for schizophrenia</div>
        <div class='boxshow' id="show-sec-0.670868988765" style="display: inline;"><a href="javascript:toggle2('sec-0.670868988765')">...more...</a></div>
        <div class='boxrstext' id="sec-0.670868988765" style="display: none;"><a href="javascript:toggle2('sec-0.670868988765')">(hide)</a>
        <div class='rstext'>23andMe blog schizophrenia *rs3131296(C) increased the odds of schizophrenia by 1.19x *rs12807809(T) increased the odds of schizophrenia by 1.15x *rs9960767(C) increased the odds of schizophrenia by 1.23 times. A study of ~2,500 Han Chinese patients with schizophrenia did find rs3131296 to be significantly associated in this population.  [GWAS:Schizophrenia]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2651899'>rs2651899</a><a href='http://www.snpedia.com/index.php/rs2651899(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFBABA;'>19.5%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.2x higher risk for migraines</div>
        <div class='boxshow' id="show-sec-0.423394466795" style="display: inline;"><a href="javascript:toggle2('sec-0.423394466795')">...more...</a></div>
        <div class='boxrstext' id="sec-0.423394466795" style="display: none;"><a href="javascript:toggle2('sec-0.423394466795')">(hide)</a>
        <div class='rstext'>rs2651899 is a SNP on ch 1p36.32 in the PRDM16 gene. A large GWAS study of over 5,000 patients with migraines and a meta-analysis concluded that this SNP was one of three influencing the condition, albeit with only slight risk increases on its own. The odds ratio for the slightly rarer rs2651899(G) allele was 1.11 (CI: 1.07 - 1.15, p = 3.8 x 10e-9). A 2013 study published by the American Headache Society reports significant influence of rs1835740, LRP1 rs11172113 and PRDM16 rs2651899 polymorphisms on migraine susceptibility in the Northern Indian population. Genome-wide-associated variants in migraine susceptibility: a replication study from North India.  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs16953002'>rs16953002</a><a href='http://www.snpedia.com/index.php/rs16953002(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFC3C3;'>24.8%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.16x increased risk of melanoma in indiiduals with low BMI  Highly significant, replicated studies, p=3.6e-12. Appears to represent a greater risk in those with low BMI, which is plausible due to the known role of FTO in obesity. This SNP is in intron 8, whereas BMI SNP's are usually in intron 1. The association was much stronger for those in the lowest quartile of BMI (1.66 per allele risk) vs those in the other 3 quartiles (1.03 per allele risk - no longer statistically significant).</div>
        <div class='boxshow' id="show-sec-0.290018399099" style="display: inline;"><a href="javascript:toggle2('sec-0.290018399099')">...more...</a></div>
        <div class='boxrstext' id="sec-0.290018399099" style="display: none;"><a href="javascript:toggle2('sec-0.290018399099')">(hide)</a>
        <div class='rstext'>[GWAS:Melanoma]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2814707'>rs2814707</a><a href='http://www.snpedia.com/index.php/rs2814707(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFC8C8;'>28.3%</span><br/>Repute:Bad<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.2x increased risk for ALS</div>
        <div class='boxshow' id="show-sec-0.665444225489" style="display: inline;"><a href="javascript:toggle2('sec-0.665444225489')">...more...</a></div>
        <div class='boxrstext' id="sec-0.665444225489" style="display: none;"><a href="javascript:toggle2('sec-0.665444225489')">(hide)</a>
        <div class='rstext'>rs2814707 is one of 2 SNPs from the ch 9p21.2 region associated in a large GWAS study with ALS risk, the other SNP being rs3849942. In this study, the reported odds ratio for rs2814707(A) was 1.22 (CI: 1.15 - 1.30, p = 4.72 x 10e-10). The odds ratio for the minor rs2814707(A) allele was 1.16 (p=7.45x10e-9). 23andMe blog rs2814707(T) 1.16x odds of ALS rs12608932(C) 1.2x odds of ALS  [PharmGKB:Curated This variant is significantly associated with the efficacy of anti-TNF treatment in rheumatoid arthritis (Adjusted P-value: 0.000002; OR: 5.2 (1.8, 16.7)). The study is a genome-wide association study using the Illumina HapMap300 SNP chip on 89 RA patients prospectively followed after beginning of anti-TNF therapy.] [OMIM:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3849942'>rs3849942</a><a href='http://www.snpedia.com/index.php/rs3849942(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFC9C9;'>29.2%</span><br/>Repute:Bad<br/>References:8<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.2x increased risk for ALS</div>
        <div class='boxshow' id="show-sec-0.190111294336" style="display: inline;"><a href="javascript:toggle2('sec-0.190111294336')">...more...</a></div>
        <div class='boxrstext' id="sec-0.190111294336" style="display: none;"><a href="javascript:toggle2('sec-0.190111294336')">(hide)</a>
        <div class='rstext'>rs3849942 is one of 2 SNPs from the ch 9p21.2 region associated in a large GWAS study with ALS risk, the other SNP being rs2814707. In this study, the reported odds ratio for rs3849942(A) was 1.22 (CI: 1.15 - 1.30, p = 4.64 x 10e-10). The odds ratio for the minor rs3849942(A) allele was 1.15 (p=1.01x10e-8). From a study of 405 Finnish patients with ALS, rs3849942 was found within a 42 SNP haplotype associated with highly increased (p = 7 x 10e-33) risk when patients with familial ALS were compared to controls (odds ratio of 21.0, CI: 11.2-39.1). This haplotype overlapped with a region reported (by others) to associate with risk for frontotemporal dementia. In terms of population attributable risk, for the 93 patients with familial ALS, this 9p21 locus accounts for 38% and that of rs1304801...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs5370'>rs5370</a><a href='http://www.snpedia.com/index.php/rs5370(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFCACA;'>30.1%</span><br/>Repute:Bad<br/>References:16<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>possibly lower HDL cholesterol in women  One report indicates that women (but not men) with this genotype may have slightly lower levels of HDL cholesterol, but other reports have not observed this.</div>
        <div class='boxshow' id="show-sec-0.28574715756" style="display: inline;"><a href="javascript:toggle2('sec-0.28574715756')">...more...</a></div>
        <div class='boxrstext' id="sec-0.28574715756" style="display: none;"><a href="javascript:toggle2('sec-0.28574715756')">(hide)</a>
        <div class='rstext'>rs5370, also known as Lys198Asn or K198N, is a SNP in the endothelin-1 EDN1 gene on chromosome 6. The generally more common rs5370(G) allele encodes the Lys (K), and the rarer rs5370(T) allele encodes the Asn (N). Among the French-Canadian founder population of the Saguenay Lac St-Jean region of Quebec, rs5370(T) was associated with lower HDL cholesterol values in women (p = 1.3 x 10e-5), but not in men. In a study of 1,000+ Western Australians, no association was seen between HDL levels and rs5370. A study of ~400 rheumatoid arthritis patients found a 3x increase in the adjusted odds of being hypertensive for rs5370(T;T) homozygotes compared to (G;G) homozygotes (odds ratio 2.89, CI: 1.02 - 8.19, p=0.011). After adjustment for multiple potential confounders, haplotype analysis revealed an...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs393152'>rs393152</a><a href='http://www.snpedia.com/index.php/rs393152(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFD5D5;'>39.8%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Slight increased risk of both PD and AD</div>
        <div class='boxshow' id="show-sec-0.421522566771" style="display: inline;"><a href="javascript:toggle2('sec-0.421522566771')">...more...</a></div>
        <div class='boxrstext' id="sec-0.421522566771" style="display: none;"><a href="javascript:toggle2('sec-0.421522566771')">(hide)</a>
        <div class='rstext'>[OMIM:?] [GWAS:Parkinson's disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs143383'>rs143383</a><a href='http://www.snpedia.com/index.php/rs143383(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFD9D9;'>43.1%</span><br/>Repute:Bad<br/>References:32<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.1x increased risk for osteoarthritis</div>
        <div class='boxshow' id="show-sec-0.182830020814" style="display: inline;"><a href="javascript:toggle2('sec-0.182830020814')">...more...</a></div>
        <div class='boxrstext' id="sec-0.182830020814" style="display: none;"><a href="javascript:toggle2('sec-0.182830020814')">(hide)</a>
        <div class='rstext'>This SNP is associated with osteoarthritis (OA). It is located in the ''five prime untranslated region'' ('''5&#8242;UTR''') of the gene encoding ''growth differentiation factor 5'' ('''GDF5'''), a chondrogenic protein active from the embryonic stage onwards. GDF5 is also known as ''cartilage-derived morphogenetic protein 1'' or ''BMP14''. The '''risk allele T''' (+ 104T/C;rs143383) causes reduction of the GDF5 promoter sequence activity. Reduction of GDF5 in human cartilage of patients with OA by up to 27% has been observed This effect is '''influenced by a second SNP ( rs143384] , C/T )''' in the same area. The C alleles of both SNPs form CpG dinucleotides. Demethylation of both SNP's increases GDF5 expression. Thus the authors conclude that epigenetic manipulation offers options in deve...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4795067'>rs4795067</a><a href='http://www.snpedia.com/index.php/rs4795067(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFD9D9;'>43.4%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>slight increase in risk for psoriatic arthritis</div>
        <div class='boxshow' id="show-sec-0.478083077907" style="display: inline;"><a href="javascript:toggle2('sec-0.478083077907')">...more...</a></div>
        <div class='boxrstext' id="sec-0.478083077907" style="display: none;"><a href="javascript:toggle2('sec-0.478083077907')">(hide)</a>
        <div class='rstext'>reports rs4795067, with risk/non-risk alleles G/A, has an odds ratio of 1.2 (p=2e&#8722;7) for psoriatic arthritis based on a case/control study of ~2,000 patients. Nitric oxide synthase genes and their interactions with environmental factors in Parkinson's disease. Lead exposure, polymorphisms in genes related to oxidative stress, and risk of adult brain tumors.  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4977756'>rs4977756</a><a href='http://www.snpedia.com/index.php/rs4977756(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFDDDD;'>47.8%</span><br/>Repute:Bad<br/>References:16<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.2x higher risk for glioma development</div>
        <div class='boxshow' id="show-sec-0.0461877819086" style="display: inline;"><a href="javascript:toggle2('sec-0.0461877819086')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0461877819086" style="display: none;"><a href="javascript:toggle2('sec-0.0461877819086')">(hide)</a>
        <div class='rstext'>rs4977756 is a SNP near the CDKN2BAS gene. A (GWA) study for open-angle glaucoma (OAG) blindness ultimately based on ~1500 cases concluded that the rather common rs4977756(A) allele was statistically associated with the condition, with a per allele odds ratio of 1.39 (CI: 1.28&#8211;1.51, p = 1.35 &#215; 10e?14).  [OMIM:?] [GWAS:Glioma]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11037909'>rs11037909</a><a href='http://www.snpedia.com/index.php/rs11037909(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFE4E4;'>55.8%</span><br/>Repute:Bad<br/>References:12<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.47x type II diabetes risk</div>
        <div class='boxshow' id="show-sec-0.551806922248" style="display: inline;"><a href="javascript:toggle2('sec-0.551806922248')">...more...</a></div>
        <div class='boxrstext' id="sec-0.551806922248" style="display: none;"><a href="javascript:toggle2('sec-0.551806922248')">(hide)</a>
        <div class='rstext'>rs11037909 increases susceptibility to Type II Diabetes 1.27 times for heterozygotes (CT) and 1.47 times for homozygotes (TT)</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3740878'>rs3740878</a><a href='http://www.snpedia.com/index.php/rs3740878(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFE4E4;'>55.8%</span><br/>Repute:Bad<br/>References:14<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.46x type II diabetes risk; common</div>
        <div class='boxshow' id="show-sec-0.357557720339" style="display: inline;"><a href="javascript:toggle2('sec-0.357557720339')">...more...</a></div>
        <div class='boxrstext' id="sec-0.357557720339" style="display: none;"><a href="javascript:toggle2('sec-0.357557720339')">(hide)</a>
        <div class='rstext'>rs3740878 increases susceptibility to Type II Diabetes 1.26 times for heterozygotes (AG) and 1.46 times for homozygotes (AA)</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs8050136'>rs8050136</a><a href='http://www.snpedia.com/index.php/rs8050136(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFE4E4;'>56.6%</span><br/>Repute:Bad<br/>References:118<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.2x increased risk for T2D in some populations</div>
        <div class='boxshow' id="show-sec-0.708690320567" style="display: inline;"><a href="javascript:toggle2('sec-0.708690320567')">...more...</a></div>
        <div class='boxrstext' id="sec-0.708690320567" style="display: none;"><a href="javascript:toggle2('sec-0.708690320567')">(hide)</a>
        <div class='rstext'>This SNP is in a linkage block in the FTO gene with rs1121980; see rs1121980 for the association of this block with early onset obesity since it showed the strongest association. This SNP did not show any association with obesity, type-2 diabetes or any other related traits in a study of 3,210 Chinese subjects. Furthermore, the minor allele frequency was much lower in Chinese populations compared to populations of European descent. However, a more recent study of 4,189 Chinese Han subjects combined with a meta-analysis of published studies concluded that rs8050136(A) was associated with both increased risk for type-2 diabetes and obesity. rs8050136 associated with increased food intake, but not of impaired energy expenditure.  [PharmGKB:Non-Curated GWAS results: Genome-wide association yie...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2223841'>rs2223841</a><a href='http://www.snpedia.com/index.php/rs2223841(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFEDED;'>69.2%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>more likely to go bald before age 40  more likely to go bald before age 40</div>
        <div class='boxshow' id="show-sec-0.0397450532741" style="display: inline;"><a href="javascript:toggle2('sec-0.0397450532741')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0397450532741" style="display: none;"><a href="javascript:toggle2('sec-0.0397450532741')">(hide)</a>
        <div class='rstext'>male-pattern baldness. Full text available. *G less likely to go bald before age 40 *A more likely to go bald before age 40</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6625163'>rs6625163</a><a href='http://www.snpedia.com/index.php/rs6625163(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFEFEF;'>72.6%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk of baldness  more likely to go bald</div>
        <div class='boxshow' id="show-sec-0.87501754011" style="display: inline;"><a href="javascript:toggle2('sec-0.87501754011')">...more...</a></div>
        <div class='boxrstext' id="sec-0.87501754011" style="display: none;"><a href="javascript:toggle2('sec-0.87501754011')">(hide)</a>
        <div class='rstext'>when used as part of gs122 this snp can influence male pattern baldness. Initial Sample Size: 578 cases, 547 controls Replication Sample Size: 1,351 cases, 2,485 controls Alopecia is readable, while the nature article is behind a paywall. This is one of the snps in gs122 Male Pattern Baldness  [PharmGKB:Non-Curated GWAS results: Male-pattern baldness susceptibility locus at 20p11. (Initial Sample Size: 578 cases, 547 controls; Replication Sample Size: 1,351 cases, 2,485 controls); (Region: Xq12; Reported Gene(s): AR; Risk Allele: rs6625163-A); (p-value= 0.00000000005).This variant is associated with Male-pattern baldness.] [GWAS:Male-pattern baldness]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6036025'>rs6036025</a><a href='http://www.snpedia.com/index.php/rs6036025(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.2<br/>Frequency: <span style='background-color: #FFF6F6;'>84.1%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>more likely to go bald  more likely to go bald</div>
        <div class='boxshow' id="show-sec-0.432313228385" style="display: inline;"><a href="javascript:toggle2('sec-0.432313228385')">...more...</a></div>
        <div class='boxrstext' id="sec-0.432313228385" style="display: none;"><a href="javascript:toggle2('sec-0.432313228385')">(hide)</a>
        <div class='rstext'>Along with rs201571 serves as a proxy for rs1160312. related to baldness</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3802842'>rs3802842</a><a href='http://www.snpedia.com/index.php/rs3802842(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 1.17<br/>Frequency: <span style='background-color: #FFD0D0;'>34.5%</span><br/>Repute:Bad<br/>References:31<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.17x increased risk of colorectal cancer</div>
        <div class='boxshow' id="show-sec-0.231623790136" style="display: inline;"><a href="javascript:toggle2('sec-0.231623790136')">...more...</a></div>
        <div class='boxrstext' id="sec-0.231623790136" style="display: none;"><a href="javascript:toggle2('sec-0.231623790136')">(hide)</a>
        <div class='rstext'>rs3802842 is a SNP in a region of ch 11q23.1 that has been associated with some cancers. IN a study of over 10,000 cases of colorectal cancer, rs3802842(C) was associated with increased risk (odds ratio 1.17, CI: 1.12-1.22, p = 1.08 x 10e-12). Additional SNPs in chromosomal regions 8q21, 8q24, 10p14, 11q, 15q13.3 and 18q21 can be combined additively to produce an overall risk, with an average odds ratio per risk allele of 1.19.  [PharmGKB:Non-Curated GWAS Results: Genome-wide association scan identifies a colorectal cancer susceptibility locus on 11q23 and replicates risk loci at 8q24 and 18q21 (Initial Sample Size: 981 cases, 1,002 controls; Replication Sample Size: 16,476 cases, 15,351 controls; Risk Allele: rs3802842-C).] [OMIM:COLORECTAL CANCER, SUSCEPTIBILITY TO, 7; CRCS7] [GWAS:None]...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs17465637'>rs17465637</a><a href='http://www.snpedia.com/index.php/rs17465637(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 1.17<br/>Frequency: <span style='background-color: #FFDCDC;'>46.4%</span><br/>Repute:Bad<br/>References:18<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.17x higher risk for myocardial infarction</div>
        <div class='boxshow' id="show-sec-0.544589848647" style="display: inline;"><a href="javascript:toggle2('sec-0.544589848647')">...more...</a></div>
        <div class='boxrstext' id="sec-0.544589848647" style="display: none;"><a href="javascript:toggle2('sec-0.544589848647')">(hide)</a>
        <div class='rstext'>rs17465637 is a SNP found to be associated with heart disease by the German MI (Myocardial infarction) Family Study group This 2011 study concluded that the only MIA3 gene SNP to be associated with increased risk for myocardial infarction is rs17465637, with an odds ratio of 1.17 (CI: 1.04-1.32) and 1.37 (CI: 1.08-1.74) for carriers of one or two (C) alleles, respectively. 23andMe blog coronary artery disease and heart attack SNP Risk Version Effect *rs646776 T 1.19 *rs17465637 C 1.14 *rs1746048 C 1.17 *rs6725887 C 1.17 *rs11206510 T 1.15 *rs3184504 T 1.13 *rs2306374 C 1.15 *rs3782886 C 1.44  [PharmGKB:Curated GWAS results: Genome-wide association of early-onset myocardial infarction with single nucleotide polymorphisms and copy number variants. (Initial Sample Size: 2,967 cases, 3,075 con...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs889312'>rs889312</a><a href='http://www.snpedia.com/index.php/rs889312(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FF9C9C;'>7.7%</span><br/>Repute:Bad<br/>References:49<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Very slightly higher risk for breast cancer</div>
        <div class='boxshow' id="show-sec-0.689840380913" style="display: inline;"><a href="javascript:toggle2('sec-0.689840380913')">...more...</a></div>
        <div class='boxrstext' id="sec-0.689840380913" style="display: none;"><a href="javascript:toggle2('sec-0.689840380913')">(hide)</a>
        <div class='rstext'>rs889312 is a SNP near the MAP3K1 gene. rs889312 was one of the four strongest associating SNPs found in a genome-wide association study of over 4,000 breast cancer samples. . From a DeCode user report, it appears that the risk allele is C; this A;A individual reportedly has 0.93x risk of breast cancer. In a study of 1,267 breast cancer patients, rs889312 heterozygotes and minor allele homozygotes were less likely to be lymph node positive at breast cancer diagnosis (P = 0.044) relative to major allele homozygote carriers. A meta-analysis including 26,015 cases and 33,962 controls concludes that the rs889312(C) allele is clearly a significant - but quite weak - contributor to increased risk for breast cancer. breast cancer *rs2981582, rs3803662, and rs889312) showed weak but significant as...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs249954'>rs249954</a><a href='http://www.snpedia.com/index.php/rs249954(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFC9C9;'>29.5%</span><br/>Repute:Bad<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>slight if any increased risk of Breast Cancer</div>
        <div class='boxshow' id="show-sec-0.00778272505688" style="display: inline;"><a href="javascript:toggle2('sec-0.00778272505688')">...more...</a></div>
        <div class='boxrstext' id="sec-0.00778272505688" style="display: none;"><a href="javascript:toggle2('sec-0.00778272505688')">(hide)</a>
        <div class='rstext'>rs249954, also known as 2586+58C&gt;T, is an intronic SNP in the PALB2 gene. There is conflicting literature about the association of this SNP with increased risk for breast cancer, but overall, it seems as if any association between rs249954 and breast cancer is likely to be either nonexistent or at least fairly small. Specifically: * ClinVar tags this SNP as benign, based on a single submission * A 2008 study of 1,000 Chinese with breast cancer reports a 36% increase in breast cancer risk (odds ratio 1.36, CI: 1.13-1.64, p = 0.001; TT/TC versus CC genotypes) * A 2010 study of 660 Chinese patients did not find an association for this SNP * A 2010 study of Polish patients reported an allele frequency of 5.7% (4 out of 70) in those patients but did not report the frequency in matched contro...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4324715'>rs4324715</a><a href='http://www.snpedia.com/index.php/rs4324715(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFD8D8;'>42.5%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.5x increased testicular cancer risk for men</div>
        <div class='boxshow' id="show-sec-0.985184718466" style="display: inline;"><a href="javascript:toggle2('sec-0.985184718466')">...more...</a></div>
        <div class='boxrstext' id="sec-0.985184718466" style="display: none;"><a href="javascript:toggle2('sec-0.985184718466')">(hide)</a>
        <div class='rstext'>Testicular germ cell tumor risk I('TGCT'; i.e. testicular cancer risk) was increased 40% per copy of the major (T) allele at rs4324715 and rs6897876 (OR = 1.37, CI: 1.14&#8211;1.64; OR = 1.39, CI: 1.16&#8211;1.66, respectively), based on a genome-wide scan of 277 primarily Caucasian TGCT cases and 919 controls. Men who have two copies of the common version of the sprouty 4 SPRY4 gene have a 1.5-fold higher risk of testicular cancer than men who have two copies of the less common or minor version of the gene. [doi|10.1038/ng.393]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6897876'>rs6897876</a><a href='http://www.snpedia.com/index.php/rs6897876(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFDDDD;'>47.8%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>3x increased testicular cancer risk for men</div>
        <div class='boxshow' id="show-sec-0.233591649772" style="display: inline;"><a href="javascript:toggle2('sec-0.233591649772')">...more...</a></div>
        <div class='boxrstext' id="sec-0.233591649772" style="display: none;"><a href="javascript:toggle2('sec-0.233591649772')">(hide)</a>
        <div class='rstext'>Testicular germ cell tumor risk I('TGCT'; i.e. testicular cancer risk) was increased 40% per copy of the major allele (C) at rs4324715 and rs6897876 (OR = 1.37, CI: 1.14&#8211;1.64; OR = 1.39, CI: 1.16&#8211;1.66, respectively), based on a genome-wide scan of 277 primarily Caucasian TGCT cases and 919 controls. Men who have two copies of the common version of the sprouty 4 SPRY4 gene have a 1.5-fold higher risk of testicular cancer than men who have two copies of the less common or minor version of the gene. [doi|10.1038/ng.393]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4977574'>rs4977574</a><a href='http://www.snpedia.com/index.php/rs4977574(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFDEDE;'>49.6%</span><br/>Repute:Bad<br/>References:28<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Some studies - but not others - report a slightly increased risk for myocardial infarction</div>
        <div class='boxshow' id="show-sec-0.119097858468" style="display: inline;"><a href="javascript:toggle2('sec-0.119097858468')">...more...</a></div>
        <div class='boxrstext' id="sec-0.119097858468" style="display: none;"><a href="javascript:toggle2('sec-0.119097858468')">(hide)</a>
        <div class='rstext'>rs4977574 increases susceptibility to Coronary artery disease 1.30 times for heterozygotes (A;G) and 1.54 times for homozygotes (G;G) rs4977574 was found to be associated with myocardial infarction based on a study by the eMERGE network in which electronic medical records were searched to find clinical associations associated with selected SNPs. . In this study of 2,000 patients, rs4977574(G;G) homozygotes had an increased risk (odds ratio) for myocardial infarction of 1.84 (CI: 1.14&#8722;3.09, p = 0.011)  [PharmGKB:Curated GWAS results: Genome-wide association of early-onset myocardial infarction with single nucleotide polymorphisms and copy number variants. (Initial Sample Size: 2,967 cases, 3,075 controls; Replication Sample Size: 9,746 cases, 9,746 controls); (Region: 9p21.3; Reported...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1344706'>rs1344706</a><a href='http://www.snpedia.com/index.php/rs1344706(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFE1E1;'>52.3%</span><br/>Repute:Bad<br/>References:41<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.1x increased risk for schizophrenia</div>
        <div class='boxshow' id="show-sec-0.0161108344344" style="display: inline;"><a href="javascript:toggle2('sec-0.0161108344344')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0161108344344" style="display: none;"><a href="javascript:toggle2('sec-0.0161108344344')">(hide)</a>
        <div class='rstext'>rs1344706 is a SNP in the ZNF804A gene, and it is probably the first SNP to have been reported as achieving genome-wide significance for psychosis. Reviews the literature, and concludes that rs1344706 is 'robustly, if modestly' associated with increased risk for schizophrenia. A genome-wide association study of schizophrenia (479 cases, 2,937 controls) followed by additional testing and a meta-analysis concluded that the strongest evidence for association was around ZNF804A (rs1344706; p = 1.61 x 10e-7) and that the finding was strengthened when the affected phenotype included bipolar disorder (p = 9.96 x 10e-9). Executive function, neural circuitry, and genetic mechanisms in schizophrenia. Apoptotic engulfment pathway and schizophrenia. Persistence criteria for susceptibility genes for sc...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7171755'>rs7171755</a><a href='http://www.snpedia.com/index.php/rs7171755(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFE6E6;'>58.4%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>very slight decrease in cortical thickness and IQ; see full text</div>
        <div class='boxshow' id="show-sec-0.558643467847" style="display: inline;"><a href="javascript:toggle2('sec-0.558643467847')">...more...</a></div>
        <div class='boxrstext' id="sec-0.558643467847" style="display: none;"><a href="javascript:toggle2('sec-0.558643467847')">(hide)</a>
        <div class='rstext'>rs7171755 is a SNP near the synaptic cell adhesion glycoprotein-encoding NPTN gene. A study of ~1,500 adolescents determined that the rs7171755(A) allele is associated with a slightly thinner left hemisphere cortex and also a slight decrease in verbal and nonverbal intelligence (IQ). The differences were really quite slight; each copy of the risk allele decreased cortical thickness by 0.7%, and decreased verbal and nonverbal IQ by 1.8 and 1.4 points, respectively. This study awaits independent replication. Also, note that eight SNPs in the NPTN locus were reported as jointly showing significant association with cortical thickness in addition to rs7171755: rs7176637, rs11854138, rs8028749, rs12185108, rs1564492, rs899981, rs7178269 and rs4075802.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3758549'>rs3758549</a><a href='http://www.snpedia.com/index.php/rs3758549(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFE8E8;'>61.9%</span><br/>Repute:Bad<br/>References:11<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased risk of parkinson's disease  While this does indicate increased risk of in one Parkinson's disease study. It is so common that it makes more sense to think of the other 2 genotypes as rare risk reducers.</div>
        <div class='boxshow' id="show-sec-0.885542885908" style="display: inline;"><a href="javascript:toggle2('sec-0.885542885908')">...more...</a></div>
        <div class='boxrstext' id="sec-0.885542885908" style="display: none;"><a href="javascript:toggle2('sec-0.885542885908')">(hide)</a>
        <div class='rstext'>CC linked to parkinson's disease We provide evidence for a novel, strong and reproducible association of the PITX3 promoter SNP rs3758549 : C&gt;T (p=0.004) with PD. The C-allele appears to be a recessive risk allele with an estimated population frequency of 83%.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2653349'>rs2653349</a><a href='http://www.snpedia.com/index.php/rs2653349(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFEEEE;'>70.5%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2-6x increased risk for cluster headaches</div>
        <div class='boxshow' id="show-sec-0.849400537818" style="display: inline;"><a href="javascript:toggle2('sec-0.849400537818')">...more...</a></div>
        <div class='boxrstext' id="sec-0.849400537818" style="display: none;"><a href="javascript:toggle2('sec-0.849400537818')">(hide)</a>
        <div class='rstext'>rs2653349, also known as G1246A, is a SNP in the HCRTR2 hypocretin-orexin gene. It has been linked primarily to cluster headaches. The more common allele (G) has been linked to increased risk; alternatively, this could be seen as implying that the less common (A) allele reduces cluster headache risk. In the (first) study, 100+ Italian patients were analyzed to reveal that rs2653349(G;G) genotypes were at 5x increased risk for the disorder, compared with (A;G) and (A;A) genotypes. A study of 200+ German patients indicated that (G;G) homozygotes were at 2x increased risk (CI: 1.32-2.92, p = 0.0007) for cluster headaches. A meta-analysis of 3 pooled studies concluded that the rs2653349(G;G) genotype showed a higher disease risk compared to the remaining genotypes (odds ratio 1.69, CI:1.11-2.5...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs925391'>rs925391</a><a href='http://www.snpedia.com/index.php/rs925391(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFF6F6;'>83.1%</span><br/>Repute:Bad<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>more likely to go bald; common</div>
        <div class='boxshow' id="show-sec-0.283748110655" style="display: inline;"><a href="javascript:toggle2('sec-0.283748110655')">...more...</a></div>
        <div class='boxrstext' id="sec-0.283748110655" style="display: none;"><a href="javascript:toggle2('sec-0.283748110655')">(hide)</a>
        <div class='rstext'>23andMe seems to consider each A at rs925391 to have lowered men's odds of baldness by 11x. baldness *lowest P value of 2.1&#215;10-12 for rs10521339 *family based rs938059 shows the lowest P value (4.03&#215;10-6) (table 3) *rs6152 (P=6.66&#215;10-10)</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11209026'>rs11209026</a><a href='http://www.snpedia.com/index.php/rs11209026(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFFAFA;'>91.8%</span><br/>Repute:Bad<br/>References:103<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal, but higher risk for certain autoimmune diseases.  Almost everyone has this form, but it indicates a slightly higher risk of Ankylosing Spondylitis, Crohn's Disease (1.09x), Psoriasis (1.06x), and Ulcerative Colitis (1.06x). The risk is much greater than for the rare version. This is the interleukin-23 (IL-23) receptor. Media:http://www.the-rheumatologist.org/details/article/2528901/Neutralizing_IL-23_and_Its_Targets_May_Improve_Ankylosing_Spondylitis.html] Neutralizing IL-23 and Its Targets May Improve Ankylosing Spondylitis by Lara C. Pullen, PhD in The Rheumatologist newsmagazine ]</div>
        <div class='boxshow' id="show-sec-0.282408009442" style="display: inline;"><a href="javascript:toggle2('sec-0.282408009442')">...more...</a></div>
        <div class='boxrstext' id="sec-0.282408009442" style="display: none;"><a href="javascript:toggle2('sec-0.282408009442')">(hide)</a>
        <div class='rstext'>A relatively rare allele at SNP rs11209026, in the IL23R gene, appears to provide a fairly strong protective effect against the development of Crohn's disease in both Jewish and non-Jewish populations. Note that in this study the (A;A) and (A;G) genotypes were combined for statistical reasons (there were too few (A;A) individuals to study by themselves), so the risk is shown as equal for both. associated With Inflammatory Bowel Diseases but Not With Celiac Disease rs11209026 had a protective effect for IBD in the case-control analysis (odds ratio [OR] 0.19, 95% confidence interval [CI] 0.10-0.37, P= 6.6E-09). Both CD (OR 0.14, CI 0.06-0.37, P= 3.9E-07) and UC (OR 0.33, CI 0.15-0.73, P= 1.4E-03) were associated with IL23R. rs2241880 was associated with CD susceptibility (OR 1.36, CI 1.12-1....</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs34516635'>rs34516635</a><a href='http://www.snpedia.com/index.php/rs34516635(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Less longevity for Ashkenazi Jewish women.</div>
        <div class='boxshow' id="show-sec-0.596507018231" style="display: inline;"><a href="javascript:toggle2('sec-0.596507018231')">...more...</a></div>
        <div class='boxrstext' id="sec-0.596507018231" style="display: none;"><a href="javascript:toggle2('sec-0.596507018231')">(hide)</a>
        <div class='rstext'>rs34516635 is one of a small number of (rare) SNPs in the IGF1R gene found in a study of 384 Ashkenazi centenarians to potentially be associated with longevity and ageing. The rs34516635(A) allele was found somewhat more frequently (p=0.02) in female centenarians than in controls. No association was found for males. Higher IGF1 levels and an average decrease in maximal lifetime height of 2.5cm were also reported to be associated with (female Ashkenazi) centenarians.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7412'>rs7412</a><a href='http://www.snpedia.com/index.php/rs7412(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Repute:Bad<br/>References:35<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>more likely to gain weight if taking olanzapine</div>
        <div class='boxshow' id="show-sec-0.995675666853" style="display: inline;"><a href="javascript:toggle2('sec-0.995675666853')">...more...</a></div>
        <div class='boxrstext' id="sec-0.995675666853" style="display: none;"><a href="javascript:toggle2('sec-0.995675666853')">(hide)</a>
        <div class='rstext'>The ancestral allele is C. The rs7412(T) allele, also known as Arg176Cys, generally indicates the presence of an Apo&#949;2 allele; see the ApoE page for a full discussion of the ApoE alleles and their association with Alzheimer's disease. Another SNP related to ApoE is rs429358. In a study of 67 mostly Caucasian patients prescribed the atypical antipsychotic olanzapine, carriers of a rs7412(C) allele were more likely to gain significant weight compared to rs7412(T;T) carriers, as assessed by physiogenomic analysis of corresponding weight profiles. Two other SNPs, rs5092 and rs4765623, were also significantly associated with weight profiles in these patients. Current limitations of SNP data from the public domain for studies of complex disorders: a test for ten candidate genes for obesity ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11110912'>rs11110912</a><a href='http://www.snpedia.com/index.php/rs11110912(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #F0CAF0;'>61.1%</span><br/>Repute:Bad<br/>References:1<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.3x high blood pressure risk</div>
        <div class='boxshow' id="show-sec-0.904765279534" style="display: inline;"><a href="javascript:toggle2('sec-0.904765279534')">...more...</a></div>
        <div class='boxrstext' id="sec-0.904765279534" style="display: none;"><a href="javascript:toggle2('sec-0.904765279534')">(hide)</a>
        <div class='rstext'>rs11110912 has been reported in a large study to be associated with high blood pressure. The risk allele (oriented to the dbSNP entry) is (G); the odds ratio associated with heterozygotes is 1.33 (CI 1.18-1.51), and for homozygotes, 1.34 (CI 0.96-1.86).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2291834'>rs2291834</a><a href='http://www.snpedia.com/index.php/rs2291834(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.05<br/>Frequency: <span style='background-color: #FFDBDB;'>46.0%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>very slightly higher risk for myocardial infarction</div>
        <div class='boxshow' id="show-sec-0.900804935187" style="display: inline;"><a href="javascript:toggle2('sec-0.900804935187')">...more...</a></div>
        <div class='boxrstext' id="sec-0.900804935187" style="display: none;"><a href="javascript:toggle2('sec-0.900804935187')">(hide)</a>
        <div class='rstext'>rs2291834 is a SNP in the MIA3 gene. Several studies, most notably perhaps by deCode scientists, have shown a (very slightly) elevated risk for myocardial infarctions to be associated with the rs2291834(C) allele.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3735684'>rs3735684</a><a href='http://www.snpedia.com/index.php/rs3735684(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FF7373;'>1.5%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>associated with increased colorectal cancer risk</div>
        <div class='boxshow' id="show-sec-0.749759986512" style="display: inline;"><a href="javascript:toggle2('sec-0.749759986512')">...more...</a></div>
        <div class='boxrstext' id="sec-0.749759986512" style="display: none;"><a href="javascript:toggle2('sec-0.749759986512')">(hide)</a>
        <div class='rstext'></div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11974297'>rs11974297</a><a href='http://www.snpedia.com/index.php/rs11974297(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFB7B7;'>17.7%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Migraine  A four-marker haplotype in DDC was specific to migraine with Aura (rs2329340A, rs11974297C, rs2044859T, rs11761683G)</div>
        <div class='boxshow' id="show-sec-0.905358050559" style="display: inline;"><a href="javascript:toggle2('sec-0.905358050559')">...more...</a></div>
        <div class='boxrstext' id="sec-0.905358050559" style="display: none;"><a href="javascript:toggle2('sec-0.905358050559')">(hide)</a>
        <div class='rstext'>a haplotype consisting of rs2329340(A), rs11974297(C), rs2044859(T) and rs11761683(G) associated with increased incidence of migraine with aura among a sample of 528 migraine patients (308 without aura, 220 with aura) and 528 sex-matched migraine-free controls</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6974491'>rs6974491</a><a href='http://www.snpedia.com/index.php/rs6974491(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFC3C3;'>25.0%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Higher risk of coeliac and/or inflammatory bowel disease</div>
        <div class='boxshow' id="show-sec-0.545558147406" style="display: inline;"><a href="javascript:toggle2('sec-0.545558147406')">...more...</a></div>
        <div class='boxrstext' id="sec-0.545558147406" style="display: none;"><a href="javascript:toggle2('sec-0.545558147406')">(hide)</a>
        <div class='rstext'>[GWAS:Celiac disease]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2546890'>rs2546890</a><a href='http://www.snpedia.com/index.php/rs2546890(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFC9C9;'>29.2%</span><br/>Repute:Bad<br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Higher risk of multiple sclerosis</div>
        <div class='boxshow' id="show-sec-0.620730063158" style="display: inline;"><a href="javascript:toggle2('sec-0.620730063158')">...more...</a></div>
        <div class='boxrstext' id="sec-0.620730063158" style="display: none;"><a href="javascript:toggle2('sec-0.620730063158')">(hide)</a>
        <div class='rstext'>23andMe blog multiple sclerosis rs2546890	IL12B	G	0.87  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6932590'>rs6932590</a><a href='http://www.snpedia.com/index.php/rs6932590(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFC9C9;'>29.2%</span><br/>Repute:Bad<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.1x increased risk for schizophrenia</div>
        <div class='boxshow' id="show-sec-0.719610993107" style="display: inline;"><a href="javascript:toggle2('sec-0.719610993107')">...more...</a></div>
        <div class='boxrstext' id="sec-0.719610993107" style="display: none;"><a href="javascript:toggle2('sec-0.719610993107')">(hide)</a>
        <div class='rstext'>rs6932590 is a SNP that is postulated to have a role in conferring increased susceptibility for schizophrenia. rs6932590 lies in the major histocompatibility complex (MHC) region on chromosome 6, a region that codes for several genes with significant roles in autoimmunity, such as human leukocyte antigens. Specifically, rs6932590 lies in an intergenic region between PRSS16 (a serine protease expressed in the thymus that plays a role in positive T cell selection) and POM121L2 (a membrane glycoprotein) . The T allele at rs6932590 purportedly confers an increased risk for schizophrenia, with reported odds ratios of 1.16 and 1.31 . === Evidence === One large-scale case-controlled genome-wide association study (GWAS) that examined differential genotypes in individuals with schizophrenia, found ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1799990'>rs1799990</a><a href='http://www.snpedia.com/index.php/rs1799990(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDADA;'>44.2%</span><br/>Repute:Bad<br/>References:18<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased chance of Prion Disease (PrP 129 Met homozygote)  This genotype encodes for a homozygous Methionine at position 129 of PRNP, the Prion Protein gene (PrP-129MM). Unlike the protective M/V heterozgotes, homozygosity at this residue is associated with an increased susceptibility to prion disease (such as Creutzfeldt-Jakob) All reported cases of vCJD, (linked to ingestion of 'mad cow disease' contaminated meat) have been 129 Met homozygous</div>
        <div class='boxshow' id="show-sec-0.882355470652" style="display: inline;"><a href="javascript:toggle2('sec-0.882355470652')">...more...</a></div>
        <div class='boxrstext' id="sec-0.882355470652" style="display: none;"><a href="javascript:toggle2('sec-0.882355470652')">(hide)</a>
        <div class='rstext'>rs1799990, also known as Met129Val or M129V, is a SNP in the prion protein PRNP gene. The more common rs1799990(A) allele encodes the Met (methionine). Historically, since 1991 this SNP has been associated with risk for sporadic Creutzfeldt-Jakob disease, with homozygotes of both types apparently at increased risk compared to heterozygotes. More recently, this SNP has been related to long-term memory with (A;A) and (A;G) individuals recalling 17% more words than (G;G) individuals at 24h following learning. A further study of 12 individuals of each of the 3 possible genotypes showed that at both 30 minutes and 24 hour time lags, correlations between retrieval-related brain activity and retrieval success were negative for (G;) homozygotes (the more activity, the worse retrieval success), whi...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs12593813'>rs12593813</a><a href='http://www.snpedia.com/index.php/rs12593813(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDBDB;'>45.5%</span><br/>Repute:Bad<br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>normal risk of restless leg syndrome</div>
        <div class='boxshow' id="show-sec-0.798102193235" style="display: inline;"><a href="javascript:toggle2('sec-0.798102193235')">...more...</a></div>
        <div class='boxrstext' id="sec-0.798102193235" style="display: none;"><a href="javascript:toggle2('sec-0.798102193235')">(hide)</a>
        <div class='rstext'>rs12593813, a SNP located in a region of chromosome 15q, has been linked to a lower frequency of restless legs syndrome, a common sleep disorder, with an overall odds ratio of 0.71 (CI: 0.60-0.83) for the (A) minor allele. Restless Legs Syndrome: Preliminary Research  [PharmGKB:Curated GWAS Results: Genome-wide association study of restless legs syndrome identifies common variants in three genomic regions (Initial Sample Size: 401 cases, 1,644 controls; Replication Sample Size: 1,158 cases, 1,178 controls; Risk Allele: rs12593813-G).] [GWAS:None] [GWAS:Restless legs syndrome]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11635424'>rs11635424</a><a href='http://www.snpedia.com/index.php/rs11635424(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDCDC;'>46.9%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>normal risk of restless leg syndrome</div>
        <div class='boxshow' id="show-sec-0.244189775681" style="display: inline;"><a href="javascript:toggle2('sec-0.244189775681')">...more...</a></div>
        <div class='boxrstext' id="sec-0.244189775681" style="display: none;"><a href="javascript:toggle2('sec-0.244189775681')">(hide)</a>
        <div class='rstext'>rs11635424, a SNP located in a region of chromosome 15q, has been linked to a lower frequency of restless legs syndrome, a common sleep disorder, with an overall odds ratio of 0.70 (CI: 0.59-0.82) for the (A) minor allele.  [PharmGKB:Curated In replicated GWAS case-control studies of Caucasian(European and French-Canadian) familial and sporadic RLS, the G allele of rs11635424 was significantly associated with Restless Legs Syndrome.]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs3784709'>rs3784709</a><a href='http://www.snpedia.com/index.php/rs3784709(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDCDC;'>46.9%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing restless legs syndrome</div>
        <div class='boxshow' id="show-sec-0.693070773272" style="display: inline;"><a href="javascript:toggle2('sec-0.693070773272')">...more...</a></div>
        <div class='boxrstext' id="sec-0.693070773272" style="display: none;"><a href="javascript:toggle2('sec-0.693070773272')">(hide)</a>
        <div class='rstext'>rs3784709, a SNP located in a region of chromosome 15q, has been linked to a lower frequency of restless legs syndrome, a common sleep disorder, with an overall odds ratio of 0.71 (CI: 0.60-0.83) for the (T) minor allele.  [PharmGKB:Curated In replicated GWAS case-control studies of Caucasian(European and French-Canadian) familial and sporadic RLS, the C allele of rs3784709 was significantly associated with Restless Legs Syndrome.]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1026732'>rs1026732</a><a href='http://www.snpedia.com/index.php/rs1026732(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDCDC;'>47.3%</span><br/>Repute:Bad<br/>References:4<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>normal risk of restless leg syndrome</div>
        <div class='boxshow' id="show-sec-0.0272610029554" style="display: inline;"><a href="javascript:toggle2('sec-0.0272610029554')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0272610029554" style="display: none;"><a href="javascript:toggle2('sec-0.0272610029554')">(hide)</a>
        <div class='rstext'>rs1026732, a SNP located in a region of chromosome 15q, has been linked to a lower frequency of restless legs syndrome, a common sleep disorder, with an overall odds ratio of 0.70 (CI: 0.59-0.82) for the (A) minor allele.  [PharmGKB:Curated In replicated GWAS case-control studies of Caucasian(European and French-Canadian) familial and sporadic RLS, the G allele of rs1026732 was significantly associated with Restless Legs Syndrome.]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs926103'>rs926103</a><a href='http://www.snpedia.com/index.php/rs926103(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDDDD;'>47.7%</span><br/>Repute:Bad<br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Possible increased risk of multiple sclerosis if SH2D2A promoter GA repeat=16  Possible increased risk of multiple sclerosis if SH2D2A promoter GA repeat=16. See The SH2D2A gene and susceptibility to multiple sclerosis. The T cell regulator gene SH2D2A contributes to the genetic susceptibility of multiple sclerosis rs926103 and a GA repeat polymorphism forming a risk haplotype for Multiple sclerosis</div>
        <div class='boxshow' id="show-sec-0.0086886699043" style="display: inline;"><a href="javascript:toggle2('sec-0.0086886699043')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0086886699043" style="display: none;"><a href="javascript:toggle2('sec-0.0086886699043')">(hide)</a>
        <div class='rstext'>Rs926103 is located within the SH2D2A gene on Chromosome 1. SH2D2A encodes an adapter protein thought to function in T-cell signal transduction. A study in the Journal for Neuroimmunology reported that 16 repeats repeat of GA after the (A;A) allele at this SNP is associated with multiple sclerosis in Norwegian, Danish and Swedish sample sets. The study concludes that the SH2D2A gene may contribute to susceptibility to MS. This study has yet to be verified or reproduced in any subsequent research. The T cell regulator gene SH2D2A contributes to the genetic susceptibility of multiple sclerosis rs926103 and a GA repeat polymorphism forming a risk haplotype for Multiple sclerosis The fact that this is a mixture of a snp and a repeat makes this an interesting test case for Promethease</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11761683'>rs11761683</a><a href='http://www.snpedia.com/index.php/rs11761683(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDDDD;'>47.8%</span><br/>Repute:Bad<br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Migraine  A four-marker haplotype in DDC was specific to migraine with Aura (rs2329340A, rs11974297C, rs2044859T, rs11761683G)</div>
        <div class='boxshow' id="show-sec-0.734222605387" style="display: inline;"><a href="javascript:toggle2('sec-0.734222605387')">...more...</a></div>
        <div class='boxrstext' id="sec-0.734222605387" style="display: none;"><a href="javascript:toggle2('sec-0.734222605387')">(hide)</a>
        <div class='rstext'>a haplotype consisting of rs2329340(A), rs11974297(C), rs2044859(T) and rs11761683(G) associated with increased incidence of migraine with aura among a sample of 528 migraine patients (308 without aura, 220 with aura) and 528 sex-matched migraine-free controls</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxbadresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4489954'>rs4489954</a><a href='http://www.snpedia.com/index.php/rs4489954(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDFDF;'>50.0%</span><br/>Repute:Bad<br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk of developing restless legs syndrome</div>
        <div class='boxshow' id="show-sec-0.83355814202" style="display: inline;"><a href="javascript:toggle2('sec-0.83355814202')">...more...</a></div>
        <div class='boxrstext' id="sec-0.83355814202" style="display: none;"><a href="javascript:toggle2('sec-0.83355814202')">(hide)</a>
        <div class='rstext'>rs4489954, a SNP located in a region of chromosome 15q, has been linked to a lower frequency of restless legs syndrome, a common sleep disorder, with an overall odds ratio of 0.69 (CI: 0.69-0.82) for the (T) minor allele.  [PharmGKB:Curated In replicated GWAS case-control studies of Caucasian(European and French-Canadian) familial and sporadic RLS, the G allele of rs4489954 was significantly associated with Restless Legs Syndrome.]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>
</div>     <div class="majorboxfooter"></div>     </div><div class="majorsection">     <div class='sectiontext'>These seem interesting, but have not been flagged as clearly Good or Bad</div>
        <div class='boxshow' id="show-withMag" style="display: inline;"><a href="javascript:toggle2('withMag')">...more...</a></div>
        <div class="boxsection" id="withMag" style="display: none;"><a href="javascript:toggle2('withMag')">(hide)</a>
  <div id="gs144"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs144'>gs144</a></div></div>
        <div class='boxmag'>Magnitude: 4</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <img class="genoimage" src="http://upload.wikimedia.org/wikipedia/commons/thumb/c/c1/Male.svg/100px-Male.svg.png"/>
        <div class='gsboxsummary'>Male</div>
        <div class='gsboxeffect'>Male.</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs227"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs227'>gs227</a></div></div>
        <div class='boxmag'>Magnitude: 3</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>fully heterozygous, age related tasting variation</div>
        <div class='gsboxeffect'>You are heterozygous at all 3 of the SNPs which are known to influence the ability to taste bitterness. This means you are better than average at detecting bitter tastes while young, but that this ability will decrease to less than average during adulthood. As a child you will probably hate brussel sprouts, and by early adulthood will discover that olives and brussel sprouts now taste good. A 2010 study shows the change bitter sensitivity which occurs over the lifespan (from bitter sensitive to less so) is more common in people with this genoset. Children with this genotype could perceive a bitter taste at lower PROP concentrations than could heterozygous adults. The threshold for adolescents was intermediate. The 3 SNPs are rs10246939, rs1726866, rs713598 in the gene TAS2R38.</div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1426654'>rs1426654</a><a href='http://www.snpedia.com/index.php/rs1426654(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2.7<br/>Frequency: <span style='background-color: #FFFFFF;'>100.0%</span><br/>References:18<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>probably light-skinned, European ancestry This SNP influences skin pigmentation and indicates indicates light-skinned European ancestry. estimates that the rs1426654(A) allele (light skin pigmentation) spread through the European population around 6,000 - 12,000 years ago. Prior to that, 'European ancestors' were most likely relatively brown-skinned.</div>
        <div class='boxshow' id="show-sec-0.505639386396" style="display: inline;"><a href="javascript:toggle2('sec-0.505639386396')">...more...</a></div>
        <div class='boxrstext' id="sec-0.505639386396" style="display: none;"><a href="javascript:toggle2('sec-0.505639386396')">(hide)</a>
        <div class='rstext'>This SNP influences skin pigmentation. The allele, A111T, rs1426654(A), indicates light-skinned european ancestry. It appears as if this SNP is a relatively new one in human evolution; one estimate is that the rs1426654(A) allele, in other words, light skin pigmentation, spread through the European population around 6,000 - 12,000 years ago. Prior to that, 'European ancestors' were most likely relatively brown-skinned. Another study () has concluded that almost individuals carrying the A111T variant can trace ancestry back to a single person who most likely lived at least 10,000 years ago. This SNP is one of three from the SLC24A5 gene that can be analyzed to categorize the ancestry of a person as either European, African, or Asian, based on a 2009 study. The 3 SNPs are: * rs1426654 * rs25...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs146"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs146'>gs146</a></div></div>
        <div class='boxmag'>Magnitude: 2.5</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>Haplogroup A (Y-DNA)</div>
        <div class='gsboxeffect'>In human genetics, Haplogroup A (M91) is a Y-chromosome haplogroup. Haplogroup A is found in the Southern Nile region & South Africa. It represents the oldest and most diverse of the human Y-chromosome haplogroups. It is believed to be the haplogroup corresponding to Y-chromosomal Adam. This genoset is supposed to detect that, but since it also think's User:Daniel_MacArthur is part of this group it seems it needs some further adjustments.</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs157"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs157'>gs157</a></div></div>
        <div class='boxmag'>Magnitude: 2.5</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>more stimulated by coffee</div>
        <div class='gsboxeffect'>You appear to have a common genotype in the gene CYP1A2 which metabolizes coffee more slowly than some other forms. The same amount of caffeine will tend to have more stimulating effect on slow metabolizers than on fast metabolizers. Ciprofloxacin is also metabolized by CYP1A2, but is unclear if your genotype should influence its effect.</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs256"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs256'>gs256</a></div></div>
        <div class='boxmag'>Magnitude: 2.5</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <img class="genoimage" src="http://www.snpedia.com/images/thumb/a/a3/IMAG1373.jpg/90px-IMAG1373.jpg"/>
        <div class='gsboxsummary'>Blue eyes</div>
        <div class='gsboxeffect'>thumb Heterozygous for what calls blue eye color haplotype #1. It seems 'blue', might be overstating it. In the eye color research community, there seems to be a distinction between the dark brown eyes typical for asian and african ancestry, and 'blue' for lighter eyes found in europeans.</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs280"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs280'>gs280</a></div></div>
        <div class='boxmag'>Magnitude: 2.2</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>Light hair color for europeans</div>
        <div class='gsboxeffect'>You have one of the SNPs linked to light color hair. *rs12821256(C;C) *rs35264875(T;T) *rs12896399(G;G)</div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs762551'>rs762551</a><a href='http://www.snpedia.com/index.php/rs762551(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFD3D3;'>38.1%</span><br/>References:39<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Carrier of one CYP1A2*1F allele; Slow Caffeine Metabolizer.  One copy of the slow caffeine metaboliser SNP, and one copy of the fast version. This makes you more strongly affected by drinking coffee, as caffeine is broken down slower in the liver. Supposedly this increases the risk of heart attacks, although other studies show caffeine is generally good for the heart. It also makes caffeine more effective at preventing Breast Cancer, Alzheimer's Disease, and Parkinson's disease. Too much caffeine will shrink your breasts.</div>
        <div class='boxshow' id="show-sec-0.196403246184" style="display: inline;"><a href="javascript:toggle2('sec-0.196403246184')">...more...</a></div>
        <div class='boxrstext' id="sec-0.196403246184" style="display: none;"><a href="javascript:toggle2('sec-0.196403246184')">(hide)</a>
        <div class='rstext'>rs762551, also known as -164A&gt;C or -163C&gt;A, is a SNP encoding the CYP1A2*1F allele of the CYP1A2 gene. For historic reasons, the rs762551(C) allele is considered the wild-type, even though it is the rarer allele in most populations. The rs762551(A) allele is the 'fast metabolizer' allele known as CYP1A2*1F; the (C) allele is by comparison a slower metabolizer of certain substrates (including caffeine). In terms of genotypes, only rs762551(A;A) individuals are considered fast metabolizers. Individuals who are rs762551(A;C) heterozygotes or rs762551(C;C) homozygotes are both considered slow metabolizers. The CYP1A2 gene encodes a member of the cytochrome p450 family of proteins, which metabolize nutrients and drugs. One well known substrate of CYP1A2 is caffeine; individuals who are ca...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs664677'>rs664677</a><a href='http://www.snpedia.com/index.php/rs664677(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>Frequency: <span style='background-color: #FFE1E1;'>53.1%</span><br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Lower lung cancer and increased breast cancer risk  '''rs664677''' polymorphism was associated with decreased lung cancer risk and increased breast cancer risk (for lung cancer: TC vs. TT: OR = 0.76, 95% CI 0.62-0.92, CC vs. TT: OR = 0.80, 95% CI 0.64-0.99 and for breast cancer: TC vs. TT: OR = 1.42, 95% CI 1.17-1.73, CC vs. TT: OR = 1.51, 95% CI 1.21-1.87).</div>
        <div class='boxshow' id="show-sec-0.0146821164987" style="display: inline;"><a href="javascript:toggle2('sec-0.0146821164987')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0146821164987" style="display: none;"><a href="javascript:toggle2('sec-0.0146821164987')">(hide)</a>
        <div class='rstext'>Haplotype and linkage disequilibrium architecture for human cancer-associated genes. Characterization of the linkage disequilibrium structure and identification of tagging-SNPs in five DNA repair genes. Explorative study to identify novel candidate genes related to oxaliplatin efficacy and toxicity using a DNA repair array. ATM polymorphisms and risk of lung cancer among never smokers.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs25531'>rs25531</a><a href='http://www.snpedia.com/index.php/rs25531(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>References:110<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>long form of 5-HTTLPR. less sensitive to pain  23andMe reports that this genotype is not reliable on their platform. 23andMe users should ignore this result. happier, less sensitive to pain You consider perpetrating unintentional harm as more acceptable The genotype of this SNP is not meaningful on 23andme v3. *https://www.23andme.com/you/community/thread/6192/</div>
        <div class='boxshow' id="show-sec-0.201890630164" style="display: inline;"><a href="javascript:toggle2('sec-0.201890630164')">...more...</a></div>
        <div class='boxrstext' id="sec-0.201890630164" style="display: none;"><a href="javascript:toggle2('sec-0.201890630164')">(hide)</a>
        <div class='rstext'>According to David Hinds of 23andMe on community forums, 'nearly everyone (99.97%) is getting called as CC, and there is no clear heterozygote cluster' ... 'the genotype calls for rs25531 on our platform are not meaningful.' Looking at OpenSNP frequency data, this seems to be universal for direct to consumer genotyping services at this stage. The minor allele (G) of rs25531 is almost always in phase with the long (L) allele of the 5-HTTLPR, which is the most commonly studied variation in this region. The (L) long allele may be associated with lower levels of serotonin. The article and video from 'The Atlantic' suggests that children with the short allele (A), may need and benefit greatly from more maternal support *http://www.theatlantic.com/doc/200912/dobbs-orchid-gene In contrast to earl...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs708272'>rs708272</a><a href='http://www.snpedia.com/index.php/rs708272(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2.1<br/>References:35<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Small reduction in heart disease risk from drinking alcohol  This is the genotype known as B1B2, and as reported in the Mehlig et al. 2014 article, () from a risk for coronary heart disease perspective this genotype does not show the large decrease in risk (odds ratio 0.21) from consuming alcohol seen in B2B2 patients. Hoever, intermediate amounts (6.5 - 13g of ethanol per day for men) of alcohol consumption were still associated with a somewhat reduced odds ratio (0.80) for both B1B1 and B1B2 genotypes compared to those with low (under 6.5g/day) ethanol intake.</div>
        <div class='boxshow' id="show-sec-0.893986828663" style="display: inline;"><a href="javascript:toggle2('sec-0.893986828663')">...more...</a></div>
        <div class='boxrstext' id="sec-0.893986828663" style="display: none;"><a href="javascript:toggle2('sec-0.893986828663')">(hide)</a>
        <div class='rstext'>rs708272, also known as the TaqIB polymorphism of the CETP gene, may influence the levels of the 'good' cholesterols, the high density lipoprotein (HDL) cholesterols. Generally, the B2 allele is considered to lead to higher HDL levels. However, high HDL cholesterol, at least when combined with another HDL raising SNP (rs1800588), doesn't appear to actually protect individuals from coronary artery disease, at least based on one study of ~800 Caucasian male patients. A similarly confusing picture emerges from a study of two populations of 10,000+ individuals each. The distribution of the CETP genotype was observed to be different in population-based studies compared with studies in populations selected by high cardiovascular risk, and contradictory results were seen when trying to figure out...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs135"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs135'>gs135</a></div></div>
        <div class='boxmag'>Magnitude: 2</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>This genoset identifies an ABO blood group where Type B is likely but Type A possible. Type B from B+O more likely (~8% Caucasian) than Type A.</div>
        <div class='gsboxeffect'>This genoset identifies an ABO blood group where Type B is likely but Type A possible. Type B from B+O more likely (~8% Caucasian) than Type A.</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs173"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs173'>gs173</a></div></div>
        <div class='boxmag'>Magnitude: 2</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>CYP2D6*10</div>
        <div class='gsboxeffect'>CYP2D6*10 reduced metabolism</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs288"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs288'>gs288</a></div></div>
        <div class='boxmag'>Magnitude: 2</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>You have two long form 5-HTTLPR.</div>
        <div class='gsboxeffect'>You likely have two long-form 5-HTTLPR (serotonin-transporter-linked polymorphic region). Variations in the region have been extensively investigated in connection with neuropsychiatric disorders. Identification of tag haplotypes for 5HTTLPR for different genome-wide SNP platforms</div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1076560'>rs1076560</a><a href='http://www.snpedia.com/index.php/rs1076560(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF7777;'>1.8%</span><br/>References:22<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>influences working memory</div>
        <div class='boxshow' id="show-sec-0.623306833151" style="display: inline;"><a href="javascript:toggle2('sec-0.623306833151')">...more...</a></div>
        <div class='boxrstext' id="sec-0.623306833151" style="display: none;"><a href="javascript:toggle2('sec-0.623306833151')">(hide)</a>
        <div class='rstext'>rs1076560 is located in intron 6 of the dopamine receptor D2 gene. In one study of Japanese males, rs1076560(A) alleles were 1.3 fold more associated with Alcoholism than the rs1076560(C) alleles. The DRD2 risk allele A was more prevalent in the alcoholic patients than in the healthy controls. These data identify rs1076560 as a potentially important variable in the development of alcoholism. rs1076560 and the DAT 3'-VNTR variant influences memory Genetic variants altering dopamine D2 receptor expression or function modulate the risk of opiate addiction and the dosage requirements of methadone substitution  [PharmGKB:Curated This variant is a potential marker for alcoholism. A allele was more prevalent in the alcoholic patients than in the healthy controls in a study consists of 248 alcohol...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs16891982'>rs16891982</a><a href='http://www.snpedia.com/index.php/rs16891982(C;G)'>(C;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF8585;'>3.2%</span><br/>References:15<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>if European, 7x more likely to have black hair</div>
        <div class='boxshow' id="show-sec-0.241136331573" style="display: inline;"><a href="javascript:toggle2('sec-0.241136331573')">...more...</a></div>
        <div class='boxrstext' id="sec-0.241136331573" style="display: none;"><a href="javascript:toggle2('sec-0.241136331573')">(hide)</a>
        <div class='rstext'>This snp influences skin pigmentation. The allele p.L374F indicates light-skinned european ancestry c.1122C&gt;G, p.Phe374Leu (NCBI dbSNP rs16891982) in SLC45A2 was associated with protection from malignant melanoma (OR, 0.41; 95% CI, 0.24-0.70; P=0.008 after adjustment for multiple testing) An association study conducted on a population of European origin concluded that the rare (in Europeans, at least) allele L374, i.e. rs16891982(C) in dbSNP orientation, significantly increases the possibility of having black hair color, with an odds ratio of around 7. This population-based case-control study investigated a number of pigmentation-related SNPs and their correlation with cutaneous malignant melanoma (CMM) in an Australian population. rs16891982 (variant F374L) was found to be associated w...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs776746'>rs776746</a><a href='http://www.snpedia.com/index.php/rs776746(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF9A9A;'>7.2%</span><br/>References:35<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>carrier of 1 nonfunctional CYP3A5 allele; drug metabolism affects</div>
        <div class='boxshow' id="show-sec-0.280626307496" style="display: inline;"><a href="javascript:toggle2('sec-0.280626307496')">...more...</a></div>
        <div class='boxrstext' id="sec-0.280626307496" style="display: none;"><a href="javascript:toggle2('sec-0.280626307496')">(hide)</a>
        <div class='rstext'>rs776746, also known as 6986A&gt;G, is a SNP encoding the (nonfunctional) CYP3A5*3 allele of the CYP3A5 gene. CYP3A5*3 has been studied especially in connection with the metabolism of tacrolimus, an immunosuppressive drug used in organ transplantation. The optimal therapeutic dose is in a relatively narrow window, so blood concentration monitoring is recommended. CYP3A5*3 carriers tend to build up higher levels of tacrolimus than CYP3A5*1 carriers, unless their tacrolimus intake is reduced. Risk of testicular germ cell cancer in relation to variation in maternal and offspring cytochrome p450 genes involved in catechol estrogen metabolism. Polymorphisms in the cytochrome P450 genes CYP1A2, CYP1B1, CYP3A4, CYP3A5, CYP11A1, CYP17A1, CYP19A1 and colorectal cancer risk. Prevalence of common dis...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2070075'>rs2070075</a><a href='http://www.snpedia.com/index.php/rs2070075(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FF9D9D;'>7.8%</span><br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'></div>
        <div class='boxshow' id="show-sec-0.117944408667" style="display: inline;"><a href="javascript:toggle2('sec-0.117944408667')">...more...</a></div>
        <div class='boxrstext' id="sec-0.117944408667" style="display: none;"><a href="javascript:toggle2('sec-0.117944408667')">(hide)</a>
        <div class='rstext'>[OMIM:LOS ANGELES VARIANT]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs17236239'>rs17236239</a><a href='http://www.snpedia.com/index.php/rs17236239(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFA6A6;'>10.6%</span><br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>higher risk for speech development delay and/or impairment</div>
        <div class='boxshow' id="show-sec-0.601470495479" style="display: inline;"><a href="javascript:toggle2('sec-0.601470495479')">...more...</a></div>
        <div class='boxrstext' id="sec-0.601470495479" style="display: none;"><a href="javascript:toggle2('sec-0.601470495479')">(hide)</a>
        <div class='rstext'>Speech development rs4431523, rs17236239 and significant associations (with P values from 0.01 to 5.0x10&#8211;5) between nonsense-word repetition and nine intronic SNPs (rs851715, rs10246256, rs2710102, rs759178, rs1922892, rs2538991, rs17236239, rs2538976, and rs2710117) 23andMe blog g2b2mh discusses this snp nejm CNTNAP2 variants affect early language development in the general population.  [PharmGKB:Curated This SNP in the CNTNAP2 gene was significantly associated with nonsense-word repetition in children with typical specific language impairment.] [OMIM:SPECIFIC LANGUAGE IMPAIRMENT 4; SLI4]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs572515'>rs572515</a><a href='http://www.snpedia.com/index.php/rs572515(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFABAB;'>12.4%</span><br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'></div>
        <div class='boxshow' id="show-sec-0.845361285903" style="display: inline;"><a href="javascript:toggle2('sec-0.845361285903')">...more...</a></div>
        <div class='boxrstext' id="sec-0.845361285903" style="display: none;"><a href="javascript:toggle2('sec-0.845361285903')">(hide)</a>
        <div class='rstext'>rs572515 was the most significantly associated of a group of SNPs in the chromosome 1q32-33 region with risk for neovascular age related macular degeneration (p &lt; 10-6).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2298383'>rs2298383</a><a href='http://www.snpedia.com/index.php/rs2298383(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFB2B2;'>15.5%</span><br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>increased anxiety in response to caffeine  associated with increased anxiety in response to caffeine</div>
        <div class='boxshow' id="show-sec-0.149757228652" style="display: inline;"><a href="javascript:toggle2('sec-0.149757228652')">...more...</a></div>
        <div class='boxrstext' id="sec-0.149757228652" style="display: none;"><a href="javascript:toggle2('sec-0.149757228652')">(hide)</a>
        <div class='rstext'>Variation at this SNP, rs2298383, appears to influence how an individual responds to moderate levels (150mg) of caffeine.  [PharmGKB:Curated Risk or phenotype-associated allele: rs2298383 T allele. Phenotype: Carriers of 1 or 2 copies of the T allele for rs2298383 showed increased risk of any (OR = 2.23, CI 1.03-5.07, p = 0.03) or gastrointestinal-specific (OR = 2.65, CI = 0.8-11.35, p = 0.09) adverse events. Study size: 309 RA patients, including 147 good MTX responders, 101 MTX inefficacy failures, and 61 adverse event (AE) MTX failures (e.g. gastrointestinal (n = 24), abnormal liver function tests (n = 20), haematological (n = 7), skin rashes (n = 6), and other (n = 17). Study population/ethnicity: Rheumatoid arthritis (RA) patients aged over 18 yrs, of White Caucasian ethnic origin, cl...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4410790'>rs4410790</a><a href='http://www.snpedia.com/index.php/rs4410790(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFB7B7;'>17.7%</span><br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>drank about a third of a cup of coffee less each day than those without any copies.</div>
        <div class='boxshow' id="show-sec-0.371652949483" style="display: inline;"><a href="javascript:toggle2('sec-0.371652949483')">...more...</a></div>
        <div class='boxrstext' id="sec-0.371652949483" style="display: none;"><a href="javascript:toggle2('sec-0.371652949483')">(hide)</a>
        <div class='rstext'>rs4410790 and rs2470893 influences habitual caffeine consumption  [GWAS:None]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6732426'>rs6732426</a><a href='http://www.snpedia.com/index.php/rs6732426(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFB9B9;'>18.6%</span><br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>curlier hair</div>
        <div class='boxshow' id="show-sec-0.0769341262493" style="display: inline;"><a href="javascript:toggle2('sec-0.0769341262493')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0769341262493" style="display: none;"><a href="javascript:toggle2('sec-0.0769341262493')">(hide)</a>
        <div class='rstext'>[GWAS:Hair morphology]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1954787'>rs1954787</a><a href='http://www.snpedia.com/index.php/rs1954787(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFBDBD;'>21.2%</span><br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>~10% less likely to respond to citalopram</div>
        <div class='boxshow' id="show-sec-0.369490947989" style="display: inline;"><a href="javascript:toggle2('sec-0.369490947989')">...more...</a></div>
        <div class='boxrstext' id="sec-0.369490947989" style="display: none;"><a href="javascript:toggle2('sec-0.369490947989')">(hide)</a>
        <div class='rstext'>rs1954787, located in an intron of the GRIK4 gene, was found to be associated with a somewhat increased rate of successful response to treatment of depression with the drug citalopram. From the abstract of this article: 'The effect size of (this) GRIK4 marker alone was modest, but homozygote carriers of the treatment-response-associated marker alleles of both the GRIK4 and HTR2A (see rs7997012) genes were 23% less likely to experience nonresponse to (citalopram) treatment relative to participants who did not carry any of these marker alleles.' * See also Anxiety Blog posting  [PharmGKB:Curated A study in 1,816 eligible patients from the STAR*D cohort found that genetic variation in a kainic acid-type glutamate receptor is reproducibly associated with response to the antidepressant citalopr...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4343'>rs4343</a><a href='http://www.snpedia.com/index.php/rs4343(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC3C3;'>24.8%</span><br/>References:38<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>usually the ACE Alu deletion; better strength vs endurance;</div>
        <div class='boxshow' id="show-sec-0.918391026588" style="display: inline;"><a href="javascript:toggle2('sec-0.918391026588')">...more...</a></div>
        <div class='boxrstext' id="sec-0.918391026588" style="display: none;"><a href="javascript:toggle2('sec-0.918391026588')">(hide)</a>
        <div class='rstext'>A and G alleles of rs4343 are strongly associated marking insertion and deletion alleles of ACE I/D respectively. For those looking for the ACE Alu Insertion/Deletion, this is a very good substitute for people searching for ACE Del16. G would mean deletion. A haplotype of rs4311, rs4343, rs699 increases risk of diabetic nephropathy 4x.  [PharmGKB:Curated While the ACE:I/D has dbSNP identifiers rs1799752, rs4340, rs13447447 and rs4646994, measurement of a single base at these positions does not give an informative genotype. In Europeans, rs4343 is in complete LD with the ACE:I/D, with the A and G alleles of rs4343 marking the insertion and deletion alleles of ACE:I/D respectively.] [GWAS:Angiotensin-converting enzyme activity]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs5888'>rs5888</a><a href='http://www.snpedia.com/index.php/rs5888(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFC7C7;'>27.7%</span><br/>References:15<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>lower HDL cholesterol but higher risk for age-related macular degeneration</div>
        <div class='boxshow' id="show-sec-0.459152358014" style="display: inline;"><a href="javascript:toggle2('sec-0.459152358014')">...more...</a></div>
        <div class='boxrstext' id="sec-0.459152358014" style="display: none;"><a href="javascript:toggle2('sec-0.459152358014')">(hide)</a>
        <div class='rstext'>rs5888 is a SNP in the scavenger receptor class B, member 1 SCARB1 gene. In a case-control study of two Caucasian populations totaling 2,498 patients, rs5888(C;T) heterozygotes had an increased odds ratio of 2.9 (CI: 1.6-5.3, p&lt;0.002) for age-related macular degeneration (ARMD) based on a pooled analysis. Subgroup analysis indicated that the odds ratio for exudative forms of ARMD was even higher (3.6; CI: 1.7-7.6, p&gt;0.0015) for rs5888(C;T) heterozygotes.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4654748'>rs4654748</a><a href='http://www.snpedia.com/index.php/rs4654748(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFCECE;'>33.6%</span><br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2.90 ng/mL lower Vitamin B6 blood concentration</div>
        <div class='boxshow' id="show-sec-0.489256433335" style="display: inline;"><a href="javascript:toggle2('sec-0.489256433335')">...more...</a></div>
        <div class='boxrstext' id="sec-0.489256433335" style="display: none;"><a href="javascript:toggle2('sec-0.489256433335')">(hide)</a>
        <div class='rstext'>This is one of the SNPs reported by NutraHacker. Genetics and genomics of human aging.  [PharmGKB:Non-Curated The C allele of this SNP is associated with lower blood concentration of vitamin B6 (Combined studies of Tuscan Italians ( inCHIANTI:N =1175 and Progetto Nutrizione study:N = 687) found 1.45 ng/mL lower vitamin B6 per copy of C (p = 8.3 x 10 (-18)))] [OMIM:?] [GWAS:Folate pathway vitamins]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9923231'>rs9923231</a><a href='http://www.snpedia.com/index.php/rs9923231(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFD6D6;'>40.7%</span><br/>References:53<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>reduced warfarin dose if treated for VTE</div>
        <div class='boxshow' id="show-sec-0.790945173171" style="display: inline;"><a href="javascript:toggle2('sec-0.790945173171')">...more...</a></div>
        <div class='boxrstext' id="sec-0.790945173171" style="display: none;"><a href="javascript:toggle2('sec-0.790945173171')">(hide)</a>
        <div class='rstext'>Several SNPs in the VKORC1 gene have been linked to warfarin sensitivity, with perhaps the most common being this SNP rs9923231. Note that the orientation as published in scientific articles is often on the opposite strand compared to the orientation in dbSNP, so you will sometimes see it as a G&gt;T snp. It is also known as *-1639G&gt;A with the minus indicating that this is in an upstream promoter *3673 based on its position in GenBank accession number AY587020. *VKORC1*2. The main findings related to the treatment of venous thromboembolism (aka VTE; from hypercoagulability) with the blood thinner warfarin for this SNP are that carriers of the rs9923231(T) allele require significantly reduced doses of warfarin, and are (otherwise) at a higher risk of serious bleeding. Clinical studies de...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4481887'>rs4481887</a><a href='http://www.snpedia.com/index.php/rs4481887(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDADA;'>44.2%</span><br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>more likely to be able to smell asparagus metabolites in urine  1.67x more likely than people with (G;G) to be able to smell the methanethiol produced after eating asparagus.</div>
        <div class='boxshow' id="show-sec-0.111327426872" style="display: inline;"><a href="javascript:toggle2('sec-0.111327426872')">...more...</a></div>
        <div class='boxrstext' id="sec-0.111327426872" style="display: none;"><a href="javascript:toggle2('sec-0.111327426872')">(hide)</a>
        <div class='rstext'>Asparagus anosmia research was performed by 23andMe and presented at the 2009 ASHG according to geneticfuture. Asparagus anosmia Asparagus Metabolite Detection  [GWAS:Common traits (Other)]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs9938025'>rs9938025</a><a href='http://www.snpedia.com/index.php/rs9938025(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFDADA;'>45.1%</span><br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Unknown effect</div>
        <div class='boxshow' id="show-sec-0.0218849670323" style="display: inline;"><a href="javascript:toggle2('sec-0.0218849670323')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0218849670323" style="display: none;"><a href="javascript:toggle2('sec-0.0218849670323')">(hide)</a>
        <div class='rstext'>23andme reports that having two copies of an 'A' at rs9938025 is associated with moderately lower odds of dry earwax and having two copies of a 'G' is associated with moderately higher odds in their customers with European ancestry compared to those who have 'AG'. https://www.23andme.com/en-eu/about/factoids/</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs460000'>rs460000</a><a href='http://www.snpedia.com/index.php/rs460000(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE4E4;'>56.6%</span><br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Better response to amphetamine  Individuals with the C/C genotype at rs460000 (N = 83) reported approximately twofold higher ratings of stimulation and euphoria relative to the A/A+A/C (N = 69) genotype group, at both the 10 and 20 mg doses.</div>
        <div class='boxshow' id="show-sec-0.184922466823" style="display: inline;"><a href="javascript:toggle2('sec-0.184922466823')">...more...</a></div>
        <div class='boxrstext' id="sec-0.184922466823" style="display: none;"><a href="javascript:toggle2('sec-0.184922466823')">(hide)</a>
        <div class='rstext'></div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2108622'>rs2108622</a><a href='http://www.snpedia.com/index.php/rs2108622(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFE7E7;'>60.7%</span><br/>References:51<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>lower warfarin dosing</div>
        <div class='boxshow' id="show-sec-0.5671065669" style="display: inline;"><a href="javascript:toggle2('sec-0.5671065669')">...more...</a></div>
        <div class='boxrstext' id="sec-0.5671065669" style="display: none;"><a href="javascript:toggle2('sec-0.5671065669')">(hide)</a>
        <div class='rstext'>rs2108622 is a SNP in the cytochrome P450, family 4, subfamily F, polypeptide 2 CYP4F2 gene. In the males rs2108622(G) was significantly higher in cerebral infarction patients (P = 0.025) A study of Italian patients concluded that rs2108622(T;T) patients require 5.49 mg/day of warfarin versus 2.93 mg/day for (C;C) patients. Analysis of variance indicates that about 7% of mean weekly warfarin dose variance is explained by CYP4F2 genotype. A study of 370 Brazilian ('admixed') patients concludes that prospective CYP4F2 genotyping is of little value to these patients, presumably due to ethnic-based differences.  [PharmGKB:Curated Risk or phenotype-associated allele: no allelic association. Phenotype: Improved pharmacogenetic algorithm-based warfarin dosing. Study size: 370. Study population/et...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2547231'>rs2547231</a><a href='http://www.snpedia.com/index.php/rs2547231(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFEAEA;'>64.3%</span><br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Increased blood metabolite levels  5alpha-androstan-3beta,17beta-diol disulfate - 0.081 unit increase X-12850 - 0.059 unit increase X-12456 - 0.041 unit increase X-11440/4-androsten-3beta,17beta-diol disulfate 2 - 0.141 unit increase</div>
        <div class='boxshow' id="show-sec-0.923307991597" style="display: inline;"><a href="javascript:toggle2('sec-0.923307991597')">...more...</a></div>
        <div class='boxrstext' id="sec-0.923307991597" style="display: none;"><a href="javascript:toggle2('sec-0.923307991597')">(hide)</a>
        <div class='rstext'>[GWAS:Blood metabolite levels]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2281617'>rs2281617</a><a href='http://www.snpedia.com/index.php/rs2281617(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #FFEDED;'>69.0%</span><br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal (better) response to amphetamine  The main finding of the study is that subjects with genotype C/C at rs2281617 scored significantly higher on Euphoria and Energy than subjects with genotypes C/T and T/T after 10 mg amphetamine, and trends were also observed with measures of Stimulation and (lower) Sedation.</div>
        <div class='boxshow' id="show-sec-0.519161291599" style="display: inline;"><a href="javascript:toggle2('sec-0.519161291599')">...more...</a></div>
        <div class='boxrstext' id="sec-0.519161291599" style="display: none;"><a href="javascript:toggle2('sec-0.519161291599')">(hide)</a>
        <div class='rstext'>Some evidence exists that the T;T and C;T genotypes are associated with less euphoria and energy in response to a given dose of a stimulant drug, e.g. dextroamphetamine, than occurs for the C;C genotype. See [especially figure 3 http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3377371/figure/F3/]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11803731'>rs11803731</a><a href='http://www.snpedia.com/index.php/rs11803731(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>Frequency: <span style='background-color: #F0CEF0;'>57.8%</span><br/>References:4<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>straighter hair</div>
        <div class='boxshow' id="show-sec-0.539761750675" style="display: inline;"><a href="javascript:toggle2('sec-0.539761750675')">...more...</a></div>
        <div class='boxrstext' id="sec-0.539761750675" style="display: none;"><a href="javascript:toggle2('sec-0.539761750675')">(hide)</a>
        <div class='rstext'>rs11803731, a SNP in the Trichohyalin gene (TCHH) gene, is estimated to account for 6% of the variance in hair morphology (ie hair curliness) based on a study of Australians of European ancestry.  [OMIM:?] [GWAS:Hair morphology]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1135840'>rs1135840</a><a href='http://www.snpedia.com/index.php/rs1135840(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 2<br/>References:5<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Homozygous for CYP2D6 variants (non-CYP2D6*1)</div>
        <div class='boxshow' id="show-sec-0.282508613057" style="display: inline;"><a href="javascript:toggle2('sec-0.282508613057')">...more...</a></div>
        <div class='boxrstext' id="sec-0.282508613057" style="display: none;"><a href="javascript:toggle2('sec-0.282508613057')">(hide)</a>
        <div class='rstext'>rs1135840, also known as 4180G&gt;C or S486T, is a SNP in the CYP2D6 gene. The wild type (normal) allele at this SNP is (G). The (C) variant indicates the presence of a non-wild type CYP2D6 variant, but it appears in many different variants so it can not be used to determine the presence of any particular variant. Please be careful when interpreting results for this SNP as it is in dbSNP and SNPedia in minus orientation where the risk allele is C, but test results are usually in plus orientation where the risk allele is G. This SNP has an ambiguous flip which can make this very confusing. Polymorphisms affecting gene transcription and mRNA processing in pharmacogenetic candidate genes: detection through allelic expression imbalance in human target tissues.  [PharmGKB:Curated Risk or phenot...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4794067'>rs4794067</a><a href='http://www.snpedia.com/index.php/rs4794067(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.9<br/>Frequency: <span style='background-color: #FFD4D4;'>38.9%</span><br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>2.1x risk for Aspirin Induced Asthma. But possibly lower risk of intractible Grave's disease.  Only tested in Japanese people. Might not be true for other races. May also be slightly protective against lupus (in Chinese people).</div>
        <div class='boxshow' id="show-sec-0.586173945014" style="display: inline;"><a href="javascript:toggle2('sec-0.586173945014')">...more...</a></div>
        <div class='boxrstext' id="sec-0.586173945014" style="display: none;"><a href="javascript:toggle2('sec-0.586173945014')">(hide)</a>
        <div class='rstext'>rs4794067(C;C) and rs4794067(C;T) genotypes for the TBX21 gene are calculated to be at 2.15 fold higher risk (CI: 1.26-3.64) for aspirin-induced asthma (AIA) compared to rs4794067(T;T) homozygotes, based on a study of 72 patients vs 640 controls in a study of a Japanese population. Note that in this population, but perhaps not others, linkage disequilibrium exists between this SNP located upstream of the gene, rs4794067, and a common synonymous SNP located in the first exon, rs2074190, with a predictive accuracy of 92%. On the other hand, rs4794067(C;C) may greatly decrease the risk of lupus if rs17250932(C;C) is also present (at least in Chinese people). And those 2 SNPs also reduce the risk of intractable Graves' disease (at least in Japanese people).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7216389'>rs7216389</a><a href='http://www.snpedia.com/index.php/rs7216389(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.8<br/>Frequency: <span style='background-color: #FFE3E3;'>54.9%</span><br/>References:26<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal risk for Childhood Asthma.</div>
        <div class='boxshow' id="show-sec-0.769773052683" style="display: inline;"><a href="javascript:toggle2('sec-0.769773052683')">...more...</a></div>
        <div class='boxrstext' id="sec-0.769773052683" style="display: none;"><a href="javascript:toggle2('sec-0.769773052683')">(hide)</a>
        <div class='rstext'>rs7216389, a SNP in the ORMDL3 gene on chromosome 17q21, was associated with susceptibility to childhood asthma in a study of ~1,000 British patients. The variation appears to be linked to altered levels of the ORMDL3 mRNA, which was shown in a cohort study of ~5,000 British and German patients to be correlated to childhood asthma. A large study of Scottish asthma patients replicated these results, finding that a single copy of the T allele conferred an odds ratio of 1.50 (CI: 1.24-1.81) and 2 copies conferred an odds ratio of 2.11 (CI: 1.71-2.61), respectively.  [PharmGKB:Non-Curated GWAS Results: Genetic variants regulating ORMDL3 expression contribute to the risk of childhood asthma (Initial Sample Size: 994 cases, 1,243 controls; Replication Sample Size: 2,320 cases, 3,301 controls; Ri...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs5400'>rs5400</a><a href='http://www.snpedia.com/index.php/rs5400(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1.7<br/>Frequency: <span style='background-color: #FFB9B9;'>19.1%</span><br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>significantly higher sugar consumption?</div>
        <div class='boxshow' id="show-sec-0.442301992519" style="display: inline;"><a href="javascript:toggle2('sec-0.442301992519')">...more...</a></div>
        <div class='boxrstext' id="sec-0.442301992519" style="display: none;"><a href="javascript:toggle2('sec-0.442301992519')">(hide)</a>
        <div class='rstext'>rs5400(T;T) carriers had a significantly higher intake of sugars blog discussion Food Preference</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2024513'>rs2024513</a><a href='http://www.snpedia.com/index.php/rs2024513(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.7<br/>Frequency: <span style='background-color: #FFCBCB;'>31.0%</span><br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.7x higher risk for schizophrenia (among Han Chinese)</div>
        <div class='boxshow' id="show-sec-0.279254776421" style="display: inline;"><a href="javascript:toggle2('sec-0.279254776421')">...more...</a></div>
        <div class='boxrstext' id="sec-0.279254776421" style="display: none;"><a href="javascript:toggle2('sec-0.279254776421')">(hide)</a>
        <div class='rstext'>rs2024513 is a SNP in the neurexin-1 NRXN1 gene on chromosome 2p16.3. Based on a case-control study of 700+ Han Chinese patients with schizophrenia, rs2024513(A) was associated with higher risk, at an odds ratio of 1.3 (CI: 1.07 - 1.56, p=0.006).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs139"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs139'>gs139</a></div></div>
        <div class='boxmag'>Magnitude: 1.5</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>NAT2 intermediate metabolizer</div>
        <div class='gsboxeffect'>NAT2 Intermediate metabolizer source Generally this means a person is carrying one rapid (normal) NAT2 allele plus one slow allele. This appears to be the most common form. Intermediate acetylators may benefit by slightly lower than average drug dosages of the drugs metabolized by NAT2 for optimal therapeutic response.</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs250"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs250'>gs250</a></div></div>
        <div class='boxmag'>Magnitude: 1.5</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>Normal Parkinson's Disease Risk</div>
        <div class='gsboxeffect'>The (G;G) genotype at this position is considered protective against Parkinson's Disease, but you don't have it. You instead have the more common rs10513789(T;T) which does not reduce your risk. see also gs249 for the more common heterozygote form http://www.genomicslawreport.com/index.php/2012/06/01/patenting-and-personal-genomics-23andme-receives-its-first-patent-and-plenty-of-questions/</div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2007153'>rs2007153</a><a href='http://www.snpedia.com/index.php/rs2007153(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFD5D5;'>39.8%</span><br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>normal risk of schizophrenia in limited study</div>
        <div class='boxshow' id="show-sec-0.755757984872" style="display: inline;"><a href="javascript:toggle2('sec-0.755757984872')">...more...</a></div>
        <div class='boxrstext' id="sec-0.755757984872" style="display: none;"><a href="javascript:toggle2('sec-0.755757984872')">(hide)</a>
        <div class='rstext'>Associated with schizophrenia in a study of Croatians. Due to the limited number and diversity of the studied population, more study is needed to confirm any link. The G allele was associated with an increased risk of schizophrenia.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4331145'>rs4331145</a><a href='http://www.snpedia.com/index.php/rs4331145(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.5<br/>Frequency: <span style='background-color: #FFE4E4;'>56.9%</span><br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>decreased risk of schizophrenia in limited study</div>
        <div class='boxshow' id="show-sec-0.397176320799" style="display: inline;"><a href="javascript:toggle2('sec-0.397176320799')">...more...</a></div>
        <div class='boxrstext' id="sec-0.397176320799" style="display: none;"><a href="javascript:toggle2('sec-0.397176320799')">(hide)</a>
        <div class='rstext'>Associated with schizophrenia in a study of Croatians. Due to the limited number and diversity of the studied population, more study is needed to confirm any link. The G allele was associated with a decreased risk of schizophrenia.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs838133'>rs838133</a><a href='http://www.snpedia.com/index.php/rs838133(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFC4C4;'>25.9%</span><br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Lower odds preferring sweet snack</div>
        <div class='boxshow' id="show-sec-0.453629805209" style="display: inline;"><a href="javascript:toggle2('sec-0.453629805209')">...more...</a></div>
        <div class='boxrstext' id="sec-0.453629805209" style="display: none;"><a href="javascript:toggle2('sec-0.453629805209')">(hide)</a>
        <div class='rstext'>&#8220;Genetic associations with traits in 23andMe customers.&#8221; 23andMe White Paper. Salty or sweet - 23andMe blog entry  [GWAS:Dietary macronutrient intake]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs997669'>rs997669</a><a href='http://www.snpedia.com/index.php/rs997669(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFD4D4;'>38.9%</span><br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>1.18x higher risk for breast cancer</div>
        <div class='boxshow' id="show-sec-0.996463053105" style="display: inline;"><a href="javascript:toggle2('sec-0.996463053105')">...more...</a></div>
        <div class='boxrstext' id="sec-0.996463053105" style="display: none;"><a href="javascript:toggle2('sec-0.996463053105')">(hide)</a>
        <div class='rstext'>rs997669 shows an association with breast cancer in a British study involving ~2300 patients. The odds ratio for the (G;G) vs (A;A) homozygotes is 1.18 (CI: 1.04-1.34, p=0.003).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs363050'>rs363050</a><a href='http://www.snpedia.com/index.php/rs363050(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFDBDB;'>46.0%</span><br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>The (A;G) genotype averages 2.8 PIQ points higher than the (G;G), but averages 2.84 PIQ points lower than (A;A) genotype</div>
        <div class='boxshow' id="show-sec-0.260849471309" style="display: inline;"><a href="javascript:toggle2('sec-0.260849471309')">...more...</a></div>
        <div class='boxrstext' id="sec-0.260849471309" style="display: none;"><a href="javascript:toggle2('sec-0.260849471309')">(hide)</a>
        <div class='rstext'>SNPs in the SNAP25 gene were initially linked to intelligence but have failed to replicate . While now suspect, the original work can be found in . The (A;A) genotype averaged 2.84 PIQ points higher than the (A;G) genotype, which averages 2.8 PIQ points higher than the (G;G) genotype (p=0.0002). Variance in this SNP may account for 3.4% of the phenotypic variance in PIQ. rs363050 is in close linkage with rs353016, with r2 = 0.98, such that the rs363050(A) allele typically is linked to the rs353016(T) allele. For SNPs rs363039 - rs363043 - rs353016, respectively, the haplotype C-T-T is most associated with higher intelligence in children, whereas the haplotype C-C-T is most associated in adults (SNPs oriented as in dbSNP). Measures of Intelligence</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs4496877'>rs4496877</a><a href='http://www.snpedia.com/index.php/rs4496877(G;T)'>(G;T)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFDDDD;'>47.8%</span><br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>For type-1 diabetics, 1.3x increased nephropathy risk</div>
        <div class='boxshow' id="show-sec-0.532729024688" style="display: inline;"><a href="javascript:toggle2('sec-0.532729024688')">...more...</a></div>
        <div class='boxrstext' id="sec-0.532729024688" style="display: none;"><a href="javascript:toggle2('sec-0.532729024688')">(hide)</a>
        <div class='rstext'>rs4496877 is a SNP near the NOS3 gene. Several Type-1 diabetes patients pools, totaling over 2,000 patients, plus a meta-analysis were involved in a study concluding that in such patients, the rs4496877 was associated with higher risk for nephropathy. The odds ratio associated with rs4496877(T) was 1.26 (p = 0.002, CI: 1.08-1.47). A proxy for this SNP (pairwise linkage of 0.96) is rs1800783.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs10495584'>rs10495584</a><a href='http://www.snpedia.com/index.php/rs10495584(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #FFFFFF;'>100.0%</span><br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Normal (higher) blood pressure.  Almost all white people and a majority of non-white people have this normal version. But it isn't as good as the other rare mutant version which would have lowered blood pressure. Tested in Asians. They had an average blood pressure of 127.4/80.1 with this Allele.</div>
        <div class='boxshow' id="show-sec-0.285830236122" style="display: inline;"><a href="javascript:toggle2('sec-0.285830236122')">...more...</a></div>
        <div class='boxrstext' id="sec-0.285830236122" style="display: none;"><a href="javascript:toggle2('sec-0.285830236122')">(hide)</a>
        <div class='rstext'>The minor (G) allele of SNP rs10495584 was associated with lower mean systolic and diastolic blood pressures in men (121.1 +/- 14.2 and 76.3 +/- 10.2 mm Hg vs. 127.4 +/- 15.2 and 80.1 +/- 10.5 mm Hg, P = 0.002 and 0.007, respectively), but not in women (P &gt; 0.05), and remained significant after correction for numerous other factors. However, its effect may be caused by a nearby, tightly linked SNP, rs11524.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1801253'>rs1801253</a><a href='http://www.snpedia.com/index.php/rs1801253(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1.1<br/>Frequency: <span style='background-color: #F0D4F0;'>50.8%</span><br/>References:28<br/><a href='http://www.snpedia.com/index.php/Ambiguous_flip'>ambig</a><br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>responds well to bucindolol; may also depend on rs1801252</div>
        <div class='boxshow' id="show-sec-0.371352613758" style="display: inline;"><a href="javascript:toggle2('sec-0.371352613758')">...more...</a></div>
        <div class='boxrstext' id="sec-0.371352613758" style="display: none;"><a href="javascript:toggle2('sec-0.371352613758')">(hide)</a>
        <div class='rstext'>Also known as Arg389, variation at this SNP, located in the ADRB1 gene, may encode either the amino acid glycine or arginine at amino acid position 389 of the corresponding protein, the beta-1 adrenergic receptor (hence why it is frequently called Arg389Gly). This protein is the target of beta blocker drugs, and so how well the drug works to help lower a patients high blood pressure depends in part on this SNP. The status of this SNP is often reported together with the status of SNP rs1801252, which encodes an amino acid variant at position 49 of the same (ADRB1) protein. The rs1801253(G) allele encodes the glycine. One of the best known studies of the effects of these 2 separate SNPs on the average efficacy of the beta blocker metoprolol in lowering blood pressure (BP) can be summarized f...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs12041331'>rs12041331</a><a href='http://www.snpedia.com/index.php/rs12041331(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FF0A0A;'>0.0%</span><br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Platelet proteins  A;A genotype not associated with increased stickiness of platelets</div>
        <div class='boxshow' id="show-sec-0.402676772095" style="display: inline;"><a href="javascript:toggle2('sec-0.402676772095')">...more...</a></div>
        <div class='boxrstext' id="sec-0.402676772095" style="display: none;"><a href="javascript:toggle2('sec-0.402676772095')">(hide)</a>
        <div class='rstext'></div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1801272'>rs1801272</a><a href='http://www.snpedia.com/index.php/rs1801272(A;T)'>(A;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFA2A2;'>9.2%</span><br/>References:10<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Reduced CYP2A6 metabolism; impaired nicotine metabolism?</div>
        <div class='boxshow' id="show-sec-0.50441018296" style="display: inline;"><a href="javascript:toggle2('sec-0.50441018296')">...more...</a></div>
        <div class='boxrstext' id="sec-0.50441018296" style="display: none;"><a href="javascript:toggle2('sec-0.50441018296')">(hide)</a>
        <div class='rstext'>minor form is a slow metaboliser according to pharmgkb  [PharmGKB:In-Depth This variant results in enzyme inactivity and is considered to be part of the %22slow metabolizer%22 group; part of the CYP2A6*2 haplotype.] [OMIM:?] [GWAS:Smoking behavior]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs679620'>rs679620</a><a href='http://www.snpedia.com/index.php/rs679620(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFB2B2;'>15.5%</span><br/>References:23<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>higher risk for Achilles tendinopathy?</div>
        <div class='boxshow' id="show-sec-0.360884264231" style="display: inline;"><a href="javascript:toggle2('sec-0.360884264231')">...more...</a></div>
        <div class='boxrstext' id="sec-0.360884264231" style="display: none;"><a href="javascript:toggle2('sec-0.360884264231')">(hide)</a>
        <div class='rstext'>A study of 1418 African-American women and men from the Genetic Epidemiology Network of Arteriopathy study found a significant sex-specific interaction effect between rs679620, a SNP in the metallopeptidase 3 MMP3 gene and body mass index (BMI) in African-American women but not men (p = 0.0009). rs679620 appears to have a protective effect on diastolic blood pressure in women with high BMI, but surprisingly, it had the opposite effect on women with low BMI, resulting in higher diastolic BP. A relatively small study (~100 patients) found associations between risk for Achilles tendinopathy and SNPs in the MMP3 gene. For rs679620, the (G;G) genotype had an odds ratio of 2.5 (CI: 1.2 - 4.90, p = 0.010). An interaction resulting in increased risk for Achilles tendinopathy was also reported betw...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs591058'>rs591058</a><a href='http://www.snpedia.com/index.php/rs591058(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFB9B9;'>18.6%</span><br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>higher risk for Achilles tendinopathy?</div>
        <div class='boxshow' id="show-sec-0.915102069288" style="display: inline;"><a href="javascript:toggle2('sec-0.915102069288')">...more...</a></div>
        <div class='boxrstext' id="sec-0.915102069288" style="display: none;"><a href="javascript:toggle2('sec-0.915102069288')">(hide)</a>
        <div class='rstext'>rs591058 is a SNP in the matrix metallopeptidase 3 MMP3 gene on ch 11. A relatively small study (~100 patients) found associations between risk for Achilles tendinopathy and SNPs in the MMP3 gene. For rs591058, the (C;C) genotype had an odds ratio of 2.3 (CI: 1.1 to 4.50, p = 0.023).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs6746030'>rs6746030</a><a href='http://www.snpedia.com/index.php/rs6746030(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFBFBF;'>22.1%</span><br/>References:7<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>reported to influence perception of pain</div>
        <div class='boxshow' id="show-sec-0.0646526428751" style="display: inline;"><a href="javascript:toggle2('sec-0.0646526428751')">...more...</a></div>
        <div class='boxrstext' id="sec-0.0646526428751" style="display: none;"><a href="javascript:toggle2('sec-0.0646526428751')">(hide)</a>
        <div class='rstext'>rs6746030 is a SNP in the sodium channel Nav1.7 SCN9A gene. In five cohorts tested, totaling 1,277 individuals, the rarer rs6746030(A) allele was associated with increased pain (p=0.0001).</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2060793'>rs2060793</a><a href='http://www.snpedia.com/index.php/rs2060793(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFD3D3;'>37.2%</span><br/>References:6<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Higher serum levels of vitamin D</div>
        <div class='boxshow' id="show-sec-0.671665922934" style="display: inline;"><a href="javascript:toggle2('sec-0.671665922934')">...more...</a></div>
        <div class='boxrstext' id="sec-0.671665922934" style="display: none;"><a href="javascript:toggle2('sec-0.671665922934')">(hide)</a>
        <div class='rstext'>Levels of 25-hydroxyvitamin D in familial longevity: the Leiden Longevity Study. Concluded that familial longevity was associated with a lower vitamin D levels and a lower frequency of the allelic variation in the CYP2R1 gene. Genome-wide association study of circulating vitamin D levels. This SNP is part of CYP2R1, which encodes a key C-25 hydroxylase that converts vitamin D3 to an active vitamin D receptor ligand. This SNP, along with rs2282679, rs3829251, and rs6599638 accounted for 2.8% of the variance in circulating vitamin D levels.  [GWAS:Vitamin D levels]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1835740'>rs1835740</a><a href='http://www.snpedia.com/index.php/rs1835740(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFD3D3;'>37.2%</span><br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>unreplicated, higher risk for migraine</div>
        <div class='boxshow' id="show-sec-0.829428407506" style="display: inline;"><a href="javascript:toggle2('sec-0.829428407506')">...more...</a></div>
        <div class='boxrstext' id="sec-0.829428407506" style="display: none;"><a href="javascript:toggle2('sec-0.829428407506')">(hide)</a>
        <div class='rstext'>rs1835740 is a SNP from the 8q22.1 chromosomal region, best known for harboring various genes involved in glutamate metabolism and transport. A large study comprising over 5,000 migraine patients and over 10,000 matched controls, primarily from Europe, found that the rs1835740(A) allele conferred a (slightly) higher risk for the disorder. The reported odds ratio was 1.18, (CI: 1.127&#8211;1.244, p = 1.69 &#215; 10e-11).. However this study has failed to replicate in 2 followups.</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1800547'>rs1800547</a><a href='http://www.snpedia.com/index.php/rs1800547(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFD5D5;'>39.8%</span><br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>MAPT H1/H2 (heterozygous) diplotype</div>
        <div class='boxshow' id="show-sec-0.88099057002" style="display: inline;"><a href="javascript:toggle2('sec-0.88099057002')">...more...</a></div>
        <div class='boxrstext' id="sec-0.88099057002" style="display: none;"><a href="javascript:toggle2('sec-0.88099057002')">(hide)</a>
        <div class='rstext'>replicates the association between rs1800547 and late-onset Parkinson's disease, calculating an odds ratio of 0.77 (CI: 0.66 - 0.88, p=3x10e-4) for the rarer rs1800547(G) allele among 1445 patients from northern Spain. significantly associated with parkinson's disease  [OMIM:MICROTUBULE-ASSOCIATED PROTEIN TAU; MAPT]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs11362'>rs11362</a><a href='http://www.snpedia.com/index.php/rs11362(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDADA;'>45.1%</span><br/>References:13<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'></div>
        <div class='boxshow' id="show-sec-0.318997183654" style="display: inline;"><a href="javascript:toggle2('sec-0.318997183654')">...more...</a></div>
        <div class='boxrstext' id="sec-0.318997183654" style="display: none;"><a href="javascript:toggle2('sec-0.318997183654')">(hide)</a>
        <div class='rstext'>Heterozygotes for SNP rs11362, located in the DEFB1 gene, are reported to be at increased risk for colonic Crohn's disease - but not the ileal or ileocolonic forms of the disease - based on a study of 190 Caucasians. The odds ratio is 2.393 (CI: 1.18-4.87, p=0.02). G-20A rs11362(A;G) 2.4x risk colonic and ileocolonic Crohn's disease but no association was detected for the ileal localization</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs5911'>rs5911</a><a href='http://www.snpedia.com/index.php/rs5911(A;C)'>(A;C)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDBDB;'>45.3%</span><br/>References:3<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Heterozygous for BAK platelet-specific alloantigen</div>
        <div class='boxshow' id="show-sec-0.693973719261" style="display: inline;"><a href="javascript:toggle2('sec-0.693973719261')">...more...</a></div>
        <div class='boxrstext' id="sec-0.693973719261" style="display: none;"><a href="javascript:toggle2('sec-0.693973719261')">(hide)</a>
        <div class='rstext'>rs5911, also known as Ile874Ser or Ile843Ser, is a SNP in the ITGA2B gene on chromosome 17. In the context of the plus strand nomenclature used by dbSNP, rs5911(A) encodes the Ile, and rs5911(G) the Ser. The rs5911(C) allele gives rise to the platelet-specific alloantigen known as BAK. Of some historic interest in the field of platelet transfusion, the BAK alloantigen (rs5911(C;C) homozygotes) has more recently been reported in a small study of healthy Chinese males to be associated with decreased ex vivo antiplatelet activity of ticagrelor.  [OMIM:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs187116'>rs187116</a><a href='http://www.snpedia.com/index.php/rs187116(C;T)'>(C;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFDBDB;'>46.0%</span><br/>References:2<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>among patients with gastric cancer, risk for more aggressive form?</div>
        <div class='boxshow' id="show-sec-0.762835205677" style="display: inline;"><a href="javascript:toggle2('sec-0.762835205677')">...more...</a></div>
        <div class='boxrstext' id="sec-0.762835205677" style="display: none;"><a href="javascript:toggle2('sec-0.762835205677')">(hide)</a>
        <div class='rstext'>rs187116, also known as +4883G&gt;A, is a SNP in the CD44 gene. A presentation at the 2010 American Society of Clinical Oncology meeting based on ~100 patients reported this SNP as being significantly associated with more aggressive gastric cancer. In dbSNP orientation, the risk allele is rs187116(C); patients carrying at least one rs187116(C) had a significantly shorter median time-to-recurrence (TTR) of 2.1 years compared to 7.0 years TTR for rs187116(T;T) patients (p = 0.036, log-rank test).[http://www.asco.org/ASCOv2/Meetings/Abstracts?&vmview=abst_detail_view&confID=72&abstractID=1689]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs7107418'>rs7107418</a><a href='http://www.snpedia.com/index.php/rs7107418(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFE1E1;'>52.7%</span><br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Less likely to think cilantro tastes like soap</div>
        <div class='boxshow' id="show-sec-0.627534463087" style="display: inline;"><a href="javascript:toggle2('sec-0.627534463087')">...more...</a></div>
        <div class='boxrstext' id="sec-0.627534463087" style="display: none;"><a href="javascript:toggle2('sec-0.627534463087')">(hide)</a>
        <div class='rstext'>rs7107418 G allele is a proxy for rs72921001 A allele. rs72921001 A allele associated with lower chance of thinking cilantro tastes like soap. OR=.81 per A allele. http://blog.23andme.com/23andme-research/cilantro-love-hate-genetic-trait/</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1800469'>rs1800469</a><a href='http://www.snpedia.com/index.php/rs1800469(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFE1E1;'>53.1%</span><br/>References:77<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>lower TGF-&#946;1 levels, higher risk for COPD?  This normal C version significantly reduces TGF-&#946;1 levels by allowing AP1 to downregulate TGF-&#946;1 production.</div>
        <div class='boxshow' id="show-sec-0.609668116114" style="display: inline;"><a href="javascript:toggle2('sec-0.609668116114')">...more...</a></div>
        <div class='boxrstext' id="sec-0.609668116114" style="display: none;"><a href="javascript:toggle2('sec-0.609668116114')">(hide)</a>
        <div class='rstext'>rs1800469, also known as -509 C&gt;T or &#8722;1347 C &gt; T, is an SNP in the promoter region of the transforming growth factor beta1 TGFB1 gene. That means it doesn't change the nature of the TGF-&#946;1 protein, but it does change the amount produced. The T version increases the amount of TGF-&#946;1 produced, by preventing AP1 from binding to this region where in the C version it would normally downregulate production. That increased TGF-&#946;1 production with the T version is probably what causes all the health effects of this SNP. Whether increased TGF-&#946;1 is good or bad probably depends on the disease you are interested in. It has been associated with increased risk for chronic obstructive pulmonary disease (COPD). Although smoking is the main risk factor, smokers with a rs1800469(C) allele (or 2 other ...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1800407'>rs1800407</a><a href='http://www.snpedia.com/index.php/rs1800407(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFF7F7;'>85.8%</span><br/>References:13<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>blue/gray eyes more possible</div>
        <div class='boxshow' id="show-sec-0.00734752975977" style="display: inline;"><a href="javascript:toggle2('sec-0.00734752975977')">...more...</a></div>
        <div class='boxrstext' id="sec-0.00734752975977" style="display: none;"><a href="javascript:toggle2('sec-0.00734752975977')">(hide)</a>
        <div class='rstext'>rs1800407 is located in exon 13 of the OCA2 gene at amino acid position 419, and it is therefore also known as the R419Q variant. The (A) allele encodes the amino acid glutamine (instead of arginine), and it is associated with increased odds of having green/hazel eye color in some Caucasian populations. One copy of of rs1800407(A) is sufficient for this effect. [PMID 12163334, PMID 15889046; OMIM 203200.0012] According to tree scanning analysis, the proportion of eye colour variation explained by this nucleotide position is merely 4%. Thus, additional variation present in the OCA2 gene and perhaps some other pigment related genes must be taken into account in order to explain the high phenotypic variation in iris colour. Among rs12913832(A;A) individuals, the penetrance for green/hazel eye...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs12003612'>rs12003612</a><a href='http://www.snpedia.com/index.php/rs12003612(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFF7F7;'>86.4%</span><br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'></div>
        <div class='boxshow' id="show-sec-0.797364094774" style="display: inline;"><a href="javascript:toggle2('sec-0.797364094774')">...more...</a></div>
        <div class='boxrstext' id="sec-0.797364094774" style="display: none;"><a href="javascript:toggle2('sec-0.797364094774')">(hide)</a>
        <div class='rstext'></div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1861046'>rs1861046</a><a href='http://www.snpedia.com/index.php/rs1861046(G;G)'>(G;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>Frequency: <span style='background-color: #FFFBFB;'>92.9%</span><br/>References:1<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Conduct disorder (CD) is one of the most prevalent childhood psychiatric conditions, and is associated with a number of serious concomitant and future problems. CD symptomatology is known to have a considerable genetic component, with heritability estimates in the range of 50%. Four SNPs reached genome-wide significance with CD symptom count, based on P &lt; 5x10^-8 Two of these SNPs (rs16891867 and rs1861046) were in the gene C1QTNF7 (C1q and tumor necrosis factor-related protein 7). These SNPs were in high LD (r^2=0.97). The other top two SNPs meeting genome-wide significance were intergenic and located on chromosomes 11 and 13. Alleles A/G were found to be associated with CD symptom count or CD case status. [http://www.downstate.edu/hbnl/documents/2010-Dick-Genome-wideassociationstudyof...</div>
        <div class='boxshow' id="show-sec-0.599474504652" style="display: inline;"><a href="javascript:toggle2('sec-0.599474504652')">...more...</a></div>
        <div class='boxrstext' id="sec-0.599474504652" style="display: none;"><a href="javascript:toggle2('sec-0.599474504652')">(hide)</a>
        <div class='rstext'>[GWAS:Conduct disorder (case status)]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/i4000189'>i4000189</a><a href='http://www.snpedia.com/index.php/i4000189(T;T)'>(T;T)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>References:0<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>P40+  Match for Haplogroup I1 (Y-DNA).  [haplogroup:I1 (Y-DNA)]</div>
        <div class='boxshow' id="show-sec-0.342899791949" style="display: inline;"><a href="javascript:toggle2('sec-0.342899791949')">...more...</a></div>
        <div class='boxrstext' id="sec-0.342899791949" style="display: none;"><a href="javascript:toggle2('sec-0.342899791949')">(hide)</a>
        <div class='rstext'>[haplogroup:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs165599'>rs165599</a><a href='http://www.snpedia.com/index.php/rs165599(A;G)'>(A;G)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>References:64<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'></div>
        <div class='boxshow' id="show-sec-0.388877862817" style="display: inline;"><a href="javascript:toggle2('sec-0.388877862817')">...more...</a></div>
        <div class='boxrstext' id="sec-0.388877862817" style="display: none;"><a href="javascript:toggle2('sec-0.388877862817')">(hide)</a>
        <div class='rstext'>anxiety-related personality traits, ADHD, schizophrenia part of a three marker haplotype rs737865-rs4680-rs165599 epistasis between SNPs in COMT (rs2097603, Val158Met (rs4680), rs165599) and polymorphisms in other schizophrenia susceptibility genes popular in pubmed is associated with bipolar disorder and influences prefrontal aspects of verbal memory in bipolar patients and healthy controls.  [PharmGKB:Curated In a study of 78 African American and 65 white patients diagnosed with schizophrenia or schizoaffective disorder, this SNP in the COMT gene was found to have significant associations with response to risperidone over 2-12 weeks in both African-American and white patients.] [OMIM:CATECHOL-O-METHYLTRANSFERASE; COMT]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs1800401'>rs1800401</a><a href='http://www.snpedia.com/index.php/rs1800401(C;C)'>(C;C)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>References:5<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>blue/gray eyes possible</div>
        <div class='boxshow' id="show-sec-0.977281236593" style="display: inline;"><a href="javascript:toggle2('sec-0.977281236593')">...more...</a></div>
        <div class='boxrstext' id="sec-0.977281236593" style="display: none;"><a href="javascript:toggle2('sec-0.977281236593')">(hide)</a>
        <div class='rstext'>rs1800401 is located in exon 7 of the OCA2 gene at amino acid position 305. The (T) allele encodes the amino acid tryptophan (instead of arginine), and it is one of several SNPs associated with increased odds of having brown or black eye color in Caucasians. One copy of of rs1800401(T) is sufficient for this effect. [PMID 12163334, PMID 15889046; OMIM 203200.0011]  [OMIM:?]</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>


  <div  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxlink'><a href='http://www.snpedia.com/index.php/rs2814778'>rs2814778</a><a href='http://www.snpedia.com/index.php/rs2814778(A;A)'>(A;A)</a></div>
        <div class='boxmag'>Magnitude: 1<br/>References:23<br/></div>
      </div>
      <div class='boxresultcol2'>
        <div class='boxeffect'>Duffy-positive  This genotype is associated with European populations and southwestern Native American populations.</div>
        <div class='boxshow' id="show-sec-0.881546162676" style="display: inline;"><a href="javascript:toggle2('sec-0.881546162676')">...more...</a></div>
        <div class='boxrstext' id="sec-0.881546162676" style="display: none;"><a href="javascript:toggle2('sec-0.881546162676')">(hide)</a>
        <div class='rstext'>rs2814778 is within the DARC gene, which encodes the Duffy blood group antigen . This SNP shows an almost perfectly fixed difference in frequency between Europeans and those with African ancestry. (One exception appears to be a certain population of Czech gypsies, and certain non-Ashkenazi Jewish populations.) Additionally the Namibian San samples of the CEPH-HGDP are, uncharacteristically for Africans, all AA homozygotes for this SNP. The rs2814778 (G) allele is associated with African populations, while rs2814778 (A) is associated with European populations and southwestern Native American populations. rs2814778(G;G) protect against Plasmodium vivax malaria and predicts of white blood cell and neutrophil count 23andMe blog discussion Malaria Resistance (Duffy Antigen)  [PharmGKB:Curated A...</div>
      </div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs235"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs235'>gs235</a></div></div>
        <div class='boxmag'>Magnitude: 0.8</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>unclassified pain sensitivity</div>
        <div class='gsboxeffect'>No information regarding pain sensitivity. you were not recognized as gs234, gs233, or gs232.</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs233"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs233'>gs233</a></div></div>
        <div class='boxmag'>Magnitude: 0.1</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>normal pain sensitivity</div>
        <div class='gsboxeffect'>You probably experience typical sensitivity to pain according to See also gs232, gs234, and gs235</div>
      </div>
      <div class="boxfooter"></div>
  </div>

  <div id="gs217"  class='boxresult'>
      <div class='boxresultcol1'>
        <div class='boxfreq'><div class='boxlink'><a href='http://www.snpedia.com/index.php/gs217'>gs217</a></div></div>
        <div class='boxmag'>Magnitude: 0</div>
        <div class='boxrep'></div>
      </div>
      <div class='boxresultcol2'>
        <div class='gsboxsummary'>one or more copies of a test CNV</div>
        <div class='gsboxeffect'>you have one or more copies of the CNV discussed at . see also gs218 tested via at least rs16889859(-;-)</div>
      </div>
      <div class="boxfooter"></div>
  </div>
</div>     <div class="majorboxfooter"></div>     </div><p><br/></p></body></html>